<?xml version='1.0' encoding='utf-8'?>
<<<<<<< HEAD
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-GB" id="nojs">
  <head>
    <title>
   BMC Genomics | Full text | 5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction
  </title>
    <link href="/images/icon/1008/favicon.ico" type="image/x-icon" rel="shortcut icon"/>
    <meta content="IE=7" http-equiv="X-UA-Compatible"/>
    <meta content="BMC Genomics" name="citation_journal_title"/>
    <meta content="BioMed Central Ltd" name="citation_publisher"/>
    <meta content="Joseph M Dhahbi" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Stephen R Spindler" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Hani Atamna" name="citation_author"/>
    <meta content="Department of Basic Sciences, Neuroscience, The Commonwealth Medical College, Scranton, Scranton, PA, 18510, USA" name="citation_author_institution"/>
    <meta content="Amy Yamakawa" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Dario Boffelli" name="citation_author"/>
    <meta content="Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA" name="citation_author_institution"/>
    <meta content="Patricia Mote" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="David IK Martin" name="citation_author"/>
    <meta content="Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA" name="citation_author_institution"/>
    <meta content="1471-2164" name="citation_issn"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="citation_title"/>
    <meta content="14" name="citation_volume"/>
    <meta content="1" name="citation_issue"/>
    <meta content="2013-05-02" name="citation_date"/>
    <meta content="298" name="citation_firstpage"/>
    <meta content="10.1186/1471-2164-14-298" name="citation_doi"/>
    <meta content="http://www.biomedcentral.com/content/pdf/1471-2164-14-298.pdf" name="citation_pdf_url"/>
    <link href="http://www.biomedcentral.com/content/pdf/1471-2164-14-298.pdf" type="application/pdf" rel="alternate" title="PDF"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="title"/>
    <meta content="Small RNAs complex with proteins to mediate a variety of functions in animals and plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may carry out a signaling function by entering target cells and modulating gene expression. The subject of this study is a set of circulating 30–33 nt RNAs that are processed derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit translation initiation in response to stress in cultured cells." name="description"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="dc.title"/>
    <meta content="Dhahbi, Joseph M" name="dc.creator"/>
    <meta content="Spindler, Stephen R" name="dc.creator"/>
    <meta content="Atamna, Hani" name="dc.creator"/>
    <meta content="Yamakawa, Amy" name="dc.creator"/>
    <meta content="Boffelli, Dario" name="dc.creator"/>
    <meta content="Mote, Patricia" name="dc.creator"/>
    <meta content="Martin, David IK" name="dc.creator"/>
    <meta content="Small RNAs complex with proteins to mediate a variety of functions in animals and plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may carry out a signaling function by entering target cells and modulating gene expression. The subject of this study is a set of circulating 30–33 nt RNAs that are processed derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit translation initiation in response to stress in cultured cells." name="dc.description"/>
    <meta content="BMC Genomics 2013 14:298" name="dc.source"/>
    <meta content="text/html" name="dc.format"/>
    <meta content="BioMed Central Ltd" name="dc.publisher"/>
    <meta content="2013-05-02" name="dc.date"/>
    <meta content="Research article" name="dc.type"/>
    <meta content="10.1186/1471-2164-14-298" name="dc.identifier"/>
    <meta content="en" name="dc.language"/>
    <meta content="2013 Dhahbi et al.; licensee BioMed Central Ltd." name="dc.copyright"/>
    <meta content="http://creativecommons.org/licenses/by/2.0/" name="dc.rights"/>
    <meta content="reprints@biomedcentral.com" name="dc.rightsAgent"/>
    <meta content="1471-2164" name="prism.issn"/>
    <meta content="BMC Genomics" name="prism.publicationName"/>
    <meta content="2013-05-02" name="prism.publicationDate"/>
    <meta content="14" name="prism.volume"/>
    <meta content="1" name="prism.number"/>
    <meta content="Research article" name="prism.section"/>
    <meta content="298" name="prism.startingPage"/>
    <meta content="2013 Dhahbi et al.; licensee BioMed Central Ltd." name="prism.copyright"/>
    <meta content="reprints@biomedcentral.com" name="prism.rightsAgent"/>
    <meta content="http://www.biomedcentral.com/1471-2164/14/298/abstract" name="citation_abstract_html_url"/>
    <meta content="http://www.biomedcentral.com/1471-2164/14/298" name="citation_fulltext_html_url"/>
    <link href="http://www.biomedcentral.com/content/download/xml/1471-2164-14-298.xml" type="text/xml" rel="alternate" title="XML version"/>
    <link href="http://www.biomedcentral.com/content/figures/1471-2164-14-298-toc.gif" rel="image_src"/>
    <link href="http://www.biomedcentral.com/content/download/supplementary/1471-2164-14-298-s1.pdf" rel="section" title="Additional file 1"/>
    <link media="screen, print" href="/css/main-0.css" type="text/css" rel="stylesheet"/>
    <link media="screen, print" href="/css/plugins-0.css" type="text/css" rel="stylesheet"/>
    <link media="screen, print" href="/css/articles-0.css" type="text/css" rel="stylesheet"/>
    <link media="screen, print" href="/bmcgenomics/css/themes/1008.css?1330424572279" type="text/css" rel="stylesheet"/>
    <link media="print" href="/css/media/print-0.css" type="text/css" rel="stylesheet"/>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  </head>
  <body class=" theme    "><div style="display:none" id="oas-campaign">
   biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/14/298
  </div>
<div style="display:none" id="oas-positions">
   Bottom,Top
  </div>
<link href="http://www.biomedcentral.com/bmcgenomics/latest/rss" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Latest articles"/>
<link href="http://www.biomedcentral.com/bmcgenomics/mostviewed/rss/" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Most viewed"/>
<link href="http://www.biomedcentral.com/bmcgenomics/latestcomments/rss" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Latest comments"/>
<style type="text/css" title="override_css">
=======
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
  <head>
    <title>
   Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text
  </title>
    <meta content="NOODP" name="googlebot"/>
    <meta content="/cgi/content/full/28/22/2963" name="HW.ad-path"/>
    <meta content="/bioinfo/28/22/2963.atom" name="HW.identifier"/>
    <meta content="text/html" name="DC.Format"/>
    <meta content="en" name="DC.Language"/>
    <meta content="Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text" name="DC.Title"/>
    <meta content="10.1093/bioinformatics/bts542" name="DC.Identifier"/>
    <meta content="2012-11-15" name="DC.Date"/>
    <meta content="Oxford University Press" name="DC.Publisher"/>
    <meta content="Leon French" name="DC.Contributor"/>
    <meta content="Suzanne Lane" name="DC.Contributor"/>
    <meta content="Lydia Xu" name="DC.Contributor"/>
    <meta content="Celia Siu" name="DC.Contributor"/>
    <meta content="Cathy Kwok" name="DC.Contributor"/>
    <meta content="Yiqi Chen" name="DC.Contributor"/>
    <meta content="Claudia Krebs" name="DC.Contributor"/>
    <meta content="Paul Pavlidis" name="DC.Contributor"/>
    <meta content="Bioinformatics" name="citation_journal_title"/>
    <meta content="Bioinformatics" name="citation_journal_abbrev"/>
    <meta content="1367-4803" name="citation_issn"/>
    <meta content="1460-2059" name="citation_issn"/>
    <meta content="Leon French" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Suzanne Lane" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Lydia Xu" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Celia Siu" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Cathy Kwok" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Yiqi Chen" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Claudia Krebs" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Paul Pavlidis" name="citation_author"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" name="citation_author_institution"/>
    <meta content="Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text" name="citation_title"/>
    <meta content="11/15/2012" name="citation_date"/>
    <meta content="28" name="citation_volume"/>
    <meta content="22" name="citation_issue"/>
    <meta content="2963" name="citation_firstpage"/>
    <meta content="2970" name="citation_lastpage"/>
    <meta content="28/22/2963" name="citation_id"/>
    <meta content="bioinfo;28/22/2963" name="citation_mjid"/>
    <meta content="10.1093/bioinformatics/bts542" name="citation_doi"/>
    <meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.abstract" name="citation_abstract_html_url"/>
    <meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.full" name="citation_fulltext_html_url"/>
    <meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.full.pdf" name="citation_pdf_url"/>
    <meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963" name="citation_public_url"/>
    <meta content="22954628" name="citation_pmid"/>
    <meta content="Original Paper" name="citation_section"/>
    <meta content="noarchive,nofollow" name="robots"/>
    <meta content="noarchive" name="googlebot"/>
    <link href="/content/28/22/2956.short" rel="prev"/>
    <link href="/content/28/22/2971.short" rel="next"/>
    <link media="all" href="/shared/css/hw-global.css" type="text/css" rel="stylesheet"/>
    <link media="print" href="/shared/css/hw-print.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/local/css/hw-local-global.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/publisher/css/hw-publisher-global.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/publisher/css/hw-publisher-ac.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/shared/css/hw-page-content.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/shared/css/jquery.fancybox-1.3.4.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/shared/css/hw-global-colexpand.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/publisher/css/hw-publisher-page-content.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/resource/css/hw20.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/resource/css/h20-oup-header.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/resource/css/h20-oup-footer.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/resource/css/h20-oup-sidebars.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="/site/resource/bioinformatics_h20.css" type="text/css" rel="stylesheet"/>
    <link media="print" href="/resource/css/print.css" type="text/css" rel="stylesheet"/>
    <link media="all" href="stylesheets/style15.css" type="text/css" rel="stylesheet"/>
    <style type="text/css" title="override_css">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
            .article_date {
                color: gray; font-family: monospace;
            }

            .article_description {
                text-indent: 0pt;
            }

            a.article {
                font-weight: bold; text-align:left;
            }

            a.feed {
                font-weight: bold;
            }

            .calibre_navbar {
                font-family:monospace;
            }
    

</style>
<<<<<<< HEAD

<div style="text-align:center" class="calibre_navbar calibre_rescale_70">| <a href="../../feed_1/index.html">Next</a> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | <a href="../article_0/index.html">Previous</a> | <hr/>
</div>
<div role="banner" id="branding">
<dl class="google-ad wide ">
<dt class="hide">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising" class="banner-ad">
</a>
</dt>
<dd>
</dd>
</dl>
<div class="sup-panel-outer">
<div class="sup-panel-inner">
<ul class="login" id="login">
<li class="greeting">
<strong>
        Welcome
        <span id="username">
         University of Louisville
        </span>
</strong>
</li>
<li id="loginLink">
<a href="/bmcgenomics/logon">
        Log on
       </a>
</li>
</ul>
<ul class="nav-sup">
<li class="current" id="BMC">
<a href="http://www.biomedcentral.com/">
<span class="img-title">
          
        </span>
<span class="text-title">
         BioMed Central
        </span>
</a>
</li>
<li>
<a href="http://www.biomedcentral.com/journals">
<span>
         Journals
        </span>
</a>
</li>
<li>
<a href="http://www.biomedcentral.com/gateways">
<span>
         Gateways
        </span>
</a>
</li>
</ul>
</div>
</div>
<div class="branding-inner">
<div class="logo">
<a href="/bmcgenomics/">
<img src="/sites/1008/images/logo.gif" alt="BMC Genomics" title="BMC Genomics Logo"/>
</a>
</div>
<div class="official" id="impact-factor">
<img src="/images/branding/official.gif" alt="official impact factor" title="4.07"/>
<span id="impact-factor-value">
      4.07
     </span>
</div>
<div role="search" class="module search gray small  ">
<div class="module-inner padded">
<form action=" /quickSearchRedirectURL " onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;" class="searchForm" method="get">
<fieldset class="search">
<span>
         Search
         <select name="url">
<option selected="selected" value="http://www.biomedcentral.com/bmcgenomics/search/results">
           BMC Genomics
          </option>
<option value="http://www.biomedcentral.com/search/results">
           BioMed Central
          </option>
</select>
         for
        </span>
<input alt="Search terms" type="text" id="searchTerms" name="terms" class="text "/>
<button type="submit" class="w37" value="Go">
         Go
        </button>
</fieldset>

</form></div>
<a href="/bmcgenomics/search" class="advanced-search">
      Advanced search
     </a>
</div>
<div id="mobile-nav-list">
<ul role="navigation" class="primary-nav">
<li>
<a href="/bmcgenomics/">
<span id="home-tab">
         Home
        </span>
</a>
</li>
<li class="current">
<a href="/bmcgenomics/content">
<span id="articles-tab">
         Articles
        </span>
</a>
</li>
<li>
<a href="/bmcgenomics/authors/instructions">
<span id="authors-tab">
         Authors
        </span>
</a>
</li>
<li>
<a href="/bmcgenomics/about/reviewers">
<span id="reviewers-tab">
         Reviewers
        </span>
</a>
</li>
<li>
<a href="/bmcgenomics/about">
<span id="about-tab">
         About this journal
        </span>
</a>
</li>
<li>
<a href="/bmcgenomics/my">
<span id="my-tab">
         My BMC Genomics
        </span>
</a>
</li>
</ul>
</div>
</div>
</div>
<hr class="hide"/>
<div class="container">
<div role="main" class="overflow-visible" id="content">
<div id="left-article-box" class="outline-wrapper block mobile-hidden">
<ul class="box" id="box-outline">
<ul class="outline" id="outline">
<li>
<a href="#">
         Top
        </a>
</li>
<li>
<a href="#abs">
         Abstract
        </a>
</li>
<li>
<a href="#sec1">
         Background
        </a>
</li>
<li>
<a href="#sec2">
         Results and discussion
        </a>
</li>
<li>
<a href="#sec3">
         Conclusions
        </a>
</li>
<li>
<a href="#sec4">
         Methods
        </a>
</li>
<li>
<a href="#sec5">
         Competing interests
        </a>
</li>
<li>
<a href="#sec6">
         Authors’ contributions
        </a>
</li>
<li>
<a href="#ack">
         Acknowledgements
        </a>
</li>
<li class="lowest">
<a href="#refs">
         References
        </a>
</li>
</ul>
</ul>
<dl id="box-outline1">
</dl>
</div>
<div id="right-panel" class="mobile-hidden">
<div id="mobile-sidebar">
<div id="mobile-sidebar-tab">
<span>
<a href="#" id="mobile-sidebar-tab-link">
<img src="/images/sidebar-hidden.png"/>
<img src="/images/sidebar-shown.png" style="display:none;"/>
</a>
</span>
</div>
<div id="mobile-sidebar-list">
<div id="article-navigation-bar">
<div class="issue-information block">
<h5>
<a href="http://www.biomedcentral.com/bmcgenomics">
<strong>
            BMC Genomics
           </strong>
</a>
</h5>
<ul class="square normal">
<li>
<a href="http://www.biomedcentral.com/bmcgenomics/content/14">
            Volume 14
           </a>
</li>
</ul>
</div>
<div id="article-info" class="article-information block">
<div id="viewing-options-links">
<h5>
           Viewing options
          </h5>
<ul class="square normal">
<li>
<a href="/1471-2164/14/298/abstract">
             Abstract
            </a>
</li>
<li>
<strong>
             Full text
            </strong>
</li>
<li class="pdfFileSize">
<a href="/content/pdf/1471-2164-14-298.pdf" onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2164/14/298', 1, true]);">
             PDF
            </a>
<span>
             (2.3MB)
            </span>
</li>
<li>
</li>
<li>
<a href="/1471-2164/14/298/additional">
             Additional files
            </a>
</li>
</ul>
</div>
<div id="associated-material-links">
<h5>
           Associated material
          </h5>
<ul class="square normal">
<li>
<a href="/pubmed/23638709">
             PubMed record
            </a>
</li>
<li>
<a href="/1471-2164/14/298/about">
             About this article
            </a>
</li>
<li>
<a href="/1471-2164/14/298/comments">
             Readers' comments
            </a>
</li>
<li>
</li>
</ul>
</div>
<div id="related-literature-links">
<h5>
           Related literature
          </h5>
<ul class="normal">
<li>
<a href="http://www.biomedcentral.com/1471-2164/14/298/about#citations">
<h6>
              Cited by
             </h6>
</a>
</li>
<li id="google-blog-search">
<a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=5%C2%BF+tRNA+halves+are+present+as+abundant+complexes+in+serum%2C+concentrated+in+blood+cells%2C+and+modulated+by+aging+and+calorie+restriction&amp;btnG=Search">
             Google blog search
            </a>
</li>
<h6>
            Other articles by authors
           </h6>
<li>
<a class="collapser">
<i>
               
             </i>
             on Google Scholar
            </a>
<ul class="hidebeforeload" id="authg">
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22JM+Dhahbi%22">
               Dhahbi JM
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22SR+Spindler%22">
               Spindler SR
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22H+Atamna%22">
               Atamna H
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22A+Yamakawa%22">
               Yamakawa A
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22D+Boffelli%22">
               Boffelli D
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22P+Mote%22">
               Mote P
              </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=author%3A%22DIK+Martin%22">
               Martin DIK
              </a>
</li>
</ul>
</li>
<li>
<a class="collapser">
<i>
               
             </i>
             on PubMed
            </a>
<ul class="hidebeforeload" id="authpm">
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Dhahbi_JM%20[Author]">
               Dhahbi JM
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Spindler_SR%20[Author]">
               Spindler SR
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Atamna_H%20[Author]">
               Atamna H
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Yamakawa_A%20[Author]">
               Yamakawa A
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Boffelli_D%20[Author]">
               Boffelli D
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Mote_P%20[Author]">
               Mote P
              </a>
</li>
<li>
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Martin_DIK%20[Author]">
               Martin DIK
              </a>
</li>
</ul>
</li>
</ul>
<h6>
           Related articles/pages
          </h6>
<ul class="square normal">
<li>
<a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2164/14/298">
             on Google
            </a>
</li>
<li>
<a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2164/14/298">
             on Google Scholar
            </a>
</li>
<li>
<a href="/pubmed/related/23638709">
             on PubMed
            </a>
</li>
</ul>
</div>
<h5>
          Tools
         </h5>
<ul class="square normal">
<li>
<a href="/1471-2164/14/298/citation">
            Download references
           </a>
</li>
<li>
<a href="/content/download/xml/1471-2164-14-298.xml">
            Download XML
           </a>
</li>
<li>
<a href="/1471-2164/14/298/email">
            Email to a friend
           </a>
</li>
<li>
<a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1471-2164-14-298&amp;volume=14&amp;issue=1&amp;title=5%26%238242%3B+tRNA+halves+are+present+as+abundant+complexes+in+serum%2C+concentrated+in+blood+cells%2C+and+modulated+by+aging+and+calorie+restriction&amp;author_name=Joseph M Dhahbi&amp;start_page=1&amp;end_page=14">
            Order reprints
           </a>
</li>
<li>
<a href="/1471-2164/14/298/postcomment">
            Post a comment
           </a>
</li>
</ul>
<h5>
          Share this article
         </h5>
<ul id="social-networking-links">
<li>
<div data-width="100" data-show-faces="false" data-action="recommend" data-send="false" data-layout="button_count" class="fb-like" data-href="http://www.biomedcentral.com/1471-2164/14/298">
</div>
</li>
<li>
<a data-url="http://www.biomedcentral.com/1471-2164/14/298" data-text="#bmcgenomics&#10;5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" href="http://twitter.com/share" class="twitter-share-button">
            Tweet
           </a>
</li>
<li>
<gU0003Aplusone width="100" size="medium">
</gU0003Aplusone>
</li>
<li>
<a href="#" class="sharethis_title btn">
<span class="icon">
              
            </span>
            More options...
           </a>
<ul id="sharethis" class="iconlist hidebeforeload">
<li>
<a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">
<span class="share-icons citeulike">
</span>
              Citeulike
             </a>
</li>
<li>
<a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=%23bmcgenomics%0D%0A5%C2%BF%20tRNA%20halves%20are%20present%20as%20abundant%20complexes%20in%20serum%2C%20concentrated%20in%20blood%20cells%2C%20and%20modulated%20by%20aging%20and%20calorie%20restriction%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2164%2F14%2F298&amp;title=5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" onclick="largepopup(this.href,'linkedin',800,600);return false">
<span class="share-icons linkedin">
</span>
              LinkedIn
             </a>
</li>
<li>
<a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2164/14/298&amp;title=5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">
<span class="share-icons delicious">
</span>
              Del.icio.us
             </a>
</li>
<li>
<a href="/1471-2164/14/298/email">
<span class="share-icons email">
</span>
              Email
             </a>
</li>
<li>
<a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'facebook',800,600);return false">
<span class="share-icons facebook">
</span>
              Facebook
             </a>
</li>
<li class="gp">
<div class="googlehider mobile-hide">
<div data-annotation="none" class="g-plusone">
</div>
</div>
<span class="share-icons googleplus mobile-hide">
</span>
<div class="googlehider2 mobile-hide">
<div data-annotation="none" class="g-plusone mobile-hide">
</div>
</div>
<span class="share-icons googleplus">
</span>
<span class="gp-link">
<a href="#">
               Google+
              </a>
</span>
</li>
<li>
<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'mendeley',800,600);return false">
<span class="share-icons mendeley">
</span>
              Mendeley
             </a>
</li>
<li>
<a href="http://twitter.com/?status=%23bmcgenomics%0D%0A5%C2%BF%20tRNA%20halves%20are%20present%20as%20abundant%20complexes%20in%20serum%2C%20concentrated%20in%20blood%20cells%2C%20and%20modulated%20by%20aging%20and%20calorie%20restriction%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2164%2F14%2F298" onclick="largepopup(this.href,'twitter',800,600);return false">
<span class="share-icons twitter">
</span>
              Twitter
             </a>
</li>
</ul>
</li>
</ul>
</div>
</div>
<div class="mobile-sidebar-gradient top">
</div>
<div class="mobile-sidebar-gradient bottom">
</div>
</div>
</div>
</div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
<div class="padded-inner">
<div id="topmatter">
<a href="/about/access">
<img src="/images/articles/openaccess-large.png" alt="Open Access" class="access mr15"/>
</a>
<span class="articletype">
         Research article
        </span>
<h1>
         5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction
        </h1>
<div class="multipleins">
<p class="authors">
<strong>
           Joseph M Dhahbi
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
<sup>
           *
          </sup>
          ,
          <strong>
           Stephen R Spindler
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          ,
          <strong>
           Hani  Atamna
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins2">
            2
           </a>
</sup>
          ,
          <strong>
           Amy  Yamakawa
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          ,
          <strong>
           Dario  Boffelli
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins3">
            3
           </a>
</sup>
          ,
          <strong>
           Patricia  Mote
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          and
          <strong>
           David IK Martin
          </strong>
<sup>
<a href="/1471-2164/14/298/#ins3">
            3
           </a>
</sup>
</p>
<div id="affiliations">
<div class="module gray  inner">
<div class="module-inner padded-inner">
<ul>
<li>
<p class="authors">
<span>
                *
               </span>
               Corresponding author:										            Joseph M Dhahbi
               <a href="mailto:jdhahbi@ucr.edu">
                jdhahbi@ucr.edu
               </a>
</p>
</li>
</ul>
<div>
<div class="collapsible-content">
<p class="options">
<a href="#" class="affiliations-toggle">
<i class="arrow">
</i>
                Author Affiliations
               </a>
</p>
<div style="display: block;" id="ins_container">
<p>
<sup id="ins1">
                 1
                </sup>
                Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA
               </p>
<p>
<sup id="ins2">
                 2
                </sup>
                Department of Basic Sciences, Neuroscience, The Commonwealth Medical College, Scranton, Scranton, PA, 18510, USA
               </p>
<p>
<sup id="ins3">
                 3
                </sup>
                Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA
               </p>
</div>
<p id="authoremails">
               For all author emails, please
               <a href="/bmcgenomics/logon">
                log on
               </a>
               .
              </p>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cit">
<div class="collapsible-content">
<p>
<em>
            BMC Genomics
           </em>
           2013,
           <strong>
            14
           </strong>
           :298 
           <span class="pseudotab">
            doi:10.1186/1471-2164-14-298
           </span>
</p>
<br/>
<p>
           The electronic version of this article is the complete one and can be found online at:
           <a href="http://www.biomedcentral.com/1471-2164/14/298">
            http://www.biomedcentral.com/1471-2164/14/298
           </a>
</p>
<br/>
<table cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td>
              Received:
             </td>
<td>
              6 February 2013
             </td>
</tr>
<tr>
<td>
              Accepted:
             </td>
<td>
              26 April 2013
             </td>
</tr>
<tr>
<td>
              Published:
             </td>
<td>
              2 May 2013
             </td>
</tr>
</tbody>
</table>
<div style="line-height:140%">
<p>
            © 2013 Dhahbi et al.; licensee BioMed Central Ltd.
            <br/>
</p>
<p>
            This is an Open Access article distributed under the terms of the Creative Commons Attribution License (
            <a href="http://creativecommons.org/licenses/by/2.0">
             http://creativecommons.org/licenses/by/2.0
            </a>
            ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
           </p>
</div>
</div>
</div>
</div>
<div id="article-body">
<div>
<a name="abs">
</a>
<h3>
          Abstract
         </h3>
<div class="collapsible-content">
<h4>
           Background
          </h4>
<p style="line-height:160%">
           Small RNAs complex with proteins to mediate a variety of functions in animals and
         plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may
         carry out a signaling function by entering target cells and modulating gene expression.
         The subject of this study is a set of circulating 30–33 nt RNAs that are processed
         derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular
         tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit
         translation initiation in response to stress in cultured cells.
          </p>
<h4>
           Results
          </h4>
<p style="line-height:160%">
           In sequencing small RNAs extracted from mouse serum, we identified abundant 5′ tRNA
         halves derived from a small subset of tRNAs, implying that they are produced by tRNA
         type-specific biogenesis and/or release. The 5′ tRNA halves are not in exosomes or
         microvesicles, but circulate as particles of 100–300 kDa. The size of these particles
         suggest that the 5′ tRNA halves are a component of a macromolecular complex; this
         is supported by the loss of 5′ tRNA halves from serum or plasma treated with EDTA,
         a chelating agent, but their retention in plasma anticoagulated with heparin or citrate.
         A survey of somatic tissues reveals that 5′ tRNA halves are concentrated within blood
         cells and hematopoietic tissues, but scant in other tissues, suggesting that they
         may be produced by blood cells. Serum levels of specific subtypes of 5′ tRNA halves
         change markedly with age, either up or down, and these changes can be prevented by
         calorie restriction.
          </p>
<h4>
           Conclusions
          </h4>
<p style="line-height:160%">
           We demonstrate that 5′ tRNA halves circulate in the blood in a stable form, most likely
         as part of a nucleoprotein complex, and their serum levels are subject to regulation
         by age and calorie restriction. They may be produced by blood cells, but their cellular
         targets are not yet known. The characteristics of these circulating molecules, and
         their known function in suppression of translation initiation, suggest that they are
         a novel form of signaling molecule.
          </p>
</div>
</div>
<span id="keywords">
<h5 class="inline">
          Keywords:
         </h5>
         Aging; Calorie restriction; Circulating small RNAs; tRNA derivatives; tRNA halves; Serum; Plasma
        </span>
<div>
<a name="sec1">
</a>
<h3>
          Background
         </h3>
<div class="collapsible-content">
<p style="line-height:160%">
           Several classes of small RNAs have been found to mediate biological functions in animals
         and plants
           <a name="d7241e192">
</a>
<a name="d7241e194">
</a>
<a name="d7241e196">
</a>
<a name="d7241e198">
</a>
<a name="d7241e200">
</a>
           [
           <a href="#B1" onclick="LoadInParent('#B1'); return false;">
            1
           </a>
           -
           <a href="#B5" onclick="LoadInParent('#B5'); return false;">
            5
           </a>
           ]. miRNAs, siRNAs, piRNAs, and others are bound by Argonaute proteins, and have the
         common property of directing protein complexes to nucleic acids with sequence complementarity,
         where they may cleave or otherwise alter the target
           <a name="d7241e204">
</a>
           [
           <a href="#B6" onclick="LoadInParent('#B6'); return false;">
            6
           </a>
           ]. In both plants and animals, some small RNAs are able to travel between tissues within
         an organism, thus transferring their functions to other cells. In vertebrates, there
         has been much recent interest in the presence of specific miRNAs in the plasma and
         serum; there is some evidence that these can be taken up by cells and alter gene expression,
         and there is also interest in the possibility that they can be markers of specific
         disease states, including cancer
           <a name="d7241e208">
</a>
<a name="d7241e210">
</a>
<a name="d7241e212">
</a>
           [
           <a href="#B7" onclick="LoadInParent('#B7'); return false;">
            7
           </a>
           -
           <a href="#B9" onclick="LoadInParent('#B9'); return false;">
            9
           </a>
           ].
          </p>
<p style="line-height:160%">
           There is also evidence for processing of non-coding RNAs into smaller RNAs, many with
         as yet poorly understood functions
           <a name="d7241e218">
</a>
<a name="d7241e220">
</a>
           [
           <a href="#B10" onclick="LoadInParent('#B10'); return false;">
            10
           </a>
           ,
           <a href="#B11" onclick="LoadInParent('#B11'); return false;">
            11
           </a>
           ]. Many of the non-coding RNAs that appear to undergo processing into smaller RNAs
         have well studied functions, although their smaller derivatives often do not. In particular,
         tRNA is processed into shorter forms termed tRNA fragments (tRFs)
           <a name="d7241e224">
</a>
<a name="d7241e226">
</a>
           [
           <a href="#B12" onclick="LoadInParent('#B12'); return false;">
            12
           </a>
           ,
           <a href="#B13" onclick="LoadInParent('#B13'); return false;">
            13
           </a>
           ]. The subject of this report is a tRNA fragment created by cleavage of tRNA near the
         anticodon loop to create a ″5′ tRNA half” (the term we will use here). Previous reports
         have described 5′ tRNA halves as intracellular molecules interacting with components
         of the translation initiation complex. 5′ tRNA halves have been shown to be induced
         by the ribonuclease angiogenin in response to stress in cultured cells, to promote
         assembly of stress granules carrying stalled preinitiation complexes, and to inhibit
         mRNA translation
           <a name="d7241e230">
</a>
<a name="d7241e232">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B15" onclick="LoadInParent('#B15'); return false;">
            15
           </a>
           ]; little more is known about their function.
          </p>
<p style="line-height:160%">
           We have sequenced small RNAs present in mouse serum; when multiple reportable alignments
         of the sequencing reads to the mouse genome were allowed, we noted the presence of
         a class of tRNA-derived 30–33 nt fragments that closely resemble the 5′ tRNA halves
         previously described in stressed cell cultures. Investigation of these 5′ tRNA halves
         reveals a novel class of circulating small RNAs whose characteristics, including changes
         with age that are antagonized by calorie restriction, strongly suggest physiologic
         regulation and function.
          </p>
</div>
</div>
<div>
<a name="sec2">
</a>
<h3>
          Results and discussion
         </h3>
<div class="collapsible-content">
<h4>
           Sequencing and computational analysis of small RNAs circulating in mouse serum
          </h4>
<p style="line-height:160%">
           While investigating the effects of aging and calorie restriction (CR) on the profiles
         of cell-free small RNAs circulating in the bloodstream, we used small RNA-Seq (Illumina
         reads of 50 nt) to compare the serum levels of small RNAs from young and old control
         mice, and old mice subjected to CR. A combined total of 196,083,881 pre-processed
         sequencing reads obtained from 9 different serum samples, were mapped to the mouse
         genome with bowtie using parameters that align reads according to a policy similar
         to Maq’s default policy
           <a name="d7241e248">
</a>
           [
           <a href="#B16" onclick="LoadInParent('#B16'); return false;">
            16
           </a>
           ]. Alignment of the combined 196,083,881 pre-processed sequencing reads generated a
         dataset of 163,078,230 mapped reads (83.2%), ranging from 5 to 48 nt. The size distribution
         of the mapped reads revealed an expected peak at 20–24 nt consistent with the size
         of miRNAs (Figure 
           <a name="d7241e251">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           ).
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','F1',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-1.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 1.
              </strong>
</a>
<strong>
              Length distribution of reads obtained by deep sequencing of small RNAs extracted from
            mouse serum.
             </strong>
             Shown here are only those reads that map to the mm10 (GRCm38) mouse genome. (
             <strong>
              A
             </strong>
             ) Length distribution is displayed by abundance of sequencing reads combined from
         9 serum samples. Reads were mapped to the mouse genome with bowtie according to Maq’s
         default policy, either allowing (blue bars) or disallowing (red bars) multiple reportable
         alignments for each read. (
             <strong>
              B
             </strong>
             ) Combined reads were mapped to the mouse genome with bowtie according to the end-to-end
         k-difference policy, either allowing (blue bars) or disallowing (red bars) multiple
         reportable alignments for each read. (
             <strong>
              C
             </strong>
             ) Length distribution of separate sequencing reads obtained from 9 individual serum
         small RNA samples. Length distribution is displayed by abundance of sequencing reads
         that were mapped to the mouse genome with bowtie according to Maq’s default policy,
         and allowing multiple reportable alignments for each read. Bars with different colors
         denote the source of the sequenced serum small RNA from the 9 different samples.
            </p>
</div>
</div>
<p style="line-height:160%">
           Only if multiple reportable alignments are allowed during bowtie mapping does an unfamiliar
         second peak emerge at 30–33 nt (Figure 
           <a name="d7241e278">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           A). The 30–33 nt peak persists when the bowtie alignment mode is changed from the
         Maq’s default policy (n option) to the end-to-end k-difference policy (v option),
         but again disappears when multiple reportable alignments are suppressed (Figure 
           <a name="d7241e281">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           B). The same two-peak pattern was observed when the 9 individual sequenced serum small
         RNA samples were mapped to the mouse genome (Figure 
           <a name="d7241e284">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           C). Dependence of the 30–33 nt peak on multiple reportable alignments indicates that
         the reads are encoded by repetitive DNA. Six percent of the 163,078,230 mapped reads,
         aligned to a group of RepeatMasker classes (DNA, LINE, LTR, Low complexity, RC, SINE,
         Satellite, and Simple repeat); these reads were mainly &lt; 20 nt in size (Additional
         file
           <a name="d7241e287">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','',630,470); return false;">
            1
           </a>
           : Figure S1) and were not considered for further analysis.
          </p>
<div class="figs">
<div class="addfile">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','S1',630,470); return false;">
</a>
<strong>
              Additional file 1.
             </strong>
<strong>
              Additional Methods and Figures S1 and S2.
             </strong>
</p>
<p>
             Format: PDF 
         	Size: 6.8MB
             <a href="/content/supplementary/1471-2164-14-298-s1.pdf">
              Download file
             </a>
</p>
<p>
             This file can be viewed with:
             <a href="http://www.adobe.com/products/acrobat/readstep.html" onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;">
              Adobe Acrobat Reader
             </a>
<span class="open-data-icon">
<a href="/about/access/#opendata">
<img src="/bmcimages/article/opendata.gif" alt="Open Data" title="Open Data"/>
</a>
</span>
</p>
</div>
</div>
<p style="line-height:160%">
           Annotation analysis of the mapped sequencing reads revealed that the 30–33 nt peak
         consists of reads mapping to tRNA genes (Figure 
           <a name="d7241e303">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           A), which are present in multiple copies in the genome. Reads in the 20–24 nt peak
         were mostly annotated as miRNAs. Further analysis showed that of the total 163,078,230
         reads that mapped to the mouse genome, 128,703,415 (79%) map to sequences encoding
         small RNAs, of which 67% and 31% were annotated as tRNAs and miRNAs, respectively
         (Figure 
           <a name="d7241e306">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           B). The remaining &lt; 3% of reads mapped to sequences annotated as encoding rRNA and
         other small RNAs (scRNA, snRNA, srpRNA).
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','F2',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-2.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 2.
              </strong>
</a>
<strong>
              Annotation of combined sequencing reads of small RNAs extracted from 9 mouse serum
            samples.
             </strong>
             (
             <strong>
              A
             </strong>
             ) Length distribution by abundance of the total reads mapped to the mouse genome (black
         bars), with reads annotated as mapping to either miRNAs (red bars), tRNAs (blue bars),
         rRNAs (green bars), or other small RNAs (yellow bars). Other small RNAs include scRNA,
         snRNA, and srpRNA. Note that the reads in the 20–24 nt and the 30–33 nt peaks are
         almost exclusively annotated as miRNAs and tRNAs, respectively. (
             <strong>
              B
             </strong>
             ) A pie chart showing the percentage of reads mapping to the specific types of small
         RNAs. (
             <strong>
              C
             </strong>
             ) Frequencies of 5′ tRNA half types represented in the aligned reads.
            </p>
</div>
</div>
<h4>
           Characterization of circulating small RNAs derived from tRNAs
          </h4>
<p style="line-height:160%">
           Since the 86,343,437 reads that align to tRNA genes are only 30 to 33 nt, and thus
         do not represent full length tRNAs, we examined the tRNA end distribution of the reads,
         and annotated the reads based on their overlap with 5′ or 3′ ends of tRNAs. More than
         99% of the tRNA-derived reads align with the 5′ end of a tRNA; this is exemplified
         in Figure 
           <a name="d7241e337">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F3','',800,470); return false;">
            3
           </a>
           for two tRNA genes.
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F3','F3',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-3.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 3.
              </strong>
</a>
<strong>
              UCSC genome browser screenshots illustrating alignment of reads to two tRNA genes.
             </strong>
             Shown are the Illumina sequencing reads (red), and the tRNA genes (blue) as annotated
         in the tRNA genes track ”Transfer RNA genes identified with tRNAscan-SE“. (
             <strong>
              A
             </strong>
             ) The alignment (number of reads, y-axis) shows that all the sequencing reads align
         to the 5′ end of chr1.tRNA704-GlyGCC gene. (
             <strong>
              B
             </strong>
             ) The majority of the sequencing reads align to the 5′ end of chr11.tRNA945-ArgCCG
         gene, whereas only a very small number of sequencing reads aligns to the 3′ end.
            </p>
</div>
</div>
<p style="line-height:160%">
           23%, 17%, 35%, and 26% of the sequencing reads that map to tRNAs are 30, 31, 32, and
         33 nucleotide in size, respectively (Table 
           <a name="d7241e361">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T1','',800,470); return false;">
            1
           </a>
           ), indicating that full length tRNAs are cleaved in the anticodon loop at more than
         one site and at varying rates to generate the 5′ tRNA halves found in serum. As an
         example, Figure 
           <a name="d7241e364">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F4" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F4','',800,470); return false;">
            4
           </a>
           depicts the size frequency of reads mapping to the 5′ end of the chr1.tRNA704-GlyGCC
         gene; this indicates that this tRNA is cleaved at different rates and at 4 different
         sites located upstream of the GCC anticodon in the anticodon loop.
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F4" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F4','F4',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-4.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 4.
              </strong>
</a>
<strong>
              Cleavage sites of tRNAs.
             </strong>
             The cloverleaf structure of chr1.tRNA704-GlyGCC gene (downloaded from
             <a href="http://gtrnadb.ucsc.edu">
              http://gtrnadb.ucsc.edu
             </a>
<a alt="" href="http://www.webcitation.org/query.php?url=http://gtrnadb.ucsc.edu&amp;refdoi=10.1186/1471-2164-14-298" class="xpushbutton" title="Archive copy of webpage">
              webcite
             </a>
             ) showing cleavage sites at the anticodon loop, with the percentage of the reads that
         map to the 5′ end of the tRNA (arrowheads). The cleavage sites are upstream of the
         GCC anticodon located at nucleotides 33 to 35. Numbers inside the anticodon loop indicate
         anticodon nucleotides positions.
            </p>
</div>
<div class="table">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T1','T1',800,470); return false;">
<strong>
               Table 1.
              </strong>
</a>
<strong>
              Total number and percentage of the different sizes of sequencing reads that map to
            tRNAs
             </strong>
</p>
</div>
</div>
<p style="line-height:160%">
           It is unlikely that this result is a sequencing artifact: the full length of most
         tRNAs is 75–90 nt, and the sequencing runs used to generate these data were 50 cycles
         while the reads occupy a narrow size range of 30–33 nt. This pattern suggests that
         the tRNA reads were derived from processed fragments of full length tRNAs; the remainder
         of the tRNA was not significantly detected in the serum small RNA libraries. In support
         of this conclusion, tRNAs have been shown to undergo cleavage within anticodon loops
         to produce tRNA-derived stress-induced fragments (tiRNAs) when cultured cells are
         subjected to stresses such as arsenite, heat shock, or ultraviolet irradiation
           <a name="d7241e453">
</a>
<a name="d7241e455">
</a>
           [
           <a href="#B17" onclick="LoadInParent('#B17'); return false;">
            17
           </a>
           ,
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. Such cleavage of the anticodon loop does not seem to be part of a tRNA degradation
         process, because the generated 5′ tRNA fragments are stable in the cell. Our findings
         indicate that tRNA fragments highly similar to tiRNAs are present under normal (unstressed)
         conditions, and can remain stable even after they are released into the peripheral
         blood. 5′ but not 3′ tRNA fragments inhibit mRNA translation initiation in cultured
         cell lines
           <a name="d7241e459">
</a>
           [
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ].
          </p>
<p style="line-height:160%">
           The individual 5′ tRNA halves present in serum are derived from a small subset of
         tRNAs (Figure 
           <a name="d7241e464">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           C). The most abundant circulating tRNA halves were derived from the isoacceptors of
         glycine (46%), valine (44%), glutamine (8%), and histidine (1%), and the remaining
         amino acids together represented &lt;1%. We contrasted the number of tRNA genes in the
         Genomic tRNA Database
           <a name="d7241e468">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ], with the relative abundances of the circulating 5′ tRNA halves, and found no correlation
         (Table 
           <a name="d7241e471">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T2','',800,470); return false;">
            2
           </a>
           ). For example, the most abundant circulating 5′ tRNA halves were derived from tRNA-Gly,
         and the copy number of tRNA-Gly gene is 29; on the other hand tRNA-Cys genes, with
         a copy number of 57, generate &lt;1% of the 5′ tRNA-Cys halves.
          </p>
<div class="figs">
<div class="table">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T2','T2',800,470); return false;">
<strong>
               Table 2.
              </strong>
</a>
<strong>
              Frequencies of circulating 5′ tRNA halves and the gene copy number of tRNAs from which
            the tRNA halves were derived
             </strong>
</p>
</div>
</div>
<p style="line-height:160%">
           This implies a tRNA type-specific biogenesis and/or release of the circulating 5′
         tRNA halves.
          </p>
<h4>
           Presence in circulating mouse blood of particles containing stable cell-free 5′ tRNA
         halves
          </h4>
<p style="line-height:160%">
           To obtain an independent validation of the sequencing results, we used Northern blotting
         to analyze small RNAs circulating in the mouse serum. As a positive control for detection
         of tRNA halves by Northern blotting, we included RNA from U2OS cells cultured in the
         absence or presence of sodium arsenite, which is known to generate tRNA halves in
         these cells
           <a name="d7241e563">
</a>
           [
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. We probed RNA from mouse serum with oligonucleotides complementary to 5′ or 3′ ends
         of specific tRNAs. Probes specific for the 5′ ends of tRNA-Gly-GCC or tRNA-Val-CAC
         detected a band migrating near the 30 nt RNA marker (Figure 
           <a name="d7241e566">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           A and C) confirming the presence of stable circulating 5′ tRNA halves. No significant
         bands migrated with the 30 nt RNA marker when the same Northern blot was probed for
         the 3′ end of tRNA-Gly-GCC or tRNA-Val-CAC (Figure 
           <a name="d7241e569">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           B and D).
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','F5',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-5.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 5.
              </strong>
</a>
<strong>
              Detection of 5′ tRNA halves in mouse serum.
             </strong>
             Northern blot analysis of RNA extracted from U2OS cells cultured in the absence (−)
         or presence (+) of sodium arsenite (AS), or from 0.4 ml of mouse serum. The blot was
         hybridized to a
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong>
              A
             </strong>
             ) or 3′ end (
             <strong>
              B
             </strong>
             ) of tRNA-Gly-GCC. The blot was also hybridized to a
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong>
              C
             </strong>
             ) or 3′ end (
             <strong>
              D
             </strong>
             ) of tRNA-Val-CAC. 5′ tRNA halves were also detected in fractionated mouse serum.
         (
             <strong>
              E
             </strong>
             -
             <strong>
              F
             </strong>
             ) Northern blotting analysis was carried out on RNA extracted from either 0.4 ml of
         mouse whole serum, from the supernatant (Sup) after ultracentrifugation of 0.4 ml
         of mouse serum at 110000g, or the ultracentrifugation pellet. The blot was hybridized
         to a
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong>
              E
             </strong>
             ) or 3′ end (
             <strong>
              F
             </strong>
             ) of tRNA-Gly-GCC. (
             <strong>
              G
             </strong>
             ) Ultrafiltration indicates a size for tRNA serum particles between 100 and 300 kDa.
         Samples of 0.2 ml serum mixed with 1.8 ml PBS were subjected to ultrafiltration through
         Vivaspin 2 columns with 30, 100, and 300 kDa MW cut-offs. Total RNAs were extracted
         from filtrate (f) and concentrate (c) fractions. Blot was hybridized to
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ end of tRNA-Gly-GCC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<p style="line-height:160%">
           We also probed RNA from mouse serum with a probe complementary to the 5′ end of tRNA-Asn-GTT
         to confirm the low abundance of circulating tRNA halves derived from tRNAs that were
         barely detected in the sequencing data. A 5-day exposure to X-ray film showed a very
         weak signal from tRNA-Asn-GTT probe compared to the strong signal from the tRNA-Gly-GCC
         probe obtained after a short (25 minute) exposure (Additional file
           <a name="d7241e628">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','',630,470); return false;">
            1
           </a>
           : Figure S2). These results are consistent with the sequencing, and inconsistent with
         a sequencing bias. They imply a tRNA type-specific biogenesis and/or release of the
         circulating 5′ tRNA halves.
          </p>
<p style="line-height:160%">
           We next asked if the 5′ tRNA halves are contained within circulating exosomes or microvesicles.
         We Northern blotted RNA extracted from pellet or supernatant after ultracentrifugation
         of mouse serum at 110000g for 2 hours. A probe for the 5′ end of tRNA-Gly-GCC detected
         an ~30 nt band present mainly in the supernatant and visible only as a trace in the
         pellet (Figure 
           <a name="d7241e633">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           E), while a probe for the 3′ end did not detect any significant signal (Figure 
           <a name="d7241e636">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           F). Identical results were obtained for the 5′ end of tRNA-Val-CAC (Figure 
           <a name="d7241e639">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','',800,470); return false;">
            6
           </a>
           A). These findings indicate that the 5′tRNA halves are mostly not included in exosomes
         or microvesicles, which would pellet in these conditions. Similarly, exosome encapsulation
         is not required for the stability of circulating miRNAs; after pelleting exosomes
         by ultracentrifugation of plasma, miRNAs were still detected in the supernatant fraction
           <a name="d7241e643">
</a>
<a name="d7241e645">
</a>
           [
           <a href="#B20" onclick="LoadInParent('#B20'); return false;">
            20
           </a>
           ,
           <a href="#B21" onclick="LoadInParent('#B21'); return false;">
            21
           </a>
           ].
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','F6',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-6.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 6.
              </strong>
</a>
<strong>
              Northern blotting analysis of tRNA-Val-CAC halves in mouse serum.
             </strong>
             (
             <strong>
              A
             </strong>
             ) RNA extracted from 0.4 ml of mouse whole serum, from supernatant or pellet after
         ultracentrifugation of 0.4 ml of mouse serum at 110000g and analyzed with northern
         blotting by hybridization to a
             <sup>
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Val-CAC.
         (
             <strong>
              B
             </strong>
             ) RNA extracted from 0.2 ml serum mixed with 1.8 ml PBS subjected to ultrafiltration
         with Vivaspin 2 columns with 30, 100, and 300 kDa MW cut-off. Total RNAs were extracted
         from filtrate (f) and concentrate (c) fractions. Blot was hybridized to a
             <sup>
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Val-CAC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<p style="line-height:160%">
           Because the tRNA halves we observe are stable in circulation but not encapsulated
         in exosomes, they are most likely complexed to carrying factors (e.g., proteins that
         protect them from degradation). To determine the size range of the putative complexes
         carrying the 5′ tRNA halves in the serum, we Northern blotted RNA extracted from concentrate
         or filtrate fractions after ultrafiltration of mouse serum samples through Vivaspin
         2 columns with 30, 100, or 300 kDa MW cut-off. A probe for the 5′ end of tRNA-Gly-GCC
         detected a ~30 nt band in the concentrates of 30 and 100 kDa MW cut-off, and in the
         filtrate of 300 kDa MW cut-off (Figure 
           <a name="d7241e675">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           G). Identical results were obtained for the 5′ end of tRNA-Val-CAC (Figure 
           <a name="d7241e678">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','',800,470); return false;">
            6
           </a>
           B).
          </p>
<p style="line-height:160%">
           Thus 5′ tRNA halves circulate as part of 100–300 kDa complexes, while the 5′ tRNA
         halves themselves are only ~10 kDa. This is reminiscent of reports that miRNAs can
         circulate in the bloodstream as components of RNA-protein/lipoprotein complexes. Stable
         argonaute2-miRNA complexes that are not part of microvesicles were recovered from
         plasma and serum, and high-density lipoprotein has been reported to carry and deliver
         miRNAs to recipient cells
           <a name="d7241e684">
</a>
<a name="d7241e686">
</a>
<a name="d7241e688">
</a>
           [
           <a href="#B20" onclick="LoadInParent('#B20'); return false;">
            20
           </a>
           -
           <a href="#B22" onclick="LoadInParent('#B22'); return false;">
            22
           </a>
           ].
          </p>
<h4>
           5′ tRNA halves are concentrated in hematopoietic and lymphoid tissues
          </h4>
<p style="line-height:160%">
           To investigate whether 5′ tRNA halves are present in tissues we extracted total RNA
         from liver, spleen, and testes, and did Northern blots with probes complementary to
         5′ and 3′ ends of tRNAs. We detected tRNA halves with a probe complementary to the
         5′ end of tRNA-Gly-GCC in the spleen, but not in the liver and testes; a probe for
         the 3′ end tRNAs detected only full length tRNAs in all 3 tissues (Figure 
           <a name="d7241e697">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','',800,470); return false;">
            7
           </a>
           A-B). This prompted us to explore the possibility that 5′ tRNA halves are present
         specifically in hematopoietic tissues. Northern blotting of several mouse tissues
         confirmed that 5′ tRNA halves are present in hematopoietic and lymphoid tissues including
         spleen, lymph nodes, fetal liver, leukocytes, bone marrow, and thymus, but almost
         absent in non-immune tissues including testes, liver, heart, brain, and kidney (Figure 
           <a name="d7241e700">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','',800,470); return false;">
            7
           </a>
           ); the presence of 3′ tRNA halves was not significant in any tissue. This finding
         is consistent with a previous report
           <a name="d7241e704">
</a>
           [
           <a href="#B23" onclick="LoadInParent('#B23'); return false;">
            23
           </a>
           ], in which tRNA halves were unexpectedly detected on cloning of microRNAs from human
         fetal liver, which is the main hematopoietic organ during fetal development. Identical
         results were obtained when the same Northern blots were probed for the 5′ and 3′ ends
         of tRNA-Val-CAC (Figure 
           <a name="d7241e707">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F8" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F8','',800,470); return false;">
            8
           </a>
           ).
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','F7',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-7.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 7.
              </strong>
</a>
<strong>
              Tissue distribution of tRNA-Gly-GCC halves.
             </strong>
             Northern blotting analysis of RNA extracted from the indicated mouse tissues. Blots
         were hybridized with
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ (
             <strong>
              A, C
             </strong>
             , and
             <strong>
              E
             </strong>
             ) or 3′ end (
             <strong>
              B, D
             </strong>
             , and
             <strong>
              F
             </strong>
             ) of tRNA-Gly-GCC. The positions of full length tRNAs and tRNA halves are indicated
         on the right. DM: decade markers.
            </p>
</div>
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F8" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F8','F8',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-8.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 8.
              </strong>
</a>
<strong>
              Tissue distribution of tRNA-Val-CAC halves.
             </strong>
             Northern blotting analysis of RNA extracted from the indicated mouse tissues. Blots
         were hybridized with
             <sup>
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ (
             <strong>
              A
             </strong>
             ,
             <strong>
              B
             </strong>
             , and
             <strong>
              D
             </strong>
             ) or 3′ end (
             <strong>
              C
             </strong>
             and
             <strong>
              E
             </strong>
             ) of tRNA-Val-CAC. The positions of full length tRNAs and tRNA halves are indicated
         on the right. DM: decade markers.
            </p>
</div>
</div>
<p style="line-height:160%">
           More extensive studies will establish if 5′ tRNA halves are concentrated in particular
         blood cell types, although the very high levels in lymph nodes point to lymphocytes
         as one such type. The evidence does not establish whether the 5′ tRNA halves are concentrated
         in hematopoietic cells because they are produced there, or because they are preferentially
         taken up from the blood: neither the origin nor the destinations of the 5′ tRNA halves
         is certain. The low levels of 5′ tRNA halves present in non-hematopoietic tissues
         may indicate low levels in those tissues, but they may also be derived from residual
         blood cells in those tissues.
          </p>
<h4>
           A chelating agent destabilizes circulating 5′ tRNA halves
          </h4>
<p style="line-height:160%">
           Because clotting has the potential to release particles that are not present in circulating
         blood, we asked if 5′ tRNA halves circulating in the mouse serum are also present
         in mouse plasma. Northern blotting with a 5′ tRNA half probe gave a very weak band
         in a plasma sample when compared to the band derived from an equal volume of serum
         from the same mouse (Figure 
           <a name="d7241e778">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','',800,470); return false;">
            9
           </a>
           ). The lack of 5′ tRNA halves in plasma is not due to a global loss of small RNAs
         during preparation of the plasma, which was anticoagulated with EDTA. We used qPCR
         to assess the integrity of two circulating miRNAs in mouse serum, serum treated with
         EDTA, and plasma collected with EDTA. As shown by amplification in all three specimens
         (Figure 
           <a name="d7241e781">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F10" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F10','',800,470); return false;">
            10
           </a>
           ), EDTA does not affect these circulating miRNAs.
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','F9',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-9.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 9.
              </strong>
</a>
<strong>
              Detection of 5′ tRNA halves in mouse plasma and serum.
             </strong>
             Northern blot of RNAs extracted from 0.4 ml of mouse serum, serum treated with EDTA,
         and heparin- or EDTA-collected plasma, hybridized to a
             <sup>
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Gly-GCC.
         EDTA sharply lowers the abundance of 5′ tRNA halves in serum; in plasma, 5′ tRNA halves
         are abundant when heparin is the anticoagulant, but are nearly absent when EDTA is
         present. 5′ tRNA halves are similarly abundant when calcium citrate is used as the
         anticoagulant (not shown).
            </p>
</div>
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F10" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F10','F10',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-10.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 10.
              </strong>
</a>
<strong>
              Real-time PCR amplification plots of circulating miRNAs.
             </strong>
             Shown are the amplification plots for miR-16 (blue), miR-24 (black), and the spiked-in
         miR-Cel-39 (red) measured in mouse serum (
             <strong>
              A
             </strong>
             ), mouse serum treated with EDTA (
             <strong>
              B
             </strong>
             ), and mouse plasma collected on EDTA (
             <strong>
              C
             </strong>
             ). The y-axis represents the relative fluorescence units (RFU) in a semi-log scale.
         The x-axis represents the cycle at which fluorescence was detected above an automatically
         determined threshold for the indicated miRNA. EDTA does not change the concentration
         of miRNAs in plasma.
            </p>
</div>
</div>
<p style="line-height:160%">
           This result could suggest that 5′ tRNA halves are an artifact of blood clotting, but
         could also be an effect of EDTA, a chelating agent that depletes ions required for
         clotting. To assess the effects of EDTA on 5′ tRNA halves, we used Northern blotting
         to analyse a sample of serum that was incubated with EDTA for 15 min before RNA extraction.
         We also analyzed a sample of plasma extracted from blood collected with heparin, a
         nonchelating anticoagulant. This analysis showed that treatment of serum with EDTA
         significantly decreased the signal corresponding the 5′ tRNA halves, while 5′ tRNA
         halves are abundant in heparinized plasma (Figure 
           <a name="d7241e824">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','',800,470); return false;">
            9
           </a>
           ). The same results were obtained with RNAs from human plasma collected on EDTA and
         from serum (Figure 
           <a name="d7241e827">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F11" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F11','',800,470); return false;">
            11
           </a>
           ). These findings suggest that chelation of ions by EDTA destabilizes the complexes
         carrying 5′ tRNA halves, exposing the RNA to ribonucleases which are abundant in plasma.
          </p>
<div class="figs">
<div class="fig">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F11" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F11','F11',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-11.gif" alt="thumbnail" class="thumbnail" align="top"/>
<strong>
               Figure 11.
              </strong>
</a>
<strong>
              5′ tRNA halves are also present in human serum and leukocytes, but not in EDTA plasma.
             </strong>
             RNAs extracted from human leukocytes, 0.4 ml of human serum, or EDTA-collected plasma.
         The northern blot was hybridized to a
             <sup>
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Gly-GCC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<h4>
           Calorie restriction offsets age-associated changes in levels of specific circulating
         5′ tRNA halves
          </h4>
<p style="line-height:160%">
           Calorie restriction (CR) can delay, prevent, or reverse many age-associated changes
         in physiologic parameters. We used aging and CR as model physiologic states to explore
         the possibility that they are associated with changes in the levels of circulating
         5′ tRNA halves. We performed pairwise comparisons between young and old control groups
         to measure the differential abundance in circulating 5′ tRNA halves associated with
         old age, and between old control and old CR groups to determine whether CR has an
         effect on any age-associated changes.
          </p>
<p style="line-height:160%">
           This analysis revealed that aging is associated with alterations, either increase
         or decrease, in the circulating levels of 5′ tRNA halves derived from specific tRNA
         isoacceptors (Table 
           <a name="d7241e854">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','',800,470); return false;">
            3
           </a>
           ). Notably, CR mitigated most of these age-related changes (Table 
           <a name="d7241e857">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','',800,470); return false;">
            3
           </a>
           ), although it did not completely prevent them. CR has been shown to oppose the molecular
         and biological markers of aging including alterations in gene expression
           <a name="d7241e861">
</a>
           [
           <a href="#B24" onclick="LoadInParent('#B24'); return false;">
            24
           </a>
           ]. A causal relationship between circulating 5′ tRNA halves and the manifestations
         of aging is not established by this study, but it does indicate that levels are regulated
         in an age-associated fashion.
          </p>
<div class="figs">
<div class="table">
<p>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','T3',800,470); return false;">
<strong>
               Table 3.
              </strong>
</a>
<strong>
              Age-associated changes in the levels of mouse circulating 5′ tRNA halves and the effects
            of CR on the age-associated changes
             </strong>
</p>
</div>
</div>
</div>
</div>
<div>
<a name="sec3">
</a>
<h3>
          Conclusions
         </h3>
<div class="collapsible-content">
<p style="line-height:160%">
           Deep sequencing of small RNAs extracted from mouse serum identifies a population of
         tRNA-derived molecules, termed 5′ tRNA halves, previously described only as stress-induced
         inhibitors of translation initiation in cultured cells. 5′ tRNA halves are more abundant
         than miRNAs in mouse serum, and are derived from distinct subset of tRNAs by cleavage
         near the anticodon loop; the 3′ portion of the tRNA molecule is present in serum only
         in trace quantities. Ultracentrifugation and size fractionation establish that the
         5′ tRNA halves circulate as part of a larger complex, but are not contained in exosomes
         or microvesicles; their sensitivity to the chelating agent EDTA provides further evidence
         that they exist as circulating nucleoprotein complexes. They are concentrated in hematopoietic
         and lymphoid tissues, and present in other tissues at very low levels that may reflect
         residual blood cells. The origin of the serum particles, and their destinations, are
         uncertain; however their concentration in blood cells suggest that they may be produced
         by these cells. Levels of serum 5′ tRNA halves are distinctly changed in aged mice,
         and calorie restriction inhibits these changes, indicating that they are subject to
         physiologic regulation. Taken together with the extant evidence that 5′ tRNA halves
         can regulate mRNA translation, the characteristics of the circulating 5′ tRNA halves
         we have discovered suggest that they function as signaling molecules with as yet unknown
         physiologic roles.
          </p>
<p style="line-height:160%">
           To date, the only known function of 5′ tRNA halves is inhibition of translation in
         cultured cells subjected to a variety of stressors; transfection of 5′ tRNA halves
         inhibits global translation in U2OS cells
           <a name="d7241e1609">
</a>
<a name="d7241e1611">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. A study published while this paper was in preparation reported induction of 5′ tRNA
         halves in human airway epithelial cells upon infection with respiratory syncytial
         virus (RSV). Induction involves cleavage at the tRNA anticodon loop by angiogenin,
         and at least one type, the 5′ tRNA-Glu-CTC half, promotes RSV replication
           <a name="d7241e1615">
</a>
           [
           <a href="#B25" onclick="LoadInParent('#B25'); return false;">
            25
           </a>
           ]. Our findings indicate that 5′ tRNA halves function on an organismal rather than
         merely a cellular level. Furthermore they are likely to function in a context much
         broader than cellular stress or infection: we find 5′ tRNA halves in unstressed conditions.
         Changes in their expression (either increased or decreased) with age are also consistent
         with a broader physiologic role, and it is particularly interesting that these changes
         are partially mitigated by calorie restriction.
          </p>
<p style="line-height:160%">
           The most extensively studied cellular tRNA halves are generated under stress conditions
         by angiogenin, which cleaves mature tRNAs within the anticodon loops
           <a name="d7241e1621">
</a>
           [
           <a href="#B26" onclick="LoadInParent('#B26'); return false;">
            26
           </a>
           ]. The stress-induced tRNA halves target the translation initiation machinery to reprogram
         protein translation in order to promote cell survival during stress
           <a name="d7241e1625">
</a>
<a name="d7241e1627">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B26" onclick="LoadInParent('#B26'); return false;">
            26
           </a>
           ]. Pull-down and mass spectrometry analyses of RNA-protein complexes have identified
         several cellular proteins (YB-1, FXR-1, and PABP1) bound to intracellular 5′ tRNA
         halves
           <a name="d7241e1631">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ]. The nature of the proteins and/or other factors that bind and stabilize the extracellular
         form of 5′ tRNAs halves has yet to be elucidated. Understanding of the origin, composition,
         and destinations of these complexes will provide insights into their role in organismal
         physiology.
          </p>
</div>
</div>
<div>
<a name="sec4">
</a>
<h3>
          Methods
         </h3>
<div class="collapsible-content">
<h4>
           Serum collection, RNA isolation, and small RNA library construction
          </h4>
<p style="line-height:160%">
           Male mice of the long-lived B6C3F1 strain were fed either control or calorie-restricted
         (CR) diet (∼40% fewer calories than the control). Three mice were studied from each
         of three groups: young (7-month) and old (27-month) mice fed the control diet, and
         old (27-month) mice fed the CR diet. Total RNA including small RNA was isolated from
         each serum sample with miRNeasy kit (Qiagen) and used to construct indexed sequencing
         libraries with the Illumina TruSeq Small RNA Sample Prep Kit. The libraries were pooled
         and sequenced on an Illumina HiSeq 2000 instrument to generate 50 base reads. Further
         details about the mice and diets, and library construction are provided in SI Methods
         and Figures.
          </p>
<h4>
           Mapping and annotation of sequencing reads
          </h4>
<p style="line-height:160%">
           Sequencing reads were pre-processed with FASTX-Toolkit (hannonlab.cshl.edu) to trim
         the adaptor sequences, and discard low quality reads. The obtained clean reads were
         mapped to the mouse reference genome (GRCm38/mm10) with bowtie version 0.12.8
           <a name="d7241e1651">
</a>
           [
           <a href="#B16" onclick="LoadInParent('#B16'); return false;">
            16
           </a>
           ] using different combinations of alignment and reporting options. We used the option
         ”-n 0 -l 14“ to align the sequencing reads according to a policy similar to Maq‘s
         default policy and requiring no mismatches in the first 14 bases (the high-quality
         end of the read). In addition, this mode of alignment was combined with options that
         define which and how many alignments should be reported; the option “-k 1 --best”
         instructed bowtie to report only the best alignment if more than one valid alignment
         exists, while the option “-m 1” instructed bowtie to refrain from reporting any alignments
         for reads having multiple reportable alignments. The “-k 1 --best” and “-m 1” modes
         of alignment reporting were also used in combination with the end-to-end k-difference
         (−v) alignment mode. Varying the alignment and reporting modes allowed the differential
         detection of two predominant peak sizes of sequencing reads as described in the results
         section.
          </p>
<p style="line-height:160%">
           Annotation analysis of the mapped sequencing reads was performed with bedtools
           <a name="d7241e1657">
</a>
           [
           <a href="#B27" onclick="LoadInParent('#B27'); return false;">
            27
           </a>
           ] using the following databases: the Genomic tRNA Database
           <a name="d7241e1661">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ] (gtrnadb.ucsc.edu), miRBase 18 (mirbase.org), and rRNA, snRNA, scRNA, and srpRNA
         which were extracted from the RepeatMasker track (genome.ucsc.edu; mm10).
          </p>
<h4>
           Analysis of differentially abundant circulating tRNA halves
          </h4>
<p style="line-height:160%">
           The bowtie alignment files generated above from the young and old control and old
         CR serum sequencing samples were analyzed with bedtools
           <a name="d7241e1671">
</a>
           [
           <a href="#B27" onclick="LoadInParent('#B27'); return false;">
            27
           </a>
           ] to obtain the coverage of the tRNA genes included in the Genomic tRNA Database
           <a name="d7241e1675">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ] (gtrnadb.ucsc.edu), and to determine the read count for each tRNA in the database.
         The tRNA read counts were further analyzed with the Bioconductor package edgeR
           <a name="d7241e1679">
</a>
           [
           <a href="#B28" onclick="LoadInParent('#B28'); return false;">
            28
           </a>
           ] to detect the changes in the levels of circulating 5′ tRNA halves in the different
         experimental groups. The algorithm of edgeR fits a negative binomial model to the
         count data, estimates dispersion, and measures differences using the generalized linear
         model likelihood ratio test which is recommended for experiments with multiple factors,
         such as the simultaneous analysis of age and diet in our study. The fitted count data
         was analyzed by performing pairwise comparisons between the different experimental
         groups: young and old control groups were compared to measure the differential abundance
         in circulating 5′ tRNA halves associated with old age; old control and old CR groups
         were compared to determine whether CR has an effect on any age-associated changes.
         The results were further filtered to keep only 5′ tRNA halves that achieved a minimum
         of 500 counts per million (cpm) in at least one of the 3 experimental groups.
          </p>
<h4>
           Northern blot assays
          </h4>
<p style="line-height:160%">
           RNAs analyzed with Northern blots were extracted from normal or sodium arsenite-treated
         U2OS and from a variety of tissues and sera harvested from one-year-old mice fed control
         diet. Before RNA extraction, some serum samples were centrifuged at 110000 g for 2
         hrs, and supernatant and pellet fractions were separated, or were separated into concentrate
         and filtrate fractions by ultrafiltration through Vivaspin 2 columns with 30, 100,
         or 300 kDa MW cut-off. RNAs were separated on 15% denaturing polyacrylamide gels,
         transferred and fixed to a membrane by chemical cross-linking
           <a name="d7241e1689">
</a>
           [
           <a href="#B29" onclick="LoadInParent('#B29'); return false;">
            29
           </a>
           ], and hybridized with probes complementary to 5’ and 3′ ends of tRNAs. Further details
         about Northern blot assays, and probe sequences are provided in SI Methods and Figures.
          </p>
</div>
</div>
<div>
<a name="sec5">
</a>
<h3>
          Competing interests
         </h3>
<div class="collapsible-content">
<p style="line-height:160%">
           The authors declare that they have no competing interests.
          </p>
</div>
</div>
<div>
<a name="sec6">
</a>
<h3>
          Authors’ contributions
         </h3>
<div class="collapsible-content">
<p style="line-height:160%">
           J.M.D conceived and designed the study, carried out the bioinformatics analysis, and
         drafted the manuscript. SRS participated in the design of the study and the interpretation
         of the results. HA participated in the data analysis and interpretation of the results.
         AY, DB, and PM performed the experiments. DIKM participated in the interpretation
         of the results, and helped to draft the manuscript. All authors read and approved
         the final manuscript.
          </p>
</div>
</div>
<div>
<a name="ack">
</a>
<h3>
          Acknowledgements
         </h3>
<div class="collapsible-content">
<p style="line-height:160%">
           We thank Noel Guerrero for his help.
          </p>
</div>
</div>
<div>
<a name="refs">
</a>
<h3>
          References
         </h3>
<div class="collapsible-content" id="article-references">
<ol id="references">
<li id="B1">
<p>
<a name="B1">
</a>
             Okamura K:
             <strong>
              Diversity of animal small RNA pathways and their biological utility.
             </strong>
</p>
<p>
<em>
              Wiley Interdiscip Rev RNA
             </em>
             2012,
             <strong>
              3
             </strong>
             (3)
             <strong>
              :
             </strong>
             351-368.
             <a href="/pubmed/22086843" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22086843" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B1','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B2">
<p>
<a name="B2">
</a>
             Wery M,  Kwapisz M,  Morillon A:
             <strong>
              Noncoding RNAs in gene regulation.
             </strong>
</p>
<p>
<em>
              Wiley Interdiscip Rev Syst Biol Med
             </em>
             2011,
             <strong>
              3
             </strong>
             (6)
             <strong>
              :
             </strong>
             728-738.
             <a href="/pubmed/21381218" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21381218" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B2','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B3">
<p>
<a name="B3">
</a>
             Zhang C:
             <strong>
              Novel functions for small RNA molecules.
             </strong>
</p>
<p>
<em>
              Curr Opin Mol Ther
             </em>
             2009,
             <strong>
              11
             </strong>
             (6)
             <strong>
              :
             </strong>
             641-651.
             <a href="/pubmed/20072941" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20072941" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B3','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B4">
<p>
<a name="B4">
</a>
             Zhang S,  Sun L,  Kragler F:
             <strong>
              The phloem-delivered RNA pool contains small noncoding RNAs and interferes with translation.
             </strong>
</p>
<p>
<em>
              Plant Physiol
             </em>
             2009,
             <strong>
              150
             </strong>
             (1)
             <strong>
              :
             </strong>
             378-387.
             <a href="/pubmed/19261735" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19261735" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19261735" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B4','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B5">
<p>
<a name="B5">
</a>
             Esteller M:
             <strong>
              Non-coding RNAs in human disease.
             </strong>
</p>
<p>
<em>
              Nat Rev Genet
             </em>
             2011,
             <strong>
              12
             </strong>
             (12)
             <strong>
              :
             </strong>
             861-874.
             <a href="/pubmed/22094949" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22094949" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B5','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B6">
<p>
<a name="B6">
</a>
             Joshua-Tor L,  Hannon GJ:
             <strong>
              Ancestral roles of small RNAs: an Ago-centric perspective.
             </strong>
</p>
<p>
<em>
              Cold Spring Harb Perspect Biol
             </em>
             2011,
             <strong>
              3
             </strong>
             (10)
             <strong>
              :
             </strong>
             a003772.
             <a href="/pubmed/20810548" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20810548" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B6','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B7">
<p>
<a name="B7">
</a>
             Allegra A,  Alonci A,  Campo S,  Penna G,  Petrungaro A,  Gerace D,  Musolino C:
             <strong>
              Circulating microRNAs: New biomarkers in diagnosis, prognosis and treatment of cancer
                  (Review).
             </strong>
</p>
<p>
<em>
              Int J Oncol
             </em>
             2012,
             <strong>
              41
             </strong>
             (6)
             <strong>
              :
             </strong>
             1897-1912.
             <a href="/pubmed/23026890" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23026890" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B7','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B8">
<p>
<a name="B8">
</a>
             Etheridge A,  Lee I,  Hood L,  Galas D,  Wang K:
             <strong>
              Extracellular microRNA: a new source of biomarkers.
             </strong>
</p>
<p>
<em>
              Mutat Res
             </em>
             2011,
             <strong>
              717
             </strong>
             (1–2)
             <strong>
              :
             </strong>
             85-90.
             <a href="/pubmed/21402084" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21402084" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21402084" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B8','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B9">
<p>
<a name="B9">
</a>
             Zen K,  Zhang CY:
             <strong>
              Circulating microRNAs: a novel class of biomarkers to diagnose and monitor human cancers.
             </strong>
</p>
<p>
<em>
              Medicinal research reviews
             </em>
             2012,
             <strong>
              32
             </strong>
             (2)
             <strong>
              :
             </strong>
             326-348.
             <a href="/pubmed/22383180" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22383180" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B9','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B10">
<p>
<a name="B10">
</a>
             Rother S,  Meister G:
             <strong>
              Small RNAs derived from longer non-coding RNAs.
             </strong>
</p>
<p>
<em>
              Biochimie
             </em>
             2011,
             <strong>
              93
             </strong>
             (11)
             <strong>
              :
             </strong>
             1905-1915.
             <a href="/pubmed/21843590" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21843590" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B10','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B11">
<p>
<a name="B11">
</a>
             Tuck AC,  Tollervey D:
             <strong>
              RNA in pieces.
             </strong>
</p>
<p>
<em>
              Trends Genetics: TIG
             </em>
             2011,
             <strong>
              27
             </strong>
             (10)
             <strong>
              :
             </strong>
             422-432.
             <a href="http://dx.doi.org/10.1016/j.tig.2011.06.001" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B11','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B12">
<p>
<a name="B12">
</a>
             Sobala A,  Hutvagner G:
             <strong>
              Transfer RNA-derived fragments: origins, processing, and functions.
             </strong>
</p>
<p>
<em>
              Wiley Interdiscip Rev RNA
             </em>
             2011,
             <strong>
              2
             </strong>
             (6)
             <strong>
              :
             </strong>
             853-862.
             <a href="/pubmed/21976287" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21976287" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B12','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B13">
<p>
<a name="B13">
</a>
             Lee YS,  Shibata Y,  Malhotra A,  Dutta A:
             <strong>
              A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).
             </strong>
</p>
<p>
<em>
              Genes Dev
             </em>
             2009,
             <strong>
              23
             </strong>
             (22)
             <strong>
              :
             </strong>
             2639-2649.
             <a href="/pubmed/19933153" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19933153" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19933153" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B13','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B14">
<p>
<a name="B14">
</a>
             Ivanov P,  Emara MM,  Villen J,  Gygi SP,  Anderson P:
             <strong>
              Angiogenin-induced tRNA fragments inhibit translation initiation.
             </strong>
</p>
<p>
<em>
              Mol Cell
             </em>
             2011,
             <strong>
              43
             </strong>
             (4)
             <strong>
              :
             </strong>
             613-623.
             <a href="/pubmed/21855800" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21855800" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21855800" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B14','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B15">
<p>
<a name="B15">
</a>
             Saikia M,  Krokowski D,  Guan BJ,  Ivanov P,  Parisien M,  Hu GF,  Anderson P,  Pan T,  Hatzoglou M:
             <strong>
              Genome-wide identification and quantitative analysis of cleaved tRNA fragments induced
                  by cellular stress.
             </strong>
</p>
<p>
<em>
              J Biol Chem
             </em>
             2012,
             <strong>
              287
             </strong>
             (51)
             <strong>
              :
             </strong>
             42708-42725.
             <a href="/pubmed/23086926" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23086926" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B15','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B16">
<p>
<a name="B16">
</a>
             Langmead B,  Trapnell C,  Pop M,  Salzberg SL:
             <strong>
              Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.
             </strong>
</p>
<p>
<em>
              Genome Biol
             </em>
             2009,
             <strong>
              10
             </strong>
             (3)
             <strong>
              :
             </strong>
             R25.
             <a href="/pubmed/19261174" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">
              BioMed Central Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19261174" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B16','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B17">
<p>
<a name="B17">
</a>
             Thompson DM,  Lu C,  Green PJ,  Parker R:
             <strong>
              tRNA cleavage is a conserved response to oxidative stress in eukaryotes.
             </strong>
</p>
<p>
<em>
              RNA
             </em>
             2008,
             <strong>
              14
             </strong>
             (10)
             <strong>
              :
             </strong>
             2095-2103.
             <a href="/pubmed/18719243" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18719243" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18719243" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B17','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B18">
<p>
<a name="B18">
</a>
             Yamasaki S,  Ivanov P,  Hu GF,  Anderson P:
             <strong>
              Angiogenin cleaves tRNA and promotes stress-induced translational repression.
             </strong>
</p>
<p>
<em>
              J Cell Biol
             </em>
             2009,
             <strong>
              185
             </strong>
             (1)
             <strong>
              :
             </strong>
             35-42.
             <a href="/pubmed/19332886" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19332886" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19332886" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B18','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B19">
<p>
<a name="B19">
</a>
             Chan PP,  Lowe TM: GRNA:
             <strong>
              a database of transfer RNA genes detected in genomic sequence.
             </strong>
</p>
<p>
<em>
              Nucleic Acids Res
             </em>
             2009,
             <strong>
              37
             </strong>
<strong>
              :
             </strong>
             93-97.
             <a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B19','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B20">
<p>
<a name="B20">
</a>
             Arroyo JD,  Chevillet JR,  Kroh EM,  Ruf IK,  Pritchard CC,  Gibson DF,  Mitchell PS,  Bennett CF,  Pogosova-Agadjanyan EL,  Stirewalt DL,
             <em>
              et al
             </em>
             .:
             <strong>
              Argonaute2 complexes carry a population of circulating microRNAs independent of vesicles
                  in human plasma.
             </strong>
</p>
<p>
<em>
              Proc Natl Acad Sci U S A
             </em>
             2011,
             <strong>
              108
             </strong>
             (12)
             <strong>
              :
             </strong>
             5003-5008.
             <a href="/pubmed/21383194" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21383194" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21383194" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B20','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B21">
<p>
<a name="B21">
</a>
             Turchinovich A,  Burwinkel B:
             <strong>
              Distinct AGO1 and AGO2 associated miRNA profiles in human cells and blood plasma.
             </strong>
</p>
<p>
<em>
              RNA Biol
             </em>
             2012,
             <strong>
              9
             </strong>
<strong>
              :
             </strong>
             8.
             <a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B21','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B22">
<p>
<a name="B22">
</a>
             Vickers KC,  Palmisano BT,  Shoucri BM,  Shamburek RD,  Remaley AT:
             <strong>
              MicroRNAs are transported in plasma and delivered to recipient cells by high-density
                  lipoproteins.
             </strong>
</p>
<p>
<em>
              Nat Cell Biol
             </em>
             2011,
             <strong>
              13
             </strong>
             (4)
             <strong>
              :
             </strong>
             423-433.
             <a href="/pubmed/21423178" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21423178" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21423178" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B22','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B23">
<p>
<a name="B23">
</a>
             Fu H,  Feng J,  Liu Q,  Sun F,  Tie Y,  Zhu J,  Xing R,  Sun Z,  Zheng X:
             <strong>
              Stress induces tRNA cleavage by angiogenin in mammalian cells.
             </strong>
</p>
<p>
<em>
              FEBS Lett
             </em>
             2009,
             <strong>
              583
             </strong>
             (2)
             <strong>
              :
             </strong>
             437-442.
             <a href="/pubmed/19114040" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19114040" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B23','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B24">
<p>
<a name="B24">
</a>
             Spindler SR,  Dhahbi JM:
             <strong>
              Conserved and tissue-specific genic and physiologic responses to caloric restriction
                  and altered IGFI signaling in mitotic and postmitotic tissues.
             </strong>
</p>
<p>
<em>
              Annu Rev Nutr
             </em>
             2007,
             <strong>
              27
             </strong>
<strong>
              :
             </strong>
             193-217.
             <a href="/pubmed/17428180" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17428180" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B24','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B25">
<p>
<a name="B25">
</a>
             Wang Q,  Lee I,  Ren J,  Ajay SS,  Lee YS,  Bao X:
             <strong>
              Identification and Functional Characterization of tRNA-derived RNA Fragments (tRFs)
                  in Respiratory Syncytial Virus Infection.
             </strong>
</p>
<p>
<em>
              Mol Ther
             </em>
             2013,
             <strong>
              21
             </strong>
             (2)
             <strong>
              :
             </strong>
             368-379.
             <a href="/pubmed/23183536" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23183536" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B25','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B26">
<p>
<a name="B26">
</a>
             Li S,  Hu GF:
             <strong>
              Emerging role of angiogenin in stress response and cell survival under adverse conditions.
             </strong>
</p>
<p>
<em>
              J Cell Physiol
             </em>
             2012,
             <strong>
              227
             </strong>
             (7)
             <strong>
              :
             </strong>
             2822-2826.
             <a href="/pubmed/22021078" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22021078" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B26','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B27">
<p>
<a name="B27">
</a>
             Quinlan AR,  Hall IM:
             <strong>
              BEDTools: a flexible suite of utilities for comparing genomic features.
             </strong>
</p>
<p>
<em>
              Bioinformatics
             </em>
             2010,
             <strong>
              26
             </strong>
             (6)
             <strong>
              :
             </strong>
             841-842.
             <a href="/pubmed/20110278" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20110278" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20110278" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B27','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B28">
<p>
<a name="B28">
</a>
             Robinson MD,  McCarthy DJ,  Smyth GK:
             <strong>
              edgeR: a Bioconductor package for differential expression analysis of digital gene
                  expression data.
             </strong>
</p>
<p>
<em>
              Bioinformatics
             </em>
             2010,
             <strong>
              26
             </strong>
             (1)
             <strong>
              :
             </strong>
             139-140.
             <a href="/pubmed/19910308" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19910308" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19910308" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B28','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
<li id="B29">
<p>
<a name="B29">
</a>
             Pall GS,  Hamilton AJ:
             <strong>
              Improved northern blot method for enhanced detection of small RNA.
             </strong>
</p>
<p>
<em>
              Nature protocols
             </em>
             2008,
             <strong>
              3
             </strong>
             (6)
             <strong>
              :
             </strong>
             1077-1084.
             <a href="/pubmed/18536652" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18536652" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B29','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" align="absmiddle" alt="OpenURL"/>
</a>
</p>
<p class="totext">
</p>
</li>
</ol>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<dl class="google-ad wide ">
<dt class="hide">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising" class="banner-ad">
</a>
</dt>
<dd>
</dd>
</dl>
<hr class="hide"/>
<div id="footer">
<span class="views">
<a href="?fmt_view=mobile">
     Mobile view
    </a>
    |
    <strong>
     Desktop view
    </strong>
</span>
<div class="content">
<div class="rounded ">
<div class="wrap-inner ">
<ul class="desktop">
<li class="noborder">
<a href="http://www.biomedcentral.com/about/tandc">
         Terms and Conditions
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/about/privacy">
         Privacy statement
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/presscenter">
         Press
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/advertisers">
         Information for advertisers
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/about/bmcjobs">
         Jobs at BMC
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/support">
         Support
        </a>
</li>
<li>
<a href="http://www.biomedcentral.com/about/contact">
         Contact us
        </a>
</li>
</ul>
<p id="copyright">
       © 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.
      </p>
</div>
</div>
</div>
</div>
<div class="springer">
<a href="http://www.springer.com/" target="blank">
<img src="/images/Springer_Branding_footer_logo.png" style="float:right; margin:0;"/>
</a>
</div>
<div id="fb-root">
</div>
<div class="hide">
<dl id="x96banner" class="google-ad">
<dt style="display: block;" class="hide">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising" class="skyscraper-ad">
      Advertisement
     </a>
</dt>
<dd>
</dd>
</dl>
</div>
<div style="text-align:center" class="calibre_navbar calibre_rescale_70">
<hr/>
<p style="text-align:left; max-width: 100%; overflow: hidden;">This article was downloaded by <strong>calibre</strong> from <a href="http://www.biomedcentral.com/1471-2164/14/298">http://www.biomedcentral.com/1471-2164/14/298</a></p>
<br/><br/> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | </div>

</body></html>
=======
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  </head>
  <body class="general_page journal bioinformatics"><div style="text-align:center" class="calibre_navbar calibre_rescale_70">| <a href="../../feed_1/index.html">Next</a> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | <a href="../article_0/index.html">Previous</a> | <hr/>
</div><div id="pageid-content" class="hw-gen-page pagetype-content hw-pub-id-article" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><div id="content-block"><div class="article fulltext-view" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<h1 id="article-title-1" itemprop="headline">
      Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text
     </h1>
<div class="contributors">
<ol id="contrib-group-1" class="contributor-list">
<li id="contrib-1" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Leon+French&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Leon French
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-2">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-2" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Suzanne+Lane&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Suzanne Lane
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-3">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-4">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-3" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Lydia+Xu&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Lydia Xu
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-5">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-6">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-4" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Celia+Siu&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Celia Siu
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-7">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-8">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-5" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Cathy+Kwok&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Cathy Kwok
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-9">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-10">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-6" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Yiqi+Chen&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Yiqi Chen
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-11">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-12">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li id="contrib-7" itemscope="itemscope" class="contributor" itemprop="author" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a href="/search?author1=Claudia+Krebs&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Claudia Krebs
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-13">
<sup>
          3
         </sup>
</a>
        and
       </li>
<li id="contrib-8" class="last">
<span class="name">
<a href="/search?author1=Paul+Pavlidis&amp;sortspec=date&amp;submit=Submit" class="name-search">
          Paul Pavlidis
         </a>
</span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-14">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#aff-1" class="xref-aff" id="xref-aff-1-15">
<sup>
          2
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a href="#corresp-1" class="xref-corresp" id="xref-corresp-1-1">
         *
        </a>
</li>
</ol>
<ol class="affiliation-list">
<li class="aff">
<a name="aff-1" id="aff-1">
</a>
<address>
<sup>
          1
         </sup>
         Department of Psychiatry,
         <sup>
          2
         </sup>
         Centre for High-Throughput Biology and
         <sup>
          3
         </sup>
         Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada
        </address>
</li>
</ol>
<ol class="corresp-list">
<li id="corresp-1" class="corresp">
<a href="#xref-corresp-1-1" class="rev-xref">
         ↵
        </a>
        *To whom correspondence should be addressed.
       </li>
</ol>
<ul class="history-list">
<li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-03">
<span class="received-label">
         Received
        </span>
        July 3, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="rev-recd" hwp:start="2012-08-13">
<span class="rev-recd-label">
         Revision received
        </span>
        August 13, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-08-28">
<span class="accepted-label">
         Accepted
        </span>
        August 28, 2012.
       </li>
</ul>
</div>
<div id="abstract-1" itemprop="description" class="section abstract">
<div class="section-nav">
<div class="nav-placeholder">
        �
       </div>
<a href="#sec-1" class="next-section-link" title="1 INTRODUCTION">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Abstract
      </h2>
<p id="p-3">
<strong>
        Motivation:
       </strong>
       Automated annotation of neuroanatomical connectivity statements from the neuroscience literature would enable accessible
                     and large-scale connectivity resources. Unfortunately, the connectivity findings are not formally encoded and occur as natural
                     language text. This hinders aggregation, indexing, searching and integration of the reports. We annotated a set of 1377 abstracts
                     for connectivity relations to facilitate automated extraction of connectivity relationships from neuroscience literature.
                     We tested several baseline measures based on co-occurrence and lexical rules. We compare results from seven machine learning
                     methods adapted from the protein interaction extraction domain that employ part-of-speech, dependency and syntax features.
      </p>
<p id="p-4">
<strong>
        Results:
       </strong>
       Co-occurrence based methods provided high recall with weak precision. The shallow linguistic kernel recalled 70.1% of the
                     sentence-level connectivity statements at 50.3% precision. Owing to its speed and simplicity, we applied the shallow linguistic
                     kernel to a large set of new abstracts. To evaluate the results, we compared 2688 extracted connections with the Brain Architecture
                     Management System (an existing database of rat connectivity). The extracted connections were connected in the Brain Architecture
                     Management System at a rate of 63.5%, compared with 51.1% for co-occurring brain region pairs. We found that precision increases
                     with the recency and frequency of the extracted relationships.
      </p>
<p id="p-5">
<strong>
        Availability and implementation:
       </strong>
       The source code, evaluations, documentation and other supplementary materials are available at
       <a href="http://www.chibi.ubc.ca/WhiteText">
        http://www.chibi.ubc.ca/WhiteText
       </a>
       .
      </p>
<p id="p-6">
<strong>
        Contact:
       </strong>
<span class="em-link">
<span class="em-addr">
         paul{at}chibi.ubc.ca
        </span>
</span>
</p>
<p id="p-7">
<strong>
        Supplementary information:
       </strong>
<a href="http://www.chibi.ubc.ca/WhiteText">
        Supplementary data
       </a>
       are available at
       <em>
        Bioinformatics
       </em>
       Online.
      </p>
</div>
<div id="sec-1" class="section intro">
<div class="section-nav">
<a href="#abstract-1" class="prev-section-link" title="Abstract">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-2" class="next-section-link" title="2 METHODS">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       1 INTRODUCTION
      </h2>
<p id="p-8">
       The brain is a vast interconnected network. Each neuron communicates with many others through chemical and electrical synapses
                     to integrate information. Groups of neurons (in structures such as nuclei or layers) make diverse connections across the brain,
                     forming pathways of information flow. This structural connectivity is a major determinant of brain function and is frequently
                     used by neuroscientists and clinicians to interpret physiological data. Examples include understanding strokes (
       <a href="#ref-18" class="xref-bibr" id="xref-ref-18-1">
        Haines, 2004
       </a>
       ) and interpreting brain imaging results. Evidence for connectivity abnormalities has been found in bipolar (
       <a href="#ref-21" class="xref-bibr" id="xref-ref-21-1">
        Houenou
        <em>
         et al.
        </em>
        , 2007
       </a>
       ), autism (
       <a href="#ref-26" class="xref-bibr" id="xref-ref-26-1">
        Koshino
        <em>
         et al.
        </em>
        , 2005
       </a>
       ), Alzheimer’s (
       <a href="#ref-43" class="xref-bibr" id="xref-ref-43-1">
        Stam
        <em>
         et al.
        </em>
        , 2007
       </a>
       ) and schizophrenia patients (
       <a href="#ref-25" class="xref-bibr" id="xref-ref-25-1">
        Karlsgodt
        <em>
         et al.
        </em>
        , 2008
       </a>
       ). A major goal of modern neuroscience is to understand the organization of the brain at all levels in as much detail as possible,
                     and to understand how this networked organization relates to brain function and ultimately behaviour and human health (
       <a href="#ref-40" class="xref-bibr" id="xref-ref-40-1">
        Sporns, 2011
       </a>
       ).
      </p>
<p id="p-9">
       The characterization of the connectivity network or wiring diagram of the brain is incomplete (
       <a href="#ref-9" class="xref-bibr" id="xref-ref-9-1">
        Crick and Jones, 1993
       </a>
       ). In part this is due to the complexity of the brain and the difficulty in collecting data. However, we suggest that informatics
                     technologies can be used to leverage existing knowledge that has already been collected to make new discoveries and guide
                     further experimentation.
      </p>
<p id="p-10">
       In this work, we are primarily concerned with ‘macroconnections’, or connections that can be identified between small brain
                     regions (as opposed to microcircuitry, which describes the connections among neurons
       <em>
        per se
       </em>
       ). These macroconnections between groups of neurons are predicted to number between 25 000 and 100 000 (
       <a href="#ref-3" class="xref-bibr" id="xref-ref-3-1">
        Bota
        <em>
         et al.
        </em>
        , 2003
       </a>
       ). This suggests a high level of complexity, although comfortably placed between the more gross levels of brain organization
                     and the microarchitecture, which encompasses billions of neurons and quadrillions of synapses (
       <a href="#ref-41" class="xref-bibr" id="xref-ref-41-1">
        Sporns
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Furthermore, this estimated number of macroconnections is smaller in scale than estimates of the human protein interactome
                     at 650 000 interactions among 25 000 proteins (
       <a href="#ref-44" class="xref-bibr" id="xref-ref-44-1">
        Stumpf
        <em>
         et al.
        </em>
        , 2008
       </a>
       ).
      </p>
<p id="p-11">
       Connectivity between brain regions can be assayed using tract tracing or electrophysiology. Tract tracing typically involves
                     injecting a dye or other tracer (e.g. horseradish peroxidase) into one brain region and tracking the fate of the tracer as
                     it follows axonal pathways (
       <a href="#ref-30" class="xref-bibr" id="xref-ref-30-1">
        Lanciego and Wouterlood, 2011
       </a>
       ). Electrophysiological methods use electrical or other stimulation in one site along with electrical recording at a second
                     site to test the functional connectivity of regions. Using these methods, a researcher can determine connections that send
                     signals to the region (afferent) or away from the region (efferent). Over many years, thousands of connectivity studies have
                     been performed, each of which typically elucidates, at most, a few connections. The presence of a deep literature on neuronal
                     connectivity is a major motivation for this work: the data are out there, they just need to be assembled.
      </p>
<p id="p-12">
       Attempts to turn this huge accumulation of knowledge into an ‘omics’ scale database have been limited, despite the potential
                     value of such a resource. Previous efforts have primarily used manual reviews of the literature to laboriously generate connectivity
                     maps for limited parts of the brain. In 1991, Felleman and Van Essen published a connectivity matrix of the macaque visual
                     cortex covering 305 pathways between 32 areas (
       <a href="#ref-13" class="xref-bibr" id="xref-ref-13-1">
        Felleman and Van Essen, 1991
       </a>
       ). Currently, a large number of collated connections are stored in the Collations of Connectivity data on the Macaque brain
                     database (CoCoMac) (
       <a href="#ref-27" class="xref-bibr" id="xref-ref-27-1">
        Kotter, 2004
       </a>
       ). CoCoMac contains detailed information from 413 literature reports regarding 7007 macaque brain regions. A fourth model
                     organism with large-scale connectivity data is the rat, with more than 40 000 formalized reports of connections in the Brain
                     Architecture Management System (BAMS) (
       <a href="#ref-4" class="xref-bibr" id="xref-ref-4-1">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Information is added to these databases manually, and therefore, they are accurate but sparse. Currently, the only complete
                     connectome scale database is the neuron-level wiring diagram of
       <em>
        Caenorhabditis elegans
       </em>
       , determined from electron micrographs (
       <a href="#ref-50" class="xref-bibr" id="xref-ref-50-1">
        White
        <em>
         et al.
        </em>
        , 1986
       </a>
       ).
      </p>
<p id="p-13">
       We sought to extend and complement manual efforts with automated text mining techniques. More than 10 years of efforts to
                     recognize gene and protein mentions and their interactions inspire our work (
       <a href="#ref-2" class="xref-bibr" id="xref-ref-2-1">
        Blaschke
        <em>
         et al.
        </em>
        , 1999
       </a>
       ;
       <a href="#ref-22" class="xref-bibr" id="xref-ref-22-1">
        Jensen
        <em>
         et al.
        </em>
        , 2006
       </a>
       ). In the gene interaction task, one must extract information from sentences such as ‘gene A interacts with gene B’ (to give
                     a toy example). Despite the difficulty of this task, great progress has been made. A comprehensive evaluation of kernel methods
                     for extracting protein–protein interactions detailed precision and recall values ranging from 45 to 70% by varying experiment
                     design, dataset and method tested (
       <a href="#ref-46" class="xref-bibr" id="xref-ref-46-1">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). At the second Critical Assessment of Information Extraction systems in Biology (BioCreAtIvE II), the top team was able
                     to extract normalized directed interaction pairs from full-text articles, with precision of 37% and recall of 33% (
       <a href="#ref-28" class="xref-bibr" id="xref-ref-28-1">
        Krallinger
        <em>
         et al.
        </em>
        , 2008
       </a>
       ). The analogy to brain connectivity is tight: we wish to extract information from sentences akin to ‘brain region A connects
                     to brain region B’. This related research gives us hope that the approaches applied to extracting gene interaction information
                     can successfully mine connectivity relations.
      </p>
<p id="p-14">
       To our knowledge, there have been no previous attempts to extract connectivity information using text mining methods. The
                     closest work to our own is that of
       <a href="#ref-6" class="xref-bibr" id="xref-ref-6-1">
        Burns
        <em>
         et al.
        </em>
        (2007)
       </a>
       , which was aimed at extracting information about tract-tracing experiments, trained and evaluated with a manually annotated
                     corpus of 1047 sentences from 21 documents. Although Burns
       <em>
        et al.
       </em>
       describe named entity recognition (e.g. identification of label injection sites), they did not report extraction of connectivity
                     statements themselves.
      </p>
<p id="p-15">
       For the work presented here, we have simplified the problem by limiting our input dataset and output results. We focus on
                     abstracts from one journal, the Journal of Comparative Neurology (JCN), because it is enriched for tract-tracing studies.
                     We used abstracts rather than full-text documents because they are enriched for high-level summary statements and are more
                     accessible. We also break our task into several subtasks, isolating the steps of brain region term recognition and normalization
                     from the evaluations (
       <a href="#ref-14" class="xref-bibr" id="xref-ref-14-1">
        French
        <em>
         et al.
        </em>
        , 2009
       </a>
       ;
       <a href="#ref-15" class="xref-bibr" id="xref-ref-15-1">
        French and Pavlidis, 2011
       </a>
       ). We only consider the presence of connectivity relations, ignoring the type, strength or direction of the connection. These
                     limitations make the task simpler and set the stage for future more detailed studies.
      </p>
<p id="p-16">
       Our results show that text mining approaches previously used to analyse protein networks can be usefully applied to brain
                     connectivity. Our large manually annotated corpus allowed testing and training of various techniques possible, and we also
                     perform extensive manual validation of the results. Beyond the corpus-based evaluations, we compared a large set of automatically
                     extracted connectivity statements with an existing connectivity database with favourable results. Together with our previous
                     work on term recognition and normalization, we present a completely automated system for extraction of brain connectivity
                     information from abstracts.
      </p>
</div>
<div id="sec-2" class="section methods">
<div class="section-nav">
<a href="#sec-1" class="prev-section-link" title="1 INTRODUCTION">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-9" class="next-section-link" title="3 RESULTS">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       2 METHODS
      </h2>
<div class="subsection" id="sec-3">
<h3>
        2.1 Annotated data
       </h3>
<p id="p-17">
        To train and test text mining algorithms, we created a large gold standard dataset. This dataset or corpus consists of abstracts
                        manually annotated by an undergraduate research assistant (S.L.) for connection verbs, species of study, brain region mentions
                        and connections between them. We annotated 1377 abstracts for 4276 connections and 17 585 brain region mentions. Abstracts
                        were randomly chosen from the JCN (years 1975–2008). This dataset has been previously used to demonstrate automated brain
                        region recognition and normalization (
        <a href="#ref-14" class="xref-bibr" id="xref-ref-14-2">
         French
         <em>
          et al.
         </em>
         , 2009
        </a>
        ;
        <a href="#ref-15" class="xref-bibr" id="xref-ref-15-2">
         French and Pavlidis, 2011
        </a>
        ), without using the connectivity annotations. Each annotated connection consists of two brain regions, text describing the
                        connection and the associated organism. This corpus provides sufficient training examples for machine-learning methods.
       </p>
<p id="p-18">
        We developed guidelines and software for the annotation process. Briefly, our main guidelines were as follows: (1) annotate
                        all brain region mentions, regardless of whether they are part of a connection; (2) annotate all connections and brain regions
                        for all organisms and organism states; (3) do not annotate mentions of white matter tracts or nerves; and (4) only annotate
                        monosynaptic or direct connections. We accepted connections that were stated in titles or introductory sentences that assume
                        connections between two high-level regions. Example relationships that were rejected are ‘may be connected’, ‘influences’,
                        ‘invaded’ and ‘alters activity’. The General Architecture for Text Engineering was used by annotators to highlight and connect
                        brain region mentions in text (
        <a href="#ref-11" class="xref-bibr" id="xref-ref-11-1">
         Cunningham
         <em>
          et al.
         </em>
         , 2002
        </a>
        ).
       </p>
</div>
<div class="subsection" id="sec-4">
<h3>
        2.2 Co-occurrence and rule-based methods
       </h3>
<p id="p-19">
        To extract neuroanatomical connections as described by the abstract authors, we must at least link two brain region mentions.
                        Our first method, acting as a naïve baseline method, predicts a stated connection between every pair of brain region mentions
                        (
        <a href="#ref-22" class="xref-bibr" id="xref-ref-22-2">
         Jensen
         <em>
          et al.
         </em>
         , 2006
        </a>
        ). We evaluate co-occurrence for single sentences and entire abstracts (including title).
       </p>
<p id="p-20">
        We created two simple rule-based extensions of the co-occurrence technique. The first simply limits co-occurrence extraction
                        to sentences with less than a set number of brain region mentions. The second requires presence of a connectivity-related
                        keyword (‘afferent’, ‘efferent’, ‘projects’, ‘projection’, ‘pathway’ or ‘inputs’).
       </p>
</div>
<div class="subsection" id="sec-5">
<h3>
        2.3 Kernel-based methods
       </h3>
<p id="p-21">
        Seven advanced kernel-based methods were applied to the dataset. These methods were originally designed for extraction of
                        protein–protein interactions. Each technique uses different features, parameters and kernel functions. Implementations were
                        brought into a common evaluation framework by Tikk and colleagues (
        <a href="#ref-46" class="xref-bibr" id="xref-ref-46-2">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ). Syntax and dependency trees for the sentences were computed by the Charniak-Lease re-ranking (
        <a href="#ref-31" class="xref-bibr" id="xref-ref-31-1">
         McClosky
         <em>
          et al.
         </em>
         , 2006
        </a>
        ) and Stanford (
        <a href="#ref-12" class="xref-bibr" id="xref-ref-12-1">
         De Marneffe
         <em>
          et al.
         </em>
         , 2006
        </a>
        ) parsers, respectively (same versions used in the Tikk
        <em>
         et al.
        </em>
        framework). The methods are categorized according to the type of features extracted. Four syntax tree-based methods use different
                        techniques to compare the sentence parse trees (
        <a href="#ref-8" class="xref-bibr" id="xref-ref-8-1">
         Collins and Duffy, 2001
        </a>
        ;
        <a href="#ref-29" class="xref-bibr" id="xref-ref-29-1">
         Kuboyama
         <em>
          et al.
         </em>
         , 2007
        </a>
        ;
        <a href="#ref-33" class="xref-bibr" id="xref-ref-33-1">
         Moschitti, 2005
        </a>
        ;
        <a href="#ref-48" class="xref-bibr" id="xref-ref-48-1">
         Vishwanathan and Smola, 2002
        </a>
        ). Going beyond syntax parsers, the all-paths graph kernel (
        <a href="#ref-1" class="xref-bibr" id="xref-ref-1-1">
         Airola
         <em>
          et al.
         </em>
         , 2008
        </a>
        ) and k-band shortest path spectrum kernel (
        <a href="#ref-46" class="xref-bibr" id="xref-ref-46-3">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ) use dependency parse information. Finally, the shallow linguistic kernel (SLK) uses only shallow parsing information such
                        as word occurrences and part-of-speech tags (
        <a href="#ref-17" class="xref-bibr" id="xref-ref-17-1">
         Giuliano
         <em>
          et al.
         </em>
         , 2006
        </a>
        ). Further details about the kernels are available on the supplement website. We used this framework to benchmark each of
                        the kernel-based methods on the brain region connectivity task. Of the nine methods described by Tikk
        <em>
         et al.
        </em>
        , we were able to successfully test seven, including the three top-performing kernels reported by
        <a href="#ref-46" class="xref-bibr" id="xref-ref-46-4">
         Tikk
         <em>
          et al.
         </em>
         (2010)
        </a>
        . The same parameter sets used by Tikk and colleagues were tested on our corpus.
       </p>
</div>
<div class="subsection" id="sec-6">
<h3>
        2.4 Experiment setup
       </h3>
<p id="p-22">
        We evaluate connection extraction independently of the previously described methods for automated brain region recognition
                        (
        <a href="#ref-14" class="xref-bibr" id="xref-ref-14-3">
         French
         <em>
          et al.
         </em>
         , 2009
        </a>
        ). This is done by providing the manually annotated brain region mentions to the relation extraction algorithm. Under this
                        design, the extraction task only requires correct linking of brain region mentions.
       </p>
<p id="p-23">
        To find a high-performing method, the different methods and varying parameters were run on a subset of 1146 abstracts. The
                        top-performing classifier and parameter set were retested on the full set of 1377 abstracts to gauge generalizability. Results
                        for the kernel methods were computed using 10-fold cross-validation. Each sentence became an input instance for the kernel
                        methods (including article title). Sentences of an abstract were not split between training and testing sets (document-level
                        split).
       </p>
</div>
<div class="subsection" id="sec-7">
<h3>
        2.5 Evaluation
       </h3>
<p id="p-24">
        Several evaluations were performed to judge the accuracy of the extracted connectivity statements. We primarily report the
                        results from the cross-validation experiments that test predictions against the manually annotated connections. Detailed evaluation
                        and annotation guidelines are provided as
        <a href="http://www.chibi.ubc.ca/WhiteText">
         Supplementary Materials
        </a>
        .
       </p>
<p id="p-25">
        Performance is measured against the number of true connectivity relations that are annotated completely within a sentence
                        or an abstract. The rule- and co-occurrence-based methods can operate at the abstract or sentence level, whereas the kernel
                        methods are limited to single sentences. Precision is computed as the proportion of predicted relations that are correct,
                        and recall is the proportion of true relations that are predicted by the method. The f-measure is the harmonic mean of these
                        two values, providing a balance of both. We also compute the area under the receiver operating curve where applicable (AUC).
                        This measure uses a ranked list of predictions with descending classification prediction scores that approximate confidence
                        in the prediction. This ranking allows computation of the true-positive and false-positive rates for a range of discrimination
                        thresholds. Previous experiments have found the AUC measure to be more robust and stable than f-measure for interaction mining
                        (
        <a href="#ref-46" class="xref-bibr" id="xref-ref-46-5">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ).
       </p>
</div>
<div class="subsection" id="sec-8">
<h3>
        2.6 Comparison with existing connectivity database
       </h3>
<p id="p-26">
        Normalization of brain region mentions to brain region concepts in formalized lexicons was targeted to the BAMS atlas (
        <a href="#ref-45" class="xref-bibr" id="xref-ref-45-1">
         Swanson, 1999
        </a>
        ). BAMS was chosen because of its wealth of curated rat-tract-tracing studies (
        <a href="#ref-4" class="xref-bibr" id="xref-ref-4-2">
         Bota
         <em>
          et al.
         </em>
         , 2005
        </a>
        ). In addition, rat is the most commonly studied species in our corpus. Our previously described Bag of Stems resolver was
                        applied, with all mention editors used, including those that map region mentions to larger enclosing brain regions (
        <a href="#ref-15" class="xref-bibr" id="xref-ref-15-3">
         French and Pavlidis, 2011
        </a>
        ). The lexical information in BAMS was expanded with synonym information to increase normalization performance. All possible
                        normalized parings are evaluated when a mention maps to more than one region. Connections in the BAMS connectivity matrices
                        were up-propagated in the anatomy hierarchy, which ensures that if there is a connection between regions A and B, then all
                        enclosing regions of A and B are also connected. Self-connections extracted from literature were ignored. The Linnaeus species
                        tagger was used to recognize species names in the abstracts (
        <a href="#ref-16" class="xref-bibr" id="xref-ref-16-1">
         Gerner
         <em>
          et al.
         </em>
         , 2010
        </a>
        ).
       </p>
</div>
</div>
<div id="sec-9" class="section results">
<div class="section-nav">
<a href="#sec-2" class="prev-section-link" title="2 METHODS">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-10" class="next-section-link" title="4 DISCUSSION">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       3 RESULTS
      </h2>
<p id="p-27">
       Our gold standard is a set of manually annotated 4276 brain region connectivity relations across a corpus of 1377 abstracts.
                     To gauge interannotator agreement, a second curator (L.X.) annotated a random subset of 231 documents. Roughly 80% of the
                     second curator’s annotations matched the primary curator (79.5% recall at 82.3% precision). Unlike the automated methods that
                     predict relations between given brain region mention spans, this evaluation required both annotators to mark the same spans
                     and relationships. By removing this restriction and allowing partially matching spans, the precision and recall reach 93.9%
                     and 91.9%, respectively.
      </p>
<p id="p-28">
       We used a co-occurrence analysis to reveal the proportion of brain region mention pairs that are co-mentioned and described
                     as connected. Co-occurrence assumes the relation is a connectivity statement. At the abstract level, this yields a precision
                     of only 2.2% at 100% recall and a f-measure of 4.3%. Within a sentence, co-occurrences between all pairs predict connected
                     pairs at 13.3% precision and 72.4% recall (the remaining relations span sentences). This level of recall means that more than
                     ¼ of all annotated connectivity relations are formed with regions in different sentences. Owing to the difficulty in extracting
                     connections spanning sentences, all of our subsequent evaluations are performed at the sentence level, with the relations
                     spanning sentences excluded. Under this evaluation framework, sentence-level co-occurrence recalls 100% of the remaining 3097
                     relations.
      </p>
<p id="p-29">
       We tested two simple modifications of the sentence-level co-occurrence technique. The first reduces co-occurrence predictions
                     to sentences with a limited number of brain region mentions. By extracting co-occurring pairs from sentences with only two
                     brain region mentions, precision reaches 23.1% and 17.2% recall (f-measure = 19.7%). This means that an average sentence with
                     two brain region mentions is reporting a connection in almost one of four cases. By varying this threshold, the f-measure
                     increases until sentences with six or more brain region mentions are included. We observed that some of these larger sentences
                     merely list brain regions involved in the study and not their relationships. By limiting the threshold at five brain region
                     mentions or less per sentence, co-occurrence provides 18.8% precision and 66.1% recall (
       <a href="#T1" class="xref-table" id="xref-table-wrap-1-1">
        Table 1
       </a>
       ). The second rule tested requires the sentences contain connectivity-related keywords (see Methods section). This keyword-based
                     rule increases recall to 17.4% and precision to 92.7% (f-measure = 29.4%). We created a new approach named ‘Keyword 5-threshold’
                     by combining these two rules. This again provides improvement, with f-measure reaching 34.1%. As expected, rule-based methods
                     increase precision at the cost of lower recall when compared with unrestricted co-occurrence.
      </p>
<div id="T1" class="table pos-float">
<div id="T1" itemprop="articleBody" class="table-expansion">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 1.
         </span>
<p id="p-30">
          Sentence level training set cross-validation results
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table rules="groups" frame="hsides" id="table-1">
<thead align="left" class="table-left" id="thead-1">
<tr id="tr-1">
<th colspan="1" rowspan="1" id="th-1">
            Kernel
           </th>
<th colspan="1" rowspan="1" id="th-2">
            Parser type
           </th>
<th colspan="1" rowspan="1" id="th-3">
            Parameter sets
           </th>
<th colspan="1" rowspan="1" id="th-4">
            Precision
           </th>
<th colspan="1" rowspan="1" id="th-5">
            Recall
           </th>
<th colspan="1" rowspan="1" id="th-6">
            F-measure
           </th>
<th colspan="1" rowspan="1" id="th-7">
            AUC
           </th>
</tr>
</thead>
<tbody align="left" class="table-left" id="tbody-1">
<tr id="tr-2">
<td id="td-1">
            Co-occurrence
           </td>
<td id="td-2">
            None
           </td>
<td id="td-3">
            1
           </td>
<td id="td-4">
            13.30%
           </td>
<td id="td-5">
            100.00%
           </td>
<td id="td-6">
            23.50%
           </td>
<td id="td-7">
</td>
</tr>
<tr id="tr-3">
<td id="td-8">
            Subset tree kernel
           </td>
<td id="td-9">
            Syntax
           </td>
<td id="td-10">
            12
           </td>
<td id="td-11">
            44.20%
           </td>
<td id="td-12">
            20.80%
           </td>
<td id="td-13">
            28.10%
           </td>
<td id="td-14">
            74.80%
           </td>
</tr>
<tr id="tr-4">
<td id="td-15">
            Co-occurrence five threshold
           </td>
<td id="td-16">
            None
           </td>
<td id="td-17">
            25
           </td>
<td id="td-18">
            18.80%
           </td>
<td id="td-19">
            66.10%
           </td>
<td id="td-20">
            29.30%
           </td>
<td id="td-21">
</td>
</tr>
<tr id="tr-5">
<td id="td-22">
            Partial tree kernel
           </td>
<td id="td-23">
            Syntax
           </td>
<td id="td-24">
            12
           </td>
<td id="td-25">
            43.30%
           </td>
<td id="td-26">
            23.10%
           </td>
<td id="td-27">
            29.80%
           </td>
<td id="td-28">
            75.20%
           </td>
</tr>
<tr id="tr-6">
<td id="td-29">
            Keyword co-occurrence
           </td>
<td id="td-30">
            None
           </td>
<td id="td-31">
            1
           </td>
<td id="td-32">
            17.40%
           </td>
<td id="td-33">
            92.70%
           </td>
<td id="td-34">
            29.40%
           </td>
<td id="td-35">
</td>
</tr>
<tr id="tr-7">
<td id="td-36">
            Spectrum tree kernel
           </td>
<td id="td-37">
            Syntax
           </td>
<td id="td-38">
            21
           </td>
<td id="td-39">
            37.40%
           </td>
<td id="td-40">
            26.10%
           </td>
<td id="td-41">
            30.20%
           </td>
<td id="td-42">
            72.90%
           </td>
</tr>
<tr id="tr-8">
<td id="td-43">
            Subtree kernel
           </td>
<td id="td-44">
            Syntax
           </td>
<td id="td-45">
            12
           </td>
<td id="td-46">
            40.70%
           </td>
<td id="td-47">
            25.20%
           </td>
<td id="td-48">
            30.80%
           </td>
<td id="td-49">
            74.60%
           </td>
</tr>
<tr id="tr-9">
<td id="td-50">
            Keyword five threshold
           </td>
<td id="td-51">
            None
           </td>
<td id="td-52">
            25
           </td>
<td id="td-53">
            23.70%
           </td>
<td id="td-54">
            60.80%
           </td>
<td id="td-55">
            34.10%
           </td>
<td id="td-56">
</td>
</tr>
<tr id="tr-10">
<td id="td-57">
            k-band shortest path spectrum
           </td>
<td id="td-58">
            Dependency
           </td>
<td id="td-59">
            288
           </td>
<td id="td-60">
            46.80%
           </td>
<td id="td-61">
            70.50%
           </td>
<td id="td-62">
            55.80%
           </td>
<td id="td-63">
            86.70%
           </td>
</tr>
<tr id="tr-11">
<td id="td-64">
            Shallow linguistic kernel (SLK)
           </td>
<td id="td-65">
            Part-of-speech tagger
           </td>
<td id="td-66">
            1
           </td>
<td id="td-67">
            50.30%
           </td>
<td id="td-68">
            70.10%
           </td>
<td id="td-69">
            58.30%
           </td>
<td id="td-70">
            88.90%
           </td>
</tr>
<tr id="tr-12">
<td id="td-71">
            All-paths graph kernel
           </td>
<td id="td-72">
            Dependency
           </td>
<td id="td-73">
            4
           </td>
<td id="td-74">
            60.40%
           </td>
<td id="td-75">
            57.90%
           </td>
<td id="td-76">
            58.40%
           </td>
<td id="td-77">
            88.40%
           </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li id="fn-2" class="fn">
<p id="p-31">
            AUC, area under the receiver operating curve; SLK, shallow linguistic kernel.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 1.
        </span>
<p class="first-child" id="p-30">
         Sentence level training set cross-validation results
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-32">
       Next, we applied seven previously published methods for extracting protein–protein interactions to our connectivity relation
                     dataset. Although the methods were designed for a different type of biomedical relation, they did not require any modification
                     for our application. The cross-validation results on the training dataset (1146 abstracts) are provided in
       <a href="#T1" class="xref-table" id="xref-table-wrap-1-2">
        Table 1
       </a>
       . For each method, the parameter set with the highest AUC score is shown. The parameter sets range in size and were reproduced
                     from Tikk
       <em>
        et al.
       </em>
       without modification (primarily grid searches of support vector machine settings). The f-measure scores for all of the seven
                     methods outperform unrestricted co-occurrence-based analysis for at least one parameter set. The simple rule-based methods
                     outperform the more complex partial tree- and subset tree-based methods. Although all of the syntax tree-based methods are
                     outperformed by the Keyword 5-threshold approach, they provide much higher precision than recall. When ranked by AUC, the
                     SLK performs best with a 58.3% f-measure and an AUC of 88.9%. The All Paths Graph and k-band shortest path spectrum kernel
                     methods rank a close second and third, respectively, with similar scores.
      </p>
<p id="p-33">
       We choose the SLK method for subsequent experiments owing to its accuracy, speed and single parameter set (global n-gram =
                     3 and local window = 2). Unlike the other kernel methods, the SLK method uses only shallow linguistic information at the local
                     (neighbouring words) and global sentence levels to predict relationships (
       <a href="#ref-17" class="xref-bibr" id="xref-ref-17-2">
        Giuliano
        <em>
         et al.
        </em>
        , 2006
       </a>
       ). This information forms feature vectors that are used to train a support vector machine classifier (scalar product kernel).
                     The performance of SLK on the complete set of 1377 abstracts is consistent with the cross-validation results (f-measure of
                     0.592).
       <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts542/DC1">
        Figure S1
       </a>
       displays the resulting receiver operating characteristic (ROC) curve (AUC = 0.899).
      </p>
<p id="p-34">
       We applied the SLK classifier to candidate sentences extracted from a set of 12 557 abstracts from the JCN (covering 1975–2011),
                     previously unseen by the algorithm. Our automatic brain region recognizer provided 33 466 sentences that mention two or more
                     brain regions (
       <a href="#ref-14" class="xref-bibr" id="xref-ref-14-4">
        French
        <em>
         et al.
        </em>
        , 2009
       </a>
       ;
       <a href="#ref-15" class="xref-bibr" id="xref-ref-15-4">
        French and Pavlidis, 2011
       </a>
       ). Within these sentences, SLK predicted 18% of the 156 484 possible brain region pairings to be connectivity relations. Of
                     these predicted relations, 9676 are in an abstract that mentions rat and can be evaluated against BAMS.
       <a href="#F1" class="xref-fig" id="xref-fig-1-1">
        Figure 1
       </a>
       shows the progression from abstracts to predicted connectivity relationships.
      </p>
<div class="fig pos-float odd" id="F1">
<div class="fig-inline">
<a href="2963/F1.expansion.html">
<img width="600" alt="Fig. 1." src="images/img1.jpg"/>
</a>
<div class="callout">
<span>
          View larger version:
         </span>
<ul class="callout-links">
<li>
<a href="2963/F1.expansion.html">
            In this window
           </a>
</li>
<li>
<a href="2963/F1.expansion.html" class="in-nw">
            In a new window
           </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/28/22/2963/F1">
            Download as PowerPoint Slide
           </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
         Fig. 1.
        </span>
<p class="first-child" id="p-35">
         Flow chart depicting the processing steps for comparison with the Brain Architecture Management System
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-36">
       To evaluate the precision of the predicted connections, we manually reviewed a random subset of 2000 abstracts. Each pair
                     was evaluated by two curators, yielding an interannotator agreement rate of 85%. Conflicts were resolved by a third curator
                     or by consensus after discussion. Overall, the SLK predictions were 55.3% precise. Errors from the automated steps of named
                     entity recognition and abbreviation expansion were 11% and 4%, respectively. These rates suggest a lesser impact of named
                     entity recognition errors when compared with assessments in the protein–protein interaction domain (
       <a href="#ref-24" class="xref-bibr" id="xref-ref-24-1">
        Kabiljo
        <em>
         et al.
        </em>
        , 2009
       </a>
       ).
       <a href="#T2" class="xref-table" id="xref-table-wrap-2-1">
        Table 2
       </a>
       presents the five most and least confident connectivity relations for the rat brain. Classification confidence is approximated
                     with the SLK prediction score (distance to classifying hyperplane), with highest values representing the cases closest to
                     positive training examples. Two of the most confident predictions are extracted from an article title and have the same form
                     (ranks 1 and 5). The sentences containing top predictions are shorter on average (192 characters) than the sentences with
                     least confident predictions (282 characters), suggesting sentence complexity affects the prediction results. Of these 10 examples,
                     only one is clearly a false-positive prediction (rank 9764), while several others point to errors in previous automated steps.
                     The mentions of ‘internal capsule’ (rank 9766) and ‘Met-enkephalin’ (rank 4) are incorrectly predicted as brain region mentions
                     (our definition of a brain region excludes fibre tracts like the internal capsule, while enkephalin is a peptide). We manually
                     compared these 10 results with the BAMS system and found it surprisingly difficult to map the mentioned regions to those in
                     BAMS. For example, ‘retrosplenial dysgranular cortex’ and ‘dorsal medullary reticular column’ were not found in BAMS. In the
                     end, corresponding connections were found in BAMS for several of the relationships, but only between enclosing regions (ranks
                     9767 and 5).
      </p>
<div id="T2" class="table pos-float">
<div id="T2" itemprop="articleBody" class="table-expansion">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 2.
         </span>
<p id="p-37">
          Top- and bottom-predicted relations from the 12 557 abstract set, ranked by SLK classification score
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table rules="groups" frame="hsides" id="table-2">
<thead align="left" class="table-left" id="thead-2">
<tr id="tr-13">
<th colspan="1" rowspan="1" id="th-8">
            Rank
           </th>
<th colspan="1" rowspan="1" id="th-9">
            Sentence
           </th>
<th colspan="1" rowspan="1" id="th-10">
            Score
           </th>
<th colspan="1" rowspan="1" id="th-11">
            Reference
           </th>
</tr>
</thead>
<tbody align="left" class="table-left" id="tbody-2">
<tr id="tr-14">
<td id="td-78">
            1
           </td>
<td id="td-79">
<strong>
             Trigeminal
            </strong>
            projections to
            <strong>
             hypoglossal and facial motor nuclei
            </strong>
            in the rat.
           </td>
<td id="td-80">
            3.47
           </td>
<td id="td-81">
<span class="xref-bibr" id="xref-ref-35-1">
             Pinganaud,
             <em>
              et al.
             </em>
             , 1999
            </span>
</td>
</tr>
<tr id="tr-15">
<td id="td-82">
            2
           </td>
<td id="td-83">
<strong>
             The cortical
            </strong>
            projections to
            <strong>
             retrosplenial dysgranular cortex
            </strong>
            (Rdg) originate primarily in the infraradiata, retrosplenial, postsubicular and areas 17 and 18b cortices.
           </td>
<td id="td-84">
            3.34
           </td>
<td id="td-85">
<span class="xref-bibr" id="xref-ref-47-1">
             van Groen and Wyss, 1992
            </span>
</td>
</tr>
<tr id="tr-16">
<td id="td-86">
            3
           </td>
<td id="td-87">
            The
            <strong>
             thalamic
            </strong>
            projections to
            <strong>
             retrosplenial dysgranular cortex
            </strong>
            (Rdg) originate in the anterior (primarily the anteromedial), lateral (primarily the laterodorsal) and reuniens nuclei.
           </td>
<td id="td-88">
            3.33
           </td>
<td id="td-89">
<span class="xref-bibr" id="xref-ref-47-2">
             van Groen and Wyss, 1992
            </span>
</td>
</tr>
<tr id="tr-17">
<td id="td-90">
            4
           </td>
<td id="td-91">
            Our results indicate that the
            <strong>
             centromedial amygdala
            </strong>
            receives
            <strong>
             Met-enkephalin
            </strong>
            afferents, as indicated by the presence of mu-opioid receptor, delta-opioid receptor and Met-enkephalin fibres in the CEA
                           and MEA, originating primarily from the bed nucleus of the stria terminalis and from other amygdaloid nuclei.
           </td>
<td id="td-92">
            3.32
           </td>
<td id="td-93">
<span class="xref-bibr" id="xref-ref-36-1">
             Poulin,
             <em>
              et al.
             </em>
             , 2006
            </span>
</td>
</tr>
<tr id="tr-18">
<td id="td-94">
            5
           </td>
<td id="td-95">
<strong>
             Thalamic
            </strong>
            projections to
            <strong>
             retrosplenial cortex
            </strong>
            in the rat.
           </td>
<td id="td-96">
            3.28
           </td>
<td id="td-97">
<span class="xref-bibr" id="xref-ref-42-1">
             Sripanidkulchai and Wyss, 1986
            </span>
</td>
</tr>
<tr id="tr-19">
<td id="td-98">
            …
           </td>
<td id="td-99">
            9757 relationships
           </td>
<td id="td-100">
</td>
<td id="td-101">
</td>
</tr>
<tr id="tr-20">
<td id="td-102">
            9763
           </td>
<td id="td-103">
            The sparse reciprocal connections to the other
            <strong>
             amygdaloid nuclei
            </strong>
            suggest that the CEA nucleus does not regulate the other amygdaloid regions, but rather executes the responses evoked by
                           the other amygdaloid nuclei that innervate the
            <strong>
             CEA nucleus
            </strong>
            .
           </td>
<td id="td-104">
            5.46 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-105">
<span class="xref-bibr" id="xref-ref-23-1">
             Jolkkonen and Pitkanen, 1998
            </span>
</td>
</tr>
<tr id="tr-21">
<td id="td-106">
            9764
           </td>
<td id="td-107">
            The majority of the endomorphin 1/fluoro-gold and endomorphin 2/fluoro-gold double-labelled neurons in the
            <strong>
             hypothalamus
            </strong>
            were distributed in the
            <strong>
             dorsomedial nucleus
            </strong>
            , areas between the dorsomedial and ventromedial nucleus and arcuate nucleus; a few were also seen in the ventromedial, periventricular
                           and posterior nucleus.
           </td>
<td id="td-108">
            4.36 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-109">
<span class="xref-bibr" id="xref-ref-7-1">
             Chen,
             <em>
              et al.
             </em>
             , 2008
            </span>
</td>
</tr>
<tr id="tr-22">
<td id="td-110">
            9765
           </td>
<td id="td-111">
            Projections from the
            <strong>
             dorsal medullary reticular column
            </strong>
            are largely bilateral and are distributed preferentially to the
            <strong>
             ventral subdivision of the fifth cranial nerve motor nuclei in the rat (MoV)
            </strong>
            , to the dorsal and intermediate subdivisions of VII and to both the dorsal and the ventral subdivision of XII.
           </td>
<td id="td-112">
            2.91 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-113">
<span class="xref-bibr" id="xref-ref-10-1">
             Cunningham and Sawchenko, 2000
            </span>
</td>
</tr>
<tr id="tr-23">
<td id="td-114">
            9766
           </td>
<td id="td-115">
            Two additional large projections leave the MEA forebrain bundle in the hypothalamus; the ansa peduncularis–ventral amygdaloid
                           bundle system turns laterally through the
            <strong>
             internal capsule
            </strong>
            into the striatal complex,
            <strong>
             amygdala
            </strong>
            and the external capsule to reach lateral and posterior cortex, and another system of fibers turns medially to innervate
                           MEA hypothalamus and median eminence and forms a contralateral projection through the supraoptic commissures.
           </td>
<td id="td-116">
            2.87 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-117">
<span class="xref-bibr" id="xref-ref-32-1">
             Moore,
             <em>
              et al.
             </em>
             , 1978
            </span>
</td>
</tr>
<tr id="tr-24">
<td id="td-118">
            9767
           </td>
<td id="td-119">
            In animals with injected horseradish peroxidase confined within the main bulb, perikarya retrogradely labelled with the protein
                           in the
            <strong>
             ipsilateral forebrain
            </strong>
            were observed in the
            <strong>
             anterior prepyriform cortex horizontal limb of the nucleus of the diagonal band
            </strong>
            , and far lateral preoptic and rostral lateral hypothalamic areas.
           </td>
<td id="td-120">
            3.36 × 10
            <sup>
             −5
            </sup>
</td>
<td id="td-121">
<span class="xref-bibr" id="xref-ref-5-1">
             Broadwell and Jacobowitz, 1976
            </span>
</td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li id="fn-3" class="fn">
<p id="p-38">
            CEA, central; MEA, medial; MoV, the fifth cranial nerve motor nuclei in the rat.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 2.
        </span>
<p class="first-child" id="p-37">
         Top- and bottom-predicted relations from the 12 557 abstract set, ranked by SLK classification score
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-39">
       Encouraged by these results, we did a comparison of the results from the 12 557 abstracts with BAMS, to gauge accuracy of
                     connections and the extent to which our approach might supplement manual curation efforts. Compared with the manual annotations,
                     this is a less precise evaluation because BAMS does not cover the complete literature and is limited to rat studies (
       <a href="#ref-4" class="xref-bibr" id="xref-ref-4-3">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). In addition, mapping errors resulting from linking brain region mentions to target regions in BAMS reduces accuracy (
       <a href="#ref-15" class="xref-bibr" id="xref-ref-15-5">
        French and Pavlidis, 2011
       </a>
       ). For example, 12% of mentions are mapped to more than one brain region owing to ambiguous synonyms. To benchmark the BAMS
                     evaluation metric, we first tested it on the manually curated connectivity relations from our training corpus of 1377 abstracts.
                     Our process first extracts abstracts that used rat (based on Linneaus analysis) and maps the brain region mentions to the
                     BAMS lexicon. These rat connectivity relationships were then compared with the BAMS connectivity matrix. Only 167 manually
                     annotated connectivity relations were testable by this method, with 70.5% having a connection in BAMS. In the same set of
                     abstracts, the 2617 brain region pairings not annotated as connections but co-occur in sentences are connected in BAMS at
                     49.8%. This is not surprising because co-occurring regions may be connected, but the author is not stating that in the sentence.
                     In the larger set of rat-related abstracts, 2688 computationally predicted connectivity statements are successfully resolved,
                     and 63.5% are, in fact, reported as true by BAMS (
       <a href="#F1" class="xref-fig" id="xref-fig-1-2">
        Fig. 1
       </a>
       ). For comparison, the remaining set of co-occurring brain region pairs is connected in BAMS at a rate of 51.1%. We noted
                     that the extracted relationships are between larger or less specific brain regions than those in BAMS. Anatomical depth, or
                     the average number of enclosing or parent brain regions for a connected pair in the BAMS matrix, is 9.6, whereas the literature-extracted
                     connections had a mean of 7.9, indicating they are larger and less specific brain regions. Along the same lines, the literature-based
                     relationships only involve 433 regions, whereas BAMS has connection reports for 633 regions. We evaluated the 899 connections
                     that are predicted by our method but not listed as connected in BAMS (provided in Supplement). Similar to the previous evaluation
                     of 2000 connections, approximately one-half of these connections are false-positive text mining errors (52.1% precision).
                     The remaining 468 connections that are true positives at the sentence level could be used to expand BAMS coverage (although
                     curation guidelines differ). Within these 468 true positives, we selected a subset of 250 for further review by a domain expert
                     (C. Krebs). Only nine of the predicted connections were rejected (3.6%). Five were rejected because a protein (pituitary adenylate
                     cyclise-activating polypeptide) was incorrectly recognized as a brain region (pituitary gland). This agreement between the
                     curators and an expert suggests our annotation guidelines are consistent and accurate.
      </p>
<p id="p-40">
       We hypothesized that owing to improvements in tracing methods, more recent reports of connectivity would be of higher quality.
                     This was suggested by a study of different eras of tract-tracing techniques that revealed large improvements in accuracy (
       <a href="#ref-3" class="xref-bibr" id="xref-ref-3-2">
        Bota
        <em>
         et al.
        </em>
        , 2003
       </a>
       ). Bota and colleagues found that limbic system connections observed using an old method, axon degeneration (
       <a href="#ref-34" class="xref-bibr" id="xref-ref-34-1">
        Nauta, 1952
       </a>
       ), are 60% accurate. In contrast, methods first applied in 1987 to exploit axonal transport are much more accurate with more
                     than 90% considered valid. By splitting our corpus into documents published before and after 1987, we tested for a similar
                     signal that separates eras of experimental techniques. In agreement with the manually quantified trend, we observe an increase
                     from 59.4 to 65.6% in the rate of connectivity statements validated in BAMS (
       <em>
        P
       </em>
       = 0.00071, hypergeometric test). We note the specificity of regions involved in the connections also increases, whereas the
                     proportion of mapped terms is unchanged.
      </p>
<p id="p-41">
       Connections predicted more than once might be more likely to be valid because of the effect of ‘confirmation’. This was feasible
                     to study because, on average, each connection was predicted more than twice. The number of extracted connections per brain
                     region (degree) provides a simple comparison with BAMS. For the 344 common brain regions, the degree vectors are strongly
                     correlated (Pearson = 0.769, Spearman = 0.433). Counting unique predicted connections, 54.7% are in BAMS (
       <a href="#T3" class="xref-table" id="xref-table-wrap-3-1">
        Table 3
       </a>
       , the value of 63.5% previously cited counts occurrences). From a recall perspective, 3.2% of BAMS connections are connected
                     in our predictions. By thresholding our connections to those predicted at least twice, precision reaches 65.9%, whereas recall
                     drops to 1.4% (
       <a href="#T3" class="xref-table" id="xref-table-wrap-3-2">
        Table 3
       </a>
       ). This accuracy is near the 67.5% precision of the hand-annotated set of connections. Precision gradually increases as the
                     threshold increases, eventually reaching 100% for nine connections that were extracted at least 12 times. Further, we note
                     the anatomical specificity of the connections increases with the average number of enclosing regions reaching 10.2 when thresholded
                     at 12 occurrences. The region pairs not predicted to form connectivity relations have precision of 33.7% and recall of 9.3%.
                     Again, this level of precision results from co-mentioned regions that are connected in BAMS, but the author is not specifying
                     that in the sentence. Further, the higher recall value results from the much larger set of pairings (6079 compared with 1286
                     SLK-predicted parings). From a co-occurrence perspective, we found that brain regions that co-occur in eight or more sentences
                     recall 1.6% of the BAMS connections at 66.4% precision. Interestingly, this naive co-occurrence-based method performs at par
                     to the SLK method that extracts direct connectivity statements. As the threshold is increased from eight co-occurrences, precision
                     continues to gain, suggesting a large number of co-occurring mentions can be used to predict connectivity, as well as a smaller
                     number of more carefully analysed connectivity statements (
       <a href="#T3" class="xref-table" id="xref-table-wrap-3-3">
        Table 3
       </a>
       ).
      </p>
<div id="T3" class="table pos-float">
<div id="T3" itemprop="articleBody" class="table-expansion">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 3.
         </span>
<p id="p-42">
          Aggregate connectivity results from several methods and relation sets
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table rules="groups" frame="hsides" id="table-3">
<thead align="left" class="table-left" id="thead-3">
<tr id="tr-25">
<th colspan="1" rowspan="1" id="th-12">
            Relation set
           </th>
<th colspan="1" rowspan="1" id="th-13">
            Method
           </th>
<th colspan="1" rowspan="1" id="th-14">
            Threshold
           </th>
<th colspan="1" rowspan="1" id="th-15">
            Anatomical depth
           </th>
<th colspan="1" rowspan="1" id="th-16">
            Connections
           </th>
<th colspan="1" rowspan="1" id="th-17">
            Precision
           </th>
<th colspan="1" rowspan="1" id="th-18">
            Recall
           </th>
<th colspan="1" rowspan="1" id="th-19">
            F-measure
           </th>
</tr>
</thead>
<tbody align="left" class="table-left" id="tbody-3">
<tr id="tr-26">
<td id="td-122">
            Positive annotated
           </td>
<td id="td-123">
            Curation
           </td>
<td id="td-124">
            1
           </td>
<td id="td-125">
            8.7
           </td>
<td id="td-126">
            200
           </td>
<td id="td-127">
            67.50%
           </td>
<td id="td-128">
            0.61%
           </td>
<td id="td-129">
            1.22%
           </td>
</tr>
<tr id="tr-27">
<td id="td-130">
            Negative annotated
           </td>
<td id="td-131">
            Curation
           </td>
<td id="td-132">
            1
           </td>
<td id="td-133">
            8.7
           </td>
<td id="td-134">
            1606
           </td>
<td id="td-135">
            41.91%
           </td>
<td id="td-136">
            3.06%
           </td>
<td id="td-137">
            5.71%
           </td>
</tr>
<tr id="tr-28">
<td id="td-138">
            Positive predictions
           </td>
<td id="td-139">
            SLK
           </td>
<td id="td-140">
            1
           </td>
<td id="td-141">
            8.4
           </td>
<td id="td-142">
            1286
           </td>
<td id="td-143">
            54.70%
           </td>
<td id="td-144">
            3.20%
           </td>
<td id="td-145">
            6.05%
           </td>
</tr>
<tr id="tr-29">
<td id="td-146">
            Positive predictions
           </td>
<td id="td-147">
            SLK
           </td>
<td id="td-148">
            2
           </td>
<td id="td-149">
            8.4
           </td>
<td id="td-150">
            454
           </td>
<td id="td-151">
            65.90%
           </td>
<td id="td-152">
            1.40%
           </td>
<td id="td-153">
            2.74%
           </td>
</tr>
<tr id="tr-30">
<td id="td-154">
            Positive predictions
           </td>
<td id="td-155">
            SLK
           </td>
<td id="td-156">
            12
           </td>
<td id="td-157">
            10.2
           </td>
<td id="td-158">
            9
           </td>
<td id="td-159">
            100.00%
           </td>
<td id="td-160">
            0.04%
           </td>
<td id="td-161">
            0.08%
           </td>
</tr>
<tr id="tr-31">
<td id="td-162">
            All pairings
           </td>
<td id="td-163">
            Co-occurrence
           </td>
<td id="td-164">
            1
           </td>
<td id="td-165">
            8.3
           </td>
<td id="td-166">
            6474
           </td>
<td id="td-167">
            34.00%
           </td>
<td id="td-168">
            10.01%
           </td>
<td id="td-169">
            15.47%
           </td>
</tr>
<tr id="tr-32">
<td id="td-170">
            All pairings
           </td>
<td id="td-171">
            Co-occurrence
           </td>
<td id="td-172">
            2
           </td>
<td id="td-173">
            8.3
           </td>
<td id="td-174">
            2865
           </td>
<td id="td-175">
            44.96%
           </td>
<td id="td-176">
            5.86%
           </td>
<td id="td-177">
            10.37%
           </td>
</tr>
<tr id="tr-33">
<td id="td-178">
            All pairings
           </td>
<td id="td-179">
            Co-occurrence
           </td>
<td id="td-180">
            8
           </td>
<td id="td-181">
            8.2
           </td>
<td id="td-182">
            515
           </td>
<td id="td-183">
            66.41%
           </td>
<td id="td-184">
            1.56%
           </td>
<td id="td-185">
            3.04%
           </td>
</tr>
<tr id="tr-34">
<td id="td-186">
            All pairings
           </td>
<td id="td-187">
            Co-occurrence
           </td>
<td id="td-188">
            16
           </td>
<td id="td-189">
            8.4
           </td>
<td id="td-190">
            189
           </td>
<td id="td-191">
            71.43%
           </td>
<td id="td-192">
            0.61%
           </td>
<td id="td-193">
            1.22%
           </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li id="fn-4" class="fn">
<p id="p-43">
            This table presents the analysis of the extracted binary connectivity matrices. The first two rows are from connectivity matrices
                           derived from the 1377 annotated abstract set. The remaining rows are from the 12 557 abstract set and are split between the
                           SLK predictions and the co-occurrence technique. The threshold column displays the required count of reported connections
                           to be marked as connected in the matrix. Anatomical depth measures how specific the connections are by averaging the number
                           of enclosing brain regions for each connected region.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 3.
        </span>
<p class="first-child" id="p-42">
         Aggregate connectivity results from several methods and relation sets
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
</div>
<div id="sec-10" class="section discussion">
<div class="section-nav">
<a href="#sec-9" class="prev-section-link" title="3 RESULTS">
<span>
         Previous Section
        </span>
</a>
<a href="#ack-1" class="next-section-link" title="Acknowledgements">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       4 DISCUSSION
      </h2>
<p id="p-44">
       We reported a complete system for extracting connectivity statements from biomedical abstracts. The method provides high recall
                     of manually annotated connectivity relations described in single sentences. Precision from two separate evaluations reached
                     50.3% and 55.3%. By comparing with an independent source of rat connectivity, we found that precision increases with the recency
                     and frequency of the extracted relationships.
      </p>
<p id="p-45">
       A limitation of our work is that we did not consider the direction of connectivity, although most of the relationships we
                     extracted have a direction described in the sentence. In addition, we did not consider negation (region A does not project
                     to region B). Extracting this information by extracting keywords such as ‘afferent’, ‘not’ or ‘input’ will require additional
                     work. These relationship modifiers are manually annotated in our training corpus and can be used to design more complex rules.
      </p>
<p id="p-46">
       Our methods also did not attempt to extract the large number of relations that span multiple sentences. When these connections
                     are taken into account, the SLK method provides only 51.7% recall of annotated connections. Application of advanced natural
                     language-processing techniques may be necessary to bridge the sentences (e.g. anaphora resolution).
      </p>
<p id="p-47">
       The comparison of seven previously published kernel-based approaches mirrored the previous results from the protein interaction
                     relationship extraction domain (
       <a href="#ref-46" class="xref-bibr" id="xref-ref-46-6">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). Further, we note that the SLK parser was applied to the drug–drug interaction domain with similar results (
       <a href="#ref-39" class="xref-bibr" id="xref-ref-39-1">
        Segura-Bedmar
        <em>
         et al.
        </em>
        , 2011
       </a>
       ). Several of the kernel methods have lower performance than our simple rule-based technique. Effort spent crafting more complex
                     rules may yield higher precision at the cost of lower recall. The top three kernel methods (SLK, all-paths graph, k-band shortest
                     path spectrum kernel) all have similar accuracy (AUC and f-measure scores) but vary in precision and recall. This difference
                     suggests higher performance may be achieved by combining the methods.
      </p>
<p id="p-48">
       Our results suggest a larger set of input abstracts will yield a larger number of precise connections. The largest extension
                     set is Medline with more than 10 million abstracts and 120 million sentences. Tikk and colleagues calculated that the SLK
                     parser could process all of Medline in 141 days (
       <a href="#ref-46" class="xref-bibr" id="xref-ref-46-7">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). A two-step process may reduce runtime and increase accuracy by first identifying abstracts with connectivity statements
                     and then by extracting the specific connections with SLK (
       <a href="#ref-37" class="xref-bibr" id="xref-ref-37-1">
        Poulter
        <em>
         et al.
        </em>
        , 2008
       </a>
       ).
      </p>
<p id="p-49">
       In natural language processing, it has been observed that simple statistical models (e.g. co-occurrence) can outperform more
                     complex models based on less data (
       <a href="#ref-19" class="xref-bibr" id="xref-ref-19-1">
        Halevy
        <em>
         et al.
        </em>
        , 2009
       </a>
       ). Our experiments confirm this. We found that brain region pairs with many co-mentions tend to be connected. This simple
                     technique produces a larger set of potential connections with reasonable precision. Although this will produce a larger set
                     of results than the SLK method, it does not target connections that can be directly curated in light of experimental evidence
                     because the co-mentions may or may not describe connectivity. Further, such co-occurrences may result from region proximity
                     or biases (i.e. popularity) that may influence research attention both in the literature and in BAMS. However, such co-occurrence
                     networks show valuable areas of focus when combined with comentions of genes and diseases (
       <a href="#ref-20" class="xref-bibr" id="xref-ref-20-1">
        Hayasaka
        <em>
         et al.
        </em>
        , 2011
       </a>
       ;
       <a href="#ref-49" class="xref-bibr" id="xref-ref-49-1">
        Voytek and Voytek, 2012
       </a>
       ).
      </p>
<p id="p-50">
       One of the most serious challenges we encountered was in mapping extracted brain region mentions to standardized lexicons.
                     In our previous work, we reported resolution rates of 63%, with the major limitation being gaps in the lexicons (
       <a href="#ref-15" class="xref-bibr" id="xref-ref-15-6">
        French and Pavlidis, 2011
       </a>
       ). For the current work, the resolution rate is greatly reduced, as both brain region mentions of a connectivity relation
                     must be mapped. It also appears that regions forming connectivity relations are harder to resolve or map, on average. For
                     this work, we managed to double the resolution rate to the BAMS lexicon by adding synonyms. Additional work to improve the
                     lexicons will lead to better resolution of connectivity statements, allowing validation and linking to other resources.
      </p>
<p id="p-51">
       For our evaluation to an outside database, we focused on BAMS (
       <a href="#ref-4" class="xref-bibr" id="xref-ref-4-4">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Although rat is the most frequent mentioned organism, other evaluations could compare the connectivity results with the
                     CoCoMac (
       <a href="#ref-27" class="xref-bibr" id="xref-ref-27-2">
        Kotter, 2004
       </a>
       ) or the Avian Brain Circuitry Database (
       <a href="#ref-38" class="xref-bibr" id="xref-ref-38-1">
        Schrott and Kabai, 2008
       </a>
       ). Beyond evaluation, our dataset and method can provide a large set of extracted connectivity relationships for other species-specific
                     databases.
      </p>
<p id="p-52">
       In conclusion, we provide the first application of large-scale text mining to neuroanatomical connectivity extraction. We
                     demonstrated that machine-learning tools designed for extraction of protein–protein interactions are generalizable to mining
                     brain region connections. From an information retrieval perspective, our large set of predicted connections can aid neuroscientists
                     in forming hypotheses and models. Future work will be aimed at further evaluating and disseminating the results before extending
                     the analysis.
      </p>
</div>
<div id="ack-1" class="section ack">
<div class="section-nav">
<a href="#sec-10" class="prev-section-link" title="4 DISCUSSION">
<span>
         Previous Section
        </span>
</a>
<a href="#fn-group-1" class="next-section-link" title="Footnotes">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Acknowledgements
      </h2>
<p id="p-53">
       The authors thank Olivia Marais and Tianna Koreman for additional curation.
      </p>
<p id="p-54">
<em>
        Funding
       </em>
       : This work was supported in part by a Discovery Grant from the Natural Sciences and Engineering Research Council of Canada
                     (371348). Further support was provided by a National Institutes of Health grant GM076990 to Paul Pavlidis, the Canadian Foundation
                     for Innovation (Leaders Opportunities Fund), the Michael Smith Foundation for Health Research (Career Investigator award to
                     Paul Pavlidis) and the Canadian Institutes of Health Research (New Investigator Salary Award to Paul Pavlidis). The funders
                     had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.
      </p>
<p id="p-55">
<em>
        Conflict of Interest
       </em>
       : none declared.
      </p>
</div>
<div id="fn-group-1" class="section fn-group">
<div class="section-nav">
<a href="#ack-1" class="prev-section-link" title="Acknowledgements">
<span>
         Previous Section
        </span>
</a>
<a href="#ref-list-1" class="next-section-link" title="REFERENCES">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Footnotes
      </h2>
<ul>
<li id="fn-1" class="fn">
<p id="p-1">
         Associate Editor: Jonathan Wren
        </p>
</li>
</ul>
</div>
<ul class="copyright-statement">
<li id="copyright-statement-1" class="fn">
       © The Author(s) 2012. Published by Oxford University Press.
      </li>
</ul>
<div id="license-1" class="license">
<p id="p-2">
       This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0),
                     which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
      </p>
</div>
<div id="ref-list-1" class="section ref-list">
<div class="section-nav">
<a href="#fn-group-1" class="prev-section-link" title="Footnotes">
<span>
         Previous Section
        </span>
</a>
<div class="nav-placeholder">
        �
       </div>
</div>
<h2>
       REFERENCES
      </h2>
<ol class="cit-list ref-use-labels">
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-1-1" class="rev-xref-ref" id="ref-1">
         ↵
        </a>
<div id="cit-28.22.2963.1" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Airola
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            All-paths graph kernel for protein-protein interaction extraction with evaluation of cross-corpus learning
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
<span class="cit-supplement">
            Suppl. 11
           </span>
           :
           <span class="cit-fpage">
            S2
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=All-paths%20graph%20kernel%20for%20protein-protein%20interaction%20extraction%20with%20evaluation%20of%20cross-corpus%20learning&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Airola&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-2-1" class="rev-xref-ref" id="ref-2">
         ↵
        </a>
<div id="cit-28.22.2963.2" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Blaschke
             </span>
<span class="cit-name-given-names">
              C
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automatic extraction of biological information from scientific text: protein-protein interactions
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Proc. Int. Conf. Intell. Syst. Mol. Biol.
           </abbr>
<span class="cit-pub-date">
            1999
           </span>
           :
           <span class="cit-fpage">
            60
           </span>
           -
           <span class="cit-lpage">
            67
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Automatic%20extraction%20of%20biological%20information%20from%20scientific%20text%3A%20protein-protein%20interactions&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Blaschke&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-3-1" class="rev-xref-ref" id="ref-3">
         ↵
        </a>
<div id="cit-28.22.2963.3" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Bota
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            From gene networks to brain networks
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Neurosci.
           </abbr>
<span class="cit-pub-date">
            2003
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            795
           </span>
           -
           <span class="cit-lpage">
            799
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nn1096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=12886225&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000184413700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-4-1" class="rev-xref-ref" id="ref-4">
         ↵
        </a>
<div id="cit-28.22.2963.4" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Bota
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Brain architecture management system
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Neuroinformatics
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            15
           </span>
           -
           <span class="cit-lpage">
            48
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1385/NI:3:1:015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15897615&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000230322400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.5" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Broadwell
             </span>
<span class="cit-name-given-names">
              RD
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jacobowitz
             </span>
<span class="cit-name-given-names">
              DM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Olfactory relationships of the telencephalon and diencephalon in the rabbit. III. The ipsilateral centrifugal fibers to the
                                    olfactory bulbar and retrobulbar formations
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1976
           </span>
           ;
           <span class="cit-vol">
            170
           </span>
           :
           <span class="cit-fpage">
            321
           </span>
           -
           <span class="cit-lpage">
            345
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.901700305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=62770&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1976CM96300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-6-1" class="rev-xref-ref" id="ref-6">
         ↵
        </a>
<div id="cit-28.22.2963.6" class="cit ref-cit ref-book">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Burns
             </span>
<span class="cit-name-given-names">
              G
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Computational Intelligence in Biomedicine
           </span>
           .
           <span class="cit-publ-loc">
            Berlin
           </span>
           :
           <span class="cit-publ-name">
            Springer-Verlag and Heidelberg
           </span>
           ;
           <span class="cit-pub-date">
            2007
           </span>
           .
           <span class="cit-article-title">
            Intelligent approaches to mining the primary research literature: techniques, systems, and examples
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Intelligent%20approaches%20to%20mining%20the%20primary%20research%20literature%3A%20techniques%2C%20systems%2C%20and%20examples&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Burns&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.7" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Chen
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Origins of endomorphin-immunoreactive fibers and terminals in different columns of the periaqueductal gray in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            509
           </span>
           :
           <span class="cit-fpage">
            72
           </span>
           -
           <span class="cit-lpage">
            87
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.21728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18421704&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-8-1" class="rev-xref-ref" id="ref-8">
         ↵
        </a>
<div id="cit-28.22.2963.8" class="cit ref-cit ref-book">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Collins
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Duffy
             </span>
<span class="cit-name-given-names">
              N
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-conf-name">
            Proceedings of Neural Information Processing Systems (NIPS'01)
           </span>
           ;
           <span class="cit-conf-loc">
            Vancouver, BC, Canada
           </span>
           .
           <span class="cit-pub-date">
            2001
           </span>
           .
           <span class="cit-article-title">
            Convolution kernels for natural language
           </span>
           ; p.
           <span class="cit-fpage">
            625
           </span>
           -
           <span class="cit-lpage">
            632
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Convolution%20kernels%20for%20natural%20language&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Collins&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-9-1" class="rev-xref-ref" id="ref-9">
         ↵
        </a>
<div id="cit-28.22.2963.9" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Crick
             </span>
<span class="cit-name-given-names">
              F
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jones
             </span>
<span class="cit-name-given-names">
              E
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Backwardness of human neuroanatomy
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nature
           </abbr>
<span class="cit-pub-date">
            1993
           </span>
           ;
           <span class="cit-vol">
            361
           </span>
           :
           <span class="cit-fpage">
            109
           </span>
           -
           <span class="cit-lpage">
            110
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/361109a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=8421513&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.10" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Cunningham
             </span>
<span class="cit-name-given-names">
              ET
             </span>
<span class="cit-name-suffix">
              Jr
             </span>
             .
            </span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sawchenko
             </span>
<span class="cit-name-given-names">
              PE
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Dorsal medullary pathways subserving oromotor reflexes in the rat: implications for the central neural control of swallowing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2000
           </span>
           ;
           <span class="cit-vol">
            417
           </span>
           :
           <span class="cit-fpage">
            448
           </span>
           -
           <span class="cit-lpage">
            466
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(20000221)417:4%3C448::AID-CNE5%3E3.0.CO;2-S&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=10701866&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000085158700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-11-1" class="rev-xref-ref" id="ref-11">
         ↵
        </a>
<div id="cit-28.22.2963.11" class="cit ref-cit ref-confproc">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Cunningham
             </span>
<span class="cit-name-given-names">
              H
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A framework and graphical development environment for robust NLP tools and applications
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of the 40th Annual Meeting of the Association for Computational Linguistics (ACL)
           </span>
           .
           <span class="cit-pub-date">
            2002
           </span>
           . p.
           <span class="cit-fpage">
            168
           </span>
           -
           <span class="cit-lpage">
            175
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=A%20framework%20and%20graphical%20development%20environment%20for%20robust%20NLP%20tools%20and%20applications&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Cunningham&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-12-1" class="rev-xref-ref" id="ref-12">
         ↵
        </a>
<div id="cit-28.22.2963.12" class="cit ref-cit ref-book">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              De Marneffe
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-conf-name">
            Proceedings of LRECVol. 6
           </span>
           .
           <span class="cit-publ-loc">
            Paris
           </span>
           :
           <span class="cit-publ-name">
            European Language Resource Association
           </span>
           ;
           <span class="cit-pub-date">
            2006
           </span>
           .
           <span class="cit-article-title">
            Generating typed dependency parses from phrase structure parses
           </span>
           ; p.
           <span class="cit-fpage">
            449
           </span>
           -
           <span class="cit-lpage">
            454
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Generating%20typed%20dependency%20parses%20from%20phrase%20structure%20parses&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=De%20Marneffe&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-13-1" class="rev-xref-ref" id="ref-13">
         ↵
        </a>
<div id="cit-28.22.2963.13" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Felleman
             </span>
<span class="cit-name-given-names">
              DJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Van Essen
             </span>
<span class="cit-name-given-names">
              DC
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Distributed hierarchical processing in the primate cerebral cortex
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Cereb. Cortex.
           </abbr>
<span class="cit-pub-date">
            1991
           </span>
           ;
           <span class="cit-vol">
            1
           </span>
           :
           <span class="cit-fpage">
            1
           </span>
           -
           <span class="cit-lpage">
            47
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=cercor&amp;resid=1/1/1-a" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-14-1" class="rev-xref-ref" id="ref-14">
         ↵
        </a>
<div id="cit-28.22.2963.14" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              French
             </span>
<span class="cit-name-given-names">
              L
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automated recognition of brain region mentions in neuroscience literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Front. Neuroinform.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            29
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=19750194&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-15-1" class="rev-xref-ref" id="ref-15">
         ↵
        </a>
<div id="cit-28.22.2963.15" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              French
             </span>
<span class="cit-name-given-names">
              L
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pavlidis
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Using text mining to link journal articles to neuroanatomical databases
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            520
           </span>
           :
           <span class="cit-fpage">
            1772
           </span>
           -
           <span class="cit-lpage">
            1783
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Using%20text%20mining%20to%20link%20journal%20articles%20to%20neuroanatomical%20databases&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=French&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-16-1" class="rev-xref-ref" id="ref-16">
         ↵
        </a>
<div id="cit-28.22.2963.16" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Gerner
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            LINNAEUS: a species name identification system for biomedical literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            11
           </span>
           :
           <span class="cit-fpage">
            85
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-11-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20149233&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-17-1" class="rev-xref-ref" id="ref-17">
         ↵
        </a>
<div id="cit-28.22.2963.17" class="cit ref-cit ref-confproc">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Giuliano
             </span>
<span class="cit-name-given-names">
              C
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Exploiting shallow linguistic information for relation extraction from biomedical literature
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of the 11st Conference of the European Chapter of the Association for Computational Linguistics (EACL’06). The
                                    Association for Computer Linguistics, Trento, Italy
           </span>
           .
           <span class="cit-pub-date">
            2006
           </span>
           . p.
           <span class="cit-fpage">
            401
           </span>
           -
           <span class="cit-lpage">
            408
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Exploiting%20shallow%20linguistic%20information%20for%20relation%20extraction%20from%20biomedical%20literature&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Giuliano&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-18-1" class="rev-xref-ref" id="ref-18">
         ↵
        </a>
<div id="cit-28.22.2963.18" class="cit ref-cit ref-book">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Haines
             </span>
<span class="cit-name-given-names">
              DE
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Neuroanatomy: an Atlas of Structures, Sections, and Systems
           </span>
           .
           <span class="cit-publ-loc">
            Baltimore and Philadelphia
           </span>
           :
           <span class="cit-publ-name">
            Lippincott Williams &amp; Wilkins
           </span>
           ;
           <span class="cit-pub-date">
            2004
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Neuroanatomy%3A%20an%20Atlas%20of%20Structures%2C%20Sections%2C%20and%20Systems&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Haines&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-19-1" class="rev-xref-ref" id="ref-19">
         ↵
        </a>
<div id="cit-28.22.2963.19" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Halevy
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The unreasonable effectiveness of data
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            IEEE Intell. Syst.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            24
           </span>
           :
           <span class="cit-fpage">
            8
           </span>
           -
           <span class="cit-lpage">
            12
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=The%20unreasonable%20effectiveness%20of%20data&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Halevy&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-20-1" class="rev-xref-ref" id="ref-20">
         ↵
        </a>
<div id="cit-28.22.2963.20" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Hayasaka
             </span>
<span class="cit-name-given-names">
              S
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A network of genes, genetic disorders, and brain areas
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PloS One
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            e20907
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pone.0020907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21695164&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-21-1" class="rev-xref-ref" id="ref-21">
         ↵
        </a>
<div id="cit-28.22.2963.21" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Houenou
             </span>
<span class="cit-name-given-names">
              J
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Increased white matter connectivity in euthymic bipolar patients: diffusion tensor tractography between the subgenual cingulate
                                    and the amygdalo-hippocampal complex
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Mol. Psychiatry
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            12
           </span>
           :
           <span class="cit-fpage">
            1001
           </span>
           -
           <span class="cit-lpage">
            1010
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/sj.mp.4002010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17471288&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000250412700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-22-1" class="rev-xref-ref" id="ref-22">
         ↵
        </a>
<div id="cit-28.22.2963.22" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jensen
             </span>
<span class="cit-name-given-names">
              LJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Literature mining for the biologist: from information retrieval to biological discovery
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Rev. Genet.
           </abbr>
<span class="cit-pub-date">
            2006
           </span>
           ;
           <span class="cit-vol">
            7
           </span>
           :
           <span class="cit-fpage">
            119
           </span>
           -
           <span class="cit-lpage">
            129
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nrg1768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16418747&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000234714000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.23" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jolkkonen
             </span>
<span class="cit-name-given-names">
              E
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pitkanen
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Intrinsic connections of the rat amygdaloid complex: projections originating in the central nucleus
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1998
           </span>
           ;
           <span class="cit-vol">
            395
           </span>
           :
           <span class="cit-fpage">
            53
           </span>
           -
           <span class="cit-lpage">
            72
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(19980525)395:1%3C53::AID-CNE5%3E3.0.CO;2-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=9590546&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000073290400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-24-1" class="rev-xref-ref" id="ref-24">
         ↵
        </a>
<div id="cit-28.22.2963.24" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kabiljo
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A realistic assessment of methods for extracting gene/protein interactions from free text
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            10
           </span>
           :
           <span class="cit-fpage">
            233
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-10-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=19635172&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-25-1" class="rev-xref-ref" id="ref-25">
         ↵
        </a>
<div id="cit-28.22.2963.25" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Karlsgodt
             </span>
<span class="cit-name-given-names">
              KH
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Diffusion tensor imaging of the superior longitudinal fasciculus and working memory in recent-onset schizophrenia
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Biol. Psychiatry
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            63
           </span>
           :
           <span class="cit-fpage">
            512
           </span>
           -
           <span class="cit-lpage">
            518
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=17720147&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-26-1" class="rev-xref-ref" id="ref-26">
         ↵
        </a>
<div id="cit-28.22.2963.26" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Koshino
             </span>
<span class="cit-name-given-names">
              H
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Functional connectivity in an fMRI working memory task in high-functioning autism
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            NeuroImage
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            24
           </span>
           :
           <span class="cit-fpage">
            810
           </span>
           -
           <span class="cit-lpage">
            821
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.neuroimage.2004.09.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15652316&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000226588500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-27-1" class="rev-xref-ref" id="ref-27">
         ↵
        </a>
<div id="cit-28.22.2963.27" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kotter
             </span>
<span class="cit-name-given-names">
              R
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Online retrieval, processing, and visualization of primate connectivity data from the CoCoMac database
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Neuroinformatics
           </abbr>
<span class="cit-pub-date">
            2004
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            127
           </span>
           -
           <span class="cit-lpage">
            144
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1385/NI:2:2:127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15319511&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000223582200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-28-1" class="rev-xref-ref" id="ref-28">
         ↵
        </a>
<div id="cit-28.22.2963.28" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Krallinger
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Overview of the protein-protein interaction annotation extraction task of BioCreative II
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genome Biol.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
<span class="cit-supplement">
            Suppl. 2
           </span>
           :
           <span class="cit-fpage">
            S4
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Overview%20of%20the%20protein-protein%20interaction%20annotation%20extraction%20task%20of%20BioCreative%20II&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Krallinger&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-29-1" class="rev-xref-ref" id="ref-29">
         ↵
        </a>
<div id="cit-28.22.2963.29" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kuboyama
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A spectrum tree kernel
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Inf. Media Technol.
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            292
           </span>
           -
           <span class="cit-lpage">
            299
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=A%20spectrum%20tree%20kernel&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Kuboyama&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-30-1" class="rev-xref-ref" id="ref-30">
         ↵
        </a>
<div id="cit-28.22.2963.30" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Lanciego
             </span>
<span class="cit-name-given-names">
              JL
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wouterlood
             </span>
<span class="cit-name-given-names">
              FG
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A half century of experimental neuroanatomical tracing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Chem. Neuroanat.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            42
           </span>
           :
           <span class="cit-fpage">
            157
           </span>
           -
           <span class="cit-lpage">
            183
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jchemneu.2011.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21782932&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-31-1" class="rev-xref-ref" id="ref-31">
         ↵
        </a>
<div id="cit-28.22.2963.31" class="cit ref-cit ref-confproc">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              McClosky
             </span>
<span class="cit-name-given-names">
              D
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Reranking and self-training for parser adaptation
           </span>
           .
           <span class="cit-conf-name">
            ACL-44 Proceedings of the 21st International Conference on Computational Linguistics and the 44th Annual Meeting of the Association
                                    for Computational Linguistics. Association for Computational Linguistics,
           </span>
           .
           <span class="cit-pub-date">
            2006
           </span>
           . p.
           <span class="cit-fpage">
            337
           </span>
           -
           <span class="cit-lpage">
            344
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Reranking%20and%20self-training%20for%20parser%20adaptation&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=McClosky&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.32" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Moore
             </span>
<span class="cit-name-given-names">
              RY
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Serotonin neurons of the midbrain raphe: ascending projections
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1978
           </span>
           ;
           <span class="cit-vol">
            180
           </span>
           :
           <span class="cit-fpage">
            417
           </span>
           -
           <span class="cit-lpage">
            438
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.901800302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=77865&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1978FD16900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-33-1" class="rev-xref-ref" id="ref-33">
         ↵
        </a>
<div id="cit-28.22.2963.33" class="cit ref-cit ref-confproc">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Moschitti
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Efficient convolution kernels for dependency and constituent syntactic trees
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of The 17th European Conference on Machine Learning
           </span>
           .
           <span class="cit-publ-loc">
            Berlin
           </span>
           :
           <span class="cit-publ-name">
            Springer and Heidelberg
           </span>
           ;
           <span class="cit-pub-date">
            2005
           </span>
           . p.
           <span class="cit-fpage">
            318
           </span>
           -
           <span class="cit-lpage">
            329
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Efficient%20convolution%20kernels%20for%20dependency%20and%20constituent%20syntactic%20trees&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Moschitti&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-34-1" class="rev-xref-ref" id="ref-34">
         ↵
        </a>
<div id="cit-28.22.2963.34" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Nauta
             </span>
<span class="cit-name-given-names">
              WJ
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Selective silver impregnation of degenerating axons in the central nervous system
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Stain Technol.
           </abbr>
<span class="cit-pub-date">
            1952
           </span>
           ;
           <span class="cit-vol">
            27
           </span>
           :
           <span class="cit-fpage">
            175
           </span>
           -
           <span class="cit-lpage">
            179
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=14958510&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1952UN01600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.35" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pinganaud
             </span>
<span class="cit-name-given-names">
              G
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Trigeminal projections to hypoglossal and facial motor nuclei in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1999
           </span>
           ;
           <span class="cit-vol">
            415
           </span>
           :
           <span class="cit-fpage">
            91
           </span>
           -
           <span class="cit-lpage">
            104
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(19991206)415:1%3C91::AID-CNE7%3E3.0.CO;2-A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=10540360&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000083518400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.36" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Poulin
             </span>
<span class="cit-name-given-names">
              JF
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Enkephalinergic afferents of the centromedial amygdala in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2006
           </span>
           ;
           <span class="cit-vol">
            496
           </span>
           :
           <span class="cit-fpage">
            859
           </span>
           -
           <span class="cit-lpage">
            876
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.20956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16628615&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000237440000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-37-1" class="rev-xref-ref" id="ref-37">
         ↵
        </a>
<div id="cit-28.22.2963.37" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Poulter
             </span>
<span class="cit-name-given-names">
              GL
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            MScanner: a classifier for retrieving Medline citations
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
           :
           <span class="cit-fpage">
            108
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-9-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18284683&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-38-1" class="rev-xref-ref" id="ref-38">
         ↵
        </a>
<div id="cit-28.22.2963.38" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Schrott
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kabai
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            ABCD: a functional database for the avian brain
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Neurosci. Methods
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            167
           </span>
           :
           <span class="cit-fpage">
            393
           </span>
           -
           <span class="cit-lpage">
            395
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jneumeth.2007.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17889371&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000252938400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-39-1" class="rev-xref-ref" id="ref-39">
         ↵
        </a>
<div id="cit-28.22.2963.39" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Segura-Bedmar
             </span>
<span class="cit-name-given-names">
              I
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Using a shallow linguistic kernel for drug-drug interaction extraction
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Biomed. Inform.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            44
           </span>
           :
           <span class="cit-fpage">
            789
           </span>
           -
           <span class="cit-lpage">
            804
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jbi.2011.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21545845&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-40-1" class="rev-xref-ref" id="ref-40">
         ↵
        </a>
<div id="cit-28.22.2963.40" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sporns
             </span>
<span class="cit-name-given-names">
              O
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The human connectome: a complex network
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Ann. NY Acad. Sci.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            1224
           </span>
           :
           <span class="cit-fpage">
            109
           </span>
           -
           <span class="cit-lpage">
            125
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1111/j.1749-6632.2010.05888.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21251014&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000290238100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-41-1" class="rev-xref-ref" id="ref-41">
         ↵
        </a>
<div id="cit-28.22.2963.41" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sporns
             </span>
<span class="cit-name-given-names">
              O
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The human connectome: a structural description of the human brain
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PLoS Comput. Biol.
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            1
           </span>
           :
           <span class="cit-fpage">
            e42
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pcbi.0010042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16201007&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.42" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sripanidkulchai
             </span>
<span class="cit-name-given-names">
              K
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wyss
             </span>
<span class="cit-name-given-names">
              JM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Thalamic projections to retrosplenial cortex in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1986
           </span>
           ;
           <span class="cit-vol">
            254
           </span>
           :
           <span class="cit-fpage">
            143
           </span>
           -
           <span class="cit-lpage">
            165
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.902540202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=3794004&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1986F117900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-43-1" class="rev-xref-ref" id="ref-43">
         ↵
        </a>
<div id="cit-28.22.2963.43" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Stam
             </span>
<span class="cit-name-given-names">
              CJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Small-world networks and functional connectivity in Alzheimer's disease
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Cereb Cortex
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            17
           </span>
           :
           <span class="cit-fpage">
            92
           </span>
           -
           <span class="cit-lpage">
            99
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=cercor&amp;resid=17/1/92" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-44-1" class="rev-xref-ref" id="ref-44">
         ↵
        </a>
<div id="cit-28.22.2963.44" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Stumpf
             </span>
<span class="cit-name-given-names">
              MP
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Estimating the size of the human interactome
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Proc. Natl Acad. Sci. USA
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            105
           </span>
           :
           <span class="cit-fpage">
            6959
           </span>
           -
           <span class="cit-lpage">
            6964
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=105/19/6959" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-45-1" class="rev-xref-ref" id="ref-45">
         ↵
        </a>
<div id="cit-28.22.2963.45" class="cit ref-cit ref-book">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Swanson
             </span>
<span class="cit-name-given-names">
              LW
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Brain maps: structure of the rat brain
           </span>
           .
           <span class="cit-publ-loc">
            Amsterdam
           </span>
           :
           <span class="cit-publ-name">
            Elsevier
           </span>
           ;
           <span class="cit-pub-date">
            1999
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Brain%20maps%3A%20structure%20of%20the%20rat%20brain&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Swanson&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-46-1" class="rev-xref-ref" id="ref-46">
         ↵
        </a>
<div id="cit-28.22.2963.46" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Tikk
             </span>
<span class="cit-name-given-names">
              D
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A comprehensive benchmark of kernel methods to extract protein-protein interactions from literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PLoS Comput. Biol.
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            e1000837
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pcbi.1000837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20617200&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div id="cit-28.22.2963.47" class="cit ref-cit ref-journal no-rev-xref">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              van Groen
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wyss
             </span>
<span class="cit-name-given-names">
              JM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Connections of the retrosplenial dysgranular cortex in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1992
           </span>
           ;
           <span class="cit-vol">
            315
           </span>
           :
           <span class="cit-fpage">
            200
           </span>
           -
           <span class="cit-lpage">
            216
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.903150207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=1545009&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1992HB75200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-48-1" class="rev-xref-ref" id="ref-48">
         ↵
        </a>
<div id="cit-28.22.2963.48" class="cit ref-cit ref-confproc">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Vishwanathan
             </span>
<span class="cit-name-given-names">
              S
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Smola
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Fast kernels for string and tree matching
           </span>
           .
           <span class="cit-conf-name">
            In: Proceedings of Neural Information Processing Systems (NIPS’02)
           </span>
           .
           <span class="cit-publ-loc">
            Cambridge, MA, USA
           </span>
           :
           <span class="cit-publ-name">
            MIT Press
           </span>
           ;
           <span class="cit-pub-date">
            2002
           </span>
           . p.
           <span class="cit-fpage">
            569
           </span>
           -
           <span class="cit-lpage">
            576
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Fast%20kernels%20for%20string%20and%20tree%20matching&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Vishwanathan&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5" target="_blank">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-49-1" class="rev-xref-ref" id="ref-49">
         ↵
        </a>
<div id="cit-28.22.2963.49" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Voytek
             </span>
<span class="cit-name-given-names">
              JB
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Voytek
             </span>
<span class="cit-name-given-names">
              B
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automated cognome construction and semi-automated hypothesis generation
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Neurosci. Methods
           </abbr>
<span class="cit-pub-date">
            2012
           </span>
           ;
           <span class="cit-vol">
            208
           </span>
           :
           <span class="cit-fpage">
            92
           </span>
           -
           <span class="cit-lpage">
            100
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jneumeth.2012.04.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=22584238&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000305865600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a title="View reference in text" href="#xref-ref-50-1" class="rev-xref-ref" id="ref-50">
         ↵
        </a>
<div id="cit-28.22.2963.50" class="cit ref-cit ref-journal">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              White
             </span>
<span class="cit-name-given-names">
              JG
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The structure of the nervous system of the nematode
            <em>
             Caenorhabditis elegans
            </em>
</span>
           .
           <abbr class="cit-jnl-abbrev">
            Philos. Trans. R. Soc. Lond. B. Biol. Sci.
           </abbr>
<span class="cit-pub-date">
            1986
           </span>
           ;
           <span class="cit-vol">
            314
           </span>
           :
           <span class="cit-fpage">
            1
           </span>
           -
           <span class="cit-lpage">
            340
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=royptb&amp;resid=314/1165/1" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
</ol>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<span id="related-urls">
</span></div></div><div style="text-align:center" class="calibre_navbar calibre_rescale_70">
<hr/>
<p style="text-align:left; max-width: 100%; overflow: hidden;">This article was downloaded by <strong>calibre</strong> from <a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2963?rss=1">http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2963?rss=1</a></p>
<br/><br/> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | </div>
</body></html>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
