
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>
   Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text
  </title>
<meta name="googlebot" content="NOODP" />
<meta name="HW.ad-path" content="/cgi/content/full/28/22/2963" />
<meta content="/bioinfo/28/22/2963.atom" name="HW.identifier" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta content="Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text" name="DC.Title" />
<meta content="10.1093/bioinformatics/bts542" name="DC.Identifier" />
<meta content="2012-11-15" name="DC.Date" />
<meta content="Oxford University Press" name="DC.Publisher" />
<meta content="Leon French" name="DC.Contributor" />
<meta content="Suzanne Lane" name="DC.Contributor" />
<meta content="Lydia Xu" name="DC.Contributor" />
<meta content="Celia Siu" name="DC.Contributor" />
<meta content="Cathy Kwok" name="DC.Contributor" />
<meta content="Yiqi Chen" name="DC.Contributor" />
<meta content="Claudia Krebs" name="DC.Contributor" />
<meta content="Paul Pavlidis" name="DC.Contributor" />
<meta content="Bioinformatics" name="citation_journal_title" />
<meta content="Bioinformatics" name="citation_journal_abbrev" />
<meta content="1367-4803" name="citation_issn" />
<meta content="1460-2059" name="citation_issn" />
<meta name="citation_author" content="Leon French" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Suzanne Lane" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Lydia Xu" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Celia Siu" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Cathy Kwok" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Yiqi Chen" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Claudia Krebs" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author" content="Paul Pavlidis" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="1Department of Psychiatry, 2Centre for High-Throughput Biology and 3Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada" />
<meta content="Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text" name="citation_title" />
<meta content="11/15/2012" name="citation_date" />
<meta content="28" name="citation_volume" />
<meta content="22" name="citation_issue" />
<meta content="2963" name="citation_firstpage" />
<meta content="2970" name="citation_lastpage" />
<meta content="28/22/2963" name="citation_id" />
<meta content="bioinfo;28/22/2963" name="citation_mjid" />
<meta content="10.1093/bioinformatics/bts542" name="citation_doi" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.abstract" name="citation_abstract_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.full" name="citation_fulltext_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963.full.pdf" name="citation_pdf_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2963" name="citation_public_url" />
<meta content="22954628" name="citation_pmid" />
<meta name="citation_section" content="Original Paper" />
<meta name="robots" content="noarchive,nofollow" />
<meta name="googlebot" content="noarchive" />
<link href="/content/28/22/2956.short" rel="prev" />
<link href="/content/28/22/2971.short" rel="next" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-global.css" />
<link rel="stylesheet" type="text/css" media="print" href="/shared/css/hw-print.css" />
<link rel="stylesheet" type="text/css" media="all" href="/local/css/hw-local-global.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-global.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-ac.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-page-content.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/jquery.fancybox-1.3.4.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-global-colexpand.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-page-content.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/hw20.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-header.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-footer.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-sidebars.css" />
<link type="text/css" media="all" rel="stylesheet" href="/site/resource/bioinformatics_h20.css" />
<link rel="stylesheet" type="text/css" href="/resource/css/print.css" media="print" />
<link rel="stylesheet" type="text/css" media="all" href="stylesheets/style15.css" />
<style type="text/css" title="override_css">
            .article_date {
                color: gray; font-family: monospace;
            }

            .article_description {
                text-indent: 0pt;
            }

            a.article {
                font-weight: bold; text-align:left;
            }

            a.feed {
                font-weight: bold;
            }

            .calibre_navbar {
                font-family:monospace;
            }
    

</style></head>
<body class="general_page journal bioinformatics"><div style="text-align:center" class="calibre_navbar calibre_rescale_70">| <a href="../../feed_1/index.html">Next</a> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | <a href="../article_0/index.html">Previous</a> | <hr />
</div><div class="hw-gen-page pagetype-content hw-pub-id-article" id="pageid-content" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><div id="content-block"><div class="article fulltext-view" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<h1 id="article-title-1" itemprop="headline">
      Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text
     </h1>
<div class="contributors">
<ol class="contributor-list" id="contrib-group-1">
<li class="contributor" id="contrib-1" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Leon+French&amp;sortspec=date&amp;submit=Submit">
          Leon French
         </a>
</span>
<a id="xref-aff-1-1" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-2" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-2" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Suzanne+Lane&amp;sortspec=date&amp;submit=Submit">
          Suzanne Lane
         </a>
</span>
<a id="xref-aff-1-3" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-4" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-3" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Lydia+Xu&amp;sortspec=date&amp;submit=Submit">
          Lydia Xu
         </a>
</span>
<a id="xref-aff-1-5" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-6" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-4" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Celia+Siu&amp;sortspec=date&amp;submit=Submit">
          Celia Siu
         </a>
</span>
<a id="xref-aff-1-7" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-8" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-5" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Cathy+Kwok&amp;sortspec=date&amp;submit=Submit">
          Cathy Kwok
         </a>
</span>
<a id="xref-aff-1-9" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-10" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-6" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Yiqi+Chen&amp;sortspec=date&amp;submit=Submit">
          Yiqi Chen
         </a>
</span>
<a id="xref-aff-1-11" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-12" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-7" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Claudia+Krebs&amp;sortspec=date&amp;submit=Submit">
          Claudia Krebs
         </a>
</span>
<a id="xref-aff-1-13" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        and
       </li>
<li class="last" id="contrib-8">
<span class="name">
<a class="name-search" href="/search?author1=Paul+Pavlidis&amp;sortspec=date&amp;submit=Submit">
          Paul Pavlidis
         </a>
</span>
<a id="xref-aff-1-14" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-aff-1-15" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
<a id="xref-corresp-1-1" class="xref-corresp" href="#corresp-1">
         *
        </a>
</li>
</ol>
<ol class="affiliation-list">
<li class="aff">
<a id="aff-1" name="aff-1">
</a>
<address>
<sup>
          1
         </sup>
         Department of Psychiatry,
         <sup>
          2
         </sup>
         Centre for High-Throughput Biology and
         <sup>
          3
         </sup>
         Department of Cellular and Physiological Sciences, University of British Columbia, Vancouver, BC, V6T 1Z4, Canada
        </address>
</li>
</ol>
<ol class="corresp-list">
<li class="corresp" id="corresp-1">
<a class="rev-xref" href="#xref-corresp-1-1">
         ↵
        </a>
        *To whom correspondence should be addressed.
       </li>
</ol>
<ul class="history-list">
<li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-03">
<span class="received-label">
         Received
        </span>
        July 3, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="rev-recd" hwp:start="2012-08-13">
<span class="rev-recd-label">
         Revision received
        </span>
        August 13, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-08-28">
<span class="accepted-label">
         Accepted
        </span>
        August 28, 2012.
       </li>
</ul>
</div>
<div class="section abstract" id="abstract-1" itemprop="description">
<div class="section-nav">
<div class="nav-placeholder">
        �
       </div>
<a href="#sec-1" title="1 INTRODUCTION" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Abstract
      </h2>
<p id="p-3">
<strong>
        Motivation:
       </strong>
       Automated annotation of neuroanatomical connectivity statements from the neuroscience literature would enable accessible
                     and large-scale connectivity resources. Unfortunately, the connectivity findings are not formally encoded and occur as natural
                     language text. This hinders aggregation, indexing, searching and integration of the reports. We annotated a set of 1377 abstracts
                     for connectivity relations to facilitate automated extraction of connectivity relationships from neuroscience literature.
                     We tested several baseline measures based on co-occurrence and lexical rules. We compare results from seven machine learning
                     methods adapted from the protein interaction extraction domain that employ part-of-speech, dependency and syntax features.
      </p>
<p id="p-4">
<strong>
        Results:
       </strong>
       Co-occurrence based methods provided high recall with weak precision. The shallow linguistic kernel recalled 70.1% of the
                     sentence-level connectivity statements at 50.3% precision. Owing to its speed and simplicity, we applied the shallow linguistic
                     kernel to a large set of new abstracts. To evaluate the results, we compared 2688 extracted connections with the Brain Architecture
                     Management System (an existing database of rat connectivity). The extracted connections were connected in the Brain Architecture
                     Management System at a rate of 63.5%, compared with 51.1% for co-occurring brain region pairs. We found that precision increases
                     with the recency and frequency of the extracted relationships.
      </p>
<p id="p-5">
<strong>
        Availability and implementation:
       </strong>
       The source code, evaluations, documentation and other supplementary materials are available at
       <a href="http://www.chibi.ubc.ca/WhiteText">
        http://www.chibi.ubc.ca/WhiteText
       </a>
       .
      </p>
<p id="p-6">
<strong>
        Contact:
       </strong>
<span class="em-link">
<span class="em-addr">
         paul{at}chibi.ubc.ca
        </span>
</span>
</p>
<p id="p-7">
<strong>
        Supplementary information:
       </strong>
<a href="http://www.chibi.ubc.ca/WhiteText">
        Supplementary data
       </a>
       are available at
       <em>
        Bioinformatics
       </em>
       Online.
      </p>
</div>
<div class="section intro" id="sec-1">
<div class="section-nav">
<a href="#abstract-1" title="Abstract" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-2" title="2 METHODS" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       1 INTRODUCTION
      </h2>
<p id="p-8">
       The brain is a vast interconnected network. Each neuron communicates with many others through chemical and electrical synapses
                     to integrate information. Groups of neurons (in structures such as nuclei or layers) make diverse connections across the brain,
                     forming pathways of information flow. This structural connectivity is a major determinant of brain function and is frequently
                     used by neuroscientists and clinicians to interpret physiological data. Examples include understanding strokes (
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
        Haines, 2004
       </a>
       ) and interpreting brain imaging results. Evidence for connectivity abnormalities has been found in bipolar (
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
        Houenou
        <em>
         et al.
        </em>
        , 2007
       </a>
       ), autism (
       <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
        Koshino
        <em>
         et al.
        </em>
        , 2005
       </a>
       ), Alzheimer’s (
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
        Stam
        <em>
         et al.
        </em>
        , 2007
       </a>
       ) and schizophrenia patients (
       <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
        Karlsgodt
        <em>
         et al.
        </em>
        , 2008
       </a>
       ). A major goal of modern neuroscience is to understand the organization of the brain at all levels in as much detail as possible,
                     and to understand how this networked organization relates to brain function and ultimately behaviour and human health (
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
        Sporns, 2011
       </a>
       ).
      </p>
<p id="p-9">
       The characterization of the connectivity network or wiring diagram of the brain is incomplete (
       <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
        Crick and Jones, 1993
       </a>
       ). In part this is due to the complexity of the brain and the difficulty in collecting data. However, we suggest that informatics
                     technologies can be used to leverage existing knowledge that has already been collected to make new discoveries and guide
                     further experimentation.
      </p>
<p id="p-10">
       In this work, we are primarily concerned with ‘macroconnections’, or connections that can be identified between small brain
                     regions (as opposed to microcircuitry, which describes the connections among neurons
       <em>
        per se
       </em>
       ). These macroconnections between groups of neurons are predicted to number between 25 000 and 100 000 (
       <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
        Bota
        <em>
         et al.
        </em>
        , 2003
       </a>
       ). This suggests a high level of complexity, although comfortably placed between the more gross levels of brain organization
                     and the microarchitecture, which encompasses billions of neurons and quadrillions of synapses (
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
        Sporns
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Furthermore, this estimated number of macroconnections is smaller in scale than estimates of the human protein interactome
                     at 650 000 interactions among 25 000 proteins (
       <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
        Stumpf
        <em>
         et al.
        </em>
        , 2008
       </a>
       ).
      </p>
<p id="p-11">
       Connectivity between brain regions can be assayed using tract tracing or electrophysiology. Tract tracing typically involves
                     injecting a dye or other tracer (e.g. horseradish peroxidase) into one brain region and tracking the fate of the tracer as
                     it follows axonal pathways (
       <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
        Lanciego and Wouterlood, 2011
       </a>
       ). Electrophysiological methods use electrical or other stimulation in one site along with electrical recording at a second
                     site to test the functional connectivity of regions. Using these methods, a researcher can determine connections that send
                     signals to the region (afferent) or away from the region (efferent). Over many years, thousands of connectivity studies have
                     been performed, each of which typically elucidates, at most, a few connections. The presence of a deep literature on neuronal
                     connectivity is a major motivation for this work: the data are out there, they just need to be assembled.
      </p>
<p id="p-12">
       Attempts to turn this huge accumulation of knowledge into an ‘omics’ scale database have been limited, despite the potential
                     value of such a resource. Previous efforts have primarily used manual reviews of the literature to laboriously generate connectivity
                     maps for limited parts of the brain. In 1991, Felleman and Van Essen published a connectivity matrix of the macaque visual
                     cortex covering 305 pathways between 32 areas (
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
        Felleman and Van Essen, 1991
       </a>
       ). Currently, a large number of collated connections are stored in the Collations of Connectivity data on the Macaque brain
                     database (CoCoMac) (
       <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
        Kotter, 2004
       </a>
       ). CoCoMac contains detailed information from 413 literature reports regarding 7007 macaque brain regions. A fourth model
                     organism with large-scale connectivity data is the rat, with more than 40 000 formalized reports of connections in the Brain
                     Architecture Management System (BAMS) (
       <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Information is added to these databases manually, and therefore, they are accurate but sparse. Currently, the only complete
                     connectome scale database is the neuron-level wiring diagram of
       <em>
        Caenorhabditis elegans
       </em>
       , determined from electron micrographs (
       <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
        White
        <em>
         et al.
        </em>
        , 1986
       </a>
       ).
      </p>
<p id="p-13">
       We sought to extend and complement manual efforts with automated text mining techniques. More than 10 years of efforts to
                     recognize gene and protein mentions and their interactions inspire our work (
       <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
        Blaschke
        <em>
         et al.
        </em>
        , 1999
       </a>
       ;
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
        Jensen
        <em>
         et al.
        </em>
        , 2006
       </a>
       ). In the gene interaction task, one must extract information from sentences such as ‘gene A interacts with gene B’ (to give
                     a toy example). Despite the difficulty of this task, great progress has been made. A comprehensive evaluation of kernel methods
                     for extracting protein–protein interactions detailed precision and recall values ranging from 45 to 70% by varying experiment
                     design, dataset and method tested (
       <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). At the second Critical Assessment of Information Extraction systems in Biology (BioCreAtIvE II), the top team was able
                     to extract normalized directed interaction pairs from full-text articles, with precision of 37% and recall of 33% (
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
        Krallinger
        <em>
         et al.
        </em>
        , 2008
       </a>
       ). The analogy to brain connectivity is tight: we wish to extract information from sentences akin to ‘brain region A connects
                     to brain region B’. This related research gives us hope that the approaches applied to extracting gene interaction information
                     can successfully mine connectivity relations.
      </p>
<p id="p-14">
       To our knowledge, there have been no previous attempts to extract connectivity information using text mining methods. The
                     closest work to our own is that of
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
        Burns
        <em>
         et al.
        </em>
        (2007)
       </a>
       , which was aimed at extracting information about tract-tracing experiments, trained and evaluated with a manually annotated
                     corpus of 1047 sentences from 21 documents. Although Burns
       <em>
        et al.
       </em>
       describe named entity recognition (e.g. identification of label injection sites), they did not report extraction of connectivity
                     statements themselves.
      </p>
<p id="p-15">
       For the work presented here, we have simplified the problem by limiting our input dataset and output results. We focus on
                     abstracts from one journal, the Journal of Comparative Neurology (JCN), because it is enriched for tract-tracing studies.
                     We used abstracts rather than full-text documents because they are enriched for high-level summary statements and are more
                     accessible. We also break our task into several subtasks, isolating the steps of brain region term recognition and normalization
                     from the evaluations (
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
        French
        <em>
         et al.
        </em>
        , 2009
       </a>
       ;
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
        French and Pavlidis, 2011
       </a>
       ). We only consider the presence of connectivity relations, ignoring the type, strength or direction of the connection. These
                     limitations make the task simpler and set the stage for future more detailed studies.
      </p>
<p id="p-16">
       Our results show that text mining approaches previously used to analyse protein networks can be usefully applied to brain
                     connectivity. Our large manually annotated corpus allowed testing and training of various techniques possible, and we also
                     perform extensive manual validation of the results. Beyond the corpus-based evaluations, we compared a large set of automatically
                     extracted connectivity statements with an existing connectivity database with favourable results. Together with our previous
                     work on term recognition and normalization, we present a completely automated system for extraction of brain connectivity
                     information from abstracts.
      </p>
</div>
<div class="section methods" id="sec-2">
<div class="section-nav">
<a href="#sec-1" title="1 INTRODUCTION" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-9" title="3 RESULTS" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       2 METHODS
      </h2>
<div id="sec-3" class="subsection">
<h3>
        2.1 Annotated data
       </h3>
<p id="p-17">
        To train and test text mining algorithms, we created a large gold standard dataset. This dataset or corpus consists of abstracts
                        manually annotated by an undergraduate research assistant (S.L.) for connection verbs, species of study, brain region mentions
                        and connections between them. We annotated 1377 abstracts for 4276 connections and 17 585 brain region mentions. Abstracts
                        were randomly chosen from the JCN (years 1975–2008). This dataset has been previously used to demonstrate automated brain
                        region recognition and normalization (
        <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
         French
         <em>
          et al.
         </em>
         , 2009
        </a>
        ;
        <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
         French and Pavlidis, 2011
        </a>
        ), without using the connectivity annotations. Each annotated connection consists of two brain regions, text describing the
                        connection and the associated organism. This corpus provides sufficient training examples for machine-learning methods.
       </p>
<p id="p-18">
        We developed guidelines and software for the annotation process. Briefly, our main guidelines were as follows: (1) annotate
                        all brain region mentions, regardless of whether they are part of a connection; (2) annotate all connections and brain regions
                        for all organisms and organism states; (3) do not annotate mentions of white matter tracts or nerves; and (4) only annotate
                        monosynaptic or direct connections. We accepted connections that were stated in titles or introductory sentences that assume
                        connections between two high-level regions. Example relationships that were rejected are ‘may be connected’, ‘influences’,
                        ‘invaded’ and ‘alters activity’. The General Architecture for Text Engineering was used by annotators to highlight and connect
                        brain region mentions in text (
        <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
         Cunningham
         <em>
          et al.
         </em>
         , 2002
        </a>
        ).
       </p>
</div>
<div id="sec-4" class="subsection">
<h3>
        2.2 Co-occurrence and rule-based methods
       </h3>
<p id="p-19">
        To extract neuroanatomical connections as described by the abstract authors, we must at least link two brain region mentions.
                        Our first method, acting as a naïve baseline method, predicts a stated connection between every pair of brain region mentions
                        (
        <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
         Jensen
         <em>
          et al.
         </em>
         , 2006
        </a>
        ). We evaluate co-occurrence for single sentences and entire abstracts (including title).
       </p>
<p id="p-20">
        We created two simple rule-based extensions of the co-occurrence technique. The first simply limits co-occurrence extraction
                        to sentences with less than a set number of brain region mentions. The second requires presence of a connectivity-related
                        keyword (‘afferent’, ‘efferent’, ‘projects’, ‘projection’, ‘pathway’ or ‘inputs’).
       </p>
</div>
<div id="sec-5" class="subsection">
<h3>
        2.3 Kernel-based methods
       </h3>
<p id="p-21">
        Seven advanced kernel-based methods were applied to the dataset. These methods were originally designed for extraction of
                        protein–protein interactions. Each technique uses different features, parameters and kernel functions. Implementations were
                        brought into a common evaluation framework by Tikk and colleagues (
        <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ). Syntax and dependency trees for the sentences were computed by the Charniak-Lease re-ranking (
        <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
         McClosky
         <em>
          et al.
         </em>
         , 2006
        </a>
        ) and Stanford (
        <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
         De Marneffe
         <em>
          et al.
         </em>
         , 2006
        </a>
        ) parsers, respectively (same versions used in the Tikk
        <em>
         et al.
        </em>
        framework). The methods are categorized according to the type of features extracted. Four syntax tree-based methods use different
                        techniques to compare the sentence parse trees (
        <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
         Collins and Duffy, 2001
        </a>
        ;
        <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
         Kuboyama
         <em>
          et al.
         </em>
         , 2007
        </a>
        ;
        <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
         Moschitti, 2005
        </a>
        ;
        <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
         Vishwanathan and Smola, 2002
        </a>
        ). Going beyond syntax parsers, the all-paths graph kernel (
        <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
         Airola
         <em>
          et al.
         </em>
         , 2008
        </a>
        ) and k-band shortest path spectrum kernel (
        <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ) use dependency parse information. Finally, the shallow linguistic kernel (SLK) uses only shallow parsing information such
                        as word occurrences and part-of-speech tags (
        <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
         Giuliano
         <em>
          et al.
         </em>
         , 2006
        </a>
        ). Further details about the kernels are available on the supplement website. We used this framework to benchmark each of
                        the kernel-based methods on the brain region connectivity task. Of the nine methods described by Tikk
        <em>
         et al.
        </em>
        , we were able to successfully test seven, including the three top-performing kernels reported by
        <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">
         Tikk
         <em>
          et al.
         </em>
         (2010)
        </a>
        . The same parameter sets used by Tikk and colleagues were tested on our corpus.
       </p>
</div>
<div id="sec-6" class="subsection">
<h3>
        2.4 Experiment setup
       </h3>
<p id="p-22">
        We evaluate connection extraction independently of the previously described methods for automated brain region recognition
                        (
        <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
         French
         <em>
          et al.
         </em>
         , 2009
        </a>
        ). This is done by providing the manually annotated brain region mentions to the relation extraction algorithm. Under this
                        design, the extraction task only requires correct linking of brain region mentions.
       </p>
<p id="p-23">
        To find a high-performing method, the different methods and varying parameters were run on a subset of 1146 abstracts. The
                        top-performing classifier and parameter set were retested on the full set of 1377 abstracts to gauge generalizability. Results
                        for the kernel methods were computed using 10-fold cross-validation. Each sentence became an input instance for the kernel
                        methods (including article title). Sentences of an abstract were not split between training and testing sets (document-level
                        split).
       </p>
</div>
<div id="sec-7" class="subsection">
<h3>
        2.5 Evaluation
       </h3>
<p id="p-24">
        Several evaluations were performed to judge the accuracy of the extracted connectivity statements. We primarily report the
                        results from the cross-validation experiments that test predictions against the manually annotated connections. Detailed evaluation
                        and annotation guidelines are provided as
        <a href="http://www.chibi.ubc.ca/WhiteText">
         Supplementary Materials
        </a>
        .
       </p>
<p id="p-25">
        Performance is measured against the number of true connectivity relations that are annotated completely within a sentence
                        or an abstract. The rule- and co-occurrence-based methods can operate at the abstract or sentence level, whereas the kernel
                        methods are limited to single sentences. Precision is computed as the proportion of predicted relations that are correct,
                        and recall is the proportion of true relations that are predicted by the method. The f-measure is the harmonic mean of these
                        two values, providing a balance of both. We also compute the area under the receiver operating curve where applicable (AUC).
                        This measure uses a ranked list of predictions with descending classification prediction scores that approximate confidence
                        in the prediction. This ranking allows computation of the true-positive and false-positive rates for a range of discrimination
                        thresholds. Previous experiments have found the AUC measure to be more robust and stable than f-measure for interaction mining
                        (
        <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">
         Tikk
         <em>
          et al.
         </em>
         , 2010
        </a>
        ).
       </p>
</div>
<div id="sec-8" class="subsection">
<h3>
        2.6 Comparison with existing connectivity database
       </h3>
<p id="p-26">
        Normalization of brain region mentions to brain region concepts in formalized lexicons was targeted to the BAMS atlas (
        <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
         Swanson, 1999
        </a>
        ). BAMS was chosen because of its wealth of curated rat-tract-tracing studies (
        <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
         Bota
         <em>
          et al.
         </em>
         , 2005
        </a>
        ). In addition, rat is the most commonly studied species in our corpus. Our previously described Bag of Stems resolver was
                        applied, with all mention editors used, including those that map region mentions to larger enclosing brain regions (
        <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
         French and Pavlidis, 2011
        </a>
        ). The lexical information in BAMS was expanded with synonym information to increase normalization performance. All possible
                        normalized parings are evaluated when a mention maps to more than one region. Connections in the BAMS connectivity matrices
                        were up-propagated in the anatomy hierarchy, which ensures that if there is a connection between regions A and B, then all
                        enclosing regions of A and B are also connected. Self-connections extracted from literature were ignored. The Linnaeus species
                        tagger was used to recognize species names in the abstracts (
        <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
         Gerner
         <em>
          et al.
         </em>
         , 2010
        </a>
        ).
       </p>
</div>
</div>
<div class="section results" id="sec-9">
<div class="section-nav">
<a href="#sec-2" title="2 METHODS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-10" title="4 DISCUSSION" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       3 RESULTS
      </h2>
<p id="p-27">
       Our gold standard is a set of manually annotated 4276 brain region connectivity relations across a corpus of 1377 abstracts.
                     To gauge interannotator agreement, a second curator (L.X.) annotated a random subset of 231 documents. Roughly 80% of the
                     second curator’s annotations matched the primary curator (79.5% recall at 82.3% precision). Unlike the automated methods that
                     predict relations between given brain region mention spans, this evaluation required both annotators to mark the same spans
                     and relationships. By removing this restriction and allowing partially matching spans, the precision and recall reach 93.9%
                     and 91.9%, respectively.
      </p>
<p id="p-28">
       We used a co-occurrence analysis to reveal the proportion of brain region mention pairs that are co-mentioned and described
                     as connected. Co-occurrence assumes the relation is a connectivity statement. At the abstract level, this yields a precision
                     of only 2.2% at 100% recall and a f-measure of 4.3%. Within a sentence, co-occurrences between all pairs predict connected
                     pairs at 13.3% precision and 72.4% recall (the remaining relations span sentences). This level of recall means that more than
                     ¼ of all annotated connectivity relations are formed with regions in different sentences. Owing to the difficulty in extracting
                     connections spanning sentences, all of our subsequent evaluations are performed at the sentence level, with the relations
                     spanning sentences excluded. Under this evaluation framework, sentence-level co-occurrence recalls 100% of the remaining 3097
                     relations.
      </p>
<p id="p-29">
       We tested two simple modifications of the sentence-level co-occurrence technique. The first reduces co-occurrence predictions
                     to sentences with a limited number of brain region mentions. By extracting co-occurring pairs from sentences with only two
                     brain region mentions, precision reaches 23.1% and 17.2% recall (f-measure = 19.7%). This means that an average sentence with
                     two brain region mentions is reporting a connection in almost one of four cases. By varying this threshold, the f-measure
                     increases until sentences with six or more brain region mentions are included. We observed that some of these larger sentences
                     merely list brain regions involved in the study and not their relationships. By limiting the threshold at five brain region
                     mentions or less per sentence, co-occurrence provides 18.8% precision and 66.1% recall (
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
        Table 1
       </a>
       ). The second rule tested requires the sentences contain connectivity-related keywords (see Methods section). This keyword-based
                     rule increases recall to 17.4% and precision to 92.7% (f-measure = 29.4%). We created a new approach named ‘Keyword 5-threshold’
                     by combining these two rules. This again provides improvement, with f-measure reaching 34.1%. As expected, rule-based methods
                     increase precision at the cost of lower recall when compared with unrestricted co-occurrence.
      </p>
<div class="table pos-float" id="T1">
<div class="table-expansion" id="T1" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 1.
         </span>
<p id="p-30">
          Sentence level training set cross-validation results
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table frame="hsides" rules="groups" id="table-1">
<thead align="left" id="thead-1" class="table-left">
<tr id="tr-1">
<th rowspan="1" colspan="1" id="th-1">
            Kernel
           </th>
<th rowspan="1" colspan="1" id="th-2">
            Parser type
           </th>
<th rowspan="1" colspan="1" id="th-3">
            Parameter sets
           </th>
<th rowspan="1" colspan="1" id="th-4">
            Precision
           </th>
<th rowspan="1" colspan="1" id="th-5">
            Recall
           </th>
<th rowspan="1" colspan="1" id="th-6">
            F-measure
           </th>
<th rowspan="1" colspan="1" id="th-7">
            AUC
           </th>
</tr>
</thead>
<tbody align="left" id="tbody-1" class="table-left">
<tr id="tr-2">
<td id="td-1">
            Co-occurrence
           </td>
<td id="td-2">
            None
           </td>
<td id="td-3">
            1
           </td>
<td id="td-4">
            13.30%
           </td>
<td id="td-5">
            100.00%
           </td>
<td id="td-6">
            23.50%
           </td>
<td id="td-7">
</td>
</tr>
<tr id="tr-3">
<td id="td-8">
            Subset tree kernel
           </td>
<td id="td-9">
            Syntax
           </td>
<td id="td-10">
            12
           </td>
<td id="td-11">
            44.20%
           </td>
<td id="td-12">
            20.80%
           </td>
<td id="td-13">
            28.10%
           </td>
<td id="td-14">
            74.80%
           </td>
</tr>
<tr id="tr-4">
<td id="td-15">
            Co-occurrence five threshold
           </td>
<td id="td-16">
            None
           </td>
<td id="td-17">
            25
           </td>
<td id="td-18">
            18.80%
           </td>
<td id="td-19">
            66.10%
           </td>
<td id="td-20">
            29.30%
           </td>
<td id="td-21">
</td>
</tr>
<tr id="tr-5">
<td id="td-22">
            Partial tree kernel
           </td>
<td id="td-23">
            Syntax
           </td>
<td id="td-24">
            12
           </td>
<td id="td-25">
            43.30%
           </td>
<td id="td-26">
            23.10%
           </td>
<td id="td-27">
            29.80%
           </td>
<td id="td-28">
            75.20%
           </td>
</tr>
<tr id="tr-6">
<td id="td-29">
            Keyword co-occurrence
           </td>
<td id="td-30">
            None
           </td>
<td id="td-31">
            1
           </td>
<td id="td-32">
            17.40%
           </td>
<td id="td-33">
            92.70%
           </td>
<td id="td-34">
            29.40%
           </td>
<td id="td-35">
</td>
</tr>
<tr id="tr-7">
<td id="td-36">
            Spectrum tree kernel
           </td>
<td id="td-37">
            Syntax
           </td>
<td id="td-38">
            21
           </td>
<td id="td-39">
            37.40%
           </td>
<td id="td-40">
            26.10%
           </td>
<td id="td-41">
            30.20%
           </td>
<td id="td-42">
            72.90%
           </td>
</tr>
<tr id="tr-8">
<td id="td-43">
            Subtree kernel
           </td>
<td id="td-44">
            Syntax
           </td>
<td id="td-45">
            12
           </td>
<td id="td-46">
            40.70%
           </td>
<td id="td-47">
            25.20%
           </td>
<td id="td-48">
            30.80%
           </td>
<td id="td-49">
            74.60%
           </td>
</tr>
<tr id="tr-9">
<td id="td-50">
            Keyword five threshold
           </td>
<td id="td-51">
            None
           </td>
<td id="td-52">
            25
           </td>
<td id="td-53">
            23.70%
           </td>
<td id="td-54">
            60.80%
           </td>
<td id="td-55">
            34.10%
           </td>
<td id="td-56">
</td>
</tr>
<tr id="tr-10">
<td id="td-57">
            k-band shortest path spectrum
           </td>
<td id="td-58">
            Dependency
           </td>
<td id="td-59">
            288
           </td>
<td id="td-60">
            46.80%
           </td>
<td id="td-61">
            70.50%
           </td>
<td id="td-62">
            55.80%
           </td>
<td id="td-63">
            86.70%
           </td>
</tr>
<tr id="tr-11">
<td id="td-64">
            Shallow linguistic kernel (SLK)
           </td>
<td id="td-65">
            Part-of-speech tagger
           </td>
<td id="td-66">
            1
           </td>
<td id="td-67">
            50.30%
           </td>
<td id="td-68">
            70.10%
           </td>
<td id="td-69">
            58.30%
           </td>
<td id="td-70">
            88.90%
           </td>
</tr>
<tr id="tr-12">
<td id="td-71">
            All-paths graph kernel
           </td>
<td id="td-72">
            Dependency
           </td>
<td id="td-73">
            4
           </td>
<td id="td-74">
            60.40%
           </td>
<td id="td-75">
            57.90%
           </td>
<td id="td-76">
            58.40%
           </td>
<td id="td-77">
            88.40%
           </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li class="fn" id="fn-2">
<p id="p-31">
            AUC, area under the receiver operating curve; SLK, shallow linguistic kernel.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 1.
        </span>
<p id="p-30" class="first-child">
         Sentence level training set cross-validation results
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-32">
       Next, we applied seven previously published methods for extracting protein–protein interactions to our connectivity relation
                     dataset. Although the methods were designed for a different type of biomedical relation, they did not require any modification
                     for our application. The cross-validation results on the training dataset (1146 abstracts) are provided in
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
        Table 1
       </a>
       . For each method, the parameter set with the highest AUC score is shown. The parameter sets range in size and were reproduced
                     from Tikk
       <em>
        et al.
       </em>
       without modification (primarily grid searches of support vector machine settings). The f-measure scores for all of the seven
                     methods outperform unrestricted co-occurrence-based analysis for at least one parameter set. The simple rule-based methods
                     outperform the more complex partial tree- and subset tree-based methods. Although all of the syntax tree-based methods are
                     outperformed by the Keyword 5-threshold approach, they provide much higher precision than recall. When ranked by AUC, the
                     SLK performs best with a 58.3% f-measure and an AUC of 88.9%. The All Paths Graph and k-band shortest path spectrum kernel
                     methods rank a close second and third, respectively, with similar scores.
      </p>
<p id="p-33">
       We choose the SLK method for subsequent experiments owing to its accuracy, speed and single parameter set (global n-gram =
                     3 and local window = 2). Unlike the other kernel methods, the SLK method uses only shallow linguistic information at the local
                     (neighbouring words) and global sentence levels to predict relationships (
       <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
        Giuliano
        <em>
         et al.
        </em>
        , 2006
       </a>
       ). This information forms feature vectors that are used to train a support vector machine classifier (scalar product kernel).
                     The performance of SLK on the complete set of 1377 abstracts is consistent with the cross-validation results (f-measure of
                     0.592).
       <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts542/DC1">
        Figure S1
       </a>
       displays the resulting receiver operating characteristic (ROC) curve (AUC = 0.899).
      </p>
<p id="p-34">
       We applied the SLK classifier to candidate sentences extracted from a set of 12 557 abstracts from the JCN (covering 1975–2011),
                     previously unseen by the algorithm. Our automatic brain region recognizer provided 33 466 sentences that mention two or more
                     brain regions (
       <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">
        French
        <em>
         et al.
        </em>
        , 2009
       </a>
       ;
       <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
        French and Pavlidis, 2011
       </a>
       ). Within these sentences, SLK predicted 18% of the 156 484 possible brain region pairings to be connectivity relations. Of
                     these predicted relations, 9676 are in an abstract that mentions rat and can be evaluated against BAMS.
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">
        Figure 1
       </a>
       shows the progression from abstracts to predicted connectivity relationships.
      </p>
<div id="F1" class="fig pos-float odd">
<div class="fig-inline">
<a href="2963/F1.expansion.html">
<img alt="Fig. 1." src="images/img1.jpg" width="600" />
</a>
<div class="callout">
<span>
          View larger version:
         </span>
<ul class="callout-links">
<li>
<a href="2963/F1.expansion.html">
            In this window
           </a>
</li>
<li>
<a class="in-nw" href="2963/F1.expansion.html">
            In a new window
           </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/28/22/2963/F1">
            Download as PowerPoint Slide
           </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
         Fig. 1.
        </span>
<p id="p-35" class="first-child">
         Flow chart depicting the processing steps for comparison with the Brain Architecture Management System
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-36">
       To evaluate the precision of the predicted connections, we manually reviewed a random subset of 2000 abstracts. Each pair
                     was evaluated by two curators, yielding an interannotator agreement rate of 85%. Conflicts were resolved by a third curator
                     or by consensus after discussion. Overall, the SLK predictions were 55.3% precise. Errors from the automated steps of named
                     entity recognition and abbreviation expansion were 11% and 4%, respectively. These rates suggest a lesser impact of named
                     entity recognition errors when compared with assessments in the protein–protein interaction domain (
       <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
        Kabiljo
        <em>
         et al.
        </em>
        , 2009
       </a>
       ).
       <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
        Table 2
       </a>
       presents the five most and least confident connectivity relations for the rat brain. Classification confidence is approximated
                     with the SLK prediction score (distance to classifying hyperplane), with highest values representing the cases closest to
                     positive training examples. Two of the most confident predictions are extracted from an article title and have the same form
                     (ranks 1 and 5). The sentences containing top predictions are shorter on average (192 characters) than the sentences with
                     least confident predictions (282 characters), suggesting sentence complexity affects the prediction results. Of these 10 examples,
                     only one is clearly a false-positive prediction (rank 9764), while several others point to errors in previous automated steps.
                     The mentions of ‘internal capsule’ (rank 9766) and ‘Met-enkephalin’ (rank 4) are incorrectly predicted as brain region mentions
                     (our definition of a brain region excludes fibre tracts like the internal capsule, while enkephalin is a peptide). We manually
                     compared these 10 results with the BAMS system and found it surprisingly difficult to map the mentioned regions to those in
                     BAMS. For example, ‘retrosplenial dysgranular cortex’ and ‘dorsal medullary reticular column’ were not found in BAMS. In the
                     end, corresponding connections were found in BAMS for several of the relationships, but only between enclosing regions (ranks
                     9767 and 5).
      </p>
<div class="table pos-float" id="T2">
<div class="table-expansion" id="T2" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 2.
         </span>
<p id="p-37">
          Top- and bottom-predicted relations from the 12 557 abstract set, ranked by SLK classification score
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table frame="hsides" rules="groups" id="table-2">
<thead align="left" id="thead-2" class="table-left">
<tr id="tr-13">
<th rowspan="1" colspan="1" id="th-8">
            Rank
           </th>
<th rowspan="1" colspan="1" id="th-9">
            Sentence
           </th>
<th rowspan="1" colspan="1" id="th-10">
            Score
           </th>
<th rowspan="1" colspan="1" id="th-11">
            Reference
           </th>
</tr>
</thead>
<tbody align="left" id="tbody-2" class="table-left">
<tr id="tr-14">
<td id="td-78">
            1
           </td>
<td id="td-79">
<strong>
             Trigeminal
            </strong>
            projections to
            <strong>
             hypoglossal and facial motor nuclei
            </strong>
            in the rat.
           </td>
<td id="td-80">
            3.47
           </td>
<td id="td-81">
<span id="xref-ref-35-1" class="xref-bibr">
             Pinganaud,
             <em>
              et al.
             </em>
             , 1999
            </span>
</td>
</tr>
<tr id="tr-15">
<td id="td-82">
            2
           </td>
<td id="td-83">
<strong>
             The cortical
            </strong>
            projections to
            <strong>
             retrosplenial dysgranular cortex
            </strong>
            (Rdg) originate primarily in the infraradiata, retrosplenial, postsubicular and areas 17 and 18b cortices.
           </td>
<td id="td-84">
            3.34
           </td>
<td id="td-85">
<span id="xref-ref-47-1" class="xref-bibr">
             van Groen and Wyss, 1992
            </span>
</td>
</tr>
<tr id="tr-16">
<td id="td-86">
            3
           </td>
<td id="td-87">
            The
            <strong>
             thalamic
            </strong>
            projections to
            <strong>
             retrosplenial dysgranular cortex
            </strong>
            (Rdg) originate in the anterior (primarily the anteromedial), lateral (primarily the laterodorsal) and reuniens nuclei.
           </td>
<td id="td-88">
            3.33
           </td>
<td id="td-89">
<span id="xref-ref-47-2" class="xref-bibr">
             van Groen and Wyss, 1992
            </span>
</td>
</tr>
<tr id="tr-17">
<td id="td-90">
            4
           </td>
<td id="td-91">
            Our results indicate that the
            <strong>
             centromedial amygdala
            </strong>
            receives
            <strong>
             Met-enkephalin
            </strong>
            afferents, as indicated by the presence of mu-opioid receptor, delta-opioid receptor and Met-enkephalin fibres in the CEA
                           and MEA, originating primarily from the bed nucleus of the stria terminalis and from other amygdaloid nuclei.
           </td>
<td id="td-92">
            3.32
           </td>
<td id="td-93">
<span id="xref-ref-36-1" class="xref-bibr">
             Poulin,
             <em>
              et al.
             </em>
             , 2006
            </span>
</td>
</tr>
<tr id="tr-18">
<td id="td-94">
            5
           </td>
<td id="td-95">
<strong>
             Thalamic
            </strong>
            projections to
            <strong>
             retrosplenial cortex
            </strong>
            in the rat.
           </td>
<td id="td-96">
            3.28
           </td>
<td id="td-97">
<span id="xref-ref-42-1" class="xref-bibr">
             Sripanidkulchai and Wyss, 1986
            </span>
</td>
</tr>
<tr id="tr-19">
<td id="td-98">
            …
           </td>
<td id="td-99">
            9757 relationships
           </td>
<td id="td-100">
</td>
<td id="td-101">
</td>
</tr>
<tr id="tr-20">
<td id="td-102">
            9763
           </td>
<td id="td-103">
            The sparse reciprocal connections to the other
            <strong>
             amygdaloid nuclei
            </strong>
            suggest that the CEA nucleus does not regulate the other amygdaloid regions, but rather executes the responses evoked by
                           the other amygdaloid nuclei that innervate the
            <strong>
             CEA nucleus
            </strong>
            .
           </td>
<td id="td-104">
            5.46 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-105">
<span id="xref-ref-23-1" class="xref-bibr">
             Jolkkonen and Pitkanen, 1998
            </span>
</td>
</tr>
<tr id="tr-21">
<td id="td-106">
            9764
           </td>
<td id="td-107">
            The majority of the endomorphin 1/fluoro-gold and endomorphin 2/fluoro-gold double-labelled neurons in the
            <strong>
             hypothalamus
            </strong>
            were distributed in the
            <strong>
             dorsomedial nucleus
            </strong>
            , areas between the dorsomedial and ventromedial nucleus and arcuate nucleus; a few were also seen in the ventromedial, periventricular
                           and posterior nucleus.
           </td>
<td id="td-108">
            4.36 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-109">
<span id="xref-ref-7-1" class="xref-bibr">
             Chen,
             <em>
              et al.
             </em>
             , 2008
            </span>
</td>
</tr>
<tr id="tr-22">
<td id="td-110">
            9765
           </td>
<td id="td-111">
            Projections from the
            <strong>
             dorsal medullary reticular column
            </strong>
            are largely bilateral and are distributed preferentially to the
            <strong>
             ventral subdivision of the fifth cranial nerve motor nuclei in the rat (MoV)
            </strong>
            , to the dorsal and intermediate subdivisions of VII and to both the dorsal and the ventral subdivision of XII.
           </td>
<td id="td-112">
            2.91 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-113">
<span id="xref-ref-10-1" class="xref-bibr">
             Cunningham and Sawchenko, 2000
            </span>
</td>
</tr>
<tr id="tr-23">
<td id="td-114">
            9766
           </td>
<td id="td-115">
            Two additional large projections leave the MEA forebrain bundle in the hypothalamus; the ansa peduncularis–ventral amygdaloid
                           bundle system turns laterally through the
            <strong>
             internal capsule
            </strong>
            into the striatal complex,
            <strong>
             amygdala
            </strong>
            and the external capsule to reach lateral and posterior cortex, and another system of fibers turns medially to innervate
                           MEA hypothalamus and median eminence and forms a contralateral projection through the supraoptic commissures.
           </td>
<td id="td-116">
            2.87 × 10
            <sup>
             −4
            </sup>
</td>
<td id="td-117">
<span id="xref-ref-32-1" class="xref-bibr">
             Moore,
             <em>
              et al.
             </em>
             , 1978
            </span>
</td>
</tr>
<tr id="tr-24">
<td id="td-118">
            9767
           </td>
<td id="td-119">
            In animals with injected horseradish peroxidase confined within the main bulb, perikarya retrogradely labelled with the protein
                           in the
            <strong>
             ipsilateral forebrain
            </strong>
            were observed in the
            <strong>
             anterior prepyriform cortex horizontal limb of the nucleus of the diagonal band
            </strong>
            , and far lateral preoptic and rostral lateral hypothalamic areas.
           </td>
<td id="td-120">
            3.36 × 10
            <sup>
             −5
            </sup>
</td>
<td id="td-121">
<span id="xref-ref-5-1" class="xref-bibr">
             Broadwell and Jacobowitz, 1976
            </span>
</td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li class="fn" id="fn-3">
<p id="p-38">
            CEA, central; MEA, medial; MoV, the fifth cranial nerve motor nuclei in the rat.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 2.
        </span>
<p id="p-37" class="first-child">
         Top- and bottom-predicted relations from the 12 557 abstract set, ranked by SLK classification score
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-39">
       Encouraged by these results, we did a comparison of the results from the 12 557 abstracts with BAMS, to gauge accuracy of
                     connections and the extent to which our approach might supplement manual curation efforts. Compared with the manual annotations,
                     this is a less precise evaluation because BAMS does not cover the complete literature and is limited to rat studies (
       <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). In addition, mapping errors resulting from linking brain region mentions to target regions in BAMS reduces accuracy (
       <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">
        French and Pavlidis, 2011
       </a>
       ). For example, 12% of mentions are mapped to more than one brain region owing to ambiguous synonyms. To benchmark the BAMS
                     evaluation metric, we first tested it on the manually curated connectivity relations from our training corpus of 1377 abstracts.
                     Our process first extracts abstracts that used rat (based on Linneaus analysis) and maps the brain region mentions to the
                     BAMS lexicon. These rat connectivity relationships were then compared with the BAMS connectivity matrix. Only 167 manually
                     annotated connectivity relations were testable by this method, with 70.5% having a connection in BAMS. In the same set of
                     abstracts, the 2617 brain region pairings not annotated as connections but co-occur in sentences are connected in BAMS at
                     49.8%. This is not surprising because co-occurring regions may be connected, but the author is not stating that in the sentence.
                     In the larger set of rat-related abstracts, 2688 computationally predicted connectivity statements are successfully resolved,
                     and 63.5% are, in fact, reported as true by BAMS (
       <a id="xref-fig-1-2" class="xref-fig" href="#F1">
        Fig. 1
       </a>
       ). For comparison, the remaining set of co-occurring brain region pairs is connected in BAMS at a rate of 51.1%. We noted
                     that the extracted relationships are between larger or less specific brain regions than those in BAMS. Anatomical depth, or
                     the average number of enclosing or parent brain regions for a connected pair in the BAMS matrix, is 9.6, whereas the literature-extracted
                     connections had a mean of 7.9, indicating they are larger and less specific brain regions. Along the same lines, the literature-based
                     relationships only involve 433 regions, whereas BAMS has connection reports for 633 regions. We evaluated the 899 connections
                     that are predicted by our method but not listed as connected in BAMS (provided in Supplement). Similar to the previous evaluation
                     of 2000 connections, approximately one-half of these connections are false-positive text mining errors (52.1% precision).
                     The remaining 468 connections that are true positives at the sentence level could be used to expand BAMS coverage (although
                     curation guidelines differ). Within these 468 true positives, we selected a subset of 250 for further review by a domain expert
                     (C. Krebs). Only nine of the predicted connections were rejected (3.6%). Five were rejected because a protein (pituitary adenylate
                     cyclise-activating polypeptide) was incorrectly recognized as a brain region (pituitary gland). This agreement between the
                     curators and an expert suggests our annotation guidelines are consistent and accurate.
      </p>
<p id="p-40">
       We hypothesized that owing to improvements in tracing methods, more recent reports of connectivity would be of higher quality.
                     This was suggested by a study of different eras of tract-tracing techniques that revealed large improvements in accuracy (
       <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
        Bota
        <em>
         et al.
        </em>
        , 2003
       </a>
       ). Bota and colleagues found that limbic system connections observed using an old method, axon degeneration (
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
        Nauta, 1952
       </a>
       ), are 60% accurate. In contrast, methods first applied in 1987 to exploit axonal transport are much more accurate with more
                     than 90% considered valid. By splitting our corpus into documents published before and after 1987, we tested for a similar
                     signal that separates eras of experimental techniques. In agreement with the manually quantified trend, we observe an increase
                     from 59.4 to 65.6% in the rate of connectivity statements validated in BAMS (
       <em>
        P
       </em>
       = 0.00071, hypergeometric test). We note the specificity of regions involved in the connections also increases, whereas the
                     proportion of mapped terms is unchanged.
      </p>
<p id="p-41">
       Connections predicted more than once might be more likely to be valid because of the effect of ‘confirmation’. This was feasible
                     to study because, on average, each connection was predicted more than twice. The number of extracted connections per brain
                     region (degree) provides a simple comparison with BAMS. For the 344 common brain regions, the degree vectors are strongly
                     correlated (Pearson = 0.769, Spearman = 0.433). Counting unique predicted connections, 54.7% are in BAMS (
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
        Table 3
       </a>
       , the value of 63.5% previously cited counts occurrences). From a recall perspective, 3.2% of BAMS connections are connected
                     in our predictions. By thresholding our connections to those predicted at least twice, precision reaches 65.9%, whereas recall
                     drops to 1.4% (
       <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">
        Table 3
       </a>
       ). This accuracy is near the 67.5% precision of the hand-annotated set of connections. Precision gradually increases as the
                     threshold increases, eventually reaching 100% for nine connections that were extracted at least 12 times. Further, we note
                     the anatomical specificity of the connections increases with the average number of enclosing regions reaching 10.2 when thresholded
                     at 12 occurrences. The region pairs not predicted to form connectivity relations have precision of 33.7% and recall of 9.3%.
                     Again, this level of precision results from co-mentioned regions that are connected in BAMS, but the author is not specifying
                     that in the sentence. Further, the higher recall value results from the much larger set of pairings (6079 compared with 1286
                     SLK-predicted parings). From a co-occurrence perspective, we found that brain regions that co-occur in eight or more sentences
                     recall 1.6% of the BAMS connections at 66.4% precision. Interestingly, this naive co-occurrence-based method performs at par
                     to the SLK method that extracts direct connectivity statements. As the threshold is increased from eight co-occurrences, precision
                     continues to gain, suggesting a large number of co-occurring mentions can be used to predict connectivity, as well as a smaller
                     number of more carefully analysed connectivity statements (
       <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">
        Table 3
       </a>
       ).
      </p>
<div class="table pos-float" id="T3">
<div class="table-expansion" id="T3" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
          Table 3.
         </span>
<p id="p-42">
          Aggregate connectivity results from several methods and relation sets
         </p>
<div class="sb-div caption-clear">
</div>
</div>
<table frame="hsides" rules="groups" id="table-3">
<thead align="left" id="thead-3" class="table-left">
<tr id="tr-25">
<th rowspan="1" colspan="1" id="th-12">
            Relation set
           </th>
<th rowspan="1" colspan="1" id="th-13">
            Method
           </th>
<th rowspan="1" colspan="1" id="th-14">
            Threshold
           </th>
<th rowspan="1" colspan="1" id="th-15">
            Anatomical depth
           </th>
<th rowspan="1" colspan="1" id="th-16">
            Connections
           </th>
<th rowspan="1" colspan="1" id="th-17">
            Precision
           </th>
<th rowspan="1" colspan="1" id="th-18">
            Recall
           </th>
<th rowspan="1" colspan="1" id="th-19">
            F-measure
           </th>
</tr>
</thead>
<tbody align="left" id="tbody-3" class="table-left">
<tr id="tr-26">
<td id="td-122">
            Positive annotated
           </td>
<td id="td-123">
            Curation
           </td>
<td id="td-124">
            1
           </td>
<td id="td-125">
            8.7
           </td>
<td id="td-126">
            200
           </td>
<td id="td-127">
            67.50%
           </td>
<td id="td-128">
            0.61%
           </td>
<td id="td-129">
            1.22%
           </td>
</tr>
<tr id="tr-27">
<td id="td-130">
            Negative annotated
           </td>
<td id="td-131">
            Curation
           </td>
<td id="td-132">
            1
           </td>
<td id="td-133">
            8.7
           </td>
<td id="td-134">
            1606
           </td>
<td id="td-135">
            41.91%
           </td>
<td id="td-136">
            3.06%
           </td>
<td id="td-137">
            5.71%
           </td>
</tr>
<tr id="tr-28">
<td id="td-138">
            Positive predictions
           </td>
<td id="td-139">
            SLK
           </td>
<td id="td-140">
            1
           </td>
<td id="td-141">
            8.4
           </td>
<td id="td-142">
            1286
           </td>
<td id="td-143">
            54.70%
           </td>
<td id="td-144">
            3.20%
           </td>
<td id="td-145">
            6.05%
           </td>
</tr>
<tr id="tr-29">
<td id="td-146">
            Positive predictions
           </td>
<td id="td-147">
            SLK
           </td>
<td id="td-148">
            2
           </td>
<td id="td-149">
            8.4
           </td>
<td id="td-150">
            454
           </td>
<td id="td-151">
            65.90%
           </td>
<td id="td-152">
            1.40%
           </td>
<td id="td-153">
            2.74%
           </td>
</tr>
<tr id="tr-30">
<td id="td-154">
            Positive predictions
           </td>
<td id="td-155">
            SLK
           </td>
<td id="td-156">
            12
           </td>
<td id="td-157">
            10.2
           </td>
<td id="td-158">
            9
           </td>
<td id="td-159">
            100.00%
           </td>
<td id="td-160">
            0.04%
           </td>
<td id="td-161">
            0.08%
           </td>
</tr>
<tr id="tr-31">
<td id="td-162">
            All pairings
           </td>
<td id="td-163">
            Co-occurrence
           </td>
<td id="td-164">
            1
           </td>
<td id="td-165">
            8.3
           </td>
<td id="td-166">
            6474
           </td>
<td id="td-167">
            34.00%
           </td>
<td id="td-168">
            10.01%
           </td>
<td id="td-169">
            15.47%
           </td>
</tr>
<tr id="tr-32">
<td id="td-170">
            All pairings
           </td>
<td id="td-171">
            Co-occurrence
           </td>
<td id="td-172">
            2
           </td>
<td id="td-173">
            8.3
           </td>
<td id="td-174">
            2865
           </td>
<td id="td-175">
            44.96%
           </td>
<td id="td-176">
            5.86%
           </td>
<td id="td-177">
            10.37%
           </td>
</tr>
<tr id="tr-33">
<td id="td-178">
            All pairings
           </td>
<td id="td-179">
            Co-occurrence
           </td>
<td id="td-180">
            8
           </td>
<td id="td-181">
            8.2
           </td>
<td id="td-182">
            515
           </td>
<td id="td-183">
            66.41%
           </td>
<td id="td-184">
            1.56%
           </td>
<td id="td-185">
            3.04%
           </td>
</tr>
<tr id="tr-34">
<td id="td-186">
            All pairings
           </td>
<td id="td-187">
            Co-occurrence
           </td>
<td id="td-188">
            16
           </td>
<td id="td-189">
            8.4
           </td>
<td id="td-190">
            189
           </td>
<td id="td-191">
            71.43%
           </td>
<td id="td-192">
            0.61%
           </td>
<td id="td-193">
            1.22%
           </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li class="fn" id="fn-4">
<p id="p-43">
            This table presents the analysis of the extracted binary connectivity matrices. The first two rows are from connectivity matrices
                           derived from the 1377 annotated abstract set. The remaining rows are from the 12 557 abstract set and are split between the
                           SLK predictions and the co-occurrence technique. The threshold column displays the required count of reported connections
                           to be marked as connected in the matrix. Anatomical depth measures how specific the connections are by averaging the number
                           of enclosing brain regions for each connected region.
           </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
         Table 3.
        </span>
<p id="p-42" class="first-child">
         Aggregate connectivity results from several methods and relation sets
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
</div>
<div class="section discussion" id="sec-10">
<div class="section-nav">
<a href="#sec-9" title="3 RESULTS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#ack-1" title="Acknowledgements" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       4 DISCUSSION
      </h2>
<p id="p-44">
       We reported a complete system for extracting connectivity statements from biomedical abstracts. The method provides high recall
                     of manually annotated connectivity relations described in single sentences. Precision from two separate evaluations reached
                     50.3% and 55.3%. By comparing with an independent source of rat connectivity, we found that precision increases with the recency
                     and frequency of the extracted relationships.
      </p>
<p id="p-45">
       A limitation of our work is that we did not consider the direction of connectivity, although most of the relationships we
                     extracted have a direction described in the sentence. In addition, we did not consider negation (region A does not project
                     to region B). Extracting this information by extracting keywords such as ‘afferent’, ‘not’ or ‘input’ will require additional
                     work. These relationship modifiers are manually annotated in our training corpus and can be used to design more complex rules.
      </p>
<p id="p-46">
       Our methods also did not attempt to extract the large number of relations that span multiple sentences. When these connections
                     are taken into account, the SLK method provides only 51.7% recall of annotated connections. Application of advanced natural
                     language-processing techniques may be necessary to bridge the sentences (e.g. anaphora resolution).
      </p>
<p id="p-47">
       The comparison of seven previously published kernel-based approaches mirrored the previous results from the protein interaction
                     relationship extraction domain (
       <a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). Further, we note that the SLK parser was applied to the drug–drug interaction domain with similar results (
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
        Segura-Bedmar
        <em>
         et al.
        </em>
        , 2011
       </a>
       ). Several of the kernel methods have lower performance than our simple rule-based technique. Effort spent crafting more complex
                     rules may yield higher precision at the cost of lower recall. The top three kernel methods (SLK, all-paths graph, k-band shortest
                     path spectrum kernel) all have similar accuracy (AUC and f-measure scores) but vary in precision and recall. This difference
                     suggests higher performance may be achieved by combining the methods.
      </p>
<p id="p-48">
       Our results suggest a larger set of input abstracts will yield a larger number of precise connections. The largest extension
                     set is Medline with more than 10 million abstracts and 120 million sentences. Tikk and colleagues calculated that the SLK
                     parser could process all of Medline in 141 days (
       <a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">
        Tikk
        <em>
         et al.
        </em>
        , 2010
       </a>
       ). A two-step process may reduce runtime and increase accuracy by first identifying abstracts with connectivity statements
                     and then by extracting the specific connections with SLK (
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
        Poulter
        <em>
         et al.
        </em>
        , 2008
       </a>
       ).
      </p>
<p id="p-49">
       In natural language processing, it has been observed that simple statistical models (e.g. co-occurrence) can outperform more
                     complex models based on less data (
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
        Halevy
        <em>
         et al.
        </em>
        , 2009
       </a>
       ). Our experiments confirm this. We found that brain region pairs with many co-mentions tend to be connected. This simple
                     technique produces a larger set of potential connections with reasonable precision. Although this will produce a larger set
                     of results than the SLK method, it does not target connections that can be directly curated in light of experimental evidence
                     because the co-mentions may or may not describe connectivity. Further, such co-occurrences may result from region proximity
                     or biases (i.e. popularity) that may influence research attention both in the literature and in BAMS. However, such co-occurrence
                     networks show valuable areas of focus when combined with comentions of genes and diseases (
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
        Hayasaka
        <em>
         et al.
        </em>
        , 2011
       </a>
       ;
       <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
        Voytek and Voytek, 2012
       </a>
       ).
      </p>
<p id="p-50">
       One of the most serious challenges we encountered was in mapping extracted brain region mentions to standardized lexicons.
                     In our previous work, we reported resolution rates of 63%, with the major limitation being gaps in the lexicons (
       <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">
        French and Pavlidis, 2011
       </a>
       ). For the current work, the resolution rate is greatly reduced, as both brain region mentions of a connectivity relation
                     must be mapped. It also appears that regions forming connectivity relations are harder to resolve or map, on average. For
                     this work, we managed to double the resolution rate to the BAMS lexicon by adding synonyms. Additional work to improve the
                     lexicons will lead to better resolution of connectivity statements, allowing validation and linking to other resources.
      </p>
<p id="p-51">
       For our evaluation to an outside database, we focused on BAMS (
       <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
        Bota
        <em>
         et al.
        </em>
        , 2005
       </a>
       ). Although rat is the most frequent mentioned organism, other evaluations could compare the connectivity results with the
                     CoCoMac (
       <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
        Kotter, 2004
       </a>
       ) or the Avian Brain Circuitry Database (
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
        Schrott and Kabai, 2008
       </a>
       ). Beyond evaluation, our dataset and method can provide a large set of extracted connectivity relationships for other species-specific
                     databases.
      </p>
<p id="p-52">
       In conclusion, we provide the first application of large-scale text mining to neuroanatomical connectivity extraction. We
                     demonstrated that machine-learning tools designed for extraction of protein–protein interactions are generalizable to mining
                     brain region connections. From an information retrieval perspective, our large set of predicted connections can aid neuroscientists
                     in forming hypotheses and models. Future work will be aimed at further evaluating and disseminating the results before extending
                     the analysis.
      </p>
</div>
<div class="section ack" id="ack-1">
<div class="section-nav">
<a href="#sec-10" title="4 DISCUSSION" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#fn-group-1" title="Footnotes" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Acknowledgements
      </h2>
<p id="p-53">
       The authors thank Olivia Marais and Tianna Koreman for additional curation.
      </p>
<p id="p-54">
<em>
        Funding
       </em>
       : This work was supported in part by a Discovery Grant from the Natural Sciences and Engineering Research Council of Canada
                     (371348). Further support was provided by a National Institutes of Health grant GM076990 to Paul Pavlidis, the Canadian Foundation
                     for Innovation (Leaders Opportunities Fund), the Michael Smith Foundation for Health Research (Career Investigator award to
                     Paul Pavlidis) and the Canadian Institutes of Health Research (New Investigator Salary Award to Paul Pavlidis). The funders
                     had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.
      </p>
<p id="p-55">
<em>
        Conflict of Interest
       </em>
       : none declared.
      </p>
</div>
<div class="section fn-group" id="fn-group-1">
<div class="section-nav">
<a href="#ack-1" title="Acknowledgements" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#ref-list-1" title="REFERENCES" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Footnotes
      </h2>
<ul>
<li class="fn" id="fn-1">
<p id="p-1">
         Associate Editor: Jonathan Wren
        </p>
</li>
</ul>
</div>
<ul class="copyright-statement">
<li class="fn" id="copyright-statement-1">
       © The Author(s) 2012. Published by Oxford University Press.
      </li>
</ul>
<div class="license" id="license-1">
<p id="p-2">
       This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0),
                     which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
      </p>
</div>
<div class="section ref-list" id="ref-list-1">
<div class="section-nav">
<a href="#fn-group-1" title="Footnotes" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<div class="nav-placeholder">
        �
       </div>
</div>
<h2>
       REFERENCES
      </h2>
<ol class="cit-list ref-use-labels">
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.1">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Airola
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            All-paths graph kernel for protein-protein interaction extraction with evaluation of cross-corpus learning
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
<span class="cit-supplement">
            Suppl. 11
           </span>
           :
           <span class="cit-fpage">
            S2
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=All-paths%20graph%20kernel%20for%20protein-protein%20interaction%20extraction%20with%20evaluation%20of%20cross-corpus%20learning&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Airola&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.2">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Blaschke
             </span>
<span class="cit-name-given-names">
              C
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automatic extraction of biological information from scientific text: protein-protein interactions
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Proc. Int. Conf. Intell. Syst. Mol. Biol.
           </abbr>
<span class="cit-pub-date">
            1999
           </span>
           :
           <span class="cit-fpage">
            60
           </span>
           -
           <span class="cit-lpage">
            67
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Automatic%20extraction%20of%20biological%20information%20from%20scientific%20text%3A%20protein-protein%20interactions&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Blaschke&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.3">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Bota
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            From gene networks to brain networks
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Neurosci.
           </abbr>
<span class="cit-pub-date">
            2003
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            795
           </span>
           -
           <span class="cit-lpage">
            799
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nn1096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=12886225&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000184413700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.4">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Bota
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Brain architecture management system
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Neuroinformatics
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            15
           </span>
           -
           <span class="cit-lpage">
            48
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1385/NI:3:1:015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15897615&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000230322400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.5">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Broadwell
             </span>
<span class="cit-name-given-names">
              RD
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jacobowitz
             </span>
<span class="cit-name-given-names">
              DM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Olfactory relationships of the telencephalon and diencephalon in the rabbit. III. The ipsilateral centrifugal fibers to the
                                    olfactory bulbar and retrobulbar formations
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1976
           </span>
           ;
           <span class="cit-vol">
            170
           </span>
           :
           <span class="cit-fpage">
            321
           </span>
           -
           <span class="cit-lpage">
            345
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.901700305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=62770&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1976CM96300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2963.6">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Burns
             </span>
<span class="cit-name-given-names">
              G
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Computational Intelligence in Biomedicine
           </span>
           .
           <span class="cit-publ-loc">
            Berlin
           </span>
           :
           <span class="cit-publ-name">
            Springer-Verlag and Heidelberg
           </span>
           ;
           <span class="cit-pub-date">
            2007
           </span>
           .
           <span class="cit-article-title">
            Intelligent approaches to mining the primary research literature: techniques, systems, and examples
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Intelligent%20approaches%20to%20mining%20the%20primary%20research%20literature%3A%20techniques%2C%20systems%2C%20and%20examples&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Burns&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.7">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Chen
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Origins of endomorphin-immunoreactive fibers and terminals in different columns of the periaqueductal gray in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            509
           </span>
           :
           <span class="cit-fpage">
            72
           </span>
           -
           <span class="cit-lpage">
            87
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.21728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18421704&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2963.8">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Collins
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Duffy
             </span>
<span class="cit-name-given-names">
              N
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-conf-name">
            Proceedings of Neural Information Processing Systems (NIPS'01)
           </span>
           ;
           <span class="cit-conf-loc">
            Vancouver, BC, Canada
           </span>
           .
           <span class="cit-pub-date">
            2001
           </span>
           .
           <span class="cit-article-title">
            Convolution kernels for natural language
           </span>
           ; p.
           <span class="cit-fpage">
            625
           </span>
           -
           <span class="cit-lpage">
            632
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Convolution%20kernels%20for%20natural%20language&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Collins&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.9">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Crick
             </span>
<span class="cit-name-given-names">
              F
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jones
             </span>
<span class="cit-name-given-names">
              E
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Backwardness of human neuroanatomy
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nature
           </abbr>
<span class="cit-pub-date">
            1993
           </span>
           ;
           <span class="cit-vol">
            361
           </span>
           :
           <span class="cit-fpage">
            109
           </span>
           -
           <span class="cit-lpage">
            110
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/361109a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=8421513&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.10">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Cunningham
             </span>
<span class="cit-name-given-names">
              ET
             </span>
<span class="cit-name-suffix">
              Jr
             </span>
             .
            </span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sawchenko
             </span>
<span class="cit-name-given-names">
              PE
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Dorsal medullary pathways subserving oromotor reflexes in the rat: implications for the central neural control of swallowing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2000
           </span>
           ;
           <span class="cit-vol">
            417
           </span>
           :
           <span class="cit-fpage">
            448
           </span>
           -
           <span class="cit-lpage">
            466
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(20000221)417:4%3C448::AID-CNE5%3E3.0.CO;2-S&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=10701866&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000085158700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">
         ↵
        </a>
<div class="cit ref-cit ref-confproc" id="cit-28.22.2963.11">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Cunningham
             </span>
<span class="cit-name-given-names">
              H
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A framework and graphical development environment for robust NLP tools and applications
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of the 40th Annual Meeting of the Association for Computational Linguistics (ACL)
           </span>
           .
           <span class="cit-pub-date">
            2002
           </span>
           . p.
           <span class="cit-fpage">
            168
           </span>
           -
           <span class="cit-lpage">
            175
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=A%20framework%20and%20graphical%20development%20environment%20for%20robust%20NLP%20tools%20and%20applications&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Cunningham&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2963.12">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              De Marneffe
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-conf-name">
            Proceedings of LRECVol. 6
           </span>
           .
           <span class="cit-publ-loc">
            Paris
           </span>
           :
           <span class="cit-publ-name">
            European Language Resource Association
           </span>
           ;
           <span class="cit-pub-date">
            2006
           </span>
           .
           <span class="cit-article-title">
            Generating typed dependency parses from phrase structure parses
           </span>
           ; p.
           <span class="cit-fpage">
            449
           </span>
           -
           <span class="cit-lpage">
            454
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Generating%20typed%20dependency%20parses%20from%20phrase%20structure%20parses&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=De%20Marneffe&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.13">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Felleman
             </span>
<span class="cit-name-given-names">
              DJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Van Essen
             </span>
<span class="cit-name-given-names">
              DC
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Distributed hierarchical processing in the primate cerebral cortex
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Cereb. Cortex.
           </abbr>
<span class="cit-pub-date">
            1991
           </span>
           ;
           <span class="cit-vol">
            1
           </span>
           :
           <span class="cit-fpage">
            1
           </span>
           -
           <span class="cit-lpage">
            47
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=cercor&amp;resid=1/1/1-a" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.14">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              French
             </span>
<span class="cit-name-given-names">
              L
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automated recognition of brain region mentions in neuroscience literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Front. Neuroinform.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            29
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=19750194&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.15">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              French
             </span>
<span class="cit-name-given-names">
              L
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pavlidis
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Using text mining to link journal articles to neuroanatomical databases
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            520
           </span>
           :
           <span class="cit-fpage">
            1772
           </span>
           -
           <span class="cit-lpage">
            1783
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Using%20text%20mining%20to%20link%20journal%20articles%20to%20neuroanatomical%20databases&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=French&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.16">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Gerner
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            LINNAEUS: a species name identification system for biomedical literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            11
           </span>
           :
           <span class="cit-fpage">
            85
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-11-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20149233&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">
         ↵
        </a>
<div class="cit ref-cit ref-confproc" id="cit-28.22.2963.17">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Giuliano
             </span>
<span class="cit-name-given-names">
              C
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Exploiting shallow linguistic information for relation extraction from biomedical literature
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of the 11st Conference of the European Chapter of the Association for Computational Linguistics (EACL’06). The
                                    Association for Computer Linguistics, Trento, Italy
           </span>
           .
           <span class="cit-pub-date">
            2006
           </span>
           . p.
           <span class="cit-fpage">
            401
           </span>
           -
           <span class="cit-lpage">
            408
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Exploiting%20shallow%20linguistic%20information%20for%20relation%20extraction%20from%20biomedical%20literature&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Giuliano&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2963.18">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Haines
             </span>
<span class="cit-name-given-names">
              DE
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Neuroanatomy: an Atlas of Structures, Sections, and Systems
           </span>
           .
           <span class="cit-publ-loc">
            Baltimore and Philadelphia
           </span>
           :
           <span class="cit-publ-name">
            Lippincott Williams & Wilkins
           </span>
           ;
           <span class="cit-pub-date">
            2004
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Neuroanatomy%3A%20an%20Atlas%20of%20Structures%2C%20Sections%2C%20and%20Systems&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Haines&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.19">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Halevy
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The unreasonable effectiveness of data
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            IEEE Intell. Syst.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            24
           </span>
           :
           <span class="cit-fpage">
            8
           </span>
           -
           <span class="cit-lpage">
            12
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=The%20unreasonable%20effectiveness%20of%20data&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Halevy&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.20">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Hayasaka
             </span>
<span class="cit-name-given-names">
              S
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A network of genes, genetic disorders, and brain areas
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PloS One
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            e20907
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pone.0020907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21695164&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.21">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Houenou
             </span>
<span class="cit-name-given-names">
              J
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Increased white matter connectivity in euthymic bipolar patients: diffusion tensor tractography between the subgenual cingulate
                                    and the amygdalo-hippocampal complex
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Mol. Psychiatry
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            12
           </span>
           :
           <span class="cit-fpage">
            1001
           </span>
           -
           <span class="cit-lpage">
            1010
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/sj.mp.4002010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17471288&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000250412700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.22">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jensen
             </span>
<span class="cit-name-given-names">
              LJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Literature mining for the biologist: from information retrieval to biological discovery
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Rev. Genet.
           </abbr>
<span class="cit-pub-date">
            2006
           </span>
           ;
           <span class="cit-vol">
            7
           </span>
           :
           <span class="cit-fpage">
            119
           </span>
           -
           <span class="cit-lpage">
            129
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nrg1768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16418747&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000234714000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.23">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Jolkkonen
             </span>
<span class="cit-name-given-names">
              E
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pitkanen
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Intrinsic connections of the rat amygdaloid complex: projections originating in the central nucleus
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1998
           </span>
           ;
           <span class="cit-vol">
            395
           </span>
           :
           <span class="cit-fpage">
            53
           </span>
           -
           <span class="cit-lpage">
            72
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(19980525)395:1%3C53::AID-CNE5%3E3.0.CO;2-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=9590546&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000073290400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.24">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kabiljo
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A realistic assessment of methods for extracting gene/protein interactions from free text
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            10
           </span>
           :
           <span class="cit-fpage">
            233
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-10-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=19635172&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.25">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Karlsgodt
             </span>
<span class="cit-name-given-names">
              KH
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Diffusion tensor imaging of the superior longitudinal fasciculus and working memory in recent-onset schizophrenia
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Biol. Psychiatry
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            63
           </span>
           :
           <span class="cit-fpage">
            512
           </span>
           -
           <span class="cit-lpage">
            518
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=17720147&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.26">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Koshino
             </span>
<span class="cit-name-given-names">
              H
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Functional connectivity in an fMRI working memory task in high-functioning autism
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            NeuroImage
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            24
           </span>
           :
           <span class="cit-fpage">
            810
           </span>
           -
           <span class="cit-lpage">
            821
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.neuroimage.2004.09.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15652316&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000226588500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.27">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kotter
             </span>
<span class="cit-name-given-names">
              R
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Online retrieval, processing, and visualization of primate connectivity data from the CoCoMac database
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Neuroinformatics
           </abbr>
<span class="cit-pub-date">
            2004
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            127
           </span>
           -
           <span class="cit-lpage">
            144
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1385/NI:2:2:127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=15319511&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000223582200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.28">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Krallinger
             </span>
<span class="cit-name-given-names">
              M
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Overview of the protein-protein interaction annotation extraction task of BioCreative II
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genome Biol.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
<span class="cit-supplement">
            Suppl. 2
           </span>
           :
           <span class="cit-fpage">
            S4
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Overview%20of%20the%20protein-protein%20interaction%20annotation%20extraction%20task%20of%20BioCreative%20II&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Krallinger&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.29">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kuboyama
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A spectrum tree kernel
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Inf. Media Technol.
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            292
           </span>
           -
           <span class="cit-lpage">
            299
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=A%20spectrum%20tree%20kernel&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Kuboyama&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.30">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Lanciego
             </span>
<span class="cit-name-given-names">
              JL
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wouterlood
             </span>
<span class="cit-name-given-names">
              FG
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A half century of experimental neuroanatomical tracing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Chem. Neuroanat.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            42
           </span>
           :
           <span class="cit-fpage">
            157
           </span>
           -
           <span class="cit-lpage">
            183
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jchemneu.2011.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21782932&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">
         ↵
        </a>
<div class="cit ref-cit ref-confproc" id="cit-28.22.2963.31">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              McClosky
             </span>
<span class="cit-name-given-names">
              D
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Reranking and self-training for parser adaptation
           </span>
           .
           <span class="cit-conf-name">
            ACL-44 Proceedings of the 21st International Conference on Computational Linguistics and the 44th Annual Meeting of the Association
                                    for Computational Linguistics. Association for Computational Linguistics,
           </span>
           .
           <span class="cit-pub-date">
            2006
           </span>
           . p.
           <span class="cit-fpage">
            337
           </span>
           -
           <span class="cit-lpage">
            344
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Reranking%20and%20self-training%20for%20parser%20adaptation&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=McClosky&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.32">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Moore
             </span>
<span class="cit-name-given-names">
              RY
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Serotonin neurons of the midbrain raphe: ascending projections
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1978
           </span>
           ;
           <span class="cit-vol">
            180
           </span>
           :
           <span class="cit-fpage">
            417
           </span>
           -
           <span class="cit-lpage">
            438
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.901800302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=77865&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1978FD16900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">
         ↵
        </a>
<div class="cit ref-cit ref-confproc" id="cit-28.22.2963.33">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Moschitti
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Efficient convolution kernels for dependency and constituent syntactic trees
           </span>
           .
           <span class="cit-conf-name">
            Proceedings of The 17th European Conference on Machine Learning
           </span>
           .
           <span class="cit-publ-loc">
            Berlin
           </span>
           :
           <span class="cit-publ-name">
            Springer and Heidelberg
           </span>
           ;
           <span class="cit-pub-date">
            2005
           </span>
           . p.
           <span class="cit-fpage">
            318
           </span>
           -
           <span class="cit-lpage">
            329
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Efficient%20convolution%20kernels%20for%20dependency%20and%20constituent%20syntactic%20trees&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Moschitti&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.34">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Nauta
             </span>
<span class="cit-name-given-names">
              WJ
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Selective silver impregnation of degenerating axons in the central nervous system
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Stain Technol.
           </abbr>
<span class="cit-pub-date">
            1952
           </span>
           ;
           <span class="cit-vol">
            27
           </span>
           :
           <span class="cit-fpage">
            175
           </span>
           -
           <span class="cit-lpage">
            179
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=14958510&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1952UN01600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.35">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Pinganaud
             </span>
<span class="cit-name-given-names">
              G
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Trigeminal projections to hypoglossal and facial motor nuclei in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1999
           </span>
           ;
           <span class="cit-vol">
            415
           </span>
           :
           <span class="cit-fpage">
            91
           </span>
           -
           <span class="cit-lpage">
            104
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/(SICI)1096-9861(19991206)415:1%3C91::AID-CNE7%3E3.0.CO;2-A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=10540360&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000083518400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.36">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Poulin
             </span>
<span class="cit-name-given-names">
              JF
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Enkephalinergic afferents of the centromedial amygdala in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            2006
           </span>
           ;
           <span class="cit-vol">
            496
           </span>
           :
           <span class="cit-fpage">
            859
           </span>
           -
           <span class="cit-lpage">
            876
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.20956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16628615&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000237440000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.37">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Poulter
             </span>
<span class="cit-name-given-names">
              GL
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            MScanner: a classifier for retrieving Medline citations
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            9
           </span>
           :
           <span class="cit-fpage">
            108
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1186/1471-2105-9-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18284683&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.38">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Schrott
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Kabai
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            ABCD: a functional database for the avian brain
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Neurosci. Methods
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            167
           </span>
           :
           <span class="cit-fpage">
            393
           </span>
           -
           <span class="cit-lpage">
            395
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jneumeth.2007.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17889371&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000252938400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.39">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Segura-Bedmar
             </span>
<span class="cit-name-given-names">
              I
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Using a shallow linguistic kernel for drug-drug interaction extraction
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Biomed. Inform.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            44
           </span>
           :
           <span class="cit-fpage">
            789
           </span>
           -
           <span class="cit-lpage">
            804
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jbi.2011.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21545845&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.40">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sporns
             </span>
<span class="cit-name-given-names">
              O
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The human connectome: a complex network
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Ann. NY Acad. Sci.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            1224
           </span>
           :
           <span class="cit-fpage">
            109
           </span>
           -
           <span class="cit-lpage">
            125
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1111/j.1749-6632.2010.05888.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21251014&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000290238100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.41">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sporns
             </span>
<span class="cit-name-given-names">
              O
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The human connectome: a structural description of the human brain
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PLoS Comput. Biol.
           </abbr>
<span class="cit-pub-date">
            2005
           </span>
           ;
           <span class="cit-vol">
            1
           </span>
           :
           <span class="cit-fpage">
            e42
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pcbi.0010042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16201007&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.42">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sripanidkulchai
             </span>
<span class="cit-name-given-names">
              K
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wyss
             </span>
<span class="cit-name-given-names">
              JM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Thalamic projections to retrosplenial cortex in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1986
           </span>
           ;
           <span class="cit-vol">
            254
           </span>
           :
           <span class="cit-fpage">
            143
           </span>
           -
           <span class="cit-lpage">
            165
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.902540202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=3794004&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1986F117900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.43">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Stam
             </span>
<span class="cit-name-given-names">
              CJ
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Small-world networks and functional connectivity in Alzheimer's disease
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Cereb Cortex
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            17
           </span>
           :
           <span class="cit-fpage">
            92
           </span>
           -
           <span class="cit-lpage">
            99
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=cercor&amp;resid=17/1/92" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.44">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Stumpf
             </span>
<span class="cit-name-given-names">
              MP
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Estimating the size of the human interactome
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Proc. Natl Acad. Sci. USA
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            105
           </span>
           :
           <span class="cit-fpage">
            6959
           </span>
           -
           <span class="cit-lpage">
            6964
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=105/19/6959" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2963.45">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Swanson
             </span>
<span class="cit-name-given-names">
              LW
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-source">
            Brain maps: structure of the rat brain
           </span>
           .
           <span class="cit-publ-loc">
            Amsterdam
           </span>
           :
           <span class="cit-publ-name">
            Elsevier
           </span>
           ;
           <span class="cit-pub-date">
            1999
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Brain%20maps%3A%20structure%20of%20the%20rat%20brain&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Swanson&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.46">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Tikk
             </span>
<span class="cit-name-given-names">
              D
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A comprehensive benchmark of kernel methods to extract protein-protein interactions from literature
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            PLoS Comput. Biol.
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            e1000837
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1371/journal.pcbi.1000837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20617200&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<div class="cit ref-cit ref-journal no-rev-xref" id="cit-28.22.2963.47">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              van Groen
             </span>
<span class="cit-name-given-names">
              T
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Wyss
             </span>
<span class="cit-name-given-names">
              JM
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Connections of the retrosplenial dysgranular cortex in the rat
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Comp. Neurol.
           </abbr>
<span class="cit-pub-date">
            1992
           </span>
           ;
           <span class="cit-vol">
            315
           </span>
           :
           <span class="cit-fpage">
            200
           </span>
           -
           <span class="cit-lpage">
            216
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1002/cne.903150207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=1545009&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1992HB75200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">
         ↵
        </a>
<div class="cit ref-cit ref-confproc" id="cit-28.22.2963.48">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Vishwanathan
             </span>
<span class="cit-name-given-names">
              S
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Smola
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Fast kernels for string and tree matching
           </span>
           .
           <span class="cit-conf-name">
            In: Proceedings of Neural Information Processing Systems (NIPS’02)
           </span>
           .
           <span class="cit-publ-loc">
            Cambridge, MA, USA
           </span>
           :
           <span class="cit-publ-name">
            MIT Press
           </span>
           ;
           <span class="cit-pub-date">
            2002
           </span>
           . p.
           <span class="cit-fpage">
            569
           </span>
           -
           <span class="cit-lpage">
            576
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Fast%20kernels%20for%20string%20and%20tree%20matching&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Vishwanathan&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.49">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Voytek
             </span>
<span class="cit-name-given-names">
              JB
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Voytek
             </span>
<span class="cit-name-given-names">
              B
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Automated cognome construction and semi-automated hypothesis generation
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Neurosci. Methods
           </abbr>
<span class="cit-pub-date">
            2012
           </span>
           ;
           <span class="cit-vol">
            208
           </span>
           :
           <span class="cit-fpage">
            92
           </span>
           -
           <span class="cit-lpage">
            100
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1016/j.jneumeth.2012.04.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=22584238&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000305865600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2963.50">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              White
             </span>
<span class="cit-name-given-names">
              JG
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            The structure of the nervous system of the nematode
            <em>
             Caenorhabditis elegans
            </em>
</span>
           .
           <abbr class="cit-jnl-abbrev">
            Philos. Trans. R. Soc. Lond. B. Biol. Sci.
           </abbr>
<span class="cit-pub-date">
            1986
           </span>
           ;
           <span class="cit-vol">
            314
           </span>
           :
           <span class="cit-fpage">
            1
           </span>
           -
           <span class="cit-lpage">
            340
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=royptb&amp;resid=314/1165/1" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
</ol>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<span id="related-urls">
</span></div></div><div style="text-align:center" class="calibre_navbar calibre_rescale_70">
<hr />
<p style="text-align:left; max-width: 100%; overflow: hidden;">This article was downloaded by <strong>calibre</strong> from <a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2963?rss=1">http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2963?rss=1</a></p>
<br /><br /> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | </div></body>
</html>