<?xml version='1.0' encoding='utf-8'?>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-GB" id="nojs">
  <head>
    <title>
   BMC Genomics | Full text | 5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction
  </title>
    <link href="/images/icon/1008/favicon.ico" type="image/x-icon" rel="shortcut icon"/>
    <meta content="IE=7" http-equiv="X-UA-Compatible"/>
    <meta content="BMC Genomics" name="citation_journal_title"/>
    <meta content="BioMed Central Ltd" name="citation_publisher"/>
    <meta content="Joseph M Dhahbi" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Stephen R Spindler" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Hani Atamna" name="citation_author"/>
    <meta content="Department of Basic Sciences, Neuroscience, The Commonwealth Medical College, Scranton, Scranton, PA, 18510, USA" name="citation_author_institution"/>
    <meta content="Amy Yamakawa" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="Dario Boffelli" name="citation_author"/>
    <meta content="Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA" name="citation_author_institution"/>
    <meta content="Patricia Mote" name="citation_author"/>
    <meta content="Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA" name="citation_author_institution"/>
    <meta content="David IK Martin" name="citation_author"/>
    <meta content="Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA" name="citation_author_institution"/>
    <meta content="1471-2164" name="citation_issn"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="citation_title"/>
    <meta content="14" name="citation_volume"/>
    <meta content="1" name="citation_issue"/>
    <meta content="2013-05-02" name="citation_date"/>
    <meta content="298" name="citation_firstpage"/>
    <meta content="10.1186/1471-2164-14-298" name="citation_doi"/>
    <meta content="http://www.biomedcentral.com/content/pdf/1471-2164-14-298.pdf" name="citation_pdf_url"/>
    <link href="http://www.biomedcentral.com/content/pdf/1471-2164-14-298.pdf" type="application/pdf" rel="alternate" title="PDF"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="title"/>
    <meta content="Small RNAs complex with proteins to mediate a variety of functions in animals and plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may carry out a signaling function by entering target cells and modulating gene expression. The subject of this study is a set of circulating 30–33 nt RNAs that are processed derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit translation initiation in response to stress in cultured cells." name="description"/>
    <meta content="5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" name="dc.title"/>
    <meta content="Dhahbi, Joseph M" name="dc.creator"/>
    <meta content="Spindler, Stephen R" name="dc.creator"/>
    <meta content="Atamna, Hani" name="dc.creator"/>
    <meta content="Yamakawa, Amy" name="dc.creator"/>
    <meta content="Boffelli, Dario" name="dc.creator"/>
    <meta content="Mote, Patricia" name="dc.creator"/>
    <meta content="Martin, David IK" name="dc.creator"/>
    <meta content="Small RNAs complex with proteins to mediate a variety of functions in animals and plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may carry out a signaling function by entering target cells and modulating gene expression. The subject of this study is a set of circulating 30–33 nt RNAs that are processed derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit translation initiation in response to stress in cultured cells." name="dc.description"/>
    <meta content="BMC Genomics 2013 14:298" name="dc.source"/>
    <meta content="text/html" name="dc.format"/>
    <meta content="BioMed Central Ltd" name="dc.publisher"/>
    <meta content="2013-05-02" name="dc.date"/>
    <meta content="Research article" name="dc.type"/>
    <meta content="10.1186/1471-2164-14-298" name="dc.identifier"/>
    <meta content="en" name="dc.language"/>
    <meta content="2013 Dhahbi et al.; licensee BioMed Central Ltd." name="dc.copyright"/>
    <meta content="http://creativecommons.org/licenses/by/2.0/" name="dc.rights"/>
    <meta content="reprints@biomedcentral.com" name="dc.rightsAgent"/>
    <meta content="1471-2164" name="prism.issn"/>
    <meta content="BMC Genomics" name="prism.publicationName"/>
    <meta content="2013-05-02" name="prism.publicationDate"/>
    <meta content="14" name="prism.volume"/>
    <meta content="1" name="prism.number"/>
    <meta content="Research article" name="prism.section"/>
    <meta content="298" name="prism.startingPage"/>
    <meta content="2013 Dhahbi et al.; licensee BioMed Central Ltd." name="prism.copyright"/>
    <meta content="reprints@biomedcentral.com" name="prism.rightsAgent"/>
    <meta content="http://www.biomedcentral.com/1471-2164/14/298/abstract" name="citation_abstract_html_url"/>
    <meta content="http://www.biomedcentral.com/1471-2164/14/298" name="citation_fulltext_html_url"/>
    <link href="http://www.biomedcentral.com/content/download/xml/1471-2164-14-298.xml" type="text/xml" rel="alternate" title="XML version"/>
    <link href="http://www.biomedcentral.com/content/figures/1471-2164-14-298-toc.gif" rel="image_src"/>
    <link href="http://www.biomedcentral.com/content/download/supplementary/1471-2164-14-298-s1.pdf" rel="section" title="Additional file 1"/>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <link href="../../stylesheet.css" type="text/css" rel="stylesheet"/>
<link href="../../page_styles.css" type="text/css" rel="stylesheet"/>
</head>
  <body class="calibre"><div id="oas-campaign" class="calibre17">
   biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/14/298
  </div>
<div id="oas-positions" class="calibre17">
   Bottom,Top
  </div>
<link href="http://www.biomedcentral.com/bmcgenomics/latest/rss" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Latest articles" class="calibre17"/>
<link href="http://www.biomedcentral.com/bmcgenomics/mostviewed/rss/" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Most viewed" class="calibre17"/>
<link href="http://www.biomedcentral.com/bmcgenomics/latestcomments/rss" type="application/rss+xml" exposetoheader="true" rel="alternate" title="Latest comments" class="calibre17"/>
<style type="text/css" title="override_css" class="calibre17">
            .article_date {
                color: gray; font-family: monospace;
            }

            .article_description {
                text-indent: 0pt;
            }

            a.article {
                font-weight: bold; text-align:left;
            }

            a.feed {
                font-weight: bold;
            }

            .calibre_navbar {
                font-family:monospace;
            }
    

</style>

<div class="calibrenavbar1">| <a href="../../feed_1/index.html">Next</a> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | <a href="../article_0/index.html">Previous</a> | <hr class="calibre6"/>
</div>
<div role="banner" id="branding" class="calibre5">
<dl class="hide">
<dt class="calibre5">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising">
</a>
</dt>
<dd class="calibre18">
</dd>
</dl>
<div class="calibre5">
<div class="calibre5">
<ul class="calibrefeedlist" id="login">
<li class="calibre4">
<strong class="calibre7">
        Welcome
        <span id="username">
         University of Louisville
        </span>
</strong>
</li>
<li id="loginLink" class="calibre4">
<a href="/bmcgenomics/logon">
        Log on
       </a>
</li>
</ul>
<ul class="calibrefeedlist">
<li class="calibre4" id="BMC">
<a href="http://www.biomedcentral.com/">
<span>
          
        </span>
<span>
         BioMed Central
        </span>
</a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/journals">
<span>
         Journals
        </span>
</a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/gateways">
<span>
         Gateways
        </span>
</a>
</li>
</ul>
</div>
</div>
<div class="calibre5">
<div class="calibre5">
<a href="/bmcgenomics/">
<img src="/sites/1008/images/logo.gif" alt="BMC Genomics" title="BMC Genomics Logo" class="calibre2"/>
</a>
</div>
<div class="calibre5" id="impact-factor">
<img src="/images/branding/official.gif" alt="official impact factor" title="4.07" class="calibre2"/>
<span id="impact-factor-value">
      4.07
     </span>
</div>
<div role="search" class="calibre5">
<div class="calibre5">
<form action=" /quickSearchRedirectURL " onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;" class="calibre5" method="get">
<fieldset>
<span>
         Search
         <select name="url">
<option selected="selected" value="http://www.biomedcentral.com/bmcgenomics/search/results">
           BMC Genomics
          </option>
<option value="http://www.biomedcentral.com/search/results">
           BioMed Central
          </option>
</select>
         for
        </span>
<input alt="Search terms" type="text" id="searchTerms" name="terms"/>
<button type="submit" value="Go">
         Go
        </button>
</fieldset>

</form></div>
<a href="/bmcgenomics/search">
      Advanced search
     </a>
</div>
<div id="mobile-nav-list" class="calibre5">
<ul role="navigation" class="calibrefeedlist">
<li class="calibre4">
<a href="/bmcgenomics/">
<span id="home-tab">
         Home
        </span>
</a>
</li>
<li class="calibre4">
<a href="/bmcgenomics/content">
<span id="articles-tab">
         Articles
        </span>
</a>
</li>
<li class="calibre4">
<a href="/bmcgenomics/authors/instructions">
<span id="authors-tab">
         Authors
        </span>
</a>
</li>
<li class="calibre4">
<a href="/bmcgenomics/about/reviewers">
<span id="reviewers-tab">
         Reviewers
        </span>
</a>
</li>
<li class="calibre4">
<a href="/bmcgenomics/about">
<span id="about-tab">
         About this journal
        </span>
</a>
</li>
<li class="calibre4">
<a href="/bmcgenomics/my">
<span id="my-tab">
         My BMC Genomics
        </span>
</a>
</li>
</ul>
</div>
</div>
</div>
<hr class="calibre6"/>
<div class="calibre5">
<div role="main" class="calibre5" id="content">
<div id="left-article-box" class="calibre5">
<ul class="calibrefeedlist" id="box-outline">
<ul class="subject-headings" id="outline">
<li class="calibre4">
<a href="#">
         Top
        </a>
</li>
<li class="calibre4">
<a href="#abs">
         Abstract
        </a>
</li>
<li class="calibre4">
<a href="#sec1">
         Background
        </a>
</li>
<li class="calibre4">
<a href="#sec2">
         Results and discussion
        </a>
</li>
<li class="calibre4">
<a href="#sec3">
         Conclusions
        </a>
</li>
<li class="calibre4">
<a href="#sec4">
         Methods
        </a>
</li>
<li class="calibre4">
<a href="#sec5">
         Competing interests
        </a>
</li>
<li class="calibre4">
<a href="#sec6">
         Authors’ contributions
        </a>
</li>
<li class="calibre4">
<a href="#ack">
         Acknowledgements
        </a>
</li>
<li class="calibre4">
<a href="#refs">
         References
        </a>
</li>
</ul>
</ul>
<dl id="box-outline1" class="hide">
</dl>
</div>
<div id="right-panel" class="calibre5">
<div id="mobile-sidebar" class="calibre5">
<div id="mobile-sidebar-tab" class="calibre5">
<span>
<a href="#" id="mobile-sidebar-tab-link">
<img src="/images/sidebar-hidden.png" class="calibre2"/>
<img src="/images/sidebar-shown.png" class="calibre19"/>
</a>
</span>
</div>
<div id="mobile-sidebar-list" class="calibre5">
<div id="article-navigation-bar" class="calibre5">
<div class="calibre5">
<h5 class="calibre20">
<a href="http://www.biomedcentral.com/bmcgenomics">
<strong class="calibre7">
            BMC Genomics
           </strong>
</a>
</h5>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="http://www.biomedcentral.com/bmcgenomics/content/14">
            Volume 14
           </a>
</li>
</ul>
</div>
<div id="article-info" class="calibre5">
<div id="viewing-options-links" class="calibre5">
<h5 class="calibre20">
           Viewing options
          </h5>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="/1471-2164/14/298/abstract">
             Abstract
            </a>
</li>
<li class="calibre4">
<strong class="calibre7">
             Full text
            </strong>
</li>
<li class="calibre4">
<a href="/content/pdf/1471-2164-14-298.pdf" onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2164/14/298', 1, true]);">
             PDF
            </a>
<span>
             (2.3MB)
            </span>
</li>
<li class="calibre4">
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/additional">
             Additional files
            </a>
</li>
</ul>
</div>
<div id="associated-material-links" class="calibre5">
<h5 class="calibre20">
           Associated material
          </h5>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="/pubmed/23638709">
             PubMed record
            </a>
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/about">
             About this article
            </a>
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/comments">
             Readers' comments
            </a>
</li>
<li class="calibre4">
</li>
</ul>
</div>
<div id="related-literature-links" class="calibre5">
<h5 class="calibre20">
           Related literature
          </h5>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="http://www.biomedcentral.com/1471-2164/14/298/about#citations">
<h6 class="calibre21">
              Cited by
             </h6>
</a>
</li>
<li id="google-blog-search" class="calibre4">
<a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=5%C2%BF+tRNA+halves+are+present+as+abundant+complexes+in+serum%2C+concentrated+in+blood+cells%2C+and+modulated+by+aging+and+calorie+restriction&amp;btnG=Search">
             Google blog search
            </a>
</li>
<h6 class="calibre21">
            Other articles by authors
           </h6>
<li class="calibre4">
<a>
<i class="calibre13">
               
             </i>
             on Google Scholar
            </a>
<ul class="subject-headings" id="authg">
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22JM+Dhahbi%22">
               Dhahbi JM
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22SR+Spindler%22">
               Spindler SR
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22H+Atamna%22">
               Atamna H
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22A+Yamakawa%22">
               Yamakawa A
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22D+Boffelli%22">
               Boffelli D
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22P+Mote%22">
               Mote P
              </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=author%3A%22DIK+Martin%22">
               Martin DIK
              </a>
</li>
</ul>
</li>
<li class="calibre4">
<a>
<i class="calibre13">
               
             </i>
             on PubMed
            </a>
<ul class="subject-headings" id="authpm">
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Dhahbi_JM%20[Author]">
               Dhahbi JM
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Spindler_SR%20[Author]">
               Spindler SR
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Atamna_H%20[Author]">
               Atamna H
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Yamakawa_A%20[Author]">
               Yamakawa A
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Boffelli_D%20[Author]">
               Boffelli D
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Mote_P%20[Author]">
               Mote P
              </a>
</li>
<li class="calibre4">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Martin_DIK%20[Author]">
               Martin DIK
              </a>
</li>
</ul>
</li>
</ul>
<h6 class="calibre21">
           Related articles/pages
          </h6>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2164/14/298">
             on Google
            </a>
</li>
<li class="calibre4">
<a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2164/14/298">
             on Google Scholar
            </a>
</li>
<li class="calibre4">
<a href="/pubmed/related/23638709">
             on PubMed
            </a>
</li>
</ul>
</div>
<h5 class="calibre20">
          Tools
         </h5>
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="/1471-2164/14/298/citation">
            Download references
           </a>
</li>
<li class="calibre4">
<a href="/content/download/xml/1471-2164-14-298.xml">
            Download XML
           </a>
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/email">
            Email to a friend
           </a>
</li>
<li class="calibre4">
<a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1471-2164-14-298&amp;volume=14&amp;issue=1&amp;title=5%26%238242%3B+tRNA+halves+are+present+as+abundant+complexes+in+serum%2C+concentrated+in+blood+cells%2C+and+modulated+by+aging+and+calorie+restriction&amp;author_name=Joseph M Dhahbi&amp;start_page=1&amp;end_page=14">
            Order reprints
           </a>
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/postcomment">
            Post a comment
           </a>
</li>
</ul>
<h5 class="calibre20">
          Share this article
         </h5>
<ul id="social-networking-links" class="calibrefeedlist">
<li class="calibre4">
<div data-width="100" data-show-faces="false" data-action="recommend" data-send="false" data-layout="button_count" class="calibre5" data-href="http://www.biomedcentral.com/1471-2164/14/298">
</div>
</li>
<li class="calibre4">
<a data-url="http://www.biomedcentral.com/1471-2164/14/298" data-text="#bmcgenomics&#10;5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" href="http://twitter.com/share">
            Tweet
           </a>
</li>
<li class="calibre4">
<gU0003Aplusone width="100" size="medium">
</gU0003Aplusone>
</li>
<li class="calibre4">
<a href="#">
<span>
              
            </span>
            More options...
           </a>
<ul id="sharethis" class="subject-headings">
<li class="calibre4">
<a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">
<span>
</span>
              Citeulike
             </a>
</li>
<li class="calibre4">
<a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=%23bmcgenomics%0D%0A5%C2%BF%20tRNA%20halves%20are%20present%20as%20abundant%20complexes%20in%20serum%2C%20concentrated%20in%20blood%20cells%2C%20and%20modulated%20by%20aging%20and%20calorie%20restriction%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2164%2F14%2F298&amp;title=5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" onclick="largepopup(this.href,'linkedin',800,600);return false">
<span>
</span>
              LinkedIn
             </a>
</li>
<li class="calibre4">
<a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2164/14/298&amp;title=5¿ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">
<span>
</span>
              Del.icio.us
             </a>
</li>
<li class="calibre4">
<a href="/1471-2164/14/298/email">
<span>
</span>
              Email
             </a>
</li>
<li class="calibre4">
<a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'facebook',800,600);return false">
<span>
</span>
              Facebook
             </a>
</li>
<li class="calibre4">
<div class="calibre5">
<div data-annotation="none" class="calibre5">
</div>
</div>
<span>
</span>
<div class="calibre5">
<div data-annotation="none" class="calibre5">
</div>
</div>
<span>
</span>
<span>
<a href="#">
               Google+
              </a>
</span>
</li>
<li class="calibre4">
<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2164/14/298" onclick="largepopup(this.href,'mendeley',800,600);return false">
<span>
</span>
              Mendeley
             </a>
</li>
<li class="calibre4">
<a href="http://twitter.com/?status=%23bmcgenomics%0D%0A5%C2%BF%20tRNA%20halves%20are%20present%20as%20abundant%20complexes%20in%20serum%2C%20concentrated%20in%20blood%20cells%2C%20and%20modulated%20by%20aging%20and%20calorie%20restriction%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2164%2F14%2F298" onclick="largepopup(this.href,'twitter',800,600);return false">
<span>
</span>
              Twitter
             </a>
</li>
</ul>
</li>
</ul>
</div>
</div>
<div class="calibre5">
</div>
<div class="calibre5">
</div>
</div>
</div>
</div>
<div class="calibre5">
<div class="calibre5">
<div class="calibre5">
<div id="topmatter" class="calibre5">
<a href="/about/access">
<img src="/images/articles/openaccess-large.png" alt="Open Access" class="calibre2"/>
</a>
<span>
         Research article
        </span>
<h1 class="calibre8">
         5′ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction
        </h1>
<div class="calibre5">
<p class="hide">
<strong class="calibre7">
           Joseph M Dhahbi
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
<sup class="calibre10">
           *
          </sup>
          ,
          <strong class="calibre7">
           Stephen R Spindler
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          ,
          <strong class="calibre7">
           Hani  Atamna
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins2">
            2
           </a>
</sup>
          ,
          <strong class="calibre7">
           Amy  Yamakawa
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          ,
          <strong class="calibre7">
           Dario  Boffelli
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins3">
            3
           </a>
</sup>
          ,
          <strong class="calibre7">
           Patricia  Mote
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins1">
            1
           </a>
</sup>
          and
          <strong class="calibre7">
           David IK Martin
          </strong>
<sup class="calibre10">
<a href="/1471-2164/14/298/#ins3">
            3
           </a>
</sup>
</p>
<div id="affiliations" class="calibre5">
<div class="calibre5">
<div class="calibre5">
<ul class="calibrefeedlist">
<li class="calibre4">
<p class="hide">
<span>
                *
               </span>
               Corresponding author:										            Joseph M Dhahbi
               <a href="mailto:jdhahbi@ucr.edu">
                jdhahbi@ucr.edu
               </a>
</p>
</li>
</ul>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="#">
<i class="calibre13">
</i>
                Author Affiliations
               </a>
</p>
<div id="ins_container" class="calibre5">
<p class="hide">
<sup id="ins1" class="calibre10">
                 1
                </sup>
                Department of Biochemistry, University of California at Riverside, Riverside, CA, 92521, USA
               </p>
<p class="hide">
<sup id="ins2" class="calibre10">
                 2
                </sup>
                Department of Basic Sciences, Neuroscience, The Commonwealth Medical College, Scranton, Scranton, PA, 18510, USA
               </p>
<p class="hide">
<sup id="ins3" class="calibre10">
                 3
                </sup>
                Center for Genetics, Childrens Hospital Oakland Research Institute, Oakland, CA, 94609, USA
               </p>
</div>
<p id="authoremails" class="hide">
               For all author emails, please
               <a href="/bmcgenomics/logon">
                log on
               </a>
               .
              </p>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<em class="calibre13">
            BMC Genomics
           </em>
           2013,
           <strong class="calibre7">
            14
           </strong>
           :298 
           <span>
            doi:10.1186/1471-2164-14-298
           </span>
</p>
<br class="calibre5"/>
<p class="hide">
           The electronic version of this article is the complete one and can be found online at:
           <a href="http://www.biomedcentral.com/1471-2164/14/298">
            http://www.biomedcentral.com/1471-2164/14/298
           </a>
</p>
<br class="calibre5"/>
<table cellpadding="0" cellspacing="0" class="calibre22">
<tbody class="calibre23">
<tr class="calibre24">
<td class="calibre25">
              Received:
             </td>
<td class="calibre25">
              6 February 2013
             </td>
</tr>
<tr class="calibre24">
<td class="calibre25">
              Accepted:
             </td>
<td class="calibre25">
              26 April 2013
             </td>
</tr>
<tr class="calibre24">
<td class="calibre25">
              Published:
             </td>
<td class="calibre25">
              2 May 2013
             </td>
</tr>
</tbody>
</table>
<div class="calibre26">
<p class="hide">
            © 2013 Dhahbi et al.; licensee BioMed Central Ltd.
            <br class="calibre5"/>
</p>
<p class="hide">
            This is an Open Access article distributed under the terms of the Creative Commons Attribution License (
            <a href="http://creativecommons.org/licenses/by/2.0">
             http://creativecommons.org/licenses/by/2.0
            </a>
            ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
           </p>
</div>
</div>
</div>
</div>
<div id="article-body" class="calibre5">
<div class="calibre5">
<a name="abs">
</a>
<h3 class="calibre14">
          Abstract
         </h3>
<div class="calibre5">
<h4 class="calibre15">
           Background
          </h4>
<p class="calibre27">
           Small RNAs complex with proteins to mediate a variety of functions in animals and
         plants. Some small RNAs, particularly miRNAs, circulate in mammalian blood and may
         carry out a signaling function by entering target cells and modulating gene expression.
         The subject of this study is a set of circulating 30–33 nt RNAs that are processed
         derivatives of the 5′ ends of a small subset of tRNA genes, and closely resemble cellular
         tRNA derivatives (tRFs, tiRNAs, half-tRNAs, 5′ tRNA halves) previously shown to inhibit
         translation initiation in response to stress in cultured cells.
          </p>
<h4 class="calibre15">
           Results
          </h4>
<p class="calibre27">
           In sequencing small RNAs extracted from mouse serum, we identified abundant 5′ tRNA
         halves derived from a small subset of tRNAs, implying that they are produced by tRNA
         type-specific biogenesis and/or release. The 5′ tRNA halves are not in exosomes or
         microvesicles, but circulate as particles of 100–300 kDa. The size of these particles
         suggest that the 5′ tRNA halves are a component of a macromolecular complex; this
         is supported by the loss of 5′ tRNA halves from serum or plasma treated with EDTA,
         a chelating agent, but their retention in plasma anticoagulated with heparin or citrate.
         A survey of somatic tissues reveals that 5′ tRNA halves are concentrated within blood
         cells and hematopoietic tissues, but scant in other tissues, suggesting that they
         may be produced by blood cells. Serum levels of specific subtypes of 5′ tRNA halves
         change markedly with age, either up or down, and these changes can be prevented by
         calorie restriction.
          </p>
<h4 class="calibre15">
           Conclusions
          </h4>
<p class="calibre27">
           We demonstrate that 5′ tRNA halves circulate in the blood in a stable form, most likely
         as part of a nucleoprotein complex, and their serum levels are subject to regulation
         by age and calorie restriction. They may be produced by blood cells, but their cellular
         targets are not yet known. The characteristics of these circulating molecules, and
         their known function in suppression of translation initiation, suggest that they are
         a novel form of signaling molecule.
          </p>
</div>
</div>
<span id="keywords">
<h5 class="calibre20">
          Keywords:
         </h5>
         Aging; Calorie restriction; Circulating small RNAs; tRNA derivatives; tRNA halves; Serum; Plasma
        </span>
<div class="calibre5">
<a name="sec1">
</a>
<h3 class="calibre14">
          Background
         </h3>
<div class="calibre5">
<p class="calibre27">
           Several classes of small RNAs have been found to mediate biological functions in animals
         and plants
           <a name="d7241e192">
</a>
<a name="d7241e194">
</a>
<a name="d7241e196">
</a>
<a name="d7241e198">
</a>
<a name="d7241e200">
</a>
           [
           <a href="#B1" onclick="LoadInParent('#B1'); return false;">
            1
           </a>
           -
           <a href="#B5" onclick="LoadInParent('#B5'); return false;">
            5
           </a>
           ]. miRNAs, siRNAs, piRNAs, and others are bound by Argonaute proteins, and have the
         common property of directing protein complexes to nucleic acids with sequence complementarity,
         where they may cleave or otherwise alter the target
           <a name="d7241e204">
</a>
           [
           <a href="#B6" onclick="LoadInParent('#B6'); return false;">
            6
           </a>
           ]. In both plants and animals, some small RNAs are able to travel between tissues within
         an organism, thus transferring their functions to other cells. In vertebrates, there
         has been much recent interest in the presence of specific miRNAs in the plasma and
         serum; there is some evidence that these can be taken up by cells and alter gene expression,
         and there is also interest in the possibility that they can be markers of specific
         disease states, including cancer
           <a name="d7241e208">
</a>
<a name="d7241e210">
</a>
<a name="d7241e212">
</a>
           [
           <a href="#B7" onclick="LoadInParent('#B7'); return false;">
            7
           </a>
           -
           <a href="#B9" onclick="LoadInParent('#B9'); return false;">
            9
           </a>
           ].
          </p>
<p class="calibre27">
           There is also evidence for processing of non-coding RNAs into smaller RNAs, many with
         as yet poorly understood functions
           <a name="d7241e218">
</a>
<a name="d7241e220">
</a>
           [
           <a href="#B10" onclick="LoadInParent('#B10'); return false;">
            10
           </a>
           ,
           <a href="#B11" onclick="LoadInParent('#B11'); return false;">
            11
           </a>
           ]. Many of the non-coding RNAs that appear to undergo processing into smaller RNAs
         have well studied functions, although their smaller derivatives often do not. In particular,
         tRNA is processed into shorter forms termed tRNA fragments (tRFs)
           <a name="d7241e224">
</a>
<a name="d7241e226">
</a>
           [
           <a href="#B12" onclick="LoadInParent('#B12'); return false;">
            12
           </a>
           ,
           <a href="#B13" onclick="LoadInParent('#B13'); return false;">
            13
           </a>
           ]. The subject of this report is a tRNA fragment created by cleavage of tRNA near the
         anticodon loop to create a ″5′ tRNA half” (the term we will use here). Previous reports
         have described 5′ tRNA halves as intracellular molecules interacting with components
         of the translation initiation complex. 5′ tRNA halves have been shown to be induced
         by the ribonuclease angiogenin in response to stress in cultured cells, to promote
         assembly of stress granules carrying stalled preinitiation complexes, and to inhibit
         mRNA translation
           <a name="d7241e230">
</a>
<a name="d7241e232">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B15" onclick="LoadInParent('#B15'); return false;">
            15
           </a>
           ]; little more is known about their function.
          </p>
<p class="calibre27">
           We have sequenced small RNAs present in mouse serum; when multiple reportable alignments
         of the sequencing reads to the mouse genome were allowed, we noted the presence of
         a class of tRNA-derived 30–33 nt fragments that closely resemble the 5′ tRNA halves
         previously described in stressed cell cultures. Investigation of these 5′ tRNA halves
         reveals a novel class of circulating small RNAs whose characteristics, including changes
         with age that are antagonized by calorie restriction, strongly suggest physiologic
         regulation and function.
          </p>
</div>
</div>
<div class="calibre5">
<a name="sec2">
</a>
<h3 class="calibre14">
          Results and discussion
         </h3>
<div class="calibre5">
<h4 class="calibre15">
           Sequencing and computational analysis of small RNAs circulating in mouse serum
          </h4>
<p class="calibre27">
           While investigating the effects of aging and calorie restriction (CR) on the profiles
         of cell-free small RNAs circulating in the bloodstream, we used small RNA-Seq (Illumina
         reads of 50 nt) to compare the serum levels of small RNAs from young and old control
         mice, and old mice subjected to CR. A combined total of 196,083,881 pre-processed
         sequencing reads obtained from 9 different serum samples, were mapped to the mouse
         genome with bowtie using parameters that align reads according to a policy similar
         to Maq’s default policy
           <a name="d7241e248">
</a>
           [
           <a href="#B16" onclick="LoadInParent('#B16'); return false;">
            16
           </a>
           ]. Alignment of the combined 196,083,881 pre-processed sequencing reads generated a
         dataset of 163,078,230 mapped reads (83.2%), ranging from 5 to 48 nt. The size distribution
         of the mapped reads revealed an expected peak at 20–24 nt consistent with the size
         of miRNAs (Figure 
           <a name="d7241e251">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           ).
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','F1',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-1.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 1.
              </strong>
</a>
<strong class="calibre7">
              Length distribution of reads obtained by deep sequencing of small RNAs extracted from
            mouse serum.
             </strong>
             Shown here are only those reads that map to the mm10 (GRCm38) mouse genome. (
             <strong class="calibre7">
              A
             </strong>
             ) Length distribution is displayed by abundance of sequencing reads combined from
         9 serum samples. Reads were mapped to the mouse genome with bowtie according to Maq’s
         default policy, either allowing (blue bars) or disallowing (red bars) multiple reportable
         alignments for each read. (
             <strong class="calibre7">
              B
             </strong>
             ) Combined reads were mapped to the mouse genome with bowtie according to the end-to-end
         k-difference policy, either allowing (blue bars) or disallowing (red bars) multiple
         reportable alignments for each read. (
             <strong class="calibre7">
              C
             </strong>
             ) Length distribution of separate sequencing reads obtained from 9 individual serum
         small RNA samples. Length distribution is displayed by abundance of sequencing reads
         that were mapped to the mouse genome with bowtie according to Maq’s default policy,
         and allowing multiple reportable alignments for each read. Bars with different colors
         denote the source of the sequenced serum small RNA from the 9 different samples.
            </p>
</div>
</div>
<p class="calibre27">
           Only if multiple reportable alignments are allowed during bowtie mapping does an unfamiliar
         second peak emerge at 30–33 nt (Figure 
           <a name="d7241e278">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           A). The 30–33 nt peak persists when the bowtie alignment mode is changed from the
         Maq’s default policy (n option) to the end-to-end k-difference policy (v option),
         but again disappears when multiple reportable alignments are suppressed (Figure 
           <a name="d7241e281">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           B). The same two-peak pattern was observed when the 9 individual sequenced serum small
         RNA samples were mapped to the mouse genome (Figure 
           <a name="d7241e284">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F1','',800,470); return false;">
            1
           </a>
           C). Dependence of the 30–33 nt peak on multiple reportable alignments indicates that
         the reads are encoded by repetitive DNA. Six percent of the 163,078,230 mapped reads,
         aligned to a group of RepeatMasker classes (DNA, LINE, LTR, Low complexity, RC, SINE,
         Satellite, and Simple repeat); these reads were mainly &lt; 20 nt in size (Additional
         file
           <a name="d7241e287">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','',630,470); return false;">
            1
           </a>
           : Figure S1) and were not considered for further analysis.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','S1',630,470); return false;">
</a>
<strong class="calibre7">
              Additional file 1.
             </strong>
<strong class="calibre7">
              Additional Methods and Figures S1 and S2.
             </strong>
</p>
<p class="hide">
             Format: PDF 
         	Size: 6.8MB
             <a href="/content/supplementary/1471-2164-14-298-s1.pdf">
              Download file
             </a>
</p>
<p class="hide">
             This file can be viewed with:
             <a href="http://www.adobe.com/products/acrobat/readstep.html" onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;">
              Adobe Acrobat Reader
             </a>
<span>
<a href="/about/access/#opendata">
<img src="/bmcimages/article/opendata.gif" alt="Open Data" title="Open Data" class="calibre2"/>
</a>
</span>
</p>
</div>
</div>
<p class="calibre27">
           Annotation analysis of the mapped sequencing reads revealed that the 30–33 nt peak
         consists of reads mapping to tRNA genes (Figure 
           <a name="d7241e303">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           A), which are present in multiple copies in the genome. Reads in the 20–24 nt peak
         were mostly annotated as miRNAs. Further analysis showed that of the total 163,078,230
         reads that mapped to the mouse genome, 128,703,415 (79%) map to sequences encoding
         small RNAs, of which 67% and 31% were annotated as tRNAs and miRNAs, respectively
         (Figure 
           <a name="d7241e306">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           B). The remaining &lt; 3% of reads mapped to sequences annotated as encoding rRNA and
         other small RNAs (scRNA, snRNA, srpRNA).
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','F2',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-2.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 2.
              </strong>
</a>
<strong class="calibre7">
              Annotation of combined sequencing reads of small RNAs extracted from 9 mouse serum
            samples.
             </strong>
             (
             <strong class="calibre7">
              A
             </strong>
             ) Length distribution by abundance of the total reads mapped to the mouse genome (black
         bars), with reads annotated as mapping to either miRNAs (red bars), tRNAs (blue bars),
         rRNAs (green bars), or other small RNAs (yellow bars). Other small RNAs include scRNA,
         snRNA, and srpRNA. Note that the reads in the 20–24 nt and the 30–33 nt peaks are
         almost exclusively annotated as miRNAs and tRNAs, respectively. (
             <strong class="calibre7">
              B
             </strong>
             ) A pie chart showing the percentage of reads mapping to the specific types of small
         RNAs. (
             <strong class="calibre7">
              C
             </strong>
             ) Frequencies of 5′ tRNA half types represented in the aligned reads.
            </p>
</div>
</div>
<h4 class="calibre15">
           Characterization of circulating small RNAs derived from tRNAs
          </h4>
<p class="calibre27">
           Since the 86,343,437 reads that align to tRNA genes are only 30 to 33 nt, and thus
         do not represent full length tRNAs, we examined the tRNA end distribution of the reads,
         and annotated the reads based on their overlap with 5′ or 3′ ends of tRNAs. More than
         99% of the tRNA-derived reads align with the 5′ end of a tRNA; this is exemplified
         in Figure 
           <a name="d7241e337">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F3','',800,470); return false;">
            3
           </a>
           for two tRNA genes.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F3','F3',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-3.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 3.
              </strong>
</a>
<strong class="calibre7">
              UCSC genome browser screenshots illustrating alignment of reads to two tRNA genes.
             </strong>
             Shown are the Illumina sequencing reads (red), and the tRNA genes (blue) as annotated
         in the tRNA genes track ”Transfer RNA genes identified with tRNAscan-SE“. (
             <strong class="calibre7">
              A
             </strong>
             ) The alignment (number of reads, y-axis) shows that all the sequencing reads align
         to the 5′ end of chr1.tRNA704-GlyGCC gene. (
             <strong class="calibre7">
              B
             </strong>
             ) The majority of the sequencing reads align to the 5′ end of chr11.tRNA945-ArgCCG
         gene, whereas only a very small number of sequencing reads aligns to the 3′ end.
            </p>
</div>
</div>
<p class="calibre27">
           23%, 17%, 35%, and 26% of the sequencing reads that map to tRNAs are 30, 31, 32, and
         33 nucleotide in size, respectively (Table 
           <a name="d7241e361">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T1','',800,470); return false;">
            1
           </a>
           ), indicating that full length tRNAs are cleaved in the anticodon loop at more than
         one site and at varying rates to generate the 5′ tRNA halves found in serum. As an
         example, Figure 
           <a name="d7241e364">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F4" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F4','',800,470); return false;">
            4
           </a>
           depicts the size frequency of reads mapping to the 5′ end of the chr1.tRNA704-GlyGCC
         gene; this indicates that this tRNA is cleaved at different rates and at 4 different
         sites located upstream of the GCC anticodon in the anticodon loop.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F4" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F4','F4',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-4.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 4.
              </strong>
</a>
<strong class="calibre7">
              Cleavage sites of tRNAs.
             </strong>
             The cloverleaf structure of chr1.tRNA704-GlyGCC gene (downloaded from
             <a href="http://gtrnadb.ucsc.edu">
              http://gtrnadb.ucsc.edu
             </a>
<a alt="" href="http://www.webcitation.org/query.php?url=http://gtrnadb.ucsc.edu&amp;refdoi=10.1186/1471-2164-14-298" title="Archive copy of webpage">
              webcite
             </a>
             ) showing cleavage sites at the anticodon loop, with the percentage of the reads that
         map to the 5′ end of the tRNA (arrowheads). The cleavage sites are upstream of the
         GCC anticodon located at nucleotides 33 to 35. Numbers inside the anticodon loop indicate
         anticodon nucleotides positions.
            </p>
</div>
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T1','T1',800,470); return false;">
<strong class="calibre7">
               Table 1.
              </strong>
</a>
<strong class="calibre7">
              Total number and percentage of the different sizes of sequencing reads that map to
            tRNAs
             </strong>
</p>
</div>
</div>
<p class="calibre27">
           It is unlikely that this result is a sequencing artifact: the full length of most
         tRNAs is 75–90 nt, and the sequencing runs used to generate these data were 50 cycles
         while the reads occupy a narrow size range of 30–33 nt. This pattern suggests that
         the tRNA reads were derived from processed fragments of full length tRNAs; the remainder
         of the tRNA was not significantly detected in the serum small RNA libraries. In support
         of this conclusion, tRNAs have been shown to undergo cleavage within anticodon loops
         to produce tRNA-derived stress-induced fragments (tiRNAs) when cultured cells are
         subjected to stresses such as arsenite, heat shock, or ultraviolet irradiation
           <a name="d7241e453">
</a>
<a name="d7241e455">
</a>
           [
           <a href="#B17" onclick="LoadInParent('#B17'); return false;">
            17
           </a>
           ,
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. Such cleavage of the anticodon loop does not seem to be part of a tRNA degradation
         process, because the generated 5′ tRNA fragments are stable in the cell. Our findings
         indicate that tRNA fragments highly similar to tiRNAs are present under normal (unstressed)
         conditions, and can remain stable even after they are released into the peripheral
         blood. 5′ but not 3′ tRNA fragments inhibit mRNA translation initiation in cultured
         cell lines
           <a name="d7241e459">
</a>
           [
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ].
          </p>
<p class="calibre27">
           The individual 5′ tRNA halves present in serum are derived from a small subset of
         tRNAs (Figure 
           <a name="d7241e464">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F2','',800,470); return false;">
            2
           </a>
           C). The most abundant circulating tRNA halves were derived from the isoacceptors of
         glycine (46%), valine (44%), glutamine (8%), and histidine (1%), and the remaining
         amino acids together represented &lt;1%. We contrasted the number of tRNA genes in the
         Genomic tRNA Database
           <a name="d7241e468">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ], with the relative abundances of the circulating 5′ tRNA halves, and found no correlation
         (Table 
           <a name="d7241e471">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T2','',800,470); return false;">
            2
           </a>
           ). For example, the most abundant circulating 5′ tRNA halves were derived from tRNA-Gly,
         and the copy number of tRNA-Gly gene is 29; on the other hand tRNA-Cys genes, with
         a copy number of 57, generate &lt;1% of the 5′ tRNA-Cys halves.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T2" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T2','T2',800,470); return false;">
<strong class="calibre7">
               Table 2.
              </strong>
</a>
<strong class="calibre7">
              Frequencies of circulating 5′ tRNA halves and the gene copy number of tRNAs from which
            the tRNA halves were derived
             </strong>
</p>
</div>
</div>
<p class="calibre27">
           This implies a tRNA type-specific biogenesis and/or release of the circulating 5′
         tRNA halves.
          </p>
<h4 class="calibre15">
           Presence in circulating mouse blood of particles containing stable cell-free 5′ tRNA
         halves
          </h4>
<p class="calibre27">
           To obtain an independent validation of the sequencing results, we used Northern blotting
         to analyze small RNAs circulating in the mouse serum. As a positive control for detection
         of tRNA halves by Northern blotting, we included RNA from U2OS cells cultured in the
         absence or presence of sodium arsenite, which is known to generate tRNA halves in
         these cells
           <a name="d7241e563">
</a>
           [
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. We probed RNA from mouse serum with oligonucleotides complementary to 5′ or 3′ ends
         of specific tRNAs. Probes specific for the 5′ ends of tRNA-Gly-GCC or tRNA-Val-CAC
         detected a band migrating near the 30 nt RNA marker (Figure 
           <a name="d7241e566">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           A and C) confirming the presence of stable circulating 5′ tRNA halves. No significant
         bands migrated with the 30 nt RNA marker when the same Northern blot was probed for
         the 3′ end of tRNA-Gly-GCC or tRNA-Val-CAC (Figure 
           <a name="d7241e569">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           B and D).
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','F5',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-5.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 5.
              </strong>
</a>
<strong class="calibre7">
              Detection of 5′ tRNA halves in mouse serum.
             </strong>
             Northern blot analysis of RNA extracted from U2OS cells cultured in the absence (−)
         or presence (+) of sodium arsenite (AS), or from 0.4 ml of mouse serum. The blot was
         hybridized to a
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong class="calibre7">
              A
             </strong>
             ) or 3′ end (
             <strong class="calibre7">
              B
             </strong>
             ) of tRNA-Gly-GCC. The blot was also hybridized to a
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong class="calibre7">
              C
             </strong>
             ) or 3′ end (
             <strong class="calibre7">
              D
             </strong>
             ) of tRNA-Val-CAC. 5′ tRNA halves were also detected in fractionated mouse serum.
         (
             <strong class="calibre7">
              E
             </strong>
             -
             <strong class="calibre7">
              F
             </strong>
             ) Northern blotting analysis was carried out on RNA extracted from either 0.4 ml of
         mouse whole serum, from the supernatant (Sup) after ultracentrifugation of 0.4 ml
         of mouse serum at 110000g, or the ultracentrifugation pellet. The blot was hybridized
         to a
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probe complementary to the 5′ (
             <strong class="calibre7">
              E
             </strong>
             ) or 3′ end (
             <strong class="calibre7">
              F
             </strong>
             ) of tRNA-Gly-GCC. (
             <strong class="calibre7">
              G
             </strong>
             ) Ultrafiltration indicates a size for tRNA serum particles between 100 and 300 kDa.
         Samples of 0.2 ml serum mixed with 1.8 ml PBS were subjected to ultrafiltration through
         Vivaspin 2 columns with 30, 100, and 300 kDa MW cut-offs. Total RNAs were extracted
         from filtrate (f) and concentrate (c) fractions. Blot was hybridized to
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ end of tRNA-Gly-GCC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<p class="calibre27">
           We also probed RNA from mouse serum with a probe complementary to the 5′ end of tRNA-Asn-GTT
         to confirm the low abundance of circulating tRNA halves derived from tRNAs that were
         barely detected in the sequencing data. A 5-day exposure to X-ray film showed a very
         weak signal from tRNA-Asn-GTT probe compared to the strong signal from the tRNA-Gly-GCC
         probe obtained after a short (25 minute) exposure (Additional file
           <a name="d7241e628">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/suppl/S1" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/suppl/S1','',630,470); return false;">
            1
           </a>
           : Figure S2). These results are consistent with the sequencing, and inconsistent with
         a sequencing bias. They imply a tRNA type-specific biogenesis and/or release of the
         circulating 5′ tRNA halves.
          </p>
<p class="calibre27">
           We next asked if the 5′ tRNA halves are contained within circulating exosomes or microvesicles.
         We Northern blotted RNA extracted from pellet or supernatant after ultracentrifugation
         of mouse serum at 110000g for 2 hours. A probe for the 5′ end of tRNA-Gly-GCC detected
         an ~30 nt band present mainly in the supernatant and visible only as a trace in the
         pellet (Figure 
           <a name="d7241e633">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           E), while a probe for the 3′ end did not detect any significant signal (Figure 
           <a name="d7241e636">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           F). Identical results were obtained for the 5′ end of tRNA-Val-CAC (Figure 
           <a name="d7241e639">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','',800,470); return false;">
            6
           </a>
           A). These findings indicate that the 5′tRNA halves are mostly not included in exosomes
         or microvesicles, which would pellet in these conditions. Similarly, exosome encapsulation
         is not required for the stability of circulating miRNAs; after pelleting exosomes
         by ultracentrifugation of plasma, miRNAs were still detected in the supernatant fraction
           <a name="d7241e643">
</a>
<a name="d7241e645">
</a>
           [
           <a href="#B20" onclick="LoadInParent('#B20'); return false;">
            20
           </a>
           ,
           <a href="#B21" onclick="LoadInParent('#B21'); return false;">
            21
           </a>
           ].
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','F6',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-6.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 6.
              </strong>
</a>
<strong class="calibre7">
              Northern blotting analysis of tRNA-Val-CAC halves in mouse serum.
             </strong>
             (
             <strong class="calibre7">
              A
             </strong>
             ) RNA extracted from 0.4 ml of mouse whole serum, from supernatant or pellet after
         ultracentrifugation of 0.4 ml of mouse serum at 110000g and analyzed with northern
         blotting by hybridization to a
             <sup class="calibre10">
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Val-CAC.
         (
             <strong class="calibre7">
              B
             </strong>
             ) RNA extracted from 0.2 ml serum mixed with 1.8 ml PBS subjected to ultrafiltration
         with Vivaspin 2 columns with 30, 100, and 300 kDa MW cut-off. Total RNAs were extracted
         from filtrate (f) and concentrate (c) fractions. Blot was hybridized to a
             <sup class="calibre10">
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Val-CAC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<p class="calibre27">
           Because the tRNA halves we observe are stable in circulation but not encapsulated
         in exosomes, they are most likely complexed to carrying factors (e.g., proteins that
         protect them from degradation). To determine the size range of the putative complexes
         carrying the 5′ tRNA halves in the serum, we Northern blotted RNA extracted from concentrate
         or filtrate fractions after ultrafiltration of mouse serum samples through Vivaspin
         2 columns with 30, 100, or 300 kDa MW cut-off. A probe for the 5′ end of tRNA-Gly-GCC
         detected a ~30 nt band in the concentrates of 30 and 100 kDa MW cut-off, and in the
         filtrate of 300 kDa MW cut-off (Figure 
           <a name="d7241e675">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F5" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F5','',800,470); return false;">
            5
           </a>
           G). Identical results were obtained for the 5′ end of tRNA-Val-CAC (Figure 
           <a name="d7241e678">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F6" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F6','',800,470); return false;">
            6
           </a>
           B).
          </p>
<p class="calibre27">
           Thus 5′ tRNA halves circulate as part of 100–300 kDa complexes, while the 5′ tRNA
         halves themselves are only ~10 kDa. This is reminiscent of reports that miRNAs can
         circulate in the bloodstream as components of RNA-protein/lipoprotein complexes. Stable
         argonaute2-miRNA complexes that are not part of microvesicles were recovered from
         plasma and serum, and high-density lipoprotein has been reported to carry and deliver
         miRNAs to recipient cells
           <a name="d7241e684">
</a>
<a name="d7241e686">
</a>
<a name="d7241e688">
</a>
           [
           <a href="#B20" onclick="LoadInParent('#B20'); return false;">
            20
           </a>
           -
           <a href="#B22" onclick="LoadInParent('#B22'); return false;">
            22
           </a>
           ].
          </p>
<h4 class="calibre15">
           5′ tRNA halves are concentrated in hematopoietic and lymphoid tissues
          </h4>
<p class="calibre27">
           To investigate whether 5′ tRNA halves are present in tissues we extracted total RNA
         from liver, spleen, and testes, and did Northern blots with probes complementary to
         5′ and 3′ ends of tRNAs. We detected tRNA halves with a probe complementary to the
         5′ end of tRNA-Gly-GCC in the spleen, but not in the liver and testes; a probe for
         the 3′ end tRNAs detected only full length tRNAs in all 3 tissues (Figure 
           <a name="d7241e697">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','',800,470); return false;">
            7
           </a>
           A-B). This prompted us to explore the possibility that 5′ tRNA halves are present
         specifically in hematopoietic tissues. Northern blotting of several mouse tissues
         confirmed that 5′ tRNA halves are present in hematopoietic and lymphoid tissues including
         spleen, lymph nodes, fetal liver, leukocytes, bone marrow, and thymus, but almost
         absent in non-immune tissues including testes, liver, heart, brain, and kidney (Figure 
           <a name="d7241e700">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','',800,470); return false;">
            7
           </a>
           ); the presence of 3′ tRNA halves was not significant in any tissue. This finding
         is consistent with a previous report
           <a name="d7241e704">
</a>
           [
           <a href="#B23" onclick="LoadInParent('#B23'); return false;">
            23
           </a>
           ], in which tRNA halves were unexpectedly detected on cloning of microRNAs from human
         fetal liver, which is the main hematopoietic organ during fetal development. Identical
         results were obtained when the same Northern blots were probed for the 5′ and 3′ ends
         of tRNA-Val-CAC (Figure 
           <a name="d7241e707">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F8" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F8','',800,470); return false;">
            8
           </a>
           ).
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F7" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F7','F7',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-7.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 7.
              </strong>
</a>
<strong class="calibre7">
              Tissue distribution of tRNA-Gly-GCC halves.
             </strong>
             Northern blotting analysis of RNA extracted from the indicated mouse tissues. Blots
         were hybridized with
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ (
             <strong class="calibre7">
              A, C
             </strong>
             , and
             <strong class="calibre7">
              E
             </strong>
             ) or 3′ end (
             <strong class="calibre7">
              B, D
             </strong>
             , and
             <strong class="calibre7">
              F
             </strong>
             ) of tRNA-Gly-GCC. The positions of full length tRNAs and tRNA halves are indicated
         on the right. DM: decade markers.
            </p>
</div>
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F8" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F8','F8',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-8.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 8.
              </strong>
</a>
<strong class="calibre7">
              Tissue distribution of tRNA-Val-CAC halves.
             </strong>
             Northern blotting analysis of RNA extracted from the indicated mouse tissues. Blots
         were hybridized with
             <sup class="calibre10">
              32
             </sup>
             P-end-labeled oligonucleotide probes complementary to the 5′ (
             <strong class="calibre7">
              A
             </strong>
             ,
             <strong class="calibre7">
              B
             </strong>
             , and
             <strong class="calibre7">
              D
             </strong>
             ) or 3′ end (
             <strong class="calibre7">
              C
             </strong>
             and
             <strong class="calibre7">
              E
             </strong>
             ) of tRNA-Val-CAC. The positions of full length tRNAs and tRNA halves are indicated
         on the right. DM: decade markers.
            </p>
</div>
</div>
<p class="calibre27">
           More extensive studies will establish if 5′ tRNA halves are concentrated in particular
         blood cell types, although the very high levels in lymph nodes point to lymphocytes
         as one such type. The evidence does not establish whether the 5′ tRNA halves are concentrated
         in hematopoietic cells because they are produced there, or because they are preferentially
         taken up from the blood: neither the origin nor the destinations of the 5′ tRNA halves
         is certain. The low levels of 5′ tRNA halves present in non-hematopoietic tissues
         may indicate low levels in those tissues, but they may also be derived from residual
         blood cells in those tissues.
          </p>
<h4 class="calibre15">
           A chelating agent destabilizes circulating 5′ tRNA halves
          </h4>
<p class="calibre27">
           Because clotting has the potential to release particles that are not present in circulating
         blood, we asked if 5′ tRNA halves circulating in the mouse serum are also present
         in mouse plasma. Northern blotting with a 5′ tRNA half probe gave a very weak band
         in a plasma sample when compared to the band derived from an equal volume of serum
         from the same mouse (Figure 
           <a name="d7241e778">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','',800,470); return false;">
            9
           </a>
           ). The lack of 5′ tRNA halves in plasma is not due to a global loss of small RNAs
         during preparation of the plasma, which was anticoagulated with EDTA. We used qPCR
         to assess the integrity of two circulating miRNAs in mouse serum, serum treated with
         EDTA, and plasma collected with EDTA. As shown by amplification in all three specimens
         (Figure 
           <a name="d7241e781">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F10" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F10','',800,470); return false;">
            10
           </a>
           ), EDTA does not affect these circulating miRNAs.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','F9',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-9.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 9.
              </strong>
</a>
<strong class="calibre7">
              Detection of 5′ tRNA halves in mouse plasma and serum.
             </strong>
             Northern blot of RNAs extracted from 0.4 ml of mouse serum, serum treated with EDTA,
         and heparin- or EDTA-collected plasma, hybridized to a
             <sup class="calibre10">
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Gly-GCC.
         EDTA sharply lowers the abundance of 5′ tRNA halves in serum; in plasma, 5′ tRNA halves
         are abundant when heparin is the anticoagulant, but are nearly absent when EDTA is
         present. 5′ tRNA halves are similarly abundant when calcium citrate is used as the
         anticoagulant (not shown).
            </p>
</div>
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F10" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F10','F10',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-10.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 10.
              </strong>
</a>
<strong class="calibre7">
              Real-time PCR amplification plots of circulating miRNAs.
             </strong>
             Shown are the amplification plots for miR-16 (blue), miR-24 (black), and the spiked-in
         miR-Cel-39 (red) measured in mouse serum (
             <strong class="calibre7">
              A
             </strong>
             ), mouse serum treated with EDTA (
             <strong class="calibre7">
              B
             </strong>
             ), and mouse plasma collected on EDTA (
             <strong class="calibre7">
              C
             </strong>
             ). The y-axis represents the relative fluorescence units (RFU) in a semi-log scale.
         The x-axis represents the cycle at which fluorescence was detected above an automatically
         determined threshold for the indicated miRNA. EDTA does not change the concentration
         of miRNAs in plasma.
            </p>
</div>
</div>
<p class="calibre27">
           This result could suggest that 5′ tRNA halves are an artifact of blood clotting, but
         could also be an effect of EDTA, a chelating agent that depletes ions required for
         clotting. To assess the effects of EDTA on 5′ tRNA halves, we used Northern blotting
         to analyse a sample of serum that was incubated with EDTA for 15 min before RNA extraction.
         We also analyzed a sample of plasma extracted from blood collected with heparin, a
         nonchelating anticoagulant. This analysis showed that treatment of serum with EDTA
         significantly decreased the signal corresponding the 5′ tRNA halves, while 5′ tRNA
         halves are abundant in heparinized plasma (Figure 
           <a name="d7241e824">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F9" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F9','',800,470); return false;">
            9
           </a>
           ). The same results were obtained with RNAs from human plasma collected on EDTA and
         from serum (Figure 
           <a name="d7241e827">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F11" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F11','',800,470); return false;">
            11
           </a>
           ). These findings suggest that chelation of ions by EDTA destabilizes the complexes
         carrying 5′ tRNA halves, exposing the RNA to ribonucleases which are abundant in plasma.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/figure/F11" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/figure/F11','F11',800,470); return false;">
<img src="/content/figures/1471-2164-14-298-11.gif" alt="thumbnail" class="thumbnail"/>
<strong class="calibre7">
               Figure 11.
              </strong>
</a>
<strong class="calibre7">
              5′ tRNA halves are also present in human serum and leukocytes, but not in EDTA plasma.
             </strong>
             RNAs extracted from human leukocytes, 0.4 ml of human serum, or EDTA-collected plasma.
         The northern blot was hybridized to a
             <sup class="calibre10">
              32
             </sup>
             P 5′-end labeled oligonucleotide probe complementary to the 5′ end of tRNA-Gly-GCC.
         The positions of full length tRNAs and tRNA halves are indicated on the right. DM:
         decade markers.
            </p>
</div>
</div>
<h4 class="calibre15">
           Calorie restriction offsets age-associated changes in levels of specific circulating
         5′ tRNA halves
          </h4>
<p class="calibre27">
           Calorie restriction (CR) can delay, prevent, or reverse many age-associated changes
         in physiologic parameters. We used aging and CR as model physiologic states to explore
         the possibility that they are associated with changes in the levels of circulating
         5′ tRNA halves. We performed pairwise comparisons between young and old control groups
         to measure the differential abundance in circulating 5′ tRNA halves associated with
         old age, and between old control and old CR groups to determine whether CR has an
         effect on any age-associated changes.
          </p>
<p class="calibre27">
           This analysis revealed that aging is associated with alterations, either increase
         or decrease, in the circulating levels of 5′ tRNA halves derived from specific tRNA
         isoacceptors (Table 
           <a name="d7241e854">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','',800,470); return false;">
            3
           </a>
           ). Notably, CR mitigated most of these age-related changes (Table 
           <a name="d7241e857">
</a>
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','',800,470); return false;">
            3
           </a>
           ), although it did not completely prevent them. CR has been shown to oppose the molecular
         and biological markers of aging including alterations in gene expression
           <a name="d7241e861">
</a>
           [
           <a href="#B24" onclick="LoadInParent('#B24'); return false;">
            24
           </a>
           ]. A causal relationship between circulating 5′ tRNA halves and the manifestations
         of aging is not established by this study, but it does indicate that levels are regulated
         in an age-associated fashion.
          </p>
<div class="calibre5">
<div class="calibre5">
<p class="hide">
<a href="http://www.biomedcentral.com/1471-2164/14/298/table/T3" onclick="popup('http://www.biomedcentral.com/1471-2164/14/298/table/T3','T3',800,470); return false;">
<strong class="calibre7">
               Table 3.
              </strong>
</a>
<strong class="calibre7">
              Age-associated changes in the levels of mouse circulating 5′ tRNA halves and the effects
            of CR on the age-associated changes
             </strong>
</p>
</div>
</div>
</div>
</div>
<div class="calibre5">
<a name="sec3">
</a>
<h3 class="calibre14">
          Conclusions
         </h3>
<div class="calibre5">
<p class="calibre27">
           Deep sequencing of small RNAs extracted from mouse serum identifies a population of
         tRNA-derived molecules, termed 5′ tRNA halves, previously described only as stress-induced
         inhibitors of translation initiation in cultured cells. 5′ tRNA halves are more abundant
         than miRNAs in mouse serum, and are derived from distinct subset of tRNAs by cleavage
         near the anticodon loop; the 3′ portion of the tRNA molecule is present in serum only
         in trace quantities. Ultracentrifugation and size fractionation establish that the
         5′ tRNA halves circulate as part of a larger complex, but are not contained in exosomes
         or microvesicles; their sensitivity to the chelating agent EDTA provides further evidence
         that they exist as circulating nucleoprotein complexes. They are concentrated in hematopoietic
         and lymphoid tissues, and present in other tissues at very low levels that may reflect
         residual blood cells. The origin of the serum particles, and their destinations, are
         uncertain; however their concentration in blood cells suggest that they may be produced
         by these cells. Levels of serum 5′ tRNA halves are distinctly changed in aged mice,
         and calorie restriction inhibits these changes, indicating that they are subject to
         physiologic regulation. Taken together with the extant evidence that 5′ tRNA halves
         can regulate mRNA translation, the characteristics of the circulating 5′ tRNA halves
         we have discovered suggest that they function as signaling molecules with as yet unknown
         physiologic roles.
          </p>
<p class="calibre27">
           To date, the only known function of 5′ tRNA halves is inhibition of translation in
         cultured cells subjected to a variety of stressors; transfection of 5′ tRNA halves
         inhibits global translation in U2OS cells
           <a name="d7241e1609">
</a>
<a name="d7241e1611">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B18" onclick="LoadInParent('#B18'); return false;">
            18
           </a>
           ]. A study published while this paper was in preparation reported induction of 5′ tRNA
         halves in human airway epithelial cells upon infection with respiratory syncytial
         virus (RSV). Induction involves cleavage at the tRNA anticodon loop by angiogenin,
         and at least one type, the 5′ tRNA-Glu-CTC half, promotes RSV replication
           <a name="d7241e1615">
</a>
           [
           <a href="#B25" onclick="LoadInParent('#B25'); return false;">
            25
           </a>
           ]. Our findings indicate that 5′ tRNA halves function on an organismal rather than
         merely a cellular level. Furthermore they are likely to function in a context much
         broader than cellular stress or infection: we find 5′ tRNA halves in unstressed conditions.
         Changes in their expression (either increased or decreased) with age are also consistent
         with a broader physiologic role, and it is particularly interesting that these changes
         are partially mitigated by calorie restriction.
          </p>
<p class="calibre27">
           The most extensively studied cellular tRNA halves are generated under stress conditions
         by angiogenin, which cleaves mature tRNAs within the anticodon loops
           <a name="d7241e1621">
</a>
           [
           <a href="#B26" onclick="LoadInParent('#B26'); return false;">
            26
           </a>
           ]. The stress-induced tRNA halves target the translation initiation machinery to reprogram
         protein translation in order to promote cell survival during stress
           <a name="d7241e1625">
</a>
<a name="d7241e1627">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ,
           <a href="#B26" onclick="LoadInParent('#B26'); return false;">
            26
           </a>
           ]. Pull-down and mass spectrometry analyses of RNA-protein complexes have identified
         several cellular proteins (YB-1, FXR-1, and PABP1) bound to intracellular 5′ tRNA
         halves
           <a name="d7241e1631">
</a>
           [
           <a href="#B14" onclick="LoadInParent('#B14'); return false;">
            14
           </a>
           ]. The nature of the proteins and/or other factors that bind and stabilize the extracellular
         form of 5′ tRNAs halves has yet to be elucidated. Understanding of the origin, composition,
         and destinations of these complexes will provide insights into their role in organismal
         physiology.
          </p>
</div>
</div>
<div class="calibre5">
<a name="sec4">
</a>
<h3 class="calibre14">
          Methods
         </h3>
<div class="calibre5">
<h4 class="calibre15">
           Serum collection, RNA isolation, and small RNA library construction
          </h4>
<p class="calibre27">
           Male mice of the long-lived B6C3F1 strain were fed either control or calorie-restricted
         (CR) diet (∼40% fewer calories than the control). Three mice were studied from each
         of three groups: young (7-month) and old (27-month) mice fed the control diet, and
         old (27-month) mice fed the CR diet. Total RNA including small RNA was isolated from
         each serum sample with miRNeasy kit (Qiagen) and used to construct indexed sequencing
         libraries with the Illumina TruSeq Small RNA Sample Prep Kit. The libraries were pooled
         and sequenced on an Illumina HiSeq 2000 instrument to generate 50 base reads. Further
         details about the mice and diets, and library construction are provided in SI Methods
         and Figures.
          </p>
<h4 class="calibre15">
           Mapping and annotation of sequencing reads
          </h4>
<p class="calibre27">
           Sequencing reads were pre-processed with FASTX-Toolkit (hannonlab.cshl.edu) to trim
         the adaptor sequences, and discard low quality reads. The obtained clean reads were
         mapped to the mouse reference genome (GRCm38/mm10) with bowtie version 0.12.8
           <a name="d7241e1651">
</a>
           [
           <a href="#B16" onclick="LoadInParent('#B16'); return false;">
            16
           </a>
           ] using different combinations of alignment and reporting options. We used the option
         ”-n 0 -l 14“ to align the sequencing reads according to a policy similar to Maq‘s
         default policy and requiring no mismatches in the first 14 bases (the high-quality
         end of the read). In addition, this mode of alignment was combined with options that
         define which and how many alignments should be reported; the option “-k 1 --best”
         instructed bowtie to report only the best alignment if more than one valid alignment
         exists, while the option “-m 1” instructed bowtie to refrain from reporting any alignments
         for reads having multiple reportable alignments. The “-k 1 --best” and “-m 1” modes
         of alignment reporting were also used in combination with the end-to-end k-difference
         (−v) alignment mode. Varying the alignment and reporting modes allowed the differential
         detection of two predominant peak sizes of sequencing reads as described in the results
         section.
          </p>
<p class="calibre27">
           Annotation analysis of the mapped sequencing reads was performed with bedtools
           <a name="d7241e1657">
</a>
           [
           <a href="#B27" onclick="LoadInParent('#B27'); return false;">
            27
           </a>
           ] using the following databases: the Genomic tRNA Database
           <a name="d7241e1661">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ] (gtrnadb.ucsc.edu), miRBase 18 (mirbase.org), and rRNA, snRNA, scRNA, and srpRNA
         which were extracted from the RepeatMasker track (genome.ucsc.edu; mm10).
          </p>
<h4 class="calibre15">
           Analysis of differentially abundant circulating tRNA halves
          </h4>
<p class="calibre27">
           The bowtie alignment files generated above from the young and old control and old
         CR serum sequencing samples were analyzed with bedtools
           <a name="d7241e1671">
</a>
           [
           <a href="#B27" onclick="LoadInParent('#B27'); return false;">
            27
           </a>
           ] to obtain the coverage of the tRNA genes included in the Genomic tRNA Database
           <a name="d7241e1675">
</a>
           [
           <a href="#B19" onclick="LoadInParent('#B19'); return false;">
            19
           </a>
           ] (gtrnadb.ucsc.edu), and to determine the read count for each tRNA in the database.
         The tRNA read counts were further analyzed with the Bioconductor package edgeR
           <a name="d7241e1679">
</a>
           [
           <a href="#B28" onclick="LoadInParent('#B28'); return false;">
            28
           </a>
           ] to detect the changes in the levels of circulating 5′ tRNA halves in the different
         experimental groups. The algorithm of edgeR fits a negative binomial model to the
         count data, estimates dispersion, and measures differences using the generalized linear
         model likelihood ratio test which is recommended for experiments with multiple factors,
         such as the simultaneous analysis of age and diet in our study. The fitted count data
         was analyzed by performing pairwise comparisons between the different experimental
         groups: young and old control groups were compared to measure the differential abundance
         in circulating 5′ tRNA halves associated with old age; old control and old CR groups
         were compared to determine whether CR has an effect on any age-associated changes.
         The results were further filtered to keep only 5′ tRNA halves that achieved a minimum
         of 500 counts per million (cpm) in at least one of the 3 experimental groups.
          </p>
<h4 class="calibre15">
           Northern blot assays
          </h4>
<p class="calibre27">
           RNAs analyzed with Northern blots were extracted from normal or sodium arsenite-treated
         U2OS and from a variety of tissues and sera harvested from one-year-old mice fed control
         diet. Before RNA extraction, some serum samples were centrifuged at 110000 g for 2
         hrs, and supernatant and pellet fractions were separated, or were separated into concentrate
         and filtrate fractions by ultrafiltration through Vivaspin 2 columns with 30, 100,
         or 300 kDa MW cut-off. RNAs were separated on 15% denaturing polyacrylamide gels,
         transferred and fixed to a membrane by chemical cross-linking
           <a name="d7241e1689">
</a>
           [
           <a href="#B29" onclick="LoadInParent('#B29'); return false;">
            29
           </a>
           ], and hybridized with probes complementary to 5’ and 3′ ends of tRNAs. Further details
         about Northern blot assays, and probe sequences are provided in SI Methods and Figures.
          </p>
</div>
</div>
<div class="calibre5">
<a name="sec5">
</a>
<h3 class="calibre14">
          Competing interests
         </h3>
<div class="calibre5">
<p class="calibre27">
           The authors declare that they have no competing interests.
          </p>
</div>
</div>
<div class="calibre5">
<a name="sec6">
</a>
<h3 class="calibre14">
          Authors’ contributions
         </h3>
<div class="calibre5">
<p class="calibre27">
           J.M.D conceived and designed the study, carried out the bioinformatics analysis, and
         drafted the manuscript. SRS participated in the design of the study and the interpretation
         of the results. HA participated in the data analysis and interpretation of the results.
         AY, DB, and PM performed the experiments. DIKM participated in the interpretation
         of the results, and helped to draft the manuscript. All authors read and approved
         the final manuscript.
          </p>
</div>
</div>
<div class="calibre5">
<a name="ack">
</a>
<h3 class="calibre14">
          Acknowledgements
         </h3>
<div class="calibre5">
<p class="calibre27">
           We thank Noel Guerrero for his help.
          </p>
</div>
</div>
<div class="calibre5">
<a name="refs">
</a>
<h3 class="calibre14">
          References
         </h3>
<div class="calibre5" id="article-references">
<ol id="references" class="contributor-list">
<li id="B1" class="calibre4">
<p class="hide">
<a name="B1">
</a>
             Okamura K:
             <strong class="calibre7">
              Diversity of animal small RNA pathways and their biological utility.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Wiley Interdiscip Rev RNA
             </em>
             2012,
             <strong class="calibre7">
              3
             </strong>
             (3)
             <strong class="calibre7">
              :
             </strong>
             351-368.
             <a href="/pubmed/22086843" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22086843" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B1','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B2" class="calibre4">
<p class="hide">
<a name="B2">
</a>
             Wery M,  Kwapisz M,  Morillon A:
             <strong class="calibre7">
              Noncoding RNAs in gene regulation.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Wiley Interdiscip Rev Syst Biol Med
             </em>
             2011,
             <strong class="calibre7">
              3
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             728-738.
             <a href="/pubmed/21381218" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21381218" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B2','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B3" class="calibre4">
<p class="hide">
<a name="B3">
</a>
             Zhang C:
             <strong class="calibre7">
              Novel functions for small RNA molecules.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Curr Opin Mol Ther
             </em>
             2009,
             <strong class="calibre7">
              11
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             641-651.
             <a href="/pubmed/20072941" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20072941" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B3','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B4" class="calibre4">
<p class="hide">
<a name="B4">
</a>
             Zhang S,  Sun L,  Kragler F:
             <strong class="calibre7">
              The phloem-delivered RNA pool contains small noncoding RNAs and interferes with translation.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Plant Physiol
             </em>
             2009,
             <strong class="calibre7">
              150
             </strong>
             (1)
             <strong class="calibre7">
              :
             </strong>
             378-387.
             <a href="/pubmed/19261735" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19261735" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19261735" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B4','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B5" class="calibre4">
<p class="hide">
<a name="B5">
</a>
             Esteller M:
             <strong class="calibre7">
              Non-coding RNAs in human disease.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Nat Rev Genet
             </em>
             2011,
             <strong class="calibre7">
              12
             </strong>
             (12)
             <strong class="calibre7">
              :
             </strong>
             861-874.
             <a href="/pubmed/22094949" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22094949" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B5','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B6" class="calibre4">
<p class="hide">
<a name="B6">
</a>
             Joshua-Tor L,  Hannon GJ:
             <strong class="calibre7">
              Ancestral roles of small RNAs: an Ago-centric perspective.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Cold Spring Harb Perspect Biol
             </em>
             2011,
             <strong class="calibre7">
              3
             </strong>
             (10)
             <strong class="calibre7">
              :
             </strong>
             a003772.
             <a href="/pubmed/20810548" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20810548" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B6','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B7" class="calibre4">
<p class="hide">
<a name="B7">
</a>
             Allegra A,  Alonci A,  Campo S,  Penna G,  Petrungaro A,  Gerace D,  Musolino C:
             <strong class="calibre7">
              Circulating microRNAs: New biomarkers in diagnosis, prognosis and treatment of cancer
                  (Review).
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Int J Oncol
             </em>
             2012,
             <strong class="calibre7">
              41
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             1897-1912.
             <a href="/pubmed/23026890" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23026890" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B7','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B8" class="calibre4">
<p class="hide">
<a name="B8">
</a>
             Etheridge A,  Lee I,  Hood L,  Galas D,  Wang K:
             <strong class="calibre7">
              Extracellular microRNA: a new source of biomarkers.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Mutat Res
             </em>
             2011,
             <strong class="calibre7">
              717
             </strong>
             (1–2)
             <strong class="calibre7">
              :
             </strong>
             85-90.
             <a href="/pubmed/21402084" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21402084" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21402084" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B8','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B9" class="calibre4">
<p class="hide">
<a name="B9">
</a>
             Zen K,  Zhang CY:
             <strong class="calibre7">
              Circulating microRNAs: a novel class of biomarkers to diagnose and monitor human cancers.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Medicinal research reviews
             </em>
             2012,
             <strong class="calibre7">
              32
             </strong>
             (2)
             <strong class="calibre7">
              :
             </strong>
             326-348.
             <a href="/pubmed/22383180" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22383180" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B9','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B10" class="calibre4">
<p class="hide">
<a name="B10">
</a>
             Rother S,  Meister G:
             <strong class="calibre7">
              Small RNAs derived from longer non-coding RNAs.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Biochimie
             </em>
             2011,
             <strong class="calibre7">
              93
             </strong>
             (11)
             <strong class="calibre7">
              :
             </strong>
             1905-1915.
             <a href="/pubmed/21843590" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21843590" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B10','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B11" class="calibre4">
<p class="hide">
<a name="B11">
</a>
             Tuck AC,  Tollervey D:
             <strong class="calibre7">
              RNA in pieces.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Trends Genetics: TIG
             </em>
             2011,
             <strong class="calibre7">
              27
             </strong>
             (10)
             <strong class="calibre7">
              :
             </strong>
             422-432.
             <a href="http://dx.doi.org/10.1016/j.tig.2011.06.001" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B11','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B12" class="calibre4">
<p class="hide">
<a name="B12">
</a>
             Sobala A,  Hutvagner G:
             <strong class="calibre7">
              Transfer RNA-derived fragments: origins, processing, and functions.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Wiley Interdiscip Rev RNA
             </em>
             2011,
             <strong class="calibre7">
              2
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             853-862.
             <a href="/pubmed/21976287" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21976287" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B12','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B13" class="calibre4">
<p class="hide">
<a name="B13">
</a>
             Lee YS,  Shibata Y,  Malhotra A,  Dutta A:
             <strong class="calibre7">
              A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Genes Dev
             </em>
             2009,
             <strong class="calibre7">
              23
             </strong>
             (22)
             <strong class="calibre7">
              :
             </strong>
             2639-2649.
             <a href="/pubmed/19933153" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19933153" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19933153" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B13','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B14" class="calibre4">
<p class="hide">
<a name="B14">
</a>
             Ivanov P,  Emara MM,  Villen J,  Gygi SP,  Anderson P:
             <strong class="calibre7">
              Angiogenin-induced tRNA fragments inhibit translation initiation.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Mol Cell
             </em>
             2011,
             <strong class="calibre7">
              43
             </strong>
             (4)
             <strong class="calibre7">
              :
             </strong>
             613-623.
             <a href="/pubmed/21855800" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21855800" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21855800" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B14','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B15" class="calibre4">
<p class="hide">
<a name="B15">
</a>
             Saikia M,  Krokowski D,  Guan BJ,  Ivanov P,  Parisien M,  Hu GF,  Anderson P,  Pan T,  Hatzoglou M:
             <strong class="calibre7">
              Genome-wide identification and quantitative analysis of cleaved tRNA fragments induced
                  by cellular stress.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              J Biol Chem
             </em>
             2012,
             <strong class="calibre7">
              287
             </strong>
             (51)
             <strong class="calibre7">
              :
             </strong>
             42708-42725.
             <a href="/pubmed/23086926" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23086926" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B15','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B16" class="calibre4">
<p class="hide">
<a name="B16">
</a>
             Langmead B,  Trapnell C,  Pop M,  Salzberg SL:
             <strong class="calibre7">
              Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Genome Biol
             </em>
             2009,
             <strong class="calibre7">
              10
             </strong>
             (3)
             <strong class="calibre7">
              :
             </strong>
             R25.
             <a href="/pubmed/19261174" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">
              BioMed Central Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19261174" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B16','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B17" class="calibre4">
<p class="hide">
<a name="B17">
</a>
             Thompson DM,  Lu C,  Green PJ,  Parker R:
             <strong class="calibre7">
              tRNA cleavage is a conserved response to oxidative stress in eukaryotes.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              RNA
             </em>
             2008,
             <strong class="calibre7">
              14
             </strong>
             (10)
             <strong class="calibre7">
              :
             </strong>
             2095-2103.
             <a href="/pubmed/18719243" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18719243" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18719243" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B17','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B18" class="calibre4">
<p class="hide">
<a name="B18">
</a>
             Yamasaki S,  Ivanov P,  Hu GF,  Anderson P:
             <strong class="calibre7">
              Angiogenin cleaves tRNA and promotes stress-induced translational repression.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              J Cell Biol
             </em>
             2009,
             <strong class="calibre7">
              185
             </strong>
             (1)
             <strong class="calibre7">
              :
             </strong>
             35-42.
             <a href="/pubmed/19332886" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19332886" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19332886" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B18','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B19" class="calibre4">
<p class="hide">
<a name="B19">
</a>
             Chan PP,  Lowe TM: GRNA:
             <strong class="calibre7">
              a database of transfer RNA genes detected in genomic sequence.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Nucleic Acids Res
             </em>
             2009,
             <strong class="calibre7">
              37
             </strong>
<strong class="calibre7">
              :
             </strong>
             93-97.
             <a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B19','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B20" class="calibre4">
<p class="hide">
<a name="B20">
</a>
             Arroyo JD,  Chevillet JR,  Kroh EM,  Ruf IK,  Pritchard CC,  Gibson DF,  Mitchell PS,  Bennett CF,  Pogosova-Agadjanyan EL,  Stirewalt DL,
             <em class="calibre13">
              et al
             </em>
             .:
             <strong class="calibre7">
              Argonaute2 complexes carry a population of circulating microRNAs independent of vesicles
                  in human plasma.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Proc Natl Acad Sci U S A
             </em>
             2011,
             <strong class="calibre7">
              108
             </strong>
             (12)
             <strong class="calibre7">
              :
             </strong>
             5003-5008.
             <a href="/pubmed/21383194" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21383194" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21383194" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B20','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B21" class="calibre4">
<p class="hide">
<a name="B21">
</a>
             Turchinovich A,  Burwinkel B:
             <strong class="calibre7">
              Distinct AGO1 and AGO2 associated miRNA profiles in human cells and blood plasma.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              RNA Biol
             </em>
             2012,
             <strong class="calibre7">
              9
             </strong>
<strong class="calibre7">
              :
             </strong>
             8.
             <a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B21','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B22" class="calibre4">
<p class="hide">
<a name="B22">
</a>
             Vickers KC,  Palmisano BT,  Shoucri BM,  Shamburek RD,  Remaley AT:
             <strong class="calibre7">
              MicroRNAs are transported in plasma and delivered to recipient cells by high-density
                  lipoproteins.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Nat Cell Biol
             </em>
             2011,
             <strong class="calibre7">
              13
             </strong>
             (4)
             <strong class="calibre7">
              :
             </strong>
             423-433.
             <a href="/pubmed/21423178" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21423178" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21423178" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B22','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B23" class="calibre4">
<p class="hide">
<a name="B23">
</a>
             Fu H,  Feng J,  Liu Q,  Sun F,  Tie Y,  Zhu J,  Xing R,  Sun Z,  Zheng X:
             <strong class="calibre7">
              Stress induces tRNA cleavage by angiogenin in mammalian cells.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              FEBS Lett
             </em>
             2009,
             <strong class="calibre7">
              583
             </strong>
             (2)
             <strong class="calibre7">
              :
             </strong>
             437-442.
             <a href="/pubmed/19114040" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19114040" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B23','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B24" class="calibre4">
<p class="hide">
<a name="B24">
</a>
             Spindler SR,  Dhahbi JM:
             <strong class="calibre7">
              Conserved and tissue-specific genic and physiologic responses to caloric restriction
                  and altered IGFI signaling in mitotic and postmitotic tissues.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Annu Rev Nutr
             </em>
             2007,
             <strong class="calibre7">
              27
             </strong>
<strong class="calibre7">
              :
             </strong>
             193-217.
             <a href="/pubmed/17428180" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17428180" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B24','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B25" class="calibre4">
<p class="hide">
<a name="B25">
</a>
             Wang Q,  Lee I,  Ren J,  Ajay SS,  Lee YS,  Bao X:
             <strong class="calibre7">
              Identification and Functional Characterization of tRNA-derived RNA Fragments (tRFs)
                  in Respiratory Syncytial Virus Infection.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Mol Ther
             </em>
             2013,
             <strong class="calibre7">
              21
             </strong>
             (2)
             <strong class="calibre7">
              :
             </strong>
             368-379.
             <a href="/pubmed/23183536" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23183536" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B25','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B26" class="calibre4">
<p class="hide">
<a name="B26">
</a>
             Li S,  Hu GF:
             <strong class="calibre7">
              Emerging role of angiogenin in stress response and cell survival under adverse conditions.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              J Cell Physiol
             </em>
             2012,
             <strong class="calibre7">
              227
             </strong>
             (7)
             <strong class="calibre7">
              :
             </strong>
             2822-2826.
             <a href="/pubmed/22021078" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22021078" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B26','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B27" class="calibre4">
<p class="hide">
<a name="B27">
</a>
             Quinlan AR,  Hall IM:
             <strong class="calibre7">
              BEDTools: a flexible suite of utilities for comparing genomic features.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Bioinformatics
             </em>
             2010,
             <strong class="calibre7">
              26
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             841-842.
             <a href="/pubmed/20110278" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20110278" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20110278" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B27','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B28" class="calibre4">
<p class="hide">
<a name="B28">
</a>
             Robinson MD,  McCarthy DJ,  Smyth GK:
             <strong class="calibre7">
              edgeR: a Bioconductor package for differential expression analysis of digital gene
                  expression data.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Bioinformatics
             </em>
             2010,
             <strong class="calibre7">
              26
             </strong>
             (1)
             <strong class="calibre7">
              :
             </strong>
             139-140.
             <a href="/pubmed/19910308" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19910308" target="_blank">
              Publisher Full Text
             </a>
             |
             <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19910308" target="_blank">
              PubMed Central Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B28','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
<li id="B29" class="calibre4">
<p class="hide">
<a name="B29">
</a>
             Pall GS,  Hamilton AJ:
             <strong class="calibre7">
              Improved northern blot method for enhanced detection of small RNA.
             </strong>
</p>
<p class="hide">
<em class="calibre13">
              Nature protocols
             </em>
             2008,
             <strong class="calibre7">
              3
             </strong>
             (6)
             <strong class="calibre7">
              :
             </strong>
             1077-1084.
             <a href="/pubmed/18536652" target="_blank">
              PubMed Abstract
             </a>
             |
             <a href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18536652" target="_blank">
              Publisher Full Text
             </a>
<a href="/sfx_links?ui=1471-2164-14-298&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2164-14-298&amp;bibl=B29','SFXMenu','460','420'); return false;">
<img src="/sfx_links?getImage" alt="OpenURL" class="calibre2"/>
</a>
</p>
<p class="hide">
</p>
</li>
</ol>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<dl class="hide">
<dt class="calibre5">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising">
</a>
</dt>
<dd class="calibre18">
</dd>
</dl>
<hr class="calibre6"/>
<div id="footer" class="calibre5">
<span>
<a href="?fmt_view=mobile">
     Mobile view
    </a>
    |
    <strong class="calibre7">
     Desktop view
    </strong>
</span>
<div class="calibre5">
<div class="calibre5">
<div class="calibre5">
<ul class="calibrefeedlist">
<li class="calibre4">
<a href="http://www.biomedcentral.com/about/tandc">
         Terms and Conditions
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/about/privacy">
         Privacy statement
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/presscenter">
         Press
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/advertisers">
         Information for advertisers
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/about/bmcjobs">
         Jobs at BMC
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/support">
         Support
        </a>
</li>
<li class="calibre4">
<a href="http://www.biomedcentral.com/about/contact">
         Contact us
        </a>
</li>
</ul>
<p id="copyright" class="hide">
       © 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.
      </p>
</div>
</div>
</div>
</div>
<div class="calibre5">
<a href="http://www.springer.com/" target="blank">
<img src="/images/Springer_Branding_footer_logo.png" class="calibre28"/>
</a>
</div>
<div id="fb-root" class="calibre5">
</div>
<div class="calibre5">
<dl id="x96banner" class="hide">
<dt class="calibre5">
<a href="http://www.biomedcentral.com/advertisers/digital_advertising">
      Advertisement
     </a>
</dt>
<dd class="calibre18">
</dd>
</dl>
</div>
<div class="calibrenavbar1">
<hr class="calibre6"/>
<p class="calibre16">This article was downloaded by <strong class="calibre7">calibre</strong> from <a href="http://www.biomedcentral.com/1471-2164/14/298">http://www.biomedcentral.com/1471-2164/14/298</a></p>
<br class="calibre5"/><br class="calibre5"/> | <a href="../index.html#article_1">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | </div>

</body></html>