
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>
<<<<<<< HEAD
   Density-based hierarchical clustering of pyro-sequences on a large scale—the case of fungal ITS1
  </title>
<meta name="googlebot" content="NOODP" />
<meta name="HW.ad-path" content="/cgi/content/full/29/10/1268" />
<meta content="/bioinfo/29/10/1268.atom" name="HW.identifier" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta content="Density-based hierarchical clustering of pyro-sequences on a large scale—the case of fungal ITS1" name="DC.Title" />
<meta content="10.1093/bioinformatics/btt149" name="DC.Identifier" />
<meta content="2013-05-15" name="DC.Date" />
<meta content="Oxford University Press" name="DC.Publisher" />
<meta content="Marco Pagni" name="DC.Contributor" />
<meta content="Hélène Niculita-Hirzel" name="DC.Contributor" />
<meta content="Loïc Pellissier" name="DC.Contributor" />
<meta content="Anne Dubuis" name="DC.Contributor" />
<meta content="Ioannis Xenarios" name="DC.Contributor" />
<meta content="Antoine Guisan" name="DC.Contributor" />
<meta content="Ian R. Sanders" name="DC.Contributor" />
<meta content="Jérôme Goudet" name="DC.Contributor" />
<meta content="Nicolas Guex" name="DC.Contributor" />
=======
   Extra-binomial variation approach for analysis of pooled DNA sequencing data
  </title>
<meta name="googlebot" content="NOODP" />
<meta name="HW.ad-path" content="/cgi/content/full/28/22/2898" />
<meta content="/bioinfo/28/22/2898.atom" name="HW.identifier" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta content="Extra-binomial variation approach for analysis of pooled DNA sequencing data" name="DC.Title" />
<meta content="10.1093/bioinformatics/bts553" name="DC.Identifier" />
<meta content="2012-11-15" name="DC.Date" />
<meta content="Oxford University Press" name="DC.Publisher" />
<meta content="Xin Yang" name="DC.Contributor" />
<meta content="John A. Todd" name="DC.Contributor" />
<meta content="David Clayton" name="DC.Contributor" />
<meta content="Chris Wallace" name="DC.Contributor" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<meta content="Bioinformatics" name="citation_journal_title" />
<meta content="Bioinformatics" name="citation_journal_abbrev" />
<meta content="1367-4803" name="citation_issn" />
<meta content="1460-2059" name="citation_issn" />
<<<<<<< HEAD
<meta name="citation_author" content="Marco Pagni" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Hélène Niculita-Hirzel" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Loïc Pellissier" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Anne Dubuis" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Ioannis Xenarios" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Antoine Guisan" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Ian R. Sanders" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Jérôme Goudet" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Nicolas Guex" />
<meta name="citation_author_institution" content="1Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland, 2Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland and 3Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland" />
<meta content="Density-based hierarchical clustering of pyro-sequences on a large scale—the case of fungal ITS1" name="citation_title" />
<meta content="05/15/2013" name="citation_date" />
<meta content="29" name="citation_volume" />
<meta content="10" name="citation_issue" />
<meta content="1268" name="citation_firstpage" />
<meta content="1274" name="citation_lastpage" />
<meta content="29/10/1268" name="citation_id" />
<meta content="bioinfo;29/10/1268" name="citation_mjid" />
<meta content="10.1093/bioinformatics/btt149" name="citation_doi" />
<meta content="http://bioinformatics.oxfordjournals.org/content/29/10/1268.abstract" name="citation_abstract_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/29/10/1268.full" name="citation_fulltext_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/29/10/1268.full.pdf" name="citation_pdf_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/29/10/1268" name="citation_public_url" />
<meta content="23539304" name="citation_pmid" />
<meta name="citation_section" content="Original Paper" />
<meta name="robots" content="noarchive,nofollow" />
<meta name="googlebot" content="noarchive" />
<link href="/content/29/10/1260.short" rel="prev" />
<link href="/content/29/10/1275.short" rel="next" />
=======
<meta name="citation_author" content="Xin Yang" />
<meta name="citation_author_institution" content="Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke’s Hospital, Cambridge CB2 0XY, UK" />
<meta name="citation_author" content="John A. Todd" />
<meta name="citation_author_institution" content="Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke’s Hospital, Cambridge CB2 0XY, UK" />
<meta name="citation_author" content="David Clayton" />
<meta name="citation_author_institution" content="Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke’s Hospital, Cambridge CB2 0XY, UK" />
<meta name="citation_author" content="Chris Wallace" />
<meta name="citation_author_institution" content="Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke’s Hospital, Cambridge CB2 0XY, UK" />
<meta content="Extra-binomial variation approach for analysis of pooled DNA sequencing data" name="citation_title" />
<meta content="11/15/2012" name="citation_date" />
<meta content="28" name="citation_volume" />
<meta content="22" name="citation_issue" />
<meta content="2898" name="citation_firstpage" />
<meta content="2904" name="citation_lastpage" />
<meta content="28/22/2898" name="citation_id" />
<meta content="bioinfo;28/22/2898" name="citation_mjid" />
<meta content="10.1093/bioinformatics/bts553" name="citation_doi" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2898.abstract" name="citation_abstract_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2898.full" name="citation_fulltext_html_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2898.full.pdf" name="citation_pdf_url" />
<meta content="http://bioinformatics.oxfordjournals.org/content/28/22/2898" name="citation_public_url" />
<meta content="22976083" name="citation_pmid" />
<meta name="citation_section" content="Original Paper" />
<meta name="robots" content="noarchive,nofollow" />
<meta name="googlebot" content="noarchive" />
<link href="/content/28/22/2891.short" rel="prev" />
<link href="/content/28/22/2905.short" rel="next" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-global.css" />
<link rel="stylesheet" type="text/css" media="print" href="/shared/css/hw-print.css" />
<link rel="stylesheet" type="text/css" media="all" href="/local/css/hw-local-global.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-global.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-ac.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-page-content.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/jquery.fancybox-1.3.4.css" />
<link rel="stylesheet" type="text/css" media="all" href="/shared/css/hw-global-colexpand.css" />
<link rel="stylesheet" type="text/css" media="all" href="/publisher/css/hw-publisher-page-content.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/hw20.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-header.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-footer.css" />
<link rel="stylesheet" type="text/css" media="all" href="/resource/css/h20-oup-sidebars.css" />
<link type="text/css" media="all" rel="stylesheet" href="/site/resource/bioinformatics_h20.css" />
<link rel="stylesheet" type="text/css" href="/resource/css/print.css" media="print" />
<<<<<<< HEAD
<link rel="stylesheet" type="text/css" media="all" href="stylesheets/style15.css" />
<link rel="stylesheet" type="text/css" media="screen" href="stylesheets/style16.css" />
=======
<link rel="stylesheet" type="text/css" media="all" href="../article_1/stylesheets/style15.css" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<style type="text/css" title="override_css">
            .article_date {
                color: gray; font-family: monospace;
            }

            .article_description {
                text-indent: 0pt;
            }

            a.article {
                font-weight: bold; text-align:left;
            }

            a.feed {
                font-weight: bold;
            }

            .calibre_navbar {
                font-family:monospace;
            }
    

</style></head>
<body class="general_page journal bioinformatics"><div style="text-align:center" class="calibre_navbar calibre_rescale_70">| <a href="../article_1/index.html">Next</a> | <a href="../index.html#article_0">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | <hr />
<<<<<<< HEAD
</div>
<div class="hw-gen-page pagetype-content hw-pub-id-article" id="pageid-content" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle">
<p class="hide">
<a href="#content">
     Skip Navigation
    </a>
</p>
<div id="secondary_nav">
<strong id="page_logo" title="Oxford Journals">
<a href="http://www.oxfordjournals.org/">
<span>
       Oxford Journals
      </span>
</a>
</strong>
<ul>
<li id="nav_contact_us" title="Contact Us">
<a href="http://www.oxfordjournals.org/contact_us.html">
<span>
        Contact Us
       </span>
</a>
</li>
<li id="nav_my_basket" title="My Basket">
<a href="https://secure.oxfordjournals.org/basket.html">
<span>
        My Basket
       </span>
</a>
</li>
<li id="nav_my_account" title="My Account">
<a href="http://services.oxfordjournals.org/cgi/tslogin?url=http://www.oxfordjournals.org/service/Register">
<span>
        My 
                        
                        Account
       </span>
</a>
</li>
</ul>
</div>
<div id="header">
<h1 id="page_title" title="Bioinformatics">
<a href="/">
<span>
       Bioinformatics
      </span>
</a>
</h1>
</div>
<div id="primary_nav">
<ul>
<li id="nav_about_this_journal" title="About This Journal">
<a href="http://www.oxfordjournals.org/bioinformatics/about.html">
<span>
        About This Journal
       </span>
</a>
</li>
<li id="nav_contact_this_journal" title="Contact This Journal">
<a href="/cgi/feedback/">
<span>
        Contact This Journal
       </span>
</a>
</li>
<li id="nav_subscriptions" title="Subscriptions">
<a href="http://www.oxfordjournals.org/bioinformatics/access_purchase/buy_online.html">
<span>
        Subscriptions
       </span>
</a>
</li>
<li id="nav_current_issue" title="Current">
<a href="/content/current">
<span>
        View Current Issue (Volume 29 Issue 10 May 15, 2013)
       </span>
</a>
</li>
<li id="nav_archive" title="Archive">
<a href="/content/by/year">
<span>
        Archive
       </span>
</a>
</li>
<li id="nav_search" title="Search">
<a href="/search">
<span>
        Search
       </span>
</a>
</li>
</ul>
</div>
<div id="user_nav">
<div class="header-ac-elements">
<div id="authstring">
<ul>
<li class="subscr-ref">
        Institution:  Journals Dept
       </li>
<li class="no-left-border">
<a href="/login?uri=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcontent%2F29%2F10%2F1268.full">
         Sign In as Personal Subscriber
        </a>
</li>
</ul>
</div>
</div>
</div>
<ul id="site-breadcrumbs">
<li class="first">
<a href="http://services.oxfordjournals.org/cgi/tslogin?url=http%3A%2F%2Fwww.oxfordjournals.org">
      Oxford Journals
     </a>
</li>
<li>
<span class="breadcrumb_subjects">
<a href="http://www.oxfordjournals.org/subject/life_sciences/" class="breadcrumb_subject">
       Life Sciences
      </a>
</span>
<span class="breadcrumb_subjects">
<a href="http://www.oxfordjournals.org/subject/mathematics/" class="breadcrumb_subject">
       Mathematics & Physical Sciences
      </a>
</span>
</li>
<li>
<a href="/">
      Bioinformatics
     </a>
</li>
<li>
<a href="/content/29/10.toc">
<span xmlns="" class="volume-value">
       Volume 29
      </span>
<span xmlns="" class="issue-value">
       Issue 10
      </span>
</a>
</li>
<li>
     Pp.
     <span class="slug-pages">
      1268-1274.
     </span>
</li>
</ul>
<div id="oas_top" class="ad_hidden">
</div>
<a name="content">
</a>
<div id="h20_page">
</div>
<div id="content-block">
<div class="article fulltext-view " itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<h1 id="article-title-1" itemprop="headline">
      Density-based hierarchical clustering of pyro-sequences on a large scale—the case of fungal ITS1
=======
</div><div class="hw-gen-page pagetype-content hw-pub-id-article" id="pageid-content" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><div id="content-block"><div class="article fulltext-view" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<h1 id="article-title-1" itemprop="headline">
      Extra-binomial variation approach for analysis of pooled DNA sequencing data
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
     </h1>
<div class="contributors">
<ol class="contributor-list" id="contrib-group-1">
<li class="contributor" id="contrib-1" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<<<<<<< HEAD
<a class="name-search" href="/search?author1=Marco+Pagni&amp;sortspec=date&amp;submit=Submit">
          Marco Pagni
         </a>
</span>
<a id="xref-aff-1-1" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
=======
<a class="name-search" href="/search?author1=Xin+Yang&amp;sortspec=date&amp;submit=Submit">
          Xin Yang
         </a>
</span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
        ,
       </li>
<li class="contributor" id="contrib-2" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<<<<<<< HEAD
<a class="name-search" href="/search?author1=H%C3%A9l%C3%A8ne+Niculita-Hirzel&amp;sortspec=date&amp;submit=Submit">
          Hélène Niculita-Hirzel
         </a>
</span>
<a id="xref-aff-1-2" class="xref-aff" href="#aff-1">
<sup>
          2
         </sup>
</a>
=======
<a class="name-search" href="/search?author1=John+A.+Todd&amp;sortspec=date&amp;submit=Submit">
          John A. Todd
         </a>
</span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
        ,
       </li>
<li class="contributor" id="contrib-3" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<<<<<<< HEAD
<a class="name-search" href="/search?author1=Lo%C3%AFc+Pellissier&amp;sortspec=date&amp;submit=Submit">
          Loïc Pellissier
         </a>
</span>
<a id="xref-aff-1-3" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-4" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Anne+Dubuis&amp;sortspec=date&amp;submit=Submit">
          Anne Dubuis
         </a>
</span>
<a id="xref-aff-1-4" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-5" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Ioannis+Xenarios&amp;sortspec=date&amp;submit=Submit">
          Ioannis Xenarios
         </a>
</span>
<a id="xref-aff-1-5" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-6" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Antoine+Guisan&amp;sortspec=date&amp;submit=Submit">
          Antoine Guisan
         </a>
</span>
<a id="xref-aff-1-6" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-7" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=Ian+R.+Sanders&amp;sortspec=date&amp;submit=Submit">
          Ian R. Sanders
         </a>
</span>
<a id="xref-aff-1-7" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        ,
       </li>
<li class="contributor" id="contrib-8" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
<span class="name" itemprop="name">
<a class="name-search" href="/search?author1=J%C3%A9r%C3%B4me+Goudet&amp;sortspec=date&amp;submit=Submit">
          Jérôme Goudet
         </a>
</span>
<a id="xref-aff-1-8" class="xref-aff" href="#aff-1">
<sup>
          3
         </sup>
</a>
        and
       </li>
<li class="last" id="contrib-9">
<span class="name">
<a class="name-search" href="/search?author1=Nicolas+Guex&amp;sortspec=date&amp;submit=Submit">
          Nicolas Guex
         </a>
</span>
<a id="xref-aff-1-9" class="xref-aff" href="#aff-1">
<sup>
          1
         </sup>
</a>
<span class="xref-sep">
         ,
        </span>
=======
<a class="name-search" href="/search?author1=David+Clayton&amp;sortspec=date&amp;submit=Submit">
          David Clayton
         </a>
</span>
        and
       </li>
<li class="last" id="contrib-4">
<span class="name">
<a class="name-search" href="/search?author1=Chris+Wallace&amp;sortspec=date&amp;submit=Submit">
          Chris Wallace
         </a>
</span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<a id="xref-corresp-1-1" class="xref-corresp" href="#corresp-1">
         *
        </a>
</li>
</ol>
<ol class="affiliation-list">
<li class="aff">
<a id="aff-1" name="aff-1">
</a>
<address>
<<<<<<< HEAD
<sup>
          1
         </sup>
         Vital-IT Group, SIB Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland,
         <sup>
          2
         </sup>
         Occupational Environment Department, Institute for Work and Health, Universities of Lausanne and Geneva, 1066 Lausanne, Switzerland
                           and
         <sup>
          3
         </sup>
         Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne, Switzerland
=======
         Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics,
                           Cambridge Institute for Medical Research, University of Cambridge, Wellcome Trust/MRC Building, Addenbrooke’s Hospital, Cambridge
                           CB2 0XY, UK
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
        </address>
</li>
</ol>
<ol class="corresp-list">
<li class="corresp" id="corresp-1">
<a class="rev-xref" href="#xref-corresp-1-1">
         ↵
        </a>
        *To whom correspondence should be addressed.
       </li>
</ol>
<ul class="history-list">
<<<<<<< HEAD
<li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-30">
<span class="received-label">
         Received
        </span>
        November 30, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="rev-recd" hwp:start="2013-03-25">
<span class="rev-recd-label">
         Revision received
        </span>
        March 25, 2013.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-03-26">
<span class="accepted-label">
         Accepted
        </span>
        March 26, 2013.
=======
<li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-11-16">
<span class="received-label">
         Received
        </span>
        November 16, 2011.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="rev-recd" hwp:start="2012-08-03">
<span class="rev-recd-label">
         Revision received
        </span>
        August 3, 2012.
       </li>
<li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-09-05">
<span class="accepted-label">
         Accepted
        </span>
        September 5, 2012.
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
       </li>
</ul>
</div>
<div class="section abstract" id="abstract-1" itemprop="description">
<div class="section-nav">
<div class="nav-placeholder">
<<<<<<< HEAD
         
=======
        �
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
       </div>
<a href="#sec-1" title="1 INTRODUCTION" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Abstract
      </h2>
<p id="p-3">
<strong>
<<<<<<< HEAD
        Motivation:
       </strong>
       Analysis of millions of pyro-sequences is currently playing a crucial role in the advance of environmental microbiology.
                     Taxonomy-independent, i.e. unsupervised, clustering of these sequences is essential for the definition of Operational Taxonomic
                     Units. For this application, reproducibility and robustness should be the most sought after qualities, but have thus far largely
                     been overlooked.
      </p>
<p id="p-4">
<strong>
        Results:
       </strong>
       More than 1 million hyper-variable internal transcribed spacer 1 (ITS1) sequences of fungal origin have been analyzed. The
                     ITS1 sequences were first properly extracted from 454 reads using generalized profiles. Then, otupipe, cd-hit-454, ESPRIT-Tree
                     and DBC454, a new algorithm presented here, were used to analyze the sequences. A numerical assay was developed to measure
                     the reproducibility and robustness of these algorithms. DBC454 was the most robust, closely followed by ESPRIT-Tree. DBC454
                     features density-based hierarchical clustering, which complements the other methods by providing insights into the structure
                     of the data.
      </p>
<p id="p-5">
<strong>
        Availability:
       </strong>
       An executable is freely available for non-commercial users at
       <a href="ftp://ftp.vital-it.ch/tools/dbc454">
        ftp://ftp.vital-it.ch/tools/dbc454
       </a>
       . It is designed to run under MPI on a cluster of 64-bit Linux machines running Red Hat 4.x, or on a multi-core OSX system.
      </p>
<p id="p-6">
<strong>
        Contact:
       </strong>
<span class="em-link">
<span class="em-addr">
         dbc454{at}vital-it.ch
        </span>
</span>
       or
       <span class="em-link">
<span class="em-addr">
         nicolas.guex{at}isb-sib.ch
        </span>
</span>
</p>
=======
        Motivation
       </strong>
       : The invention of next-generation sequencing technology has made it possible to study the rare variants that are more likely
                     to pinpoint causal disease genes. To make such experiments financially viable, DNA samples from several subjects are often
                     pooled before sequencing. This induces large between-pool variation which, together with other sources of experimental error,
                     creates over-dispersed data. Statistical analysis of pooled sequencing data needs to appropriately model this additional variance
                     to avoid inflating the false-positive rate.
      </p>
<p id="p-4">
<strong>
        Results
       </strong>
       : We propose a new statistical method based on an extra-binomial model to address the over-dispersion and apply it to pooled
                     case-control data. We demonstrate that our model provides a better fit to the data than either a standard binomial model or
                     a traditional extra-binomial model proposed by Williams and can analyse both rare and common variants with lower or more variable
                     pool depths compared to the other methods.
      </p>
<p id="p-5">
<strong>
        Availability
       </strong>
       : Package ‘extraBinomial’ is on
       <a href="http://cran.r-project.org/">
        http://cran.r-project.org/
       </a>
</p>
<p id="p-6">
<strong>
        Contact
       </strong>
       :
       <span class="em-link">
<span class="em-addr">
         chris.wallace{at}cimr.cam.ac.uk
        </span>
</span>
</p>
<p id="p-7">
<strong>
        Supplementary information:
       </strong>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
        Supplementary data
       </a>
       are available at
       <em>
        Bioinformatics
       </em>
       Online.
      </p>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
<div class="section intro" id="sec-1">
<div class="section-nav">
<a href="#abstract-1" title="Abstract" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#sec-2" title="2 METHODS" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       1 INTRODUCTION
      </h2>
<<<<<<< HEAD
<p id="p-7">
       Environmental microbiology has advanced greatly in the past few years with the advent of next-generation sequencing technologies.
                     A microbial community can now be sampled by the exhaustive sequencing of the PCR products obtained from a carefully chosen
                     pair of primers. Currently, the Roche 454 pyro-sequencing is the favored technology because it produces relatively long and
                     sufficiently numerous reads at an acceptable cost. However,
       <em>
        pyro-sequences
       </em>
       suffer from a non-negligible rate of error (
       <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
        Huse
        <em>
         et al.
        </em>
        , 2007
       </a>
       ), which often affects the sequence regions in which a given nucleotide is repeated several times.
      </p>
<p id="p-8">
       In this study, we have used a dataset of the fungal internal transcribed spacer 1 (ITS1) coming from soil samples obtained
                     from 198 different locations in the western Swiss Alps. Each location represented a 4 m
       <sup>
        2
       </sup>
       of grassland. The locations were selected by following a random-stratified design and, thus, are distributed over a great
                     variety of soil types and altitudes. Overall, more than a million pyro-sequences have been obtained, with the aim of identifying
                     how biotic and abiotic factors influence the fungal diversity at different scales. Ecological results of this study will be
                     reported elsewhere, the focus of this article being to introduce a novel classification method capable of robustly analyzing
                     this large-scale dataset.
      </p>
<p id="p-9">
       There are currently three taxonomy-independent, i.e. unsupervised, clustering algorithms that can determine Operational Taxonomic
                     Units (OTUs) on a large scale, say in the range of a million of pyro-sequences. These algorithms are cd-hit-454 (
       <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
        Niu
        <em>
         et al.
        </em>
        , 2010
       </a>
       ), otupipe (
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
        Edgar, 2010
       </a>
       ;
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
        Edgar
=======
<p id="p-8">
       To date, numerous common genetic variants associated with common disease [e.g. Type 1 diabetes (T1D)] have been successfully
                     discovered by genome-wide association studies (
       <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
        Barrett
        <em>
         et al.
        </em>
        , 2009
       </a>
       ;
       <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
        Cooper
        <em>
         et al.
        </em>
        , 2008
       </a>
       ;
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
        Smyth
        <em>
         et al.
        </em>
        , 2006
       </a>
       ;
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
        Wellcome Trust Case Control Consortium, 2007
       </a>
       ). However, linkage disequilibrium (LD) means these results may only be used to identify an associated region, which usually
                     encompasses several genes. To pinpoint the exact causal genes, under the assumption that there may be multiple variants of
                     common and low frequency which alter disease risk, attention has turned to low-frequency variants that are unlikely to be
                     in LD with other variants. Associated low-frequency variants are considered more likely to be causal. For example, several
                     independent rare risk variants with implied functional roles were recently associated with inflammatory bowel disease using
                     next-generation sequencing of pooled samples (
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
        Rivas
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
        <em>
         et al.
        </em>
        , 2011
       </a>
<<<<<<< HEAD
       ) and ESPRIT-Tree (
       <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
        Cai and Sun, 2011
       </a>
       ). They have been recently reviewed and numerically benchmarked (
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
        Sun
        <em>
         et al.
        </em>
        , 2012
       </a>
       ) in an article that constitutes an excellent introduction to the field of taxonomy-independent microbial community analysis
                     and that summarizes what is the state of art in the field.
      </p>
<p id="p-10">
       Here, we introduce DBC454, a density-based hierarchical clustering algorithm that departs on several points from the previously
                     mentioned ones. It was designed with the following requirements in mind: it should (i) determine the optimal number of clusters,
                     (ii) properly separate arbitrarily shaped distributions, (iii) not be affected by the order in which the data are presented,
                     (iv) be robust and resistant to noise (e.g. not assign outliers to clusters), (v) use intuitive-free parameters, (vi) be capable
                     of clustering millions of observations in thousands of clusters and (vii) return a solution in a reasonable amount of time.
      </p>
<p id="p-11">
       We also introduce a new numerical benchmark for unsupervised clustering algorithms, which measures their robustness with respect
                     to the errors that are likely to affect the input sequences.
      </p>
</div>
<div class="section materials-methods" id="sec-2">
=======
       ), a design similar to that of
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
        Nejentsev
        <em>
         et al.
        </em>
        (2009)
       </a>
       who undertook the first such study and whose data we re-examine here.
      </p>
<p id="p-9">
<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
        Nejentsev
        <em>
         et al.
        </em>
        (2009)
       </a>
       re-sequenced 144 target regions covering exons and splice sites of 10 T1D candidate genes that were previously found to be
                     associated with T1D or related diseases. The study used 454 sequencing of 20 pools of DNA from 480 patients and 480 healthy
                     controls (48 samples per pool). Four rare variants, single-nucleotide polymorphisms (SNPs) located in the
       <em>
        interferon induced with helicase C domain 1
       </em>
       (
       <em>
        IFIH1
       </em>
       ) and the
       <em>
        C-type lectin domain family 16, member A
       </em>
       (
       <em>
        CLEC16A
       </em>
       ) genes, displayed different frequencies in cases and controls according to Fisher’s exact test, and evidence for association
                     of two of these variants within the gene
       <em>
        IFIH1
       </em>
       was replicated in independent samples. This directly implicates
       <em>
        IFIH1
       </em>
       as causal in T1D.
      </p>
<p id="p-10">
       A major concern with this study is that huge variation exists between pool depths and the variation is especially large for
                     rare SNPs (
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">
        Fig. 1
       </a>
       ). The variation is derived from both the pooling and the sequencing, and this is not accounted for by Fisher’s exact test
                     which assumes a binomial variance within pools and neglects between-pool variation.
      </p>
<div id="F1" class="fig pos-float odd">
<div class="fig-inline">
<a href="2898/F1.expansion.html">
<img alt="Fig. 1." src="images/img1.jpg" width="600" />
</a>
<div class="callout">
<span>
          View larger version:
         </span>
<ul class="callout-links">
<li>
<a href="2898/F1.expansion.html">
            In this window
           </a>
</li>
<li>
<a class="in-nw" href="2898/F1.expansion.html">
            In a new window
           </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/28/22/2898/F1">
            Download as PowerPoint Slide
           </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
         Fig. 1.
        </span>
<p id="p-11" class="first-child">
         Variance analysis of the SNPs detected in Nejentsev
         <em>
          et al.
         </em>
         ’s article. (
         <strong>
          a
         </strong>
         ) Variance–mean depth comparison. Variance of depth increases with mean depths. (
         <strong>
          b
         </strong>
         ) Variance of depth can be extreme, particularly for low minor allele frequency
        </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-12">
       The Poisson model has also been employed for sequence count data as well as analysis of differential gene expression (
       <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
        Marioni
        <em>
         et al.
        </em>
        , 2008
       </a>
       ). With only one parameter, the Poisson distribution assumes the variance of reads is equal to its mean. However, the variance
                     may exceed the mean, requiring an over-dispersed model as in our data. A negative binomial model has been used as an alternative
                     to model the larger variance, for example in testing differential expression for digital gene expression data (
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
        Robinson and Smyth, 2007
       </a>
       ), given it has one more parameter than the Poisson distribution. The addition of this parameter, however, means the negative
                     binomial model requires a large number of replicates to estimate the two parameters properly.
      </p>
<p id="p-13">
       Another fully mathematically specified over-dispersed model is the beta-binomial model, which, applied to variants derived
                     from pooled case-control sequence data, would assume a binomial distribution within pools and that the expected allele frequency
                     of each pool is beta-distributed.
       <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
        Crowder (1978)
       </a>
       used this model and calculated maximized log-likelihoods to analyse the effects from seeds and extracts on germination. Other
                     applications of this model include the analysis of false discovery rates (FDRs) in microarray data to model the number of
                     false rejections, yielding a less biased FDR estimator than the empirical estimator following a binomial distribution (
       <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
        Hunt
        <em>
         et al.
        </em>
        , 2009
       </a>
       ). The advantage of this approach is obvious: meaningful parameters and a computable likelihood which could be maximized by
                     standard methods to calculate the relating parameters. However, maximizing a beta-binomial likelihood can be computationally
                     difficult.
      </p>
<p id="p-14">
       An alternative is to use a model in which only the first two moments are defined. We propose to use and extend such an extra-binomial
                     model originally described by
       <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
        Williams (1982)
       </a>
       . The extra-binomial model still assumes the allele counts obey a binomial distribution within pools and also takes the between-pool
                     variance into consideration. In this article, we compare the performance of statistical tests assuming simple binomial variation,
                     Williams’ extra-binomial variation and our modified extra-binomial model using quantile–quantile (Q–Q) plots. We also extend
                     our study beyond the original minor allele frequency (MAF) range used by Nejentsev
       <em>
        et al.
       </em>
       (1–3%) to include all observed single-nucleotide variation.
      </p>
</div>
<div class="section methods" id="sec-2">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="section-nav">
<a href="#sec-1" title="1 INTRODUCTION" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<<<<<<< HEAD
<a href="#sec-8" title="3 RESULTS" class="next-section-link">
=======
<a href="#sec-10" title="3 RESULTS" class="next-section-link">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       2 METHODS
      </h2>
<div id="sec-3" class="subsection">
<h3>
<<<<<<< HEAD
        2.1 Source material
       </h3>
<p id="p-12">
        Fungal ITS1 sequences were obtained from 198 soil samples gathered from 4 m
        <sup>
         2
        </sup>
        plots in grasslands along wide soil and altitudinal gradients in the western Swiss Alps (
        <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
         Dubuis
         <em>
          et al.
         </em>
         , 2012
        </a>
        ). From each plot, five subsamples of 250 mg soil samples were taken, from which DNA was extracted and used for fungi barcoding
                        using the Roche Genome Sequencer FLX Titanium 454 System. The technical details of this experiment will be described in another
                        paper. In total, 1 649 376 reads were generated, of which 1 152 121 were full-length ITS1 sequences (see below).
       </p>
</div>
<div id="sec-4" class="subsection">
<h3>
        2.2 Extraction of full-length fungal ITS1
       </h3>
<p id="p-13">
        Two generalized profiles of a length of ∼50 residues were built with the PFTOOLS v2.3 package (
        <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
         Luthy
         <em>
          et al.
         </em>
         , 1994
        </a>
        ;
        <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
         Sigrist
         <em>
          et al.
         </em>
         , 2002
        </a>
        ) (
        <a href="http://web.expasy.org/pftools">
         http://web.expasy.org/pftools
        </a>
        ) to recognize the highly conserved regions that flank the ITS1 regions, located at the end of the 18S rRNA and at the beginning
                        of the 5.8S rRNA, respectively (
        <a id="xref-fig-1-1" class="xref-fig" href="#F1">
         Fig. 1
        </a>
        ). These profiles were manually edited to produce global matches on the profile side and local matches on the sequence side.
                        In addition, because of the early termination of many pyro-sequences in the 18S region, the profile scoring system at the
                        corresponding extremity (left of profile 1 on
        <a id="xref-fig-1-2" class="xref-fig" href="#F1">
         Fig. 1
        </a>
        ) was modified to permit the termination of the alignment at the end of the sequence while within the profile. As far as we
                        know, the generalized profile as implemented in the PFTOOLS v2.3 package remains the only sequence alignment method that permits
                        such a degree of detail in the design of an alignment scoring system for DNA sequences. The two profiles were calibrated on
                        shuffled 454 reads to determine the most appropriate score thresholds (
        <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
         Pagni and Jongeneel, 2001
        </a>
        ). The pyro-sequences were searched independently with both profiles, and when both were matching on a sequence, the ITS1
                        was defined as the intervening sequence between them. With this strategy, 1 152 121 full-length ITS1 were recovered, among
                        which 341 585 were unique, out of the 1 649 376 raw pyro-sequences.
       </p>
<div id="F1" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F1.expansion.html">
<img alt="Fig. 1." src="1268/F1.small.gif" />
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<a href="1268/F1.expansion.html">
             In this window
            </a>
</li>
<li>
<a class="in-nw" href="1268/F1.expansion.html">
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/29/10/1268/F1">
             Download as PowerPoint Slide
            </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
          Fig. 1.
         </span>
<p id="p-14" class="first-child">
          Extraction of ITS1 full-length sequences from the raw 454 reads. Two generalized profiles were built in the two highly conserved
                              regions flanking the ITS1
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-15">
        The same profile-based search strategy was applied on the FUN, PLN, ENV and INV sections of the release 109 of the EMBL: both
                        strands were searched and 127 577 intervening sequences with a length from 1 to 500 nt were retained as ITS1 to constitute
                        a reference dataset. The biological conclusions drawn with the latter sequences will be discussed in the companion paper.
       </p>
<p id="p-16">
        The only drawback of this profile-based extraction is that it rejects the ITS1 that do not have enough of the 18S region sequenced,
                        i.e. <20 nt. Hence, the longer ITS1 are likely to be under-sampled, given the observed distribution of the lengths of the
                        raw 454 reads (not shown). In contrast, this eliminates the difficulties linked to the analysis of incomplete sequences, which
                        is especially relevant here, as one will only use algorithms that rely explicitly or implicitly on
        <em>
         global sequence alignments
        </em>
        .
       </p>
</div>
<div id="sec-5" class="subsection">
<h3>
        2.3 Computer programs and parameters
       </h3>
<p id="p-17">
        The DBC454 algorithm is presented in Section 3. DBC454 was run using default parameter values (23 steps, using starting and
                        ending distances of 3.317 and 7.141, respectively). We also tested five alternate starting distance levels (3.464, 3.606,
                        3.873, 4.000 and 4.243). The resulting partitions of the data are named dbc-s0 for the default parameters and dbc-s1 to dbc-s6
                        for the alternate starting levels.
       </p>
<p id="p-18">
        cd-hit-454 v4.5.4 was run using default parameters, but different values were tested for the sequence identity threshold:
                        98, 97, 95 and 90%. The resulting partitions of the input data are designated as cdh-98, cdh-97, cdh-95 and cdh-90, respectively.
       </p>
<p id="p-19">
        otupipe v9, with usearch v5.1.221, was run using default parameters, and with different values (98, 97, 95 and 90%) for PCTID_ERR,
                        PCTID_OTU and PCTID_BIN parameters. The resulting partitions of the input data are designated as otu-98, otu-97, otu-95 and
                        otu-90, respectively.
       </p>
<p id="p-20">
        ESPRIT-Tree 11152011 was obtained from the author Yjun Sun and run using default parameters, which produces a hierarchical
                        clustering with 15 levels. By cutting this hierarchy at different heights, one can obtain 15 different partitions of the data.
                        These are named etr-00 to etr-14 (from lower to higher levels).
       </p>
<p id="p-21">
        The sequences of every cluster were aligned using mafft (
        <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
         Katoh and Standley, 2013
        </a>
        ), and the cluster ‘radius’ was computed as the average of the percent divergences of every sequence pairs in the cluster.
       </p>
<p id="p-22">
        Perl and R scripts were extensively and creatively used throughout this study to re-format, glue together and analyze all
                        the data.
       </p>
</div>
<div id="sec-6" class="subsection">
<h3>
        2.4 Robustness benchmark
       </h3>
<p id="p-23">
        Artificially mutated versions of the ITS1 dataset were created using the msbar program from the EMBOSS suite (
        <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
         Rice
         <em>
          et al.
         </em>
         , 2000
        </a>
        ). To simulate the errors that occur during the 454 sequencing, we followed the quantitative analysis of
        <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
         Huse
         <em>
          et al.
         </em>
         (2007)
        </a>
        . Briefly, 18% of the original sequences were randomly selected, and among them, 61% were given a single error, 17% two errors
                        and 22% three errors. The types of errors introduced were insertions, deletions or substitutions in 46, 33 and 21% of the
                        cases, respectively. We repeated this procedure five times, to obtain five distinct versions of the mutated dataset.
       </p>
<p id="p-24">
        Two different partitions of the same dataset (e.g. from two different algorithms), or the partitions of two related datasets
                        (e.g. original versus mutated with the same algorithm), were compared by computing the adjusted Rand index (ARI;
        <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
         Hubert and Arabie, 1985
        </a>
        ) with the R package ‘clues’.
       </p>
</div>
<div id="sec-7" class="subsection">
<h3>
        2.5 Accuracy benchmark
       </h3>
<p id="p-25">
        For every algorithm, the 1 152 121 ITS1 were clustered together with the 127 577 reference ITS1 previously extracted from
                        the EMBL. The taxa of EMBL entries at the species, genus and family ranks were assigned whenever possible using the NCBI taxonomy.
                        Reference sequences corresponding to environmental samples and uncultured organisms were clustered, but not used to assess
                        the accuracy.
       </p>
</div>
</div>
<div class="section results" id="sec-8">
=======
        2.1 Experimental data pre-processing
       </h3>
<p id="p-15">
        The experimental data used for the following analysis were drawn from the article by
        <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
         Nejentsev
         <em>
          et al.
         </em>
         (2009)
        </a>
        . The sequencing data were processed using existing software or R (
        <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
         R Development Core Team, 2010
        </a>
        ) as described below. BWA-SW (
        <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
         Li and Durbin, 2010
        </a>
        ) was used to align query 454 sequences to the NCBI36 human assembly, yielding sam files which were processed by SAMtools
                        (
        <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
         Li
         <em>
          et al.
         </em>
         , 2009
        </a>
        ) to remove duplicate reads. SNPs were identified and counted by VarScan (
        <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
         Koboldt
         <em>
          et al.
         </em>
         , 2009
        </a>
        ), filtered by minimum base quality (≥25) and minimum total read depth (≥8).
       </p>
<p id="p-16">
        In the absence of a variant, errors should randomly distribute across multiple pools while for a genuine rare variant excess
                        reads of the alternative allele are expected to concentrate in one or several pools. Thus, to distinguish the errors from
                        the true SNPs, we first excluded the common variants with MAF
        <span class="inline-formula" id="inline-formula-1">
<img class="inline-graphic" alt="Graphic" src="images/img2.jpg" />
</span>
        and then compared the observed pool read distribution for each remaining rare SNP to that expected under a null hypothesis
                        of random errors only (
        <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
         Bansal, 2010
        </a>
        ). We assumed variants to be true rare SNPs if (i) they were not consistent with a random error model with a significant
        <em>
         P
        </em>
        -value less than
        <span class="inline-formula" id="inline-formula-2">
<img class="inline-graphic" alt="Graphic" src="images/img3.jpg" />
</span>
        (Bonferroni correction with a prior significance level equal to 0.05) and (ii) there was at least one pool with more than
                        two reads for the minor alleles.
       </p>
<p id="p-17">
        For the binomial model only, SNPs and pools were further filtered by chromosome coverage—only pools with more than 80% probability
                        to cover (include at least one read from) at least 80 out of 96 chromosomes were used. This reduced the variation between
                        different pools by dropping pools with poor PCR or sequencing quality. SNPs were dropped if there were no valid pools for
                        one group (case or control) at a position.
       </p>
</div>
<div id="sec-4" class="subsection">
<h3>
        2.2 Statistical models
       </h3>
<p id="p-18">
        Our aim is to identify the SNPs that are associated with T1D status. We define an observation as a success if it corresponds
                        to the major allele, otherwise it is a failure. Let
        <span class="inline-formula" id="inline-formula-3">
<img class="inline-graphic" alt="Graphic" src="images/img4.jpg" />
</span>
        and
        <span class="inline-formula" id="inline-formula-4">
<img class="inline-graphic" alt="Graphic" src="images/img5.jpg" />
</span>
        denote the expected major allele frequency for SNP
        <em>
         i
        </em>
        in control and case chromosomes, respectively. Then our hypothesis is written as
        <span class="disp-formula" id="disp-formula-1">
<img class="graphic" alt="Formula" src="images/img6.jpg" />
<span class="disp-formula-label">
          (1)
         </span>
</span>
<span class="disp-formula" id="disp-formula-2">
<img class="graphic" alt="Formula" src="images/img7.jpg" />
<span class="disp-formula-label">
          (2)
         </span>
</span>
        In the following models, we assume there is no systematic error in the sequencing.
       </p>
<div id="sec-5" class="subsection">
<h4>
         2.2.1 Binomial variation
        </h4>
<p id="p-19">
         For significance testing,
         <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">
          Nejentsev
          <em>
           et al.
          </em>
          (2009)
         </a>
         assumed a simple binomial model within pools, estimated overall allele frequencies by treating each pool equally and used
                           these estimated allele frequencies to estimate chromosome counts of each allele. Allele counts were compared in case and control
                           groups in a
         <span class="inline-formula" id="inline-formula-5">
<img class="inline-graphic" alt="Graphic" src="images/img8.jpg" />
</span>
         table at each SNP by Fisher’s exact test (see Section A of the
         <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
          Supplementary Material
         </a>
         ).
        </p>
</div>
<div id="sec-6" class="subsection">
<h4>
         2.2.2 Williams’ extra-binomial variation model (EB1)
        </h4>
<p id="p-20">
         We can relax the assumptions above by considering an over-dispersed binomial model. Here, we applied an extra-binomial model
                           proposed by
         <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
          Williams (1982)
         </a>
         to our data. With the assumption that the variation within each pool is binomial unchanged, we introduce a continuous variable
         <span class="inline-formula" id="inline-formula-6">
<img class="inline-graphic" alt="Graphic" src="images/img9.jpg" />
</span>
         as the allele frequency of SNP
         <em>
          i
         </em>
         in pool
         <em>
          j
         </em>
         to reflect the discrepancy between different pools in allele frequency.
         <span class="inline-formula" id="inline-formula-7">
<img class="inline-graphic" alt="Graphic" src="images/img10.jpg" />
</span>
         is independently distributed on (0,1) and its first two moments are defined as
         <span class="disp-formula" id="disp-formula-3">
<img class="graphic" alt="Formula" src="images/img11.jpg" />
<span class="disp-formula-label">
           (3)
          </span>
</span>
<span class="disp-formula" id="disp-formula-4">
<img class="graphic" alt="Formula" src="images/img12.jpg" />
<span class="disp-formula-label">
           (4)
          </span>
</span>
</p>
<p id="p-21">
         As shown below, the variance is over-dispersed when
         <span class="inline-formula" id="inline-formula-8">
<img class="inline-graphic" alt="Graphic" src="images/img13.jpg" />
</span>
         . Conditionally, in pool
         <em>
          j
         </em>
         when
         <span class="inline-formula" id="inline-formula-9">
<img class="inline-graphic" alt="Graphic" src="images/img14.jpg" />
</span>
         , the number of reads of the major allele
         <span class="inline-formula" id="inline-formula-10">
<img class="inline-graphic" alt="Graphic" src="images/img15.jpg" />
</span>
         follow Bin
         <span class="inline-formula" id="inline-formula-11">
<img class="inline-graphic" alt="Graphic" src="images/img16.jpg" />
</span>
         where
         <span class="inline-formula" id="inline-formula-12">
<img class="inline-graphic" alt="Graphic" src="images/img17.jpg" />
</span>
         is the depth of the
         <em>
          j
         </em>
         -th pool at the
         <em>
          i
         </em>
         -th SNP and
         <span class="inline-formula" id="inline-formula-13">
<img class="inline-graphic" alt="Graphic" src="images/img18.jpg" />
</span>
         is estimated by
         <span class="inline-formula" id="inline-formula-14">
<img class="inline-graphic" alt="Graphic" src="images/img19.jpg" />
</span>
         . Unconditionally, deriving from
         <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
          Equation (4)
         </a>
         , the variance of
         <span class="inline-formula" id="inline-formula-15">
<img class="inline-graphic" alt="Graphic" src="images/img20.jpg" />
</span>
         can be written as
         <span class="disp-formula" id="disp-formula-5">
</span>
<span class="disp-formula" id="disp-formula-6">
<img class="graphic" alt="Formula" src="images/img21.jpg" />
<span class="disp-formula-label">
           (5)
          </span>
</span>
         Thus, the extra-binomial model allows for between-pool variation by scaling the variance by a heterogeneity factor,
         <span class="inline-formula" id="inline-formula-16">
<img class="inline-graphic" alt="Graphic" src="images/img22.jpg" />
</span>
         , which increases with read depth. We could choose a specific distribution for
         <span class="inline-formula" id="inline-formula-17">
<img class="inline-graphic" alt="Graphic" src="images/img23.jpg" />
</span>
         such as the beta-distribution which could induce
         <span class="inline-formula" id="inline-formula-18">
<img class="inline-graphic" alt="Graphic" src="images/img24.jpg" />
</span>
         to follow a beta-binomial distribution (
         <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
          Zhou
          <em>
           et al.
          </em>
          , 2011
         </a>
         ). Instead, we adopt a quasi-likelihood approach and follow Williams’ method to yield an estimator of allele frequency,
         <span class="inline-formula" id="inline-formula-19">
<img class="inline-graphic" alt="Graphic" src="images/img25.jpg" />
</span>
         , by first fitting it into a logistic linear model and then maximizing the quasi-likelihood by weighted least square iteration
                           (see Section B of the
         <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
          Supplementary Material
         </a>
         ). The weighted sum of squares of residuals yields the goodness of fit statistic
         <span class="disp-formula" id="disp-formula-7">
<img class="graphic" alt="Formula" src="images/img26.jpg" />
<span class="disp-formula-label">
           (6)
          </span>
</span>
         where the weight
         <span class="inline-formula" id="inline-formula-20">
<img class="inline-graphic" alt="Graphic" src="images/img27.jpg" />
</span>
         . To test the significance of
         <span class="inline-formula" id="inline-formula-21">
<img class="inline-graphic" alt="Graphic" src="images/img28.jpg" />
</span>
         , we tested the increase in
         <span class="inline-formula" id="inline-formula-22">
<img class="inline-graphic" alt="Graphic" src="images/img29.jpg" />
</span>
         when deleting the case-control factor as suggested by Williams. This is distributed as a
         <span class="inline-formula" id="inline-formula-23">
<img class="inline-graphic" alt="Graphic" src="images/img30.jpg" />
</span>
         variable with one degree of freedom under the null hypothesis of no association.
        </p>
</div>
<div id="sec-7" class="subsection">
<h4>
         2.2.3 Modified extra-binomial model (EB2)
        </h4>
<p id="p-22">
         In Williams’ extra-binomial model, the parameter
         <span class="inline-formula" id="inline-formula-24">
<img class="inline-graphic" alt="Graphic" src="images/img31.jpg" />
</span>
         is specific to each SNP, which makes it difficult to estimate
         <span class="inline-formula" id="inline-formula-25">
<img class="inline-graphic" alt="Graphic" src="images/img32.jpg" />
</span>
         accurately given only 20 data points for each SNP. In contrast, we have many SNPs. We adopted two universal parameters
         <em>
          a
         </em>
         and
         <em>
          b
         </em>
         here instead of
         <span class="inline-formula" id="inline-formula-26">
<img class="inline-graphic" alt="Graphic" src="images/img33.jpg" />
</span>
         to scale the variance, such that
         <span class="disp-formula" id="disp-formula-8">
<img class="graphic" alt="Formula" src="images/img34.jpg" />
<span class="disp-formula-label">
           (7)
          </span>
</span>
         where
         <em>
          s
         </em>
         is the number of distinct chromosomes sampled in one pool, namely, 96 in this study. Ideally with no other over-dispersion,
                           as
         <span class="inline-formula" id="inline-formula-27">
<img class="inline-graphic" alt="Graphic" src="images/img35.jpg" />
</span>
         . However, sequencing errors and additional variation described earlier mean
         <span class="inline-formula" id="inline-formula-28">
<img class="inline-graphic" alt="Graphic" src="images/img36.jpg" />
</span>
         . We write
         <span class="disp-formula" id="disp-formula-9">
<img class="graphic" alt="Formula" src="images/img37.jpg" />
<span class="disp-formula-label">
           (8)
          </span>
</span>
         and by comparing
         <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
          Equation (7)
         </a>
         with a standard binomial variance function, we see that
         <span class="inline-formula" id="inline-formula-29">
<img class="inline-graphic" alt="Graphic" src="images/img38.jpg" />
</span>
         may be interpreted as the adjusted depth of pool
         <em>
          j
         </em>
         for SNP
         <em>
          i
         </em>
         .
        </p>
<p id="p-23">
         Considering the quantities of this model,
         <span class="disp-formula" id="disp-formula-10">
<img class="graphic" alt="Formula" src="images/img39.jpg" />
<span class="disp-formula-label">
           (9)
          </span>
</span>
         where
         <em>
          n
         </em>
         is the number of pools, namely, 20 in this study, should have expectation [see
         <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
          Equation (5)
         </a>
         in the
         <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
          Supplementary Material
         </a>
         ]:
         <span class="disp-formula" id="disp-formula-11">
<img class="graphic" alt="Formula" src="images/img40.jpg" />
<span class="disp-formula-label">
           (10)
          </span>
</span>
         We therefore estimate the parameters
         <em>
          a
         </em>
         and
         <em>
          b
         </em>
         by linear regression of
         <span class="inline-formula" id="inline-formula-30">
<img class="inline-graphic" alt="Graphic" src="images/img41.jpg" />
</span>
         on
         <span class="inline-formula" id="inline-formula-31">
<img class="inline-graphic" alt="Graphic" src="images/img42.jpg" />
</span>
         , yielding
         <span class="inline-formula" id="inline-formula-32">
<img class="inline-graphic" alt="Graphic" src="images/img43.jpg" />
</span>
         as the intercept and
         <span class="inline-formula" id="inline-formula-33">
<img class="inline-graphic" alt="Graphic" src="images/img44.jpg" />
</span>
         as the slope.
        </p>
<p id="p-24">
         Given the interpretation of
         <span class="inline-formula" id="inline-formula-34">
<img class="inline-graphic" alt="Graphic" src="images/img45.jpg" />
</span>
         as an adjusted depth, the allele counts in pool
         <em>
          j
         </em>
         were estimated as
         <span class="inline-formula" id="inline-formula-35">
<img class="inline-graphic" alt="Graphic" src="images/img46.jpg" />
</span>
         and
         <span class="inline-formula" id="inline-formula-36">
<img class="inline-graphic" alt="Graphic" src="images/img47.jpg" />
</span>
         . These were summed over case and control pools to form a
         <span class="inline-formula" id="inline-formula-37">
<img class="inline-graphic" alt="Graphic" src="images/img48.jpg" />
</span>
         table on which a
         <span class="inline-formula" id="inline-formula-38">
<img class="inline-graphic" alt="Graphic" src="images/img49.jpg" />
</span>
         test was performed to test the significance of
         <span class="inline-formula" id="inline-formula-39">
<img class="inline-graphic" alt="Graphic" src="images/img50.jpg" />
</span>
         .
        </p>
</div>
<div id="sec-8" class="subsection">
<h4>
         2.2.4 Accounting for sequencing error
        </h4>
<p id="p-25">
         To examine the potential influence of sequencing errors on the models, a base-specific error rate,
         <span class="disp-formula" id="disp-formula-12">
<img class="graphic" alt="Formula" src="images/img51.jpg" />
<span class="disp-formula-label">
           (11)
          </span>
</span>
         where allele
         <em>
          a
         </em>
         is the reference allele and allele
         <span class="inline-formula" id="inline-formula-40">
<img class="inline-graphic" alt="Graphic" src="images/img52.jpg" />
</span>
         is the alternative allele was estimated by summing allele counts over all positions in our target sequence that were either
                           not called as SNPs by VarScan or failed subsequent filtering.
        </p>
<p id="p-26">
         We used these estimated error rates which were of the order 1–10 errors per 100 000 bases (see
         <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
          Supplementary Table S1
         </a>
         ) to adjust read counts of the major allele and fitted all models to the adjusted read counts
         <span class="disp-formula" id="disp-formula-13">
<img class="graphic" alt="Formula" src="images/img53.jpg" />
<span class="disp-formula-label">
           (12)
          </span>
</span>
</p>
</div>
</div>
<div id="sec-9" class="subsection">
<h3>
        2.3 Simulation studies
       </h3>
<p id="p-27">
        To further evaluate the statistical performance of our models, a series of simulation studies were carried out and both Type
                        1 error and power under various situations were examined. The sample size was set as 500 cases and 500 controls divided into
                        20 pools of 50 people. The allele counts of SNP
        <em>
         i
        </em>
        in pool
        <em>
         j
        </em>
        per person were assumed to follow a gamma distribution with shape
        <span class="inline-formula" id="inline-formula-41">
<img class="inline-graphic" alt="Graphic" src="images/img54.jpg" />
</span>
        and scale
        <span class="inline-formula" id="inline-formula-42">
<img class="inline-graphic" alt="Graphic" src="images/img55.jpg" />
</span>
        average coverage
        <span class="inline-formula" id="inline-formula-43">
<img class="inline-graphic" alt="Graphic" src="images/img56.jpg" />
</span>
        (
        <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
         Sarin
         <em>
          et al.
         </em>
         , 2008
        </a>
        ), where the average coverage per person was set at 40×.
       </p>
<p id="p-28">
        We simulated three sets of 700 SNPs in each dataset (i.e. 2100 in total). The first was a set of ‘null’ SNPs, with MAF = 0;
                        the second was a set of ‘neutral’ SNPs, with MAF in controls ranging from very rare (MAF = 0.005) to common (MAF = 0.5) and
                        equal MAF in cases; the third was a set of ‘disease’ SNPs, with MAF in controls again ranging from 0.005 to 0.5 and in cases
                        determined according to the control MAF and assuming a multiplicative genetic with an allele relative risk (rr) set at 1.5.
                        As we specify our model with rr, we do not need to define the population disease rate. However, for a rare disease such as
                        T1D, the rr is approximately equal to the odds ratio. A range of symmetric error rates (the chance of calling the alternative
                        allele in error) from 0 to 5% was considered and applied to simulated data assuming a binomial distribution. We blinded the
                        analysis to which SNPs were truly ‘null’ and applied the same analysis as described above for the 454 data to identify false
                        SNPs. From these we estimated the error rates and corrected the read counts of the remaining SNPs according to
        <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">
         Equation (12)
        </a>
        .
       </p>
<p id="p-29">
        We performed a total of 1000 simulations in this manner, and Type 1 error rates and power were estimated from the results
                        for the ‘neutral’ and ‘disease’ SNPs, respectively.
       </p>
</div>
</div>
<div class="section results" id="sec-10">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="section-nav">
<a href="#sec-2" title="2 METHODS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<<<<<<< HEAD
<a href="#sec-18" title="4 DISCUSSION" class="next-section-link">
=======
<a href="#sec-13" title="4 DISCUSSION" class="next-section-link">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       3 RESULTS
      </h2>
<<<<<<< HEAD
<div id="sec-9" class="subsection">
<h3>
        3.1 The DBC454 algorithm
       </h3>
<div id="sec-10" class="subsection">
<h4>
         3.1.1 The choice of a metric
        </h4>
<p id="p-26">
         The DBC454 algorithm requires that every sequence be represented as a point in an
         <em>
          n
         </em>
         -dimensional space with integer coordinates. Then, the algorithm relies on the computation of the Euclidian distance between
                           two points
         <strong>
          X
         </strong>
         and
         <strong>
          Y
         </strong>
         , defined by
         <span class="disp-formula" id="disp-formula-1">
<img class="graphic" alt="Formula" src="1268/embed/graphic-2.gif" />
</span>
</p>
<p id="p-27">
         Although sequences are often believed to ‘live in some pseudo-metric space’ (
         <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
          Cai and Sun, 2011
         </a>
         ), there is no generally accepted manner to encode a sequence as a point in an
         <em>
          n
         </em>
         -dimensional space. Regarding our specific problem with fungal ITS1, which are nucleotide sequences with a length between
                           ∼50 and 300 nt, a relatively simple encoding was investigated and proved a posteriori to be successful.
        </p>
<p id="p-28">
         The simplest encoding is certainly to count the numbers of A, C, G and T of a sequence and to represent it as a point in a
                           four-dimensional space. Obviously, this is a poor encoding, as it is likely that two distinct sequences of the same length
                           can share the same nucleotide composition. However, this principle of encoding can be improved by increasing the number of
                           dimensions, for example, by counting the number of adjacent dinucleotides, which encode a sequence as a point in a 16-dimensional
                           space. The encoding used by DBC454 goes one step further, also counting all the dinucleotide separated by one or two residues.
                           Thus, every sequence is encoded as a point in a 48-dimensional space. For example, the dinucleotide counts for the sequence
                           ATAATA are AT = 2, TA = 2, AA = 1; the counts for dinucleotide separated by one nucleotide are A?A = 2, T?A = 1, A?T = 1;
                           counts for those separated by two nucleotides are A??A = 2, T??T = 1; and all the other counts are zero. In theory, this encoding
                           strategy does not exclude that two unrelated sequences of the same length can share exactly the same encoding. However, we
                           have never observed such a case while working with real data. In contrast, the DBC454 encoding has also some interesting properties,
                           particularly regarding the comparison of pyro-sequences, which are especially prone to insertion/deletion errors that affect
                           repeated stretches of a given nucleotide. The examples given in
         <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
          Table 1
         </a>
         illustrate that the metric featured by DBC454 is, for example, less sensitive to this type of errors than a metric based
                           on pairwise sequence identity. This is also somehow reminiscent of the flowgram comparison of pyro-sequences that have been
                           used by previous authors (
         <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
          Quince
          <em>
           et al.
          </em>
          , 2009
         </a>
         ).
        </p>
<div class="table pos-float" id="T1">
<div class="table-inline">
<div class="callout">
<span>
            View this table:
           </span>
<ul class="callout-links">
<li>
<a href="1268/T1.expansion.html">
              In this window
             </a>
</li>
<li>
<a class="in-nw" href="1268/T1.expansion.html">
              In a new window
             </a>
=======
<div id="sec-11" class="subsection">
<h3>
        3.1 Simulation studies
       </h3>
<p id="p-30">
        The first two moments of our simulated data were examined, resulting in a mean sequence depth of around 1950 reads per pool
                        and variance of 12 026.
       </p>
<p id="p-31">
<a id="xref-fig-2-1" class="xref-fig" href="#F2">
         Figure 2
        </a>
        shows estimated Type 1 error rates and power from our simulated data at a significance level
        <span class="inline-formula" id="inline-formula-44">
<img class="inline-graphic" alt="Graphic" src="images/img57.jpg" />
</span>
        . Overall, the EB2 model maintains good control of Type 1 error, across allele frequencies and error rates although tends
                        to be conservative for rare SNPs (MAF
        <span class="inline-formula" id="inline-formula-45">
<img class="inline-graphic" alt="Graphic" src="images/img58.jpg" />
</span>
        ). On the other hand, under Fisher’s exact test, nominal Type 1 error rates may be exceeded by a factor of nine. Note that
                        Fisher’s test is anti-conservative in this situation, in contrast with the typical behaviour of non-parametric tests. We found
                        the expected conservative behaviour when we simulated under a simple binomial model (data not shown), so the anti-conservative
                        behaviour shown here appears to result from the over-dispersion. We can also see that the Type 1 error rate for Fisher’s but
                        not EB2 is affected by the sequencing error rate. This is likely because our error correction method does a good job of correcting
                        the simple error on average, but at the expense of increasing the noise, which is correctly dealt with by EB2 but not Fisher’s
                        test. Power appears slightly lower for EB2 compared to Fisher’s, but not substantially so, given the difference in Type 1
                        error rate control. Some differentiation in power can be seen for our EB2 model with changes in sequencing error rates, with
                        higher error rates having slightly lower power. However, the effect of sequencing error, once corrected, seems minimal given
                        the relatively high coverage depth simulated.
       </p>
<div id="F2" class="fig pos-float odd">
<div class="fig-inline">
<a href="2898/F2.expansion.html">
<img alt="Fig. 2." src="images/img59.jpg" width="600" />
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<a href="2898/F2.expansion.html">
             In this window
            </a>
</li>
<li>
<a class="in-nw" href="2898/F2.expansion.html">
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/28/22/2898/F2">
             Download as PowerPoint Slide
            </a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ul>
</div>
</div>
<<<<<<< HEAD
<div class="table-caption">
<span class="table-label">
           Table 1.
          </span>
<p id="p-29" class="first-child">
           Example of distances obtained with the DBC454 metric (see Section 3.1.1) compared with the percent of divergence [100 − percent
                                 identity (%ID)], when aligning the wild-type sequence with a sequence in which the GGC (underlined) was replaced with various
                                 short sequences
          </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
</div>
<div id="sec-11" class="subsection">
<h4>
         3.1.2 Density-based hierarchical clustering
        </h4>
<p id="p-31">
         The DBC454 algorithm depends on two essential parameters:
         <em>
          N
         </em>
         , the minimal number of points that are required to find a valid cluster; and
         <em>
          d
         </em>
         , the maximal distance between two points for them to be connected together. For a given distance threshold
         <em>
          d
         </em>
         , all the points that can be connected together will form a valid cluster if their count reaches, or exceeds,
         <em>
          N
         </em>
         . Taken together, the
         <em>
          N
         </em>
         and
         <em>
          d
         </em>
         parameters implicitly specify the
         <em>
          density constraint
         </em>
         that is placed on the definition of a valid cluster. However, no constraint is placed on the ‘shape’ of a cluster, in that
                           it can be compact and spherical, doughnut shaped or thread shaped. For DBC454, there is no such thing as a cluster ‘center’
                           as defined for classical hierarchical clustering algorithms, nor of a cluster seed, as implemented in cd-hit-454 and otupipe.
                           Furthermore, as all distances are evaluated, independent runs of the algorithm on the same dataset provide the exact same
                           partition, independently of the sequence order.
        </p>
<p id="p-32">
         For
         <em>
          d
         </em>
         = 0, no clusters are returned, unless a unique sequence is present at least
         <em>
          N
         </em>
         times in the input dataset. For
         <em>
          d
         </em>
         = ∞, there exists only a single cluster containing all the sequences. A flowchart of the algorithm is presented in
         <a id="xref-fig-2-1" class="xref-fig" href="#F2">
          Figure 2
         </a>
         . The hierarchical clustering starts by providing DBC454 with a low value for
         <em>
          d
         </em>
         , which is progressively increased by a user-specified amount up to the greatest distance value desired. As
         <em>
          d
         </em>
         increases, three observations can be made: (i) new clusters with at least
         <em>
          N
         </em>
         sequences can be discovered; (ii) existing clusters can grow; and (iii) existing clusters can merge. Eventually, a tree of
                           merge events, i.e. a hierarchical clustering, is obtained, and it is possible to retrace the order (according to
         <em>
          d
         </em>
         ), in which the clusters have been discovered.
         <a id="xref-fig-3-1" class="xref-fig" href="#F3">
          Figure 3
         </a>
         presents a typical example of a clustering hierarchy observed in the ITS1 dataset. In addition, it is possible to retrieve
                           the level
         <em>
          d
         </em>
         at which a sequence first entered the classification. This is a convenient way to distinguish the sequences that are representative
                           of the core of a cluster (i.e. they entered the hierarchy at a low
         <em>
          d
         </em>
         value), from outlier sequences that entered at a greater
         <em>
          d
         </em>
         value. When the clustering is stopped at the largest
         <em>
          d
         </em>
         value, the sequences that have not been attributed to any cluster are considered to be noise.
        </p>
<div id="F2" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F2.expansion.html">
<img alt="Fig. 2." src="1268/F2.small.gif" />
</a>
<div class="callout">
<span>
            View larger version:
           </span>
<ul class="callout-links">
<li>
<a href="1268/F2.expansion.html">
              In this window
             </a>
</li>
<li>
<a class="in-nw" href="1268/F2.expansion.html">
              In a new window
             </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/29/10/1268/F2">
              Download as PowerPoint Slide
             </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
           Fig. 2.
          </span>
<p id="p-33" class="first-child">
           Algorithm flowchart. A: Encoding of the FASTA sequences using dinucleotide counts (see Section 3.1.1 for details). B: Identify
                                 new clusters and add sequences to previously identified clusters by single linkage using Euclidian distance cutoff
           <em>
            d
           </em>
           . C: When any previously identified clusters are merging at the distance cutoff
           <em>
            d
           </em>
           , record the cluster content before merging, and update the cluster parent–child hierarchy. D: Remove newly identified clusters
                                 containing less than
           <em>
            N
           </em>
           sequences. E: Increase the Euclidian distance cutoff
           <em>
            d
           </em>
           for the next pass. F: Traverse the cluster parent–child merging tree to re-create each cluster discovered in its last state
                                 before its merging with another cluster (identification of seed clusters). G: Re-assign sequences to closest seed clusters
                                 (see text for detail)
          </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<div id="F3" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F3.expansion.html">
<img alt="Fig. 3." src="1268/F3.small.gif" />
</a>
<div class="callout">
<span>
            View larger version:
           </span>
<ul class="callout-links">
<li>
<a href="1268/F3.expansion.html">
              In this window
             </a>
</li>
<li>
<a class="in-nw" href="1268/F3.expansion.html">
              In a new window
             </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/29/10/1268/F3">
              Download as PowerPoint Slide
             </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
           Fig. 3.
          </span>
<p id="p-34" class="first-child">
           Example of the clustering hierarchy obtained for 5 of the 1199 clusters discovered in the ITS1 dataset. C1–C5: clusters; R1–R6:
                                 EMBL reference sequences. For each of the 23 levels, the actual clustering distance used during the computation is indicated
                                 at the bottom of the figure. The area of each circle is proportional to the cluster size, ranging from 102 to 2549 sequences.
                                 In this example, three clusters are discovered at the lowest level, one at distance 4.359 and one at distance 5.099. The so-called
                                 seed clusters are represented in gray. They are the larger clusters found in a branch, just before the first merge event,
                                 and play a pivotal role for the post-assignment of the sequences that enter the classification at a higher level
          </p>
=======
<div class="fig-caption">
<span class="fig-label">
          Fig. 2.
         </span>
<p id="p-32" class="first-child">
          Type 1 error rate (left) and power (right) for Fisher’s exact test (Fisher) and our proposed EB2 model with the significance
                              level
          <span class="inline-formula" id="inline-formula-46">
<img class="inline-graphic" alt="Graphic" src="images/img60.jpg" />
</span>
          . The allelic relative risk is set at rr = 1.5 for the power calculation. Sequencing error rates vary from 0% to 5%
         </p>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="sb-div caption-clear">
</div>
</div>
</div>
</div>
<div id="sec-12" class="subsection">
<<<<<<< HEAD
<h4>
         3.1.3 Re-assignment to the final partition
        </h4>
<p id="p-35">
         The final number of clusters is taken as equal to the number of leaves found in the hierarchical clustering, independently
                           of the level
         <em>
          d
         </em>
         at which the leaf has appeared. The sequences placed in the leaf clusters account for only a fraction of all the sequences,
                           but represent the core of the classification. The cluster sizes tend to increase while climbing every branch of the tree.
                           The particular clusters found just before their first merging event will be referred to as the
         <em>
          seed clusters
         </em>
         and are shown in gray in
         <a id="xref-fig-3-2" class="xref-fig" href="#F3">
          Figure 3
         </a>
         . At the end of the clustering process, the seed clusters are used to attract and re-assign the sequences that entered the
                           clustering hierarchy in a branch above the first merge event. More precisely, this re-assignment is based on the shortest
                           distance between the sequence to re-assign and every sequence already placed in seed clusters.
        </p>
<p id="p-36">
         When all the sequences have been assigned to a cluster, or left in the noise, DBC454 can optionally process a set of reference
                           sequences supplied by the user, and assigns them to the closest cluster at the proper
         <em>
          d
         </em>
         value. Only reference sequences sufficiently close to an already established cluster are retained, and all the other reference
                           sequences are rejected as noise. Again, the level at which a reference sequence enters the hierarchy is indicative of how
                           close this sequence is to the core of the cluster. When taxonomically curated sequences are used for this optional last step,
                           this allows the annotation of the clusters. By separating the reference sequences (if any) from the main dataset, one ensures
                           that they will not contribute to the cluster discovery and thus alter the results. However, to ease the comparison of the
                           algorithms in the accuracy analysis (see below), this feature was not used (e.g. the whole dataset, including the reference
                           sequences, was clustered all at once).
        </p>
</div>
</div>
<div id="sec-13" class="subsection">
<h3>
        3.2 Number of clusters and percentage of classified sequences
       </h3>
<p id="p-37">
        The goal of this study was to identify how the environmental factors influence the fungal diversity at a large geographic
                        scale from a particularly large number of samples. Thus, within this project, it was essential to consider the dominant species
                        and not the rare ones. In other words, clusters should contain a representative number of members to be worth considering
                        in subsequent ecological analyses. In our example of ITS1 sequences generated from 198 geographically different sites, we
                        decided that this minimum number will be 100 (roughly the half of the number of the geographic plots considered in the project).
                        Note that this number corresponds to <0.01% of the total number of sequences. Thus, for DBC454, the minimal number of sequences
                        to seed a cluster was set to 100 (
        <em>
         N
        </em>
        = 100). We have applied the same condition to filter the results of all the other algorithms at the post-processing stage,
                        i.e. all the sequences from clusters smaller than 100 were pooled as noise.
       </p>
<p id="p-38">
<a id="xref-fig-4-1" class="xref-fig" href="#F4">
         Figure 4
        </a>
        summarizes the number of clusters that was found by each algorithm, as well as the percentage of classified sequences, i.e.
                        not assigned to the noise. DBC454 and ESPRIT-Tree tend to classify more sequences in a smaller number of clusters than cd-hit-454
                        and Otupipe. For each algorithm, the number of clusters decreases and the number of classified sequences increases when the
                        clustering parameters are relaxed. This effect is the least pronounced with DBC454. An analysis of the average radius of the
                        clusters discovered shows that DBC454 radii are in the range of those obtained with cd-hit-454 and otupipe. ESPRIT-Tree spans
                        a much larger range of cluster radii.
       </p>
<div id="F4" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F4.expansion.html">
<img alt="Fig. 4." src="1268/F4.small.gif" />
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<a href="1268/F4.expansion.html">
             In this window
            </a>
</li>
<li>
<a class="in-nw" href="1268/F4.expansion.html">
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<a href="/powerpoint/29/10/1268/F4">
             Download as PowerPoint Slide
            </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
          Fig. 4.
         </span>
<p id="p-39" class="first-child">
          Number of clusters, percentage of sequences classified and average cluster radius for the various algorithms/parameters tested.
                              Left: number of clusters containing at least 100 sequences. Right: percentage of the 1 152 122 input sequences classified
                              in clusters containing at least 100 sequences. The value for the original ITS1 dataset is shown with a white diamond; boxplots
                              show the values obtained for the five mutated datasets. The average cluster radius for the original ITS1 dataset is shown
                              with white squares on the right plot
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
</div>
<div id="sec-14" class="subsection">
<h3>
        3.3 Robustness analysis
       </h3>
<p id="p-40">
        In our view, the most important criterion to evaluate a method of clustering metagenomic data is
        <em>
         robustness
        </em>
        . Ideally, if an experiment is repeated, e.g. if the same samples are re-sequenced, one hopes to retrieve exactly the same
                        cluster structure/OTU definition. In other words, the final partition should be resistant to the errors that actually occur
                        during the sequencing. To measure this quantitatively, we have designed the following numerical benchmark. Five ‘mutated’
                        datasets were derived from the original dataset, by introducing typical 454 sequencing errors (
        <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
=======
<h3>
        3.2 Experimental data
       </h3>
<p id="p-33">
        After initial filtering, 473 SNPs were selected for analysis. Our estimated error rates were of the order of one error per
                        100 000 bases (see
        <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
         Supplementary Table S1
        </a>
        ), consistent with previous observations (
        <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
         Huse
         <em>
          et al.
         </em>
         , 2007
        </a>
<<<<<<< HEAD
        ). See Section 2 for details. We performed the clustering with each one of the five mutated datasets, and compared the partition
                        obtained with the original one. The measure of robustness was given by the ARI, which equals 1 when two partitions are identical
                        and approaches 0 when the partitions are random with respect to each other.
       </p>
<p id="p-41">
        As explained earlier, we were only interested in clusters containing at least 100 sequences. Thus, before proceeding with
                        the ARI computation, two distinct treatments can be applied to the sequences left in the noise: (i) consider the noise as
                        a single additional cluster; and (ii) reject from the analysis all the sequences that appear in the noise in either one of
                        the two compared partitions.
        <a id="xref-fig-5-1" class="xref-fig" href="#F5">
         Figure 5
        </a>
        A presents the numerical results obtained while considering the noise as a single additional cluster: DBC454 is clearly the
                        most robust method, but ESPRIT-Tree also achieves a good robustness for some intermediate tree-depth values.
        <a id="xref-fig-5-2" class="xref-fig" href="#F5">
         Figure 5
        </a>
        B presents the results, while ignoring the sequences that are unclassified (e.g. appear in the noise) in either one of the
                        two compared partitions. The ARI values are, on average, improved, but the global picture of the comparison of different algorithms
                        remains the same. It should be noted that cd-hit-454 and otupipe are especially sensitive to sequencing errors when their
                        default parameters are used.
       </p>
<div id="F5" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F5.expansion.html">
<img alt="Fig. 5." src="1268/F5.small.gif" />
=======
        ), and had a negligible impact on association results; only the results without error correction are shown.
       </p>
<p id="p-34">
        Although we expect many SNPs will be associated with T1D, given their location in established T1D-associated regions, we still
                        expect them to show only limited departure from the null hypothesis due to the sample size available (even for genuinely associated
                        SNPs, odds ratios for T1D are in the range 1–2). In contrast, a Q–Q plot of the Fisher’s exact test results (
        <a id="xref-fig-3-1" class="xref-fig" href="#F3">
         Fig. 3
        </a>
        a) shows a large slope (slope = 2.481) and observed maximum
        <span class="inline-formula" id="inline-formula-47">
<img class="inline-graphic" alt="Graphic" src="images/img61.jpg" />
</span>
        value,
        <span class="inline-formula" id="inline-formula-48">
<img class="inline-graphic" alt="Graphic" src="images/img62.jpg" />
</span>
        , around 600 suggesting that the simple binomial model does not provide a good fit to the data. Although the slope decreases
                        to 1.285 and
        <span class="inline-formula" id="inline-formula-49">
<img class="inline-graphic" alt="Graphic" src="images/img63.jpg" />
</span>
        to about 150 (
        <a id="xref-fig-3-2" class="xref-fig" href="#F3">
         Fig. 3
        </a>
        b) by filtering pools and SNPs to include only those with higher coverage (
        <em>
         n
        </em>
        = 431), this improvement was achieved at the cost of discarding data—only pools and SNPs passing the filters were counted
                        in the test.
       </p>
<div id="F3" class="fig pos-float odd">
<div class="fig-inline">
<a href="2898/F3.expansion.html">
<img alt="Fig. 3." src="images/img64.jpg" width="600" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<<<<<<< HEAD
<a href="1268/F5.expansion.html">
=======
<a href="2898/F3.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In this window
            </a>
</li>
<li>
<<<<<<< HEAD
<a class="in-nw" href="1268/F5.expansion.html">
=======
<a class="in-nw" href="2898/F3.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<<<<<<< HEAD
<a href="/powerpoint/29/10/1268/F5">
=======
<a href="/powerpoint/28/22/2898/F3">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             Download as PowerPoint Slide
            </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
<<<<<<< HEAD
          Fig. 5.
         </span>
<p id="p-42" class="first-child">
          Robustness analysis. Each box represents the ARIs of five comparisons of the original partition with the partitions obtained
                              with the five artificially mutated datasets, for the various algorithms/parameters, as given in Section 2. Above: all clusters
                              with less than 100 members were attributed to noise, and this was treated as a single additional cluster. Below: sequences
                              appearing in the noise either in the original dataset or in the mutated dataset were not taken into account
=======
          Fig. 3.
         </span>
<p id="p-35" class="first-child">
          Fisher’s exact test statistics. The observed
          <span class="inline-formula" id="inline-formula-50">
<img class="inline-graphic" alt="Graphic" src="images/img65.jpg" />
</span>
          values were calculated by using Fisher’s
          <em>
           P
          </em>
          -value to calculate quantiles of
          <span class="inline-formula" id="inline-formula-51">
<img class="inline-graphic" alt="Graphic" src="images/img66.jpg" />
</span>
          distribution. Expected values were random quantiles in
          <span class="inline-formula" id="inline-formula-52">
<img class="inline-graphic" alt="Graphic" src="images/img67.jpg" />
</span>
          distribution with df = 1. In this and subsequent quantile–quantile plots, the shaded region is the 95% concentration band.
                              (
          <strong>
           a
          </strong>
          ) 473 SNPs after initial filtering. (
          <strong>
           b
          </strong>
          ) 431 SNPs after further filtering by chromosome coverage
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<<<<<<< HEAD
</div>
<div id="sec-15" class="subsection">
<h3>
        3.4 Comparison of the different partitions
       </h3>
<p id="p-43">
<a id="xref-fig-6-1" class="xref-fig" href="#F6">
         Figure 6
        </a>
        shows the results of the comparison of the dbc-s0 partition with the partitions obtained with all the other methods. The
                        DBC454 partitions appear insensitive to the starting distance parameter. Regarding ESPRIT-Tree, the partition etr-04 appears
                        remarkably similar to dbc-s0 (ARI = 0.88 and 0.95 when noise is or is not considered, respectively). Noticeably, etr-04 was
                        previously shown to be one of the most robust partitions produced by ESPRIT-Tree (
        <a id="xref-fig-5-3" class="xref-fig" href="#F5">
         Fig. 5
        </a>
        ).
       </p>
<div id="F6" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F6.expansion.html">
<img alt="Fig. 6." src="1268/F6.small.gif" />
=======
<p id="p-36">
<a id="xref-fig-4-1" class="xref-fig" href="#F4">
         Figure 4
        </a>
        shows the Q–Q plot result based on Williams’ extra-binomial model. Despite most observed
        <span class="inline-formula" id="inline-formula-53">
<img class="inline-graphic" alt="Graphic" src="images/img68.jpg" />
</span>
        values dropping below 30, the slope remained large (slope = 24.88). Note that because variance of reads is required to estimate
                        the over-dispersion parameter
        <span class="inline-formula" id="inline-formula-54">
<img class="inline-graphic" alt="Graphic" src="images/img69.jpg" />
</span>
        , SNPs with total minor allele counts equal to zero in either cases or controls were excluded, leaving 420 SNPs to test.
       </p>
<div id="F4" class="fig pos-float odd">
<div class="fig-inline">
<a href="2898/F4.expansion.html">
<img alt="Fig. 4." src="images/img70.jpg" width="600" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<<<<<<< HEAD
<a href="1268/F6.expansion.html">
=======
<a href="2898/F4.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In this window
            </a>
</li>
<li>
<<<<<<< HEAD
<a class="in-nw" href="1268/F6.expansion.html">
=======
<a class="in-nw" href="2898/F4.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<<<<<<< HEAD
<a href="/powerpoint/29/10/1268/F6">
=======
<a href="/powerpoint/28/22/2898/F4">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             Download as PowerPoint Slide
            </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
<<<<<<< HEAD
          Fig. 6.
         </span>
<p id="p-44" class="first-child">
          Comparison of the dbc-s0 partition of the ITS1 dataset with the partitions obtained with various algorithms/parameters as
                              described in Section 2. Black plain circle: considering all sequences. White open circle: ignoring sequences that appear in
                              the noise in either of the two partitions compared
=======
          Fig. 4.
         </span>
<p id="p-37" class="first-child">
          Q–Q plots were drawn by plotting observed
          <span class="inline-formula" id="inline-formula-55">
<img class="inline-graphic" alt="Graphic" src="images/img71.jpg" />
</span>
          values against expected quantiles in
          <span class="inline-formula" id="inline-formula-56">
<img class="inline-graphic" alt="Graphic" src="images/img72.jpg" />
</span>
          distribution with df = 1 based on Williams’ extra-binomial model (EB1). The triangles in the charts stand for the SNPs with
                              extremely large
          <span class="inline-formula" id="inline-formula-57">
<img class="inline-graphic" alt="Graphic" src="images/img73.jpg" />
</span>
          values beyond the boundary shown in vertical axis
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<<<<<<< HEAD
<p id="p-45">
        We have conducted a detailed investigation into the discrepancies that exist between the dbc-s0 and etr-04 partitions. Multiple
                        sequence alignments of homologous ITS1 were produced and analyzed using phylogenetic trees and multidimensional scaling (
        <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
         Borg and Groenen, 2005
        </a>
        ). These strongly suggest to us that both algorithms are essentially producing coherent classifications of the sequences,
                        but sometimes they produce clusters with distinct levels of granularity.
       </p>
</div>
<div id="sec-16" class="subsection">
<h3>
        3.5 Accuracy analysis
       </h3>
<p id="p-46">
        Another round of clustering was performed on an extended dataset consisting of the previously investigated ITS1 sequences
                        plus 127 577 reference ITS1 from the EMBL, for which the taxa at the species, genus and family ranks were obtained from the
                        NCBI taxonomy. It must be noted that most of the reference ITS1 ended up in the noise, i.e. in clusters with less than 100
                        members, because they were not present in the investigated environmental samples. To evaluate the relative clustering accuracy,
                        we focused on the subset of reference sequences that were always classified by every single method/parameters. This represented
                        a subset of 3365 (62), 3709 (113) and 4022 (1122) sequences (taxa) for family, genus and species, respectively. We used the
                        NCBI taxonomy as the reference partition, and computed the ARI for each partition. Our results (
        <a id="xref-fig-7-1" class="xref-fig" href="#F7">
         Fig. 7
        </a>
        ) show that overall, each method behaves similarly, albeit poorly. In particular, the accuracy achieved at the species rank
                        is clearly worse than at the other taxonomic ranks. This might reflect the fact that the ITS1 itself, considering the sequencing
                        errors, is not informative enough to reliably assign the species rank. cdh-98, otu-90 and etr-12 have the best accuracy at
                        the species, genus and family ranks, respectively. However these partitions are not among the most robust, as shown previously.
                        We are aware that such an accuracy analysis is delicate, as it depends both on the investigated dataset and on the quality
                        of the reference taxonomy. This analysis is certainly not devoid of a circularity problem, as part of the reference fungal
                        taxonomy used here was likely established using numerical methods that convey implicit or explicit assumptions on the definition
                        of a fungal taxon.
       </p>
<div id="F7" class="fig pos-float  odd">
<div class="fig-inline">
<a href="1268/F7.expansion.html">
<img alt="Fig. 7." src="1268/F7.small.gif" />
=======
<p id="p-38">
        The Q–Q plot in
        <a id="xref-fig-5-1" class="xref-fig" href="#F5">
         Figure 5
        </a>
        a illustrates that EB2 model suits our data better with slope equal to 1.26, but the maximum sample quantile, although lower
                        than the EB1 model, is still above 100. We recognize that despite filtering and improved models, sequencing data are still
                        noisy and a proportion of SNPs identified may be errors. If we restrict to 270 non-novel SNPs [those in dbSNP version 128
                        (
        <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
         Sherry
         <em>
          et al.
         </em>
         , 2001
        </a>
        )] which are less likely to be errors, all sample quantiles drop below 40 (
        <a id="xref-fig-5-2" class="xref-fig" href="#F5">
         Fig. 5
        </a>
        b). The dominance of the EB2 model does not depend on MAF, as can be seen when SNPs are divided according to MAF (
        <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
         Supplementary Fig. 1
        </a>
        ).
       </p>
<div id="F5" class="fig pos-float odd">
<div class="fig-inline">
<a href="2898/F5.expansion.html">
<img alt="Fig. 5." src="images/img74.jpg" width="600" />
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</a>
<div class="callout">
<span>
           View larger version:
          </span>
<ul class="callout-links">
<li>
<<<<<<< HEAD
<a href="1268/F7.expansion.html">
=======
<a href="2898/F5.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In this window
            </a>
</li>
<li>
<<<<<<< HEAD
<a class="in-nw" href="1268/F7.expansion.html">
=======
<a class="in-nw" href="2898/F5.expansion.html">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             In a new window
            </a>
</li>
</ul>
<ul class="fig-services">
<li class="ppt-link">
<<<<<<< HEAD
<a href="/powerpoint/29/10/1268/F7">
=======
<a href="/powerpoint/28/22/2898/F5">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             Download as PowerPoint Slide
            </a>
</li>
</ul>
</div>
</div>
<div class="fig-caption">
<span class="fig-label">
<<<<<<< HEAD
          Fig. 7.
         </span>
<p id="p-47" class="first-child">
          Analysis of the clustering accuracy at the family (white), genus (gray) and species (black) ranks for the various algorithms
                              tested, using the NCBI taxonomy as an external reference
=======
          Fig. 5.
         </span>
<p id="p-39" class="first-child">
          Q–Q plots of the results using EB2 model. (
          <strong>
           a
          </strong>
          ) 473 SNPs after initial filtering were employed in the test with estimated over-dispersion parameters
          <em>
           a
          </em>
          = 0.40,
          <em>
           b
          </em>
          = 13.66. (
          <strong>
           b
          </strong>
          ) 270 db SNPs within our target regions;
          <em>
           a
          </em>
          = 0.59,
          <em>
           b
          </em>
          = 1.27
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<<<<<<< HEAD
</div>
<div id="sec-17" class="subsection">
<h3>
        3.6 Computation times
       </h3>
<p id="p-48">
        The computational complexity of DBC454 is quadratic with the number of sequences and proportional to the number of levels,
                        in the worst case. The algorithm is fully deterministic and has been parallelized with MPI to run up to 256 CPUs. Actual performance
                        depends on the nature of the dataset. For the dataset reported in this article, all computation times were determined on Intel
                        E5540 Xeon CPU at 2.53 GHz. The computation of the partition of the 1 152 121 input sequences by cd-hit-454 and otupipe both
                        took ∼5 min. The DBC454 computation was executed in parallel on 16 CPUs and took ∼13 min (wall clock time), which actually
                        corresponds to a total of 3.6 h of CPU to generate a hierarchical classification with 23 different levels (i.e. ∼10 min per
                        level). ESPRIT-Tree required ∼9 h of CPU to generate a hierarchical classification with 15 levels.
       </p>
<p id="p-49">
        When considering the mutated dataset, the average computation times increased by 69% (cd-hit-454), 48% (otupipe), 61% (ESPRIT-tree)
                        and 90% (DBC454). This can be explained by the number of unique sequences that are increased in the mutated datasets compared
                        with the original one.
       </p>
</div>
</div>
<div class="section discussion" id="sec-18">
<div class="section-nav">
<a href="#sec-8" title="3 RESULTS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#ack-1" title="ACKNOWLEDGEMENTS" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       4 DISCUSSION
      </h2>
<p id="p-50">
       The use of generalized profiles for defining the hyper-variable ITS1 by its highly conserved flanking regions alleviates most
                     of the sensitivity problems encountered with pattern-based search methods (
       <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
        Bellemain
        <em>
         et al.
        </em>
        , 2010
       </a>
       ), including sequencing errors, such as indels. In addition, this permits the recovery of all the ITS1 from the EMBL, irrespective
                     of the particular primers that have been used by the original authors.
      </p>
<p id="p-51">
       DBC454 relies on an arbitrarily defined encoding of every sequence as a point in a 48-dimensional space. When this encoding
                     is used for hierarchical density-based clustering, an especially robust partition of the input sequences is obtained, which
                     a posteriori corroborates its relevance.
      </p>
<p id="p-52">
       DBC454 uses density-based clustering to identify groups of related sequences that are naturally dense in the input dataset.
                     As a density-based method, DBC454 places no constraints on the cluster shape and radius, and has no concept of a cluster center.
                     In contrast to all the other algorithms considered here, which remove the redundancy among the sequences as an initial step,
                     DBC454 explicitly uses this information for the clustering (see the note on ESPRIT-Tree below). One of the unintuitive aspects
                     of density-based clustering is that its performance tends to improve with the size of the input dataset, contrary to many
                     other clustering approaches. Our results show that this approach leads to especially robust partitions without being detrimental
                     to the overall accuracy.
      </p>
<p id="p-53">
       This study demonstrates that DBC454 and ESPRIT-tree (at a carefully chosen level) are currently the two most robust methods
                     for the definition of OTUs in the dataset we have studied. The partitions produced by these two methods are only slightly
                     different and essentially compatible. The key observation here is that DBC454 and ESPRIT-Tree rely on completely different
                     algorithmic approaches. This indicates that both methods are actually capable of ‘recognizing’ the same structure in the input
                     dataset, which is an encouraging result regarding the future reproducibility of the experiments in the field of metagenomics.
                     It must be noted that the ESPRIT-Tree algorithm also includes a ‘density-sensitive’ computation step, i.e. the weighting scheme
                     that is used to update the probabilistic sequences at level greater than 0 (
       <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
        Cai and Sun, 2011
       </a>
       ). This might help to explain the convergent results obtained for DBC454 and ESPRIT-Tree.
      </p>
<p id="p-54">
       Recently, much attention has been placed on the benefits of ‘denoising’, i.e. correct the suspected sequencing errors of the
                     pyro-sequences before performing the taxonomy-independent analysis itself (
       <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
        Bonder
        <em>
         et al.
        </em>
        , 2012
       </a>
       ). Considering their particular sensitivity to the sequencing errors demonstrated here, this recommendation obviously makes
                     sense when cd-hit-454, otupipe or ESPRIT-Tree (at the lowest level) is used for the clustering analysis. However, the introduction
                     of an additional pre-processing step implies the optimization of additional parameters and has a cost in terms of CPU, which
                     is currently prohibitive for large datasets. Instead of requiring two successive steps (‘denoising’ followed by clustering),
                     we have shown here that the clustering process itself can be used at the same time to produce a robust partition (insensitive
                     to sequencing errors) and to identify the outlier sequences, i.e. those that were pooled as ‘noise’ in this article.
=======
<p id="p-40">
        Compared with Nejentsev
        <em>
         et al.
        </em>
        ’s results, the exact values of our Fisher’s
        <em>
         P
        </em>
        -values were slightly different due to different alignment software and filters used in this article to process data. However,
                        Nejentsev
        <em>
         et al.
        </em>
        identified four SNPs with Fisher’s
        <em>
         P
        </em>
        -values < 0.05, consistent with our results (both Fisher’s exact test and EB2 model;
        <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
         Table 1
        </a>
        ).
       </p>
<div class="table pos-float" id="T1">
<div class="table-expansion" id="T1" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
           Table 1.
          </span>
<p id="p-41">
           Comparison of different approaches on the four SNPs identified in Nejentsev
           <em>
            et al.
           </em>
           ’s article of which the first two SNPs within
           <em>
            IFIH1
           </em>
           gene were replicated in their follow-up genotyping experiment
          </p>
<div class="sb-div caption-clear">
</div>
</div>
<table frame="hsides" rules="groups" id="table-1">
<thead align="left" id="thead-1" class="table-left">
<tr id="tr-1">
<th rowspan="1" colspan="1" id="th-1">
             SNP
            </th>
<th rowspan="1" colspan="1" id="th-2">
             Location
            </th>
<th rowspan="1" colspan="1" id="th-3">
             Chr.
            </th>
<th rowspan="1" colspan="1" id="th-4">
             Alleles
            </th>
<th colspan="4" rowspan="1" id="th-5" class="table-bborder">
<em>
              P
             </em>
             -value
            </th>
</tr>
<tr id="tr-2">
<th rowspan="1" colspan="1" id="th-6">
</th>
<th rowspan="1" colspan="1" id="th-7">
</th>
<th rowspan="1" colspan="1" id="th-8">
</th>
<th rowspan="1" colspan="1" id="th-9">
</th>
<th rowspan="1" colspan="1" id="th-10">
             Nejentsev’s Fisher’s
            </th>
<th rowspan="1" colspan="1" id="th-11">
             Our Fisher’s
            </th>
<th rowspan="1" colspan="1" id="th-12">
             Our EB1
            </th>
<th rowspan="1" colspan="1" id="th-13">
             Our EB2
            </th>
</tr>
</thead>
<tbody align="left" id="tbody-1" class="table-left">
<tr id="tr-3">
<td id="td-1">
             rs35337543
            </td>
<td id="td-2">
<em>
              IFIH1
             </em>
             , intron 8
            </td>
<td id="td-3">
             chr2
            </td>
<td id="td-4">
             G
             <span class="inline-formula" id="inline-formula-58">
<img class="inline-graphic" alt="Graphic" src="T1/embed/inline-graphic-58.gif" />
</span>
             C
            </td>
<td id="td-5">
             0.000044
            </td>
<td id="td-6">
             0.0032
            </td>
<td id="td-7">
             0.0029
            </td>
<td id="td-8">
             0.016
            </td>
</tr>
<tr id="tr-4">
<td id="td-9">
             rs35667974
            </td>
<td id="td-10">
<em>
              IFIH1
             </em>
             , exon 14
            </td>
<td id="td-11">
             chr2
            </td>
<td id="td-12">
             A
             <span class="inline-formula" id="inline-formula-59">
<img class="inline-graphic" alt="Graphic" src="T1/embed/inline-graphic-59.gif" />
</span>
             G
            </td>
<td id="td-13">
             0.0049
            </td>
<td id="td-14">
             0.076
            </td>
<td id="td-15">
             0.017
            </td>
<td id="td-16">
             0.011
            </td>
</tr>
<tr id="tr-5">
<td id="td-17">
             ss107794688
            </td>
<td id="td-18">
<em>
              CLEC16A
             </em>
             , intron 23
            </td>
<td id="td-19">
             chr16
            </td>
<td id="td-20">
             C
             <span class="inline-formula" id="inline-formula-60">
<img class="inline-graphic" alt="Graphic" src="T1/embed/inline-graphic-60.gif" />
</span>
             T
            </td>
<td id="td-21">
             0.016
            </td>
<td id="td-22">
             0.045
            </td>
<td id="td-23">
             0.0047
            </td>
<td id="td-24">
             0.034
            </td>
</tr>
<tr id="tr-6">
<td id="td-25">
             ss107794687
            </td>
<td id="td-26">
<em>
              CLEC16A
             </em>
             , intron 11
            </td>
<td id="td-27">
             chr16
            </td>
<td id="td-28">
             C
             <span class="inline-formula" id="inline-formula-61">
<img class="inline-graphic" alt="Graphic" src="T1/embed/inline-graphic-61.gif" />
</span>
             T
            </td>
<td id="td-29">
             0.023
            </td>
<td id="td-30">
             0.000046
            </td>
<td id="td-31">
             0.00021
            </td>
<td id="td-32">
             0.000005
            </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li class="fn" id="fn-2">
<p id="p-42">
             Since different alignment software and filters were used, our Fisher’s exact results were slightly different from Nejentsev
             <em>
              et al.
             </em>
             rsquo;s. Overall, all the four SNPs still showed significance in our preferred EB2 model (
             <span class="inline-formula" id="inline-formula-62">
<img class="inline-graphic" alt="Graphic" src="T1/embed/inline-graphic-62.gif" />
</span>
             ).
            </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
          Table 1.
         </span>
<p id="p-41" class="first-child">
          Comparison of different approaches on the four SNPs identified in Nejentsev
          <em>
           et al.
          </em>
          ’s article of which the first two SNPs within
          <em>
           IFIH1
          </em>
          gene were replicated in their follow-up genotyping experiment
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-43">
        Apart from the SNPs identified by Nejentsev
        <em>
         et al.
        </em>
        previously, there were 85 new SNPs in the EB2 model with
        <span class="inline-formula" id="inline-formula-63">
<img class="inline-graphic" alt="Graphic" src="images/img75.jpg" />
</span>
        . After examining the flanking 5′- and 3′-sequences (to see whether there were base repeats that are associated with erroneous
                        SNP calls), read distribution across all pools and base qualities, seven SNPs listed in
        <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
         Table 2
        </a>
        were considered most likely to be real.
       </p>
<div class="table pos-float" id="T2">
<div class="table-expansion" id="T2" itemprop="articleBody">
<span class="highwire-journal-article-marker-start">
</span>
<div class="table-caption">
<span class="table-label">
           Table 2.
          </span>
<p id="p-44">
           The SNPs identified by EB2 model after initial filtering
          </p>
<div class="sb-div caption-clear">
</div>
</div>
<table frame="hsides" rules="groups" id="table-2">
<thead align="left" id="thead-2" class="table-left">
<tr id="tr-7">
<th rowspan="1" colspan="1" id="th-14">
             SNP
            </th>
<th rowspan="1" colspan="1" id="th-15">
             Gene
            </th>
<th rowspan="1" colspan="1" id="th-16">
             Alleles
            </th>
<th colspan="2" rowspan="1" id="th-17" class="table-bborder">
             MAF
            </th>
<th colspan="2" rowspan="1" id="th-18" class="table-bborder">
<em>
              P
             </em>
             -value
            </th>
<th rowspan="1" colspan="1" id="th-19">
             OR
            </th>
<th rowspan="1" colspan="1" id="th-20">
             T1D SNP
            </th>
<th rowspan="1" colspan="1" id="th-21">
<em>
              r
             </em>
<sup>
              2
             </sup>
</th>
</tr>
<tr id="tr-8">
<th rowspan="1" colspan="1" id="th-22">
</th>
<th rowspan="1" colspan="1" id="th-23">
</th>
<th rowspan="1" colspan="1" id="th-24">
</th>
<th rowspan="1" colspan="1" id="th-25">
             Control
            </th>
<th rowspan="1" colspan="1" id="th-26">
             Case
            </th>
<th rowspan="1" colspan="1" id="th-27">
             Fisher’s
            </th>
<th rowspan="1" colspan="1" id="th-28">
             EB2
            </th>
<th rowspan="1" colspan="1" id="th-29">
</th>
<th rowspan="1" colspan="1" id="th-30">
</th>
<th rowspan="1" colspan="1" id="th-31">
</th>
</tr>
</thead>
<tbody align="left" id="tbody-2" class="table-left">
<tr id="tr-9">
<td id="td-33">
             rs3184504
            </td>
<td id="td-34">
<em>
              SH2B3
             </em>
</td>
<td id="td-35">
             T
             <span class="inline-formula" id="inline-formula-64">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-64.gif" />
</span>
             C
            </td>
<td id="td-36">
             0.53
            </td>
<td id="td-37">
             0.41
            </td>
<td id="td-38">
<span class="inline-formula" id="inline-formula-65">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-65.gif" />
</span>
</td>
<td id="td-39">
<span class="inline-formula" id="inline-formula-66">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-66.gif" />
</span>
</td>
<td id="td-40">
             0.62
            </td>
<td id="td-41">
             rs3184504
            </td>
<td id="td-42">
             1
            </td>
</tr>
<tr id="tr-10">
<td id="td-43">
             rs72650660
            </td>
<td id="td-44">
<em>
              CLEC16A
             </em>
</td>
<td id="td-45">
             C
             <span class="inline-formula" id="inline-formula-67">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-67.gif" />
</span>
             T
            </td>
<td id="td-46">
             0.043
            </td>
<td id="td-47">
             0.011
            </td>
<td id="td-48">
<span class="inline-formula" id="inline-formula-68">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-68.gif" />
</span>
</td>
<td id="td-49">
<span class="inline-formula" id="inline-formula-69">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-69.gif" />
</span>
</td>
<td id="td-50">
             0.25
            </td>
<td id="td-51">
             –
            </td>
<td id="td-52">
             –
            </td>
</tr>
<tr id="tr-11">
<td id="td-53">
             rs8052325
            </td>
<td id="td-54">
<em>
              CLEC16A
             </em>
</td>
<td id="td-55">
             A
             <span class="inline-formula" id="inline-formula-70">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-70.gif" />
</span>
             G
            </td>
<td id="td-56">
             0.13
            </td>
<td id="td-57">
             0.082
            </td>
<td id="td-58">
<span class="inline-formula" id="inline-formula-71">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-71.gif" />
</span>
</td>
<td id="td-59">
<span class="inline-formula" id="inline-formula-72">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-72.gif" />
</span>
</td>
<td id="td-60">
             0.60
            </td>
<td id="td-61">
             rs12708716
            </td>
<td id="td-62">
             0.26
            </td>
</tr>
<tr id="tr-12">
<td id="td-63">
             rs2476601
            </td>
<td id="td-64">
<em>
              PTPN22
             </em>
</td>
<td id="td-65">
             G
             <span class="inline-formula" id="inline-formula-73">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-73.gif" />
</span>
             A
            </td>
<td id="td-66">
             0.093
            </td>
<td id="td-67">
             0.15
            </td>
<td id="td-68">
<span class="inline-formula" id="inline-formula-74">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-74.gif" />
</span>
</td>
<td id="td-69">
<span class="inline-formula" id="inline-formula-75">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-75.gif" />
</span>
</td>
<td id="td-70">
             1.72
            </td>
<td id="td-71">
             rs2476601
            </td>
<td id="td-72">
             1
            </td>
</tr>
<tr id="tr-13">
<td id="td-73">
             rs3827734
            </td>
<td id="td-74">
<em>
              PTPN22
             </em>
</td>
<td id="td-75">
             A
             <span class="inline-formula" id="inline-formula-76">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-76.gif" />
</span>
             T
            </td>
<td id="td-76">
             0.0025
            </td>
<td id="td-77">
             0.020
            </td>
<td id="td-78">
<span class="inline-formula" id="inline-formula-77">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-77.gif" />
</span>
</td>
<td id="td-79">
<span class="inline-formula" id="inline-formula-78">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-78.gif" />
</span>
</td>
<td id="td-80">
             8.14
            </td>
<td id="td-81">
             –
            </td>
<td id="td-82">
             –
            </td>
</tr>
<tr id="tr-14">
<td id="td-83">
             rs1800521
            </td>
<td id="td-84">
<em>
              AIRE
             </em>
</td>
<td id="td-85">
             T
             <span class="inline-formula" id="inline-formula-79">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-79.gif" />
</span>
             C
            </td>
<td id="td-86">
             0.23
            </td>
<td id="td-87">
             0.32
            </td>
<td id="td-88">
<span class="inline-formula" id="inline-formula-80">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-80.gif" />
</span>
</td>
<td id="td-89">
<span class="inline-formula" id="inline-formula-81">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-81.gif" />
</span>
</td>
<td id="td-90">
             1.58
            </td>
<td id="td-91">
             rs760426
            </td>
<td id="td-92">
             0.04
            </td>
</tr>
<tr id="tr-15">
<td id="td-93">
</td>
<td id="td-94">
</td>
<td id="td-95">
</td>
<td id="td-96">
</td>
<td id="td-97">
</td>
<td id="td-98">
</td>
<td id="td-99">
</td>
<td id="td-100">
</td>
<td id="td-101">
             rs11594656
            </td>
<td id="td-102">
             0.0029
            </td>
</tr>
<tr id="tr-16">
<td id="td-103">
             rs28360489
            </td>
<td id="td-104">
<em>
              IL2RA
             </em>
</td>
<td id="td-105">
             C
             <span class="inline-formula" id="inline-formula-82">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-82.gif" />
</span>
             T
            </td>
<td id="td-106">
             0.11
            </td>
<td id="td-107">
             0.084
            </td>
<td id="td-108">
             0.039
            </td>
<td id="td-109">
<span class="inline-formula" id="inline-formula-83">
<img class="inline-graphic" alt="Graphic" src="T2/embed/inline-graphic-83.gif" />
</span>
</td>
<td id="td-110">
             0.74
            </td>
<td id="td-111">
             rs2104286
            </td>
<td id="td-112">
             0.10
            </td>
</tr>
<tr id="tr-17">
<td id="td-113">
</td>
<td id="td-114">
</td>
<td id="td-115">
</td>
<td id="td-116">
</td>
<td id="td-117">
</td>
<td id="td-118">
</td>
<td id="td-119">
</td>
<td id="td-120">
</td>
<td id="td-121">
             rs12722495
            </td>
<td id="td-122">
             0.62
            </td>
</tr>
</tbody>
</table>
<div class="table-foot">
<ul class="table-footnotes">
<li class="fn" id="fn-3">
<p id="p-45">
             OR is odds ratio of minor allele for T1D.
             <em>
              r
             </em>
<sup>
              2
             </sup>
             is the correlation coefficient with the most associated independent T1D SNPs (marker SNPs that have been identified associated
                           with T1D before) in each region (
             <a href="http://www.t1dbase.org/page/Regions">
              http://www.t1dbase.org/page/Regions
             </a>
             ).
            </p>
</li>
</ul>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<div class="table-caption">
<span class="table-label">
          Table 2.
         </span>
<p id="p-44" class="first-child">
          The SNPs identified by EB2 model after initial filtering
         </p>
<div class="sb-div caption-clear">
</div>
</div>
</div>
<p id="p-46">
        Six of these seven reside in known T1D regions, while rs1800521 (
        <em>
         AIRE
        </em>
        ) is in a T1D candidate region. rs3184504 (
        <em>
         SH2B3
        </em>
        ) and rs2476601 (
        <em>
         PTPN22
        </em>
        ) are known T1D marker SNPs and rs28360489 (
        <em>
         IL2RA
        </em>
        ) is in LD with a known T1D marker SNP (rs12722495,
        <em>
         r
        </em>
<sup>
         2
        </sup>
        = 0.62) (
        <a href="http://www.t1dbase.org/page/Regions">
         http://www.t1dbase.org/page/Regions
        </a>
        ). Two of the remaining four, rs8052325 (
        <em>
         CLEC16A
        </em>
        ) and rs1800521 (
        <em>
         AIRE
        </em>
        ), are not in tight LD with known T1D SNPs (correlation coefficient
        <span class="inline-formula" id="inline-formula-84">
<img class="inline-graphic" alt="Graphic" src="images/img76.jpg" />
</span>
        ), while the final two, rs72650660 (
        <em>
         CLEC16A
        </em>
        ) and rs3827734 (
        <em>
         PTPN22
        </em>
        ), were not found by 1000 genomes presumably due to the small MAF (
        <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
         Table 2
        </a>
        ), which prevents us from calculating LD.
       </p>
</div>
</div>
<div class="section discussion" id="sec-13">
<div class="section-nav">
<a href="#sec-10" title="3 RESULTS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#ack-1" title="ACKNOWLEDGEMENTS" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       4 DISCUSSION
      </h2>
<p id="p-47">
       All statistical analysis models for pooled case-control data are built on the probability distribution of allele frequency
       <span class="inline-formula" id="inline-formula-85">
<img class="inline-graphic" alt="Graphic" src="images/img77.jpg" />
</span>
       which depends upon depth (
       <span class="inline-formula" id="inline-formula-86">
<img class="inline-graphic" alt="Graphic" src="images/img78.jpg" />
</span>
       ), chromosome number per pool (
       <em>
        s
       </em>
       ) and treatment group. Different assumptions about the distribution of
       <span class="inline-formula" id="inline-formula-87">
<img class="inline-graphic" alt="Graphic" src="images/img79.jpg" />
</span>
       lead to different statistical tests. The binomial model used in Fisher’s exact test assumes
       <span class="inline-formula" id="inline-formula-88">
<img class="inline-graphic" alt="Graphic" src="images/img80.jpg" />
</span>
       is constant across pools within the same treatment group [see
       <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
        Equation (2)
       </a>
       in the
       <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
        Supplementary Material
       </a>
       ], while the extra-binomial variation models used in this article assume
       <span class="inline-formula" id="inline-formula-89">
<img class="inline-graphic" alt="Graphic" src="images/img81.jpg" />
</span>
       is a continuous variable that varies from pool to pool.
      </p>
<p id="p-48">
       Note that allele frequencies estimated by sequencing pools of PCR-amplified DNA may be biased, as the PCR may preferentially
                     amplify one allele compared to the other (
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
        Sham
        <em>
         et al.
        </em>
        , 2002
       </a>
       ). This means that allele frequencies presented in
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
        Tables
       </a>
       1 and
       <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
        2
       </a>
       may deviate from the true population frequency, which can be an issue when considering the sample size required to replicate
                     findings for rare SNPs, in particular. As this bias is likely to affect cases and controls equally when there is no underlying
                     difference in allele frequency, it cannot be expected to inflate Type 1 error, although it may act to amplify or depress any
                     true difference between cases and controls depending on which allele is subject to bias.
      </p>
<p id="p-49">
       Other methods allowing for the variance in pooled sequencing have been explored.
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
        Wang
        <em>
         et al.
        </em>
        (2010)
       </a>
       estimated allele frequencies by weighting sequencing read counts using a weight inversely proportional to the variance of
                     the allele frequency estimate and compared these weighted estimates between cases and controls. However, no published software
                     is available to implement this approach. An alternative approach adopted by
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
        Kim
        <em>
         et al.
        </em>
        (2010)
       </a>
       is to estimate allele frequency by maximum likelihood and then use a likelihood ratio test to test the MAF difference between
                     cases and controls. However, this method is affected heavily by pool size. As the authors pointed out, their approach is not
                     applicable to pools with more than five individuals per pool. Given the main purpose of pooled sequencing is to decrease the
                     experimental cost, a large pool size may be expected in real experimental data.
      </p>
<p id="p-50">
       Here, instead of specifying the statistical distribution of
       <span class="inline-formula" id="inline-formula-90">
<img class="inline-graphic" alt="Graphic" src="images/img82.jpg" />
</span>
       , we propose a ‘quasi-likelihood’ approach which only defines the mean and variance of
       <span class="inline-formula" id="inline-formula-91">
<img class="inline-graphic" alt="Graphic" src="images/img83.jpg" />
</span>
       . The two EB models in this article share the same mean functions of
       <span class="inline-formula" id="inline-formula-92">
<img class="inline-graphic" alt="Graphic" src="images/img84.jpg" />
</span>
       with different variance functions where Williams’ EB1 model adopted one parameter,
       <span class="inline-formula" id="inline-formula-93">
<img class="inline-graphic" alt="Graphic" src="images/img85.jpg" />
</span>
       , for each SNP
       <em>
        i
       </em>
       and our EB2 model adopted two universal parameters,
       <em>
        a
       </em>
       and
       <em>
        b
       </em>
       , for all SNPs.
      </p>
<p id="p-51">
       The EB models have the advantage of including between-pool variation, compared to the standard binomial model which ignores
                     heterogeneity of pools and thus provides a poor fit of our sequencing data (
       <a id="xref-fig-3-3" class="xref-fig" href="#F3">
        Fig. 3
       </a>
       ). However, several shortcomings are exposed in EB1: (i) the significance test is based on increases in the
       <span class="inline-formula" id="inline-formula-94">
<img class="inline-graphic" alt="Graphic" src="images/img86.jpg" />
</span>
       statistic, not a statistically powerful approach (
       <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
        McCullagh, 1983
       </a>
       ); (ii) parameter estimation is complicated requiring iteratively re-weighted least squares (see Section C of the
       <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts553/DC1">
        Supplementary Material
       </a>
       ) and (iii) the small sample size (only 20 data points for each SNP
       <em>
        i
       </em>
       ) means
       <span class="inline-formula" id="inline-formula-95">
<img class="inline-graphic" alt="Graphic" src="images/img87.jpg" />
</span>
       cannot be estimated precisely.
      </p>
<p id="p-52">
       In EB2, we assumed that the parameters in the variance function are constant across all SNPs tested. By doing this, we expanded
                     the number of data points for parameter estimation which allowed more efficient estimation and our use of GLM means it may
                     be fitted in standard software. This could, of course, be extended to include other SNP-specific features by changing the
                     form of
       <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
        Equation (8)
       </a>
       , but no other features were found to be predictive of error structure in our 454 dataset.
      </p>
<p id="p-53">
       The improvement of EB2 over EB1 is obvious in the Q–Q plot—the slope declines to 1.26 (
       <a id="xref-fig-4-2" class="xref-fig" href="#F4">
        Fig. 4
       </a>
       versus
       <a id="xref-fig-5-3" class="xref-fig" href="#F5">
        Fig. 5
       </a>
       a). Given the fact that of the 10 re-sequenced genes, 6 were in known T1D regions, we expect some SNPs would be genuinely
                     associated with T1D and therefore expect the slope to be greater than one.
      </p>
<p id="p-54">
       The simulation results showed that across different situations, our preferred EB2 model has an overall excellent control of
                     Type 1 error rate. This is in contrast to Fisher’s exact test, where the observed Type 1 error rate may be as much as 9-fold
                     the nominal rate at
       <span class="inline-formula" id="inline-formula-96">
<img class="inline-graphic" alt="Graphic" src="images/img88.jpg" />
</span>
       .
      </p>
<p id="p-55">
       In our 454 data, base error rates were very low. However, we included a wider range of errors in our simulations to allow
                     us to examine the potential for our model to be applied to other technologies. Because our model only deals with increased
                     variance, and not systematic bias induced by base calling errors, we undertake a separate error correction step which should
                     correct the error in expectation, but may itself increase variance. This can be seen in the dependence of Type 1 error rate
                     on the sequencing error for Fisher’s exact test, whereas our model appears to have a good control of Type 1 error rates across
                     the range of sequencing errors considered, despite both methods being applied to the same, error-corrected, data. Note that
                     any estimates of error rates are likely to be uncertain. Our experience is that the effect of correcting for a poorly estimated
                     error rate in simulated data depends on MAF, the magnitude of the true error and how poorly it is estimated. In the case of
                     our 454 data, we are lucky to be dealing with low estimated error rates which are consistent with previous estimates for this
                     technology. Correction for these estimated error rates made no substantial difference to our conclusions. However, if the
                     estimated error rates in other applications were larger, it might be wise to conduct some sensitivity analysis, varying the
                     estimated error rate by factors of two or more, to examine whether associations identified are robust to misspecification
                     of error rates.
      </p>
<p id="p-56">
       A shortage of any model used here is the assumption of binomial error within each pool, which allows a zero count for major
                     alleles. This is in contrast to our SNP detection principle which requires at least two supporting reads at a position to
                     call a SNP (i.e. we should model
       <span class="inline-formula" id="inline-formula-97">
<img class="inline-graphic" alt="Graphic" src="images/img89.jpg" />
</span>
       for some
       <em>
        k
       </em>
       ,
       <span class="inline-formula" id="inline-formula-98">
<img class="inline-graphic" alt="Graphic" src="images/img90.jpg" />
</span>
       ). When we removed our detection criteria and instead examined all SNPs in dbSNPs version 128 (
       <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
        Sherry
        <em>
         et al.
        </em>
        , 2001
       </a>
       ) within our 144 target regions, the Q–Q plot showed further improvement (
       <a id="xref-fig-5-4" class="xref-fig" href="#F5">
        Fig. 5
       </a>
       b). Additionally, a much smaller maximum sample quantile was observed in dbSNP Q–Q plot, indicating there are still errors
                     among the full set of 473 called SNPs.
      </p>
<p id="p-57">
       Overall, extra-binomial models appear to have better properties than the naive binomial model. They could analyse a larger
                     range of variants with lower or more variant pool depths and our new EB2 model is more appropriate and easier to apply compared
                     with the EB1 model proposed by
       <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
        Williams (1982)
       </a>
       . Work such as this is used as the basis for further confirmatory experiments and we intend to follow up the four new SNPs
                     identified in
       <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">
        Table 2
       </a>
       . More accurate results lead to better targeting of these experiments and thus faster and more efficient progress to identify
                     the causal genes in T1D.
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
      </p>
</div>
<div class="section ack" id="ack-1">
<div class="section-nav">
<<<<<<< HEAD
<a href="#sec-18" title="4 DISCUSSION" class="prev-section-link">
=======
<a href="#sec-13" title="4 DISCUSSION" class="prev-section-link">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<span>
         Previous Section
        </span>
</a>
<a href="#fn-group-1" title="Footnotes" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       ACKNOWLEDGEMENTS
      </h2>
<<<<<<< HEAD
<p id="p-55">
       We wish to thank the anonymous reviewers for their pertinent and constructive remarks, which helped us to improve this manuscript.
                     All the computations were performed at the Vital-IT Center for high-performance computing of the SIB Swiss Institute of Bioinformatics
                     (
       <a href="http://www.vital-it.ch">
        http://www.vital-it.ch
       </a>
       ).
      </p>
<p id="p-56">
<em>
        Funding
       </em>
       : SIB receives financial supports from the
       <span class="funding-source">
        Swiss Federal Government
       </span>
       through the State Secretariat for Education and Research (SER).
       <span class="funding-source">
        Swiss National Science Foundation
       </span>
       (
       <span class="award-id" id="award-id-1">
        31003A-125145
       </span>
       ) and
       <span class="funding-source">
        University of Lausanne
       </span>
       to A.G.;
       <span class="funding-source">
        Swiss National Science Foundation
       </span>
       (
       <span class="award-id" id="award-id-2">
        PMPDP3-129027
       </span>
       to H.N.-H.).
      </p>
<p id="p-57">
=======
<p id="p-58">
       The data used in this work were generated by Sergey Nejentsev. I would like to thank colleagues in the Juvenile Diabetes Research
                     Foundation/Wellcome Trust Diabetes and Inflammation Laboratory (DIL) for helpful comments on the work and earlier drafts.
      </p>
<p id="p-59">
<em>
        Funding:
       </em>
       X.Y. was supported by Human Molecular Genetics Course in Imperial College London and the Juvenile Diabetes Research Foundation/Wellcome
                     Trust Diabetes and Inflammation Laboratory (DIL). The DIL is funded by the Juvenile Diabetes Research Foundation (9-2011-253),
                     the Wellcome Trust (091157/Z/10/Z) and the National Institute for Health Cambridge Biomedical Research Centre. The Cambridge
                     Institute for Medical Research is in receipt of a Wellcome Trust Strategic Award (079895). D.C. is supported by a Wellcome
                     Trust Principal Research Fellowship (091388/Z/10/Z) and C.W. is supported by a Wellcome Trust Research Career Development
                     Fellowship (089989/Z/09/Z).
      </p>
<p id="p-60">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<em>
        Conflict of Interest
       </em>
       : none declared.
      </p>
</div>
<div class="section fn-group" id="fn-group-1">
<div class="section-nav">
<a href="#ack-1" title="ACKNOWLEDGEMENTS" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<a href="#ref-list-1" title="REFERENCES" class="next-section-link">
<span>
         Next Section
        </span>
</a>
</div>
<h2>
       Footnotes
      </h2>
<ul>
<li class="fn" id="fn-1">
<p id="p-1">
<<<<<<< HEAD
         Associate Editor: Martin Bishop
=======
         Associate Editor: Michael Brudno
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
        </p>
</li>
</ul>
</div>
<ul class="copyright-statement">
<li class="fn" id="copyright-statement-1">
<<<<<<< HEAD
       © The Author 2013. Published by Oxford University Press.
=======
       © The Author 2012. Published by Oxford University Press.
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
      </li>
</ul>
<div class="license" id="license-1">
<p id="p-2">
       This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/),
                     which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.
      </p>
</div>
<div class="section ref-list" id="ref-list-1">
<div class="section-nav">
<a href="#fn-group-1" title="Footnotes" class="prev-section-link">
<span>
         Previous Section
        </span>
</a>
<div class="nav-placeholder">
<<<<<<< HEAD
         
=======
        �
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
       </div>
</div>
<h2>
       REFERENCES
      </h2>
<ol class="cit-list ref-use-labels">
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.1">
=======
<a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.1">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Bellemain
             </span>
<span class="cit-name-given-names">
              E
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
=======
              Bansal
             </span>
<span class="cit-name-given-names">
              V
             </span>
</span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            ITS as an environmental DNA barcode for fungi: an in silico approach reveals potential PCR biases
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Microbiol.
=======
            A statistical method for the detection of variants from next-generation resequencing of DNA pools
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
<<<<<<< HEAD
            10
           </span>
           :
           <span class="cit-fpage">
            189
=======
            26
           </span>
           :
           <span class="cit-fpage">
            i318
           </span>
           -
           <span class="cit-lpage">
            i324
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1186/1471-2180-10-189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20618939&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=26/12/i318" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.2">
=======
<a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.2">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Bonder
             </span>
<span class="cit-name-given-names">
              MJ
=======
              Barrett
             </span>
<span class="cit-name-given-names">
              JC
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Comparing clustering and pre-processing in taxonomy analysis
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2012
           </span>
           ;
           <span class="cit-vol">
            28
           </span>
           :
           <span class="cit-fpage">
            2891
           </span>
           -
           <span class="cit-lpage">
            2897
=======
            Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Genet.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            41
           </span>
           :
           <span class="cit-fpage">
            703
           </span>
           -
           <span class="cit-lpage">
            707
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=28/22/2891" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
=======
<a href="/external-ref?access_num=10.1038/ng.381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=19430480&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000266411700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-29.10.1268.3">
=======
<a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.3">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Borg
             </span>
<span class="cit-name-given-names">
              I
=======
              Cooper
             </span>
<span class="cit-name-given-names">
              JD
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<<<<<<< HEAD
<span class="cit-auth">
<span class="cit-name-surname">
              Groenen
             </span>
<span class="cit-name-given-names">
              PJF
=======
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Meta-analysis of genome-wide association study data identifies additional type 1 diabetes risk loci
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Genet.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            40
           </span>
           :
           <span class="cit-fpage">
            1399
           </span>
           -
           <span class="cit-lpage">
            1401
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/ng.249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18978792&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000261215900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.4">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Crowder
             </span>
<span class="cit-name-given-names">
              MJ
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
</li>
</ol>
<cite>
           .
<<<<<<< HEAD
           <span class="cit-source">
            Modern Multidimensional Scaling Theory and Applications
           </span>
           .
           <span class="cit-publ-loc">
            New York
           </span>
           :
           <span class="cit-publ-name">
            Springer series in statistics. Springer
           </span>
           ;
           <span class="cit-pub-date">
            2005
=======
           <span class="cit-article-title">
            Beta-binomial ANOVA for proportions
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Roy. Stat. Soc. C
           </abbr>
<span class="cit-pub-date">
            1978
           </span>
           ;
           <span class="cit-vol">
            27
           </span>
           :
           <span class="cit-fpage">
            34
           </span>
           -
           <span class="cit-lpage">
            37
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Modern%20Multidimensional%20Scaling%20Theory%20and%20Applications&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Borg&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
=======
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Beta-binomial%20ANOVA%20for%20proportions&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Crowder&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           Search Google Scholar
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.4">
=======
<a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.5">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Cai
             </span>
<span class="cit-name-given-names">
              Y
=======
              Hunt
             </span>
<span class="cit-name-given-names">
              DL
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<<<<<<< HEAD
<span class="cit-auth">
<span class="cit-name-surname">
              Sun
             </span>
<span class="cit-name-given-names">
              Y
             </span>
</span>
=======
<span class="cit-etal">
             et al
            </span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            ESPRIT-Tree: hierarchical clustering analysis of millions of 16S rRNA pyrosequences in quasilinear computational time
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nucleic Acids Res.
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            39
           </span>
           :
           <span class="cit-fpage">
            e95
=======
            The beta-binomial distribution for estimating the number of false rejections in microarray gene expression studies
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Comput. Stat. Data Anal.
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            53
           </span>
           :
           <span class="cit-fpage">
            1688
           </span>
           -
           <span class="cit-lpage">
            1700
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=39/14/e95" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
=======
<a href="/external-ref?access_num=10.1016/j.csda.2008.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20352014&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000264751000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.5">
=======
<a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.6">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Dubuis
             </span>
<span class="cit-name-given-names">
              A
=======
              Huse
             </span>
<span class="cit-name-given-names">
              SM
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Improving the prediction of plant species distribution and community composition by adding edaphic to topo-climatic variables
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Veg. Sci.
           </abbr>
<span class="cit-pub-date">
            2012
           </span>
           .
           <span class="cit-comment">
            [Epub ahead of print, doi: 10.1111/jvs.12002, October 29, 2012]
=======
            Accuracy and quality of massively parallel DNA pyrosequencing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genome Biol.
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            8
           </span>
           :
           <span class="cit-fpage">
            R143
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Improving%20the%20prediction%20of%20plant%20species%20distribution%20and%20community%20composition%20by%20adding%20edaphic%20to%20topo-climatic%20variables&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Dubuis&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
           Search Google Scholar
=======
<a href="/external-ref?access_num=10.1186/gb-2007-8-7-r143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17659080&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.6">
=======
<a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.7">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Edgar
             </span>
<span class="cit-name-given-names">
              RC
             </span>
</span>
=======
              Kim
             </span>
<span class="cit-name-given-names">
              SY
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Search and clustering orders of magnitude faster than BLAST
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
=======
            Design of association studies with pooled or un-pooled next-generation sequencing data
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genet. Epidemiol.
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
<<<<<<< HEAD
            26
           </span>
           :
           <span class="cit-fpage">
            2460
           </span>
           -
           <span class="cit-lpage">
            2461
=======
            34
           </span>
           :
           <span class="cit-fpage">
            479
           </span>
           -
           <span class="cit-lpage">
            491
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=26/19/2460" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
=======
<a href="/external-ref?access_num=10.1002/gepi.20501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20552648&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.7">
=======
<a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.8">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Edgar
             </span>
<span class="cit-name-given-names">
              RC
=======
              Koboldt
             </span>
<span class="cit-name-given-names">
              DC
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            UCHIME improves sensitivity and speed of chimera detection
=======
            Varscan: variant detection in massively parallel sequencing of individual and pooled samples
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
<<<<<<< HEAD
            2011
           </span>
           ;
           <span class="cit-vol">
            27
           </span>
           :
           <span class="cit-fpage">
            2194
           </span>
           -
           <span class="cit-lpage">
            2200
=======
            2009
           </span>
           ;
           <span class="cit-vol">
            25
           </span>
           :
           <span class="cit-fpage">
            2283
           </span>
           -
           <span class="cit-lpage">
            2285
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=27/16/2194" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=25/17/2283" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.8">
=======
<a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.9">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Hubert
             </span>
<span class="cit-name-given-names">
              L
=======
              Li
             </span>
<span class="cit-name-given-names">
              H
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Arabie
             </span>
<span class="cit-name-given-names">
              P
=======
              Durbin
             </span>
<span class="cit-name-given-names">
              R
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Comparing partitions
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Classif.
           </abbr>
<span class="cit-pub-date">
            1985
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            193
           </span>
           -
           <span class="cit-lpage">
            218
=======
            Fast and accurate long-read alignment with burrows-wheeler transform
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            26
           </span>
           :
           <span class="cit-fpage">
            589
           </span>
           -
           <span class="cit-lpage">
            595
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1007/BF01908075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=A1985AVF9100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=26/5/589" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.9">
=======
<a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.10">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Huse
             </span>
<span class="cit-name-given-names">
              SM
=======
              Li
             </span>
<span class="cit-name-given-names">
              H
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Accuracy and quality of massively parallel DNA pyrosequencing
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genome Biol.
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            8
           </span>
           :
           <span class="cit-fpage">
            R143
=======
            The sequence alignment/map format and samtools
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            25
           </span>
           :
           <span class="cit-fpage">
            2078
           </span>
           -
           <span class="cit-lpage">
            2079
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1186/gb-2007-8-7-r143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17659080&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=25/16/2078" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.10">
=======
<a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.11">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Katoh
             </span>
<span class="cit-name-given-names">
              K
=======
              Marioni
             </span>
<span class="cit-name-given-names">
              JC
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<<<<<<< HEAD
<span class="cit-auth">
<span class="cit-name-surname">
              Standley
             </span>
<span class="cit-name-given-names">
              DM
             </span>
</span>
=======
<span class="cit-etal">
             et al
            </span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            MAFFT multiple sequence alignment software version 7: improvements in performance and usability
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Mol. Biol. Evol.
           </abbr>
<span class="cit-pub-date">
            2013
           </span>
           ;
           <span class="cit-vol">
            30
           </span>
           :
           <span class="cit-fpage">
            772
           </span>
           -
           <span class="cit-lpage">
            780
=======
            Rna-seq: an assessment of technical reproducibility and comparison with gene expression arrays
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genome Res.
           </abbr>
<span class="cit-pub-date">
            2008
           </span>
           ;
           <span class="cit-vol">
            18
           </span>
           :
           <span class="cit-fpage">
            1509
           </span>
           -
           <span class="cit-lpage">
            1517
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=molbiolevol&amp;resid=30/4/772" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/9/1509" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.11">
=======
<a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.12">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Luthy
             </span>
<span class="cit-name-given-names">
              R
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Xenarios
             </span>
<span class="cit-name-given-names">
              I
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Bucher
=======
              McCullagh
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Improving the sensitivity of the sequence profile method
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Protein Sci.
           </abbr>
<span class="cit-pub-date">
            1994
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            139
           </span>
           -
           <span class="cit-lpage">
            146
=======
            Quasi-likelihood functions
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Ann. Stat.
           </abbr>
<span class="cit-pub-date">
            1983
           </span>
           ;
           <span class="cit-vol">
            11
           </span>
           :
           <span class="cit-fpage">
            59
           </span>
           -
           <span class="cit-lpage">
            67
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=7511453&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=A1994MT52100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
=======
<a href="/external-ref?access_num=10.1214/aos/1176346056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=A1983QE68600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.12">
=======
<a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.13">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Niu
             </span>
<span class="cit-name-given-names">
              B
=======
              Nejentsev
             </span>
<span class="cit-name-given-names">
              S
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Artificial and natural duplicates in pyrosequencing reads of metagenomic data
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            BMC Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            11
           </span>
           :
           <span class="cit-fpage">
            187
=======
            Rare variants of
            <em>
             IFIH1
            </em>
            , a gene implicated in antiviral responses, protect against type 1 diabetes
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Science
           </abbr>
<span class="cit-pub-date">
            2009
           </span>
           ;
           <span class="cit-vol">
            324
           </span>
           :
           <span class="cit-fpage">
            387
           </span>
           -
           <span class="cit-lpage">
            389
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1186/1471-2105-11-187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20388221&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.13">
<div class="cit-metadata">
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=324/5925/387" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">
         ↵
        </a>
<div class="cit ref-cit ref-book" id="cit-28.22.2898.14">
<div class="cit-metadata">
<cite>
<span class="cit-auth cit-collab">
            R Development Core Team
           </span>
           .
           <span class="cit-source">
            R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing
           </span>
           .
           <span class="cit-publ-loc">
            Austria
           </span>
           :
           <span class="cit-publ-name">
            Vienna
           </span>
           ;
           <span class="cit-pub-date">
            2010
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.15">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Rivas
             </span>
<span class="cit-name-given-names">
              MA
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Deep resequencing of GWAS loci identifies independent rare variants associated with inflammatory bowel disease
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Genet
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            43
           </span>
           :
           <span class="cit-fpage">
            1066
           </span>
           -
           <span class="cit-lpage">
            1073
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/ng.952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=21983784&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.16">
<div class="cit-metadata">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Pagni
             </span>
<span class="cit-name-given-names">
              M
=======
              Robinson
             </span>
<span class="cit-name-given-names">
              MD
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Jongeneel
             </span>
<span class="cit-name-given-names">
              CV
=======
              Smyth
             </span>
<span class="cit-name-given-names">
              GK
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Making sense of score statistics for sequence alignments
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Brief. Bioinfo.
           </abbr>
<span class="cit-pub-date">
            2001
           </span>
           ;
           <span class="cit-vol">
            2
           </span>
           :
           <span class="cit-fpage">
            51
           </span>
           -
           <span class="cit-lpage">
            67
=======
            Moderated statistical tests for assessing differences in tag abundance
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            23
           </span>
           :
           <span class="cit-fpage">
            2881
           </span>
           -
           <span class="cit-lpage">
            2887
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1093/bib/2.1.51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=23/21/2881" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.14">
=======
<a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.17">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Quince
             </span>
<span class="cit-name-given-names">
              C
=======
              Sarin
             </span>
<span class="cit-name-given-names">
              S
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            Accurate determination of microbial diversity from 454 pyrosequencing data
=======
            Caenorhabditis elegans mutant allele identification by whole-genome sequencing
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Methods
           </abbr>
<span class="cit-pub-date">
<<<<<<< HEAD
            2009
           </span>
           ;
           <span class="cit-vol">
            6
           </span>
           :
           <span class="cit-fpage">
            639
           </span>
           -
           <span class="cit-lpage">
            641
=======
            2008
           </span>
           ;
           <span class="cit-vol">
            5
           </span>
           :
           <span class="cit-fpage">
            865
           </span>
           -
           <span class="cit-lpage">
            867
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1038/nmeth.1361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=19668203&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000269382900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
=======
<a href="/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=18677319&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.15">
=======
<a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.18">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Rice
=======
              Sham
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
<span class="cit-name-given-names">
              P
             </span>
</span>
            ,
           </li>
<li>
<<<<<<< HEAD
<span class="cit-auth">
<span class="cit-name-surname">
              Longden
             </span>
<span class="cit-name-given-names">
              I
=======
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            DNA pooling: a tool for large-scale association studies
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Rev. Genet.
           </abbr>
<span class="cit-pub-date">
            2002
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            862
           </span>
           -
           <span class="cit-lpage">
            871
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nrg930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=12415316&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000179041600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.19">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Sherry
             </span>
<span class="cit-name-given-names">
              ST
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<<<<<<< HEAD
<span class="cit-auth">
<span class="cit-name-surname">
              Bleasby
             </span>
<span class="cit-name-given-names">
              A
             </span>
</span>
=======
<span class="cit-etal">
             et al
            </span>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            EMBOSS: the European Molecular Biology Open Software Suite
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Trends Genet.
           </abbr>
<span class="cit-pub-date">
            2000
           </span>
           ;
           <span class="cit-vol">
            16
           </span>
           :
           <span class="cit-fpage">
            276
           </span>
           -
           <span class="cit-lpage">
            277
=======
            dbSNP: the NCBI database of genetic variation
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nucleic Acids Res.
           </abbr>
<span class="cit-pub-date">
            2001
           </span>
           ;
           <span class="cit-vol">
            29
           </span>
           :
           <span class="cit-fpage">
            308
           </span>
           -
           <span class="cit-lpage">
            311
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=10827456&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
=======
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=29/1/308" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.16">
=======
<a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.20">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Sigrist
             </span>
<span class="cit-name-given-names">
              CJ
=======
              Smyth
             </span>
<span class="cit-name-given-names">
              DJ
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            PROSITE: a documented database using patterns and profiles as motif descriptors
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Brief. Bioinfo.
           </abbr>
<span class="cit-pub-date">
            2002
           </span>
           ;
           <span class="cit-vol">
            3
           </span>
           :
           <span class="cit-fpage">
            265
           </span>
           -
           <span class="cit-lpage">
            274
=======
            A genome-wide association study of nonsynonymous snps identifies a type 1 diabetes locus in the interferon-induced helicase
                                    (
            <em>
             IFIH1
            </em>
            ) region
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nat. Genet.
           </abbr>
<span class="cit-pub-date">
            2006
           </span>
           ;
           <span class="cit-vol">
            38
           </span>
           :
           <span class="cit-fpage">
            617
           </span>
           -
           <span class="cit-lpage">
            619
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1093/bib/3.3.265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
=======
<a href="/external-ref?access_num=10.1038/ng1800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=16699517&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000237954800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<<<<<<< HEAD
<a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-29.10.1268.17">
=======
<a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.21">
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
<<<<<<< HEAD
              Sun
             </span>
<span class="cit-name-given-names">
              Y
=======
              Wang
             </span>
<span class="cit-name-given-names">
              T
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
<<<<<<< HEAD
            A large-scale benchmark study of existing algorithms for taxonomy-independent microbial community analysis
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Brief. Bioinfo.
           </abbr>
<span class="cit-pub-date">
            2012
           </span>
           ;
           <span class="cit-vol">
            13
           </span>
           :
           <span class="cit-fpage">
            107
           </span>
           -
           <span class="cit-lpage">
            121
=======
            Resequencing of pooled DNA for detecting disease associations with rare variants
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Genet. Epidemiol.
           </abbr>
<span class="cit-pub-date">
            2010
           </span>
           ;
           <span class="cit-vol">
            34
           </span>
           :
           <span class="cit-fpage">
            492
           </span>
           -
           <span class="cit-lpage">
            501
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<<<<<<< HEAD
<a href="/external-ref?access_num=10.1093/bib/bbr009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
</div>
</div>
</li>
</ol>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<span id="related-urls">
</span>
</div>
<div id="col-2">
<div class="article-nav sidebar-nav">
<a href="/content/29/10/1260.short" title="Previous article" class="previous">
      « Previous
     </a>
<span class="article-nav-sep">
      |
     </span>
<a href="/content/29/10/1275.short" title="Next article" class="next">
      Next Article »
     </a>
<span class="toc-link">
<a href="/content/29/10.toc" title="Table of Contents">
       Table of Contents
      </a>
</span>
</div>
<div class="content-box" id="article-cb-main">
<div class="cb-contents">
<h3 class="cb-contents-header">
<span>
        This Article
       </span>
</h3>
<div class="cb-section cb-slug">
<ol>
<li>
<div id="slugline">
<cite>
<abbr title="Bioinformatics" class="slug-jnl-abbrev">
<nlm:abbrev-journal-title xmlns:nlm="http://schema.highwire.org/NLM/Journal" abbrev-type="publisher">
             Bioinformatics
            </nlm:abbrev-journal-title>
</abbr>
<span class="slug-pub-date" itemprop="datePublished">
            (2013)
           </span>
<span class="slug-vol">
            29
           </span>
<span class="slug-issue">
            (10):
           </span>
<span class="slug-pages">
            1268-1274.
           </span>
<span class="slug-doi-wrapper">
            doi:
            <span title="10.1093/bioinformatics/btt149" class="slug-doi">
             10.1093/bioinformatics/btt149
            </span>
</span>
</cite>
<span class="slug-metadata-note ahead-of-print">
           First published online:
           <span class="slug-ahead-of-print-date">
            March 28, 2013
           </span>
</span>
</div>
</li>
</ol>
</div>
<div class="cb-section cb-views">
<ol>
<li>
<span class="creative-commons-article">
          This article is Open Access
          <span>
           OA
          </span>
</span>
</li>
<li class="abstract-view-link primary">
<a href="/content/29/10/1268.abstract" rel="view-abstract">
          Abstract
         </a>
<span class="viewspecificaccesscheck bioinfo;29/10/1268 abstract">
</span>
</li>
<li class="notice full-text-view-link">
<span class="variant-indicator">
          »
          <span>
           Full Text (HTML)
          </span>
</span>
<span class="viewspecificaccesscheck bioinfo;29/10/1268 full">
</span>
</li>
<li class="notice full-text-pdf-view-link">
<a href="/content/29/10/1268.full.pdf+html" rel="view-full-text.pdf">
          Full Text (PDF)
         </a>
<span class="viewspecificaccesscheck bioinfo;29/10/1268 reprint">
</span>
</li>
<li class="cb-versions">
<div>
<span id="pap-all-versions-label">
           All Versions of this Article:
          </span>
<ol class="version-list">
<li>
<a href="/content/early/2013/03/28/bioinformatics.btt149">
             btt149v1
            </a>
</li>
<li>
<a href="/content/early/2013/04/24/bioinformatics.btt149">
             btt149v2
            </a>
</li>
<li class="current-li">
<span>
             29/10/1268
            </span>
<span class="current-version">
             most recent
            </span>
</li>
</ol>
</div>
</li>
</ol>
</div>
<div class="cb-section collapsible" id="cb-art-cat">
<h4 class="cb-section-header">
<span>
         Classifications
        </span>
</h4>
<ol>
<li>
<ul class="subject-headings">
<li>
           Original Paper
           <ul class="subject-sub-headings last-child">
<li>
<a class="tocsection-search" href="/search?tocsectionid=SEQUENCE+ANALYSIS&amp;sortspec=date&amp;submit=Submit">
              SEQUENCE ANALYSIS
             </a>
</li>
</ul>
</li>
</ul>
</li>
</ol>
</div>
<div class="cb-section collapsible" id="cb-art-svcs">
<h4 class="cb-section-header">
<span>
         Services
        </span>
</h4>
<ol>
<li class="alert-link icon-link">
<a href="/cgi/alerts/ctalert?alertType=citedby&amp;addAlert=cited_by&amp;cited_by_criteria_resid=bioinfo;29/10/1268&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/29/10/1268.full">
          Alert me when cited
         </a>
</li>
<li class="alert-link icon-link">
<a href="/cgi/alerts/ctalert?alertType=correction&amp;addAlert=correction&amp;correction_criteria_value=29/10/1268&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/29/10/1268.full">
          Alert me if corrected
         </a>
</li>
<li>
<a href="/cgi/alerts/ctalert?alertType=eletter&amp;addAlert=eletter&amp;eletter_criteria_value=29/10/1268&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/29/10/1268.full">
          Alert me if commented
         </a>
</li>
<li class="similar-link">
<a href="/search?qbe=bioinfo;btt149&amp;citation=Pagni%20et%20al.%2029%20%2810%29:%201268&amp;submit=yes">
          Find similar articles
         </a>
</li>
<li class="similar-link">
<a href="/external-ref?access_num=bioinfo%3B29%2F10%2F1268&amp;link_type=ISI_RELATEDRECORDS" id="cb-isi-similar-articles" class="similar-link">
          Similar articles in Web of Science
         </a>
</li>
<li class="similar-link">
<a href="/external-ref?access_num=23539304&amp;link_type=MED_NBRS" class="similar-link">
          Similar articles in PubMed
         </a>
</li>
<li class="folder-link icon-link">
<a href="/cgi/folders?action=addtofolder&amp;wherefrom=JOURNALS&amp;wrapped_id=bioinfo;29/10/1268">
          Add to my archive
         </a>
</li>
<li class="cit-man-link icon-link cite-link">
<a href="/citmgr?gca=bioinfo;29/10/1268">
          Download citation
         </a>
</li>
<li>
<a class="request-permissions" href="https://s100.copyright.com/AppDispatchServlet?publisherName=oup&amp;publication=bioinformatics&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1%3A%20&amp;publicationDate=05%2F15%2F2013&amp;author=Marco%20Pagni%2C%20H%C3%A9l%C3%A8ne%20Niculita-Hirzel%2C%20Lo%C3%AFc%20Pellissier%2C%20Anne%20Dubuis%2C%20Ioannis%20Xenarios%2C%20Antoine%20Guisan%2C%20Ian%20R.%20Sanders%2C%20J%C3%A9r%C3%B4me%20Goudet%2C%20Nicolas%20Guex&amp;copyright=Oxford%20University%20Press&amp;contentID=10.1093%2Fbioinformatics%2Fbtt149&amp;volumeNum=29&amp;issueNum=10&amp;startPage=1268&amp;endPage=1274&amp;orderBeanReset=true" target="_blank">
          Request Permissions
         </a>
</li>
</ol>
</div>
<div class="cb-section collapsible default-closed" id="cb-letter-cit">
<h4 class="cb-section-header">
<span>
         Responses
        </span>
</h4>
<ol>
<li>
<a href="/letters/submit/bioinfo;29/10/1268">
          Submit a response
         </a>
</li>
<li>
<span class="eletters-no-responses-published">
          No responses published
         </span>
</li>
</ol>
</div>
<div class="cb-section collapsible default-closed" id="cb-art-cit">
<h4 class="cb-section-header">
<span>
         Citing Articles
        </span>
</h4>
<ol>
<li class="hw-citing-link">
<a href="/content/29/10/1268.full?cited-by=yes&amp;legid=bioinfo;29/10/1268#cited-by" id="cb-hw-citing-articles">
          Load citing article information
         </a>
</li>
<li>
<a href="/cgi/crossref-forward-links/29/10/1268" rel="external-nw" id="cb-crossref-citing-articles">
          Citing articles via CrossRef
         </a>
</li>
<li>
<a href="/scopus-links/bioinfo/29/10/1268?access_num=/bioinfo/29/10/1268" rel="external-nw" id="cb-scopus-citing-articles">
          Citing articles via Scopus
         </a>
</li>
<li>
<a href="/external-ref?access_num=%2Fbioinfo%2F29%2F10%2F1268&amp;link_type=ISI_CITING&amp;accnum_type=native" rel="external-nw" id="cb-isi-citing-articles">
          Citing articles via Web of Science
         </a>
</li>
</ol>
</div>
<div class="cb-section collapsible default-closed" id="cb-art-gs">
<h4 class="cb-section-header">
<span>
         Google Scholar
        </span>
</h4>
<ol>
<li class="cb-art-gs-auth author-link">
<a href="http://scholar.google.com/scholar?q=%22author%3APagni%20author%3AM.%22" class="cb-art-gs-auth author-link">
          Articles by  Pagni, M.
         </a>
</li>
<li class="cb-art-gs-auth author-link">
<a href="http://scholar.google.com/scholar?q=%22author%3AGuex%20author%3AN.%22" class="cb-art-gs-auth author-link">
          Articles by  Guex, N.
         </a>
</li>
</ol>
</div>
<div class="cb-section collapsible default-closed" id="cb-art-pm">
<h4 class="cb-section-header">
<span>
         PubMed
        </span>
</h4>
<ol>
<li class="cb-art-pm-cite cite-link icon-link">
<a href="/external-ref?access_num=23539304&amp;link_type=PUBMED" class="cb-art-pm-cite cite-link icon-link">
          PubMed citation
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Pagni%20M&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Pagni, M.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Niculita-Hirzel%20H&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Niculita-Hirzel, H.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Pellissier%20L&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Pellissier, L.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Dubuis%20A&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Dubuis, A.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Xenarios%20I&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Xenarios, I.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Guisan%20A&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Guisan, A.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Sanders%20IR&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Sanders, I. R.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Goudet, J.
         </a>
</li>
<li class="cb-art-pm-auth author-link">
<a href="/external-ref?access_num=Guex%20N&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">
          Articles by  Guex, N.
         </a>
</li>
</ol>
</div>
<div class="cb-section collapsible default-closed" id="cb-art-rel">
<h4 class="cb-section-header">
<span>
         Related Content
        </span>
</h4>
<ol>
<li>
<a href="/content/29/10/1268.full?related-urls=yes&amp;legid=bioinfo;29/10/1268#related-urls" id="cb-related-urls">
          Load related web page information
         </a>
</li>
</ol>
</div>
<div class="cb-section collapsible" id="cb-art-soc">
<h4 class="cb-section-header">
<span>
         Share
        </span>
</h4>
<ol>
<li>
<a href="/email?gca=bioinfo;29/10/1268&amp;current-view-path=/content/29/10/1268.full">
          Email this article
         </a>
</li>
<li>
<div class="social-bookmarking">
<ul class="social-bookmark-links">
<li class="social-bookmarking-item social-bookmarking-item-citeulike">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=CITEULIKE">
<img src="/shared/img/common/social-bookmarking/citeulike.gif" alt="Add to CiteULike" title="CiteULike" />
</a>
<span class="soc-bm-link-text">
             CiteULike
            </span>
</li>
<li class="social-bookmarking-item social-bookmarking-item-delicious">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=DEL_ICIO_US">
<img src="/shared/img/common/social-bookmarking/delicious.gif" alt="Add to Delicious" title="Delicious" />
</a>
<span class="soc-bm-link-text">
             Delicious
            </span>
</li>
<li class="social-bookmarking-item social-bookmarking-item-facebook">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/short/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=FACEBOOK" class="sb-facebook" rel="external-nw">
<img src="/shared/img/common/social-bookmarking/facebook.gif" alt="Add to Facebook" title="Facebook" />
</a>
<span class="soc-bm-link-text">
             Facebook
            </span>
</li>
<li class="social-bookmarking-item social-bookmarking-item-googleplus">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=GOOGLEPLUS&amp;log_only=yes">
<img src="/shared/img/common/social-bookmarking/googleplus.jpg" alt="Add to Google+" title="Google+" />
</a>
<span class="soc-bm-link-text">
             Google+
            </span>
</li>
<li class="social-bookmarking-item social-bookmarking-item-mendeley">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=MENDELEY">
<img src="/shared/img/common/social-bookmarking/mendeley.gif" alt="Add to Mendeley" title="Mendeley" />
</a>
<span class="soc-bm-link-text">
             Mendeley
            </span>
</li>
<li class="social-bookmarking-item social-bookmarking-item-twitter">
<a href="/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/29/10/1268&amp;title=Density-based%20hierarchical%20clustering%20of%20pyro-sequences%20on%20a%20large%20scale%E2%80%94the%20case%20of%20fungal%20ITS1+--+Pagni%20et%20al.%2029%20%2810%29%3A%201268+--+bioinformatics&amp;doi=10.1093/bioinformatics/btt149&amp;link_type=TWITTER">
<img src="/shared/img/common/social-bookmarking/twitter.gif" alt="Add to Twitter" title="Twitter" />
</a>
<span class="soc-bm-link-text">
             Twitter
            </span>
</li>
</ul>
<p class="social-bookmarking-help">
<a href="/help/social_bookmarks.dtl">
            What's this?
           </a>
</p>
</div>
</li>
</ol>
</div>
</div>
</div>
<div class="content-box" id="article-dyn-nav">
<div class="cb-contents">
<h3 class="cb-contents-header">
<span>
        Navigate This Article
       </span>
</h3>
<div class="cb-section" id="cb-art-nav">
<ol>
<li>
<a href="#content-block">
          Top
         </a>
</li>
<li>
<a href="#abstract-1">
          Abstract
         </a>
</li>
<li>
<a href="#sec-1">
          1 INTRODUCTION
         </a>
</li>
<li>
<a href="#sec-2">
          2 METHODS
         </a>
</li>
<li>
<a href="#sec-8">
          3 RESULTS
         </a>
</li>
<li>
<a href="#sec-18">
          4 DISCUSSION
         </a>
</li>
<li>
<a href="#ack-1">
          ACKNOWLEDGEMENTS
         </a>
</li>
<li>
<a href="#fn-group-1">
          Footnotes
         </a>
</li>
<li>
<a href="#ref-list-1">
          REFERENCES
         </a>
</li>
</ol>
</div>
</div>
</div>
<div class="content-box" id="article-cb-main">
<div class="cb-contents">
<div class="cb-section collapsible default-open" id="cb-art-bn">
<h4 class="cb-section-header">
<span>
         People also read [Beta]
        </span>
</h4>
<ul>
<li>
<div id="bnArticleRecs">
</div>
</li>
<li>
<p id="bnHelp">
<a href="http://www.oxfordjournals.org/our_journals/help.html">
           What's this?
          </a>
</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div id="col-3">
<div id="col-3-qs" class="sidebar-qs">
<form class="searchbox" action="/search" method="get">
<p>
       Search this journal:
      </p>
<div>
<input value="" type="text" name="fulltext" id="header-qs-input" />
<input type="hidden" name="submit" value="yes" />
<input class="inline_button" type="image" alt="Go" src="/publisher/img/buttons/go.gif" />
<div class="adv-search-link">
<a href="/search">
         Advanced »
        </a>
</div>
</div>
</form>
</div>
<div class="cb-contents">
<div id="sidebar-current-issue" class="content-box">
<h3 class="cb-contents-header">
       Current Issue
      </h3>
<div class="cb-section">
<ol>
<li>
<a href="/content/current">
<span>
<span class="current-issue-date">
            May 15, 2013
           </span>
<span class="toc-citation-volume">
            29
           </span>
<span class="toc-citation-issue">
            (10)
           </span>
</span>
</a>
</li>
</ol>
</div>
<div class="cb-section">
<ol>
<li>
<a href="/content/current">
<img src="/content/29/10.cover.gif" alt="Bioinformatics" id="cover" />
</a>
</li>
</ol>
</div>
<div class="cb-section sidebar-etoc-link">
<ol>
<li>
<a href="/cgi/alerts/etoc">
          Alert me to new issues
         </a>
</li>
</ol>
</div>
</div>
</div>
<div id="second">
<div class="features">
<div class="feature separator_after first">
<h2 id="the_journal" title="The Journal">
<span>
         The Journal
        </span>
</h2>
<ul class="emphasised">
<li>
<a href="http://www.oxfordjournals.org/bioinformatics/about.html" title="About this journal">
          About this journal
         </a>
</li>
<li>
<a href="/cgi/calendarcontent/index.html?target=hw.dtl" title="Calendar of events">
          Calendar of events
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/access_purchase/rights_permissions.html" title="Rights &amp; Permissions">
          Rights & Permissions
         </a>
</li>
<li>
<a href="HTTP://www.oxfordjournals.org/access_purchase/dispatch_dates.html#CABIOSP" title="Dispatch date of the next issue">
          Dispatch date of the next issue
         </a>
</li>
<li>
<a href="http://publicationethics.org/about" title="This journal is a member of the Committee on Publication Ethics (COPE)">
          This journal is a member of the Committee on Publication Ethics (COPE)
         </a>
</li>
<li>
<a href="/cgi/eletters?lookup=by_date&amp;days=84&amp;target=hw.dtl" title="Recent Comments">
          Recent Comments
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/mobile_faqs.html" title="We are mobile – find out more">
          We are mobile – find out more
         </a>
</li>
</ul>
<a href="http://www.oxfordjournals.org/page/4638/1">
<img alt="Next Generation Sequencing " src="images/img1.jpg" />
</a>
</div>
<div class="feature separator_after">
<h3>
        An official journal of
       </h3>
<ul class="emphasised">
<li>
<a href="http://www.iscb.org/" title="The International Society for Computational Biology">
          The International Society for Computational Biology
         </a>
</li>
</ul>
</div>
<div class="feature separator_after">
<h3>
        Impact factor:  5.468
       </h3>
</div>
<div class="feature">
<h3>
        Editors-in-Chief
       </h3>
<div class="contact">
<div class="name">
         J Kelso
        </div>
<div class="clear">
</div>
</div>
<div class="contact">
<div class="name">
         A Valencia
        </div>
<div class="clear">
</div>
</div>
<ul class="emphasised">
<li>
<a href="http://www.oxfordjournals.org/bioinformatics/editorial_board.html" title="View full editorial board">
          View full editorial board
         </a>
</li>
</ul>
</div>
<div class="feature">
<h2 id="for_authors" title="For Authors">
<span>
         For Authors
        </span>
</h2>
<ul class="emphasised">
<li>
<a href="http://www.oxfordjournals.org/bioinformatics/for_authors/index.html" title="Instructions to authors">
          Instructions to authors
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/our_journals/bioinformatics/for_authors/submission_online.html" title="Online submission">
          Online submission
         </a>
</li>
<li>
<a href="http://mc.manuscriptcentral.com/bioinformatics" title="Submit Now!">
          Submit Now!
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/access_purchase/self-archiving_policyb.html" title="Self-archiving policy">
          Self-archiving policy
         </a>
</li>
</ul>
<a href="http://www.oxfordjournals.org/oxfordopen/ ">
<img alt="Open access options for authors - visit Oxford Open" src="images/img2.png" />
</a>
<a href="http://www.oxfordjournals.org/for_authors/repositories.html">
<img alt="RCUK Wellcome Trust Open Access" src="images/img3.jpg" />
</a>
<p>
        Open access options for authors - visit
        <a href="http://www.oxfordjournals.org/oxfordopen/">
         Oxford Open
        </a>
</p>
<a href="http://publicaccess.nih.gov/submit_process_journals.htm">
<img alt="PMC Logo" src="images/img4.png" />
</a>
<ul class="emphasised">
<li>
<a href="http://publicaccess.nih.gov/submit_process_journals.htm" title="This journal enables compliance with the NIH Public Access Policy">
          This journal enables compliance with the NIH Public Access Policy
         </a>
</li>
</ul>
</div>
<div class="feature">
<h2 id="alerting_services" title="Alerting Services">
<span>
         Alerting Services
        </span>
</h2>
<ul class="emphasised">
<li>
<a href="/cgi/alerts/etoc" title="Email table of contents">
          Email table of contents
         </a>
</li>
<li>
<a href="/cgi/alerts/etoc" title="Email Advance Access">
          Email Advance Access
         </a>
</li>
<li>
<a href="/help/citetrack" title="CiteTrack">
          CiteTrack
         </a>
</li>
<li>
<a href="/rss" title="XML RSS feed">
          XML RSS feed
         </a>
</li>
</ul>
</div>
<div class="feature">
<h2 id="corporate_services" title="Corporate Services">
<span>
         Corporate Services
        </span>
</h2>
<ul class="emphasised">
<li>
<a href="http://www.oxfordjournals.org/corporate_services/advertising.html" title="Advertising sales">
          Advertising sales
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/corporate_services/reprints.html" title="Reprints">
          Reprints
         </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/corporate_services/supplements.html" title="Supplements">
          Supplements
         </a>
</li>
</ul>
</div>
<div class="feature">
<h2>
        Widget
       </h2>
<ul class="emphasised">
<li>
<a href="http://www.oxfordjournals.org/our_journals/bioinformatics/widget.html" title="Get a widget">
          Get a widget
         </a>
</li>
</ul>
</div>
</div>
<div class="ad_hidden" id="oas_right1">
</div>
</div>
<div class="most-links-box js-marker">
<div class="most-header">
<h3>
       Most
      </h3>
</div>
<ul>
<li class="most-cur-sel">
<h4>
        Most Read
       </h4>
<div class="most-list">
<ol>
<li class="first-item">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/29/8/965?rss=1&amp;ssource=mfr">
           An approximate Bayesian approach for mapping paired-end DNA reads to a reference genome
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/15/2156?rss=1&amp;ssource=mfr">
           The variant call format and VCFtools
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/29/1/15?rss=1&amp;ssource=mfr">
           STAR: ultrafast universal RNA-seq aligner
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/29/8/1072?rss=1&amp;ssource=mfr">
           QUAST: quality assessment tool for genome assemblies
          </a>
</li>
<li class="last-item">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/29/8/1060?rss=1&amp;ssource=mfr">
           Robust data-driven incorporation of prior knowledge into the inference of dynamic regulatory networks
          </a>
</li>
</ol>
<a href="http://bioinformatics.oxfordjournals.org/reports/mfr1.dtl" class="view-all">
         » View all Most Read articles
        </a>
</div>
</li>
<li>
<h4>
        Most Cited
       </h4>
<div class="most-list">
<ol>
<li class="first-item">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/15/2156?rss=1&amp;ssource=mfc">
           The variant call format and VCFtools
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/16/2194?rss=1&amp;ssource=mfc">
           UCHIME improves sensitivity and speed of chimera detection
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/12/1696?rss=1&amp;ssource=mfc">
           MEME-ChIP: motif analysis of large DNA datasets
          </a>
</li>
<li>
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/17/2325?rss=1&amp;ssource=mfc">
           Identification of novel transcripts in annotated genomes using RNA-Seq
          </a>
</li>
<li class="last-item">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/11/1571?rss=1&amp;ssource=mfc">
           Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications
          </a>
</li>
</ol>
<a href="http://bioinformatics.oxfordjournals.org/reports/mfc1.dtl" class="view-all">
         » View all Most Cited articles
        </a>
</div>
</li>
</ul>
</div>
</div>
<div id="oas_bottom" class="ad_hidden">
</div>
<div id="secondary_footer">
<div id="issn">
     Online ISSN 1460-2059 - Print ISSN 1367-4803
    </div>
<div id="copyright">
<a href="http://www.oxfordjournals.org/our_journals/terms.html">
      Copyright ©
     </a>
      
     <span>
      2013
     </span>
      Oxford University Press
    </div>
</div>
<div id="primary_footer">
<div id="site_logo">
<strong>
<span>
       Oxford Journals
      </span>
</strong>
<em>
<span>
       Oxford University Press
      </span>
</em>
</div>
<div id="third_nav">
<ul>
<li>
<a href="http://www.oxfordjournals.org/site_map.html">
        Site Map
       </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/legal/privacy_policy.html">
        Privacy Policy
       </a>
</li>
<li>
<a href="http://global.oup.com/cookiepolicy/?siteid=journals">
        Cookie Policy
       </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/legal/legal_notices.html">
        Legal Notices
       </a>
</li>
<li>
<a href="http://www.oxfordjournals.org/faq/">
        Frequently Asked Questions
       </a>
</li>
</ul>
</div>
<div id="other_oup_sites">
<form action="http://www.oxfordjournals.org/service/redirect">
<div>
<label for="site_list">
        Other Oxford University Press sites:
       </label>
<select name="url" id="site_list">
<option value="http://www.oup.com">
         Oxford University Press
        </option>
<option value="http://www.oxfordjournals.org/china/">
         Oxford Journals China
        </option>
<option value="http://www.oxfordjournals.org/japan/">
         Oxford Journals Japan
        </option>
<option value="http://www.oup.com/uk/academic">
         Academic & Professional books
        </option>
<option value="http://www.oup.com/oxed">
         Children's & Schools Books
        </option>
<option value="http://www.oup.com/uk/dictionaries">
         Dictionaries & Reference
        </option>
<option value="http://www.oup.co.uk/newdnb/">
         Dictionary of National Biography
        </option>
<option value="http://www.oup.com/uk/digital_reference">
         Digital Reference
        </option>
<option value="http://www.oup.com/elt/">
         English Language Teaching
        </option>
<option value="http://www.oup.com/uk/textbooks">
         Higher Education Textbooks
        </option>
<option value="http://www.oup.co.uk/ieu/">
         International Education Unit
        </option>
<option value="http://www.oup.com/uk/law">
         Law
        </option>
<option value="http://www.oup.com/uk/medicine">
         Medicine
        </option>
<option value="http://www.oup.com/uk/music">
         Music
        </option>
<option value="http://www.oup.com/online">
         Online Products & Publishing
        </option>
<option value="http://oxfordbibliographiesonline.com">
         Oxford Bibliographies Online
        </option>
<option value="http://oxforddictionaries.com">
         Oxford Dictionaries Online
        </option>
<option value="http://www.oed.com">
         Oxford English Dictionary
        </option>
<option value="http://www.oxfordlanguagedictionaries.com">
         Oxford Language Dictionaries Online
        </option>
<option value="http://www.oxfordscholarship.com">
         Oxford Scholarship Online
        </option>
<option value="http://www.oup.com/uk/reference">
         Reference
        </option>
<option value="http://www.oup.com/uk/rights">
         Rights and Permissions
        </option>
<option value="http://www.oup.com/uk/booksellers">
         Resources for Retailers & Wholesalers
        </option>
<option value="http://www.oup.com/uk/corporate">
         Resources for the Healthcare Industry
        </option>
<option value="http://www.oup.com/vsi">
         Very Short Introductions
        </option>
<option value="http://www.oup.com/worldsclassics">
         World's Classics
        </option>
</select>
<input type="image" src="/resource/image/buttons/go.gif" alt="Go" class="inline_button" />
</div>
</form>
</div>
</div>
</div>
<div style="text-align:center" class="calibre_navbar calibre_rescale_70">
<hr />
<p style="text-align:left; max-width: 100%; overflow: hidden;">This article was downloaded by <strong>calibre</strong> from <a href="http://bioinformatics.oxfordjournals.org/content/29/10/1268.full">http://bioinformatics.oxfordjournals.org/content/29/10/1268.full</a></p>
=======
<a href="/external-ref?access_num=10.1002/gepi.20502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=20578089&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000280349600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.22">
<div class="cit-metadata">
<cite>
<span class="cit-auth cit-collab">
            Wellcome Trust Case Control Consortium
           </span>
           .
           <span class="cit-article-title">
            Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Nature
           </abbr>
<span class="cit-pub-date">
            2007
           </span>
           ;
           <span class="cit-vol">
            447
           </span>
           :
           <span class="cit-fpage">
            661
           </span>
           -
           <span class="cit-lpage">
            678
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
<a href="/external-ref?access_num=17554300&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
           Medline
          </a>
<a href="/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           Web of Science
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.23">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Williams
             </span>
<span class="cit-name-given-names">
              D
             </span>
</span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            Extra-binomial variation in logistic linear models
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            J. Roy. Stat. Soc. C (Appl. Stat.)
           </abbr>
<span class="cit-pub-date">
            1982
           </span>
           ;
           <span class="cit-vol">
            31
           </span>
           :
           <span class="cit-fpage">
            144
           </span>
           -
           <span class="cit-lpage">
            148
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/external-ref?access_num=10.2307/2347977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">
           CrossRef
          </a>
</div>
</div>
</li>
<li>
<span class="ref-label ref-label-empty">
</span>
<a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">
         ↵
        </a>
<div class="cit ref-cit ref-journal" id="cit-28.22.2898.24">
<div class="cit-metadata">
<ol class="cit-auth-list">
<li>
<span class="cit-auth">
<span class="cit-name-surname">
              Zhou
             </span>
<span class="cit-name-given-names">
              Y-H
             </span>
</span>
            ,
           </li>
<li>
<span class="cit-etal">
             et al
            </span>
</li>
</ol>
<cite>
           .
           <span class="cit-article-title">
            A powerful and flexible approach to the analysis of RNA sequence count data
           </span>
           .
           <abbr class="cit-jnl-abbrev">
            Bioinformatics
           </abbr>
<span class="cit-pub-date">
            2011
           </span>
           ;
           <span class="cit-vol">
            27
           </span>
           :
           <span class="cit-fpage">
            2672
           </span>
           -
           <span class="cit-lpage">
            2678
           </span>
           .
          </cite>
</div>
<div class="cit-extra">
<a href="/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=27/19/2672" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks">
<span class="cit-reflinks-abstract">
            Abstract
           </span>
<span class="cit-sep cit-reflinks-variant-name-sep">
            /
           </span>
<span class="cit-reflinks-full-text">
<span class="free-full-text">
             FREE
            </span>
            Full Text
           </span>
</a>
</div>
</div>
</li>
</ol>
</div>
<span class="highwire-journal-article-marker-end">
</span>
</div>
<span id="related-urls">
</span></div></div><div style="text-align:center" class="calibre_navbar calibre_rescale_70">
<hr />
<p style="text-align:left; max-width: 100%; overflow: hidden;">This article was downloaded by <strong>calibre</strong> from <a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2898?rss=1">http://bioinformatics.oxfordjournals.org/cgi/content/short/28/22/2898?rss=1</a></p>
>>>>>>> 4770c2fa2cc3fbe020c130e94b4d1692ec8ad491
<br /><br /> | <a href="../index.html#article_0">Section Menu</a> | <a href="../../index.html#feed_0">Main Menu</a> | </div></body>
</html>